\begin{savequote}[8cm]
‘After all, it is a common weakness of young authors to put too much into their papers.’

  \qauthor{--- Ronald Fisher, \textit{\usebibentry{fisher1950contributions}{title}} \citeyearpar{fisher1950contributions}}
\end{savequote}

% \chapter{\label{app:list-hostpots}Human and mouse recombination hotspots individually analysed with sperm-typing studies}
\chapter{\label{app:data-and-figs}Supplementary data and figures}

\minitoc{}

\section{Supplementary data}
\subsection{PRDM9\textsuperscript{Dom2/Cst}-targeted hotspots studied}

The table below gives the list of mouse hotspots targeted by either PRDM9\textsuperscript{Dom2} or PRDM9\textsuperscript{Cst} that have been individually studied.


\begin{table}[h]
    \centering
	\begin{adjustbox}{width = 1\textwidth}
    \begin{tabular}{rrrr}
        \toprule
        \textbf{Name} & \textbf{Target allele} & \textbf{Chromosome} & \textbf{Reference} \\

        % \textbf{category} & \textbf{targets} & \textbf{fragments} & \textbf{events} & \textbf{($\times$ 10\textsuperscript{-6})} \\

        \midrule
        A3 & PRDM9\textsuperscript{Dom2} & 1 & \citet{kelmenson2005torrid, cole2010comprehensive} \\
        G7c & PRDM9\textsuperscript{Dom2} & 17 & \citet{snoek1998molecular} \\
        E\textsubscript{\textgreek{β}} & PRDM9\textsuperscript{Dom2} & 17 & \citet{steinmetz1982molecular} \\ % strain B6
        Esrrg1 & PRDM9\textsuperscript{Cst} & 1 & \citet{billings2013dna} \\
        Hlx1 & PRDM9\textsuperscript{Cst} & 1 & \citet{ng2008quantitative,billings2013dna} \\
        HS9 & PRDM9\textsuperscript{Dom2} & 19 & \citet{bois2007highly,getun2010nucleosome} \\%B6/DBA2 strain
        HS22 & PRDM9\textsuperscript{Dom2} & 19 & \citet{getun2010nucleosome} \\
        HS59.4 & PRDM9\textsuperscript{Dom2} & 19 & \citet{getun2010nucleosome} \\
        HS61.1 & PRDM9\textsuperscript{Dom2} & 19 & \citet{wu2010anatomy,getun2010nucleosome} \\
        Pbx1 & PRDM9\textsuperscript{Dom2} & 1 & \citet{billings2013dna,baker2015multimer} \\
        Psmb9 & PRDM9\textsuperscript{Cst} & 17 & \citet{guillon2002initiation,baudat2007cis} \\
        \bottomrule
    \end{tabular}
	\end{adjustbox}
    \caption[List of PRDM9\textsuperscript{Dom2}- and PRDM9\textsuperscript{Cst}-targeted hotspots individually studied]
    {\textbf{List of PRDM9\textsuperscript{Dom2}- and PRDM9\textsuperscript{Cst}-targeted hotspots individually studied.}
    }
\label{tab:hotspots-studied-sperm-typing}
\end{table}


\subsection{Disclaimer for the resources used}
% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389181/
% The ensuing 25 years saw the number of identified mammalian hot spots increase incrementally, often by chance. The list of recognized hot spots now includes Ath1 (also known as Tnfsf4)9, Scnm1 (REF. 10) and the HS22 region11 in mice, at least four hot spots in the H2 region12,

This work was performed using the computing facilities of the CC LBBE/PRABI\@.

\subsection{Source code to reproduce figures}

The source code to reproduce figures will be put online shortly.



% SRA et reproductibilite des figures
% mettre aussi la liste des positions genomiques obtenues

\hypersetup{linkcolor=titlepagecolorsection}
\section{Supplementary figures for Chapter~\ref{ch:6-recombination-parameters}}
\hypersetup{linkcolor=black}

\subsection{Recombination rate and DMC1 binding affinity}

\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/Correlation_intensity_DMC1.eps}
    \caption[Proportionality between the recombination rate and DMC1 binding intensity]
    {\textbf{Proportionality between the recombination rate and DMC1 binding intensity.}
        \par All 1,018 hotspots were divided into 10 classes of increasing DMC1 signal (x-axis), i.e.\ the number of DMC1 ChIP-seq tags on each PRDM9 ChIP-seq peak (DMC1 ChIP-seq data on B6xCAST hybrid mice from \citealp{smagulova2016evolutionary}).
        The observed number of recombination events identified per sequenced Mb (left y-axis) was converted into a CO rate (right y-axis) as detailed in Chapter~\ref{ch:6-recombination-parameters}.
        The points and error bars respectively represent the mean number of events (or CO rate) and the standard error on the mean for hotspots of each class.
    }
\label{fig:correlation-DMC1-standard-error}
\end{figure}



\subsection{Hotspot asymmetry and DMC1 binding affinity}


\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/Recombination_rates_2_axes_sym_vs_Asym_DMC1_with_legend_and_outliers.eps}
    \caption[Asymmetric hotspots display lower recombinational activity than expected by their DMC1 binding affinity]
    {\textbf{Asymmetric hotspots display lower recombinational activity than expected by their DMC1 binding affinity.}
        \par All 1,018 hotspots were divided into 10 classes of increasing DMC1 signal (x-axis), i.e.\ the number of DMC1 ChIP-seq tags on each DMC1 ChIP-seq peak (DMC1 ChIP-seq data on B6xCAST hybrid mice from \citealp{smagulova2016evolutionary}).
        The observed number of recombination events identified per sequenced Mb (left y-axis) was converted into a CO rate (right y-axis) as detailed in Chapter~\ref{ch:6-recombination-parameters}.
        Symmetric hotspots (green) were distinguished from asymmetric hotspots (orange) as detailed in Chapter~\ref{ch:7-quantification-BGC}.
        The linear regression model for symmetric (slope~$= 1.8$; intercept~$=0$; \textit{p}-val~$< 2.2 \times 10^{-16}$) and asymmetric (slope~$= 8.6$; intercept~$=0$; \textit{p}-val~$< 2.2 \times 10^{-16}$) hotspots are drawn as dotted lines.
    }
\label{fig:asymmetry-and-DMC1}
\end{figure}



\newpage
\subsection{Distribution of switch points}


\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/density_switch_points.eps}
    \caption[Distribution of switch points along hotspots for Rec-1S and Rec-2S events]
    {\textbf{Distribution of switch points along hotspots for Rec-1S and Rec-2S events.}
}
\label{fig:density-switch-points}
\end{figure}



\newpage
\hypersetup{linkcolor=titlepagecolorsection}
\section{Supplementary figures for Chapters~\ref{ch:7-quantification-BGC} and~\ref{ch:8-HFM1}}
\hypersetup{linkcolor=black}

\subsection{Correlation between expected and observed donor}



\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/CorrelationDMC1_FINAL_BIS_on_lab_computer_COLOURS.eps}
    \caption[Correlation between the expected and observed proportions of CAST-donor fragments across hotspots displaying at least 5 events, coloured per PRDM9 target]
    {\textbf{Correlation between the expected and observed proportions of CAST-donor fragments across hotspots displaying at least 5 events, coloured per PRDM9 target.}
        \par The expected proportion of CAST-donor fragments (x-axis) was based on the probability that the DSB initiates on the B6 haplotype from DMC1 ssDNA-sequencing (SSDS) data by \citet{smagulova2016evolutionary} (see main text).
        Only the 582 hotspots displaying a minimum of 5 recombination events were reported in this figure.
        The Pearson correlation between the two measures gave: $R^2 = 0.66$; {\textit{p}-val $< 2.2 \times 10^{-16}$}.
}
\label{fig:correl-donor-DMC1-with-colour}
\end{figure}



\subsection{Genetic background of all chromosomes}


% LEFT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-3.4cm
	\rightskip-2.7cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth]{figures/chap8/HFM1_background_28355.eps}
	\captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Mosaic of genetic backgrounds inferred at each target along the chromosomes of mouse 28355]
	{\textbf{Mosaic of genetic backgrounds inferred at each target along the chromosomes of mouse 28355.}
		\par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		Each segment corresponds to the position of a target (hotspot or control region) and was coloured in red when the background inferred was BD/BD (homozygous) and in blue when the background inferred was BD/CAST (heterozygous).
	}
\label{fig:mosaic-backgrounds}
\end{sidewaysfigure}


% RIGHT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-2.4cm
	\rightskip-2.4cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth]{figures/chap8/HFM1_background_28367.eps}
	\captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Mosaic of genetic backgrounds inferred at each target along the chromosomes of mouse 28367]
	{\textbf{Mosaic of genetic backgrounds inferred at each target along the chromosomes of mouse 28367.}
		\par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		Each segment corresponds to the position of a target (hotspot or control region) and was coloured in red when the background inferred was BD/BD (homozygous) and in blue when the background inferred was BD/CAST (heterozygous).
	}
\label{fig:mosaic-backgrounds}
\end{sidewaysfigure}


% LEFT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-3.4cm
	\rightskip-2.7cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth]{figures/chap8/HFM1_background_28371.eps}
	\captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Mosaic of genetic backgrounds inferred at each target along the chromosomes of mouse 28371]
	{\textbf{Mosaic of genetic backgrounds inferred at each target along the chromosomes of mouse 28371.}
		\par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		Each segment corresponds to the position of a target (hotspot or control region) and was coloured in red when the background inferred was BD/BD (homozygous) and in blue when the background inferred was BD/CAST (heterozygous).
	}
\label{fig:mosaic-backgrounds}
\end{sidewaysfigure}





\newpage
% \subsection{Pairwise recombination rates of shared hotspots}
\subsection{Pairwise comparison of the RR in shared hotspots}



\begin{figure}[p]
    \centering
    \begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28371 (WT) and 28353 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28371_vs_28353.eps}
    \end{subfigure}

    \vspace{0.5cm}

    \begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28371 (WT) and 28367 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28371_vs_28367.eps}
    \end{subfigure}

    \caption[Correlation of the number of recombination events in shared hotspots for all pairs of mice]
    {\textbf{Correlation of the number of recombination events in shared hotspots for all pairs of mice.}
        % \par The linear regression was significant for the WT mice (slope $=1.03$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 257$) and for the two mutant mice (slope = $0.69$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 241$).
    }
\label{fig:pairwise-RR-shared-BIS-1}
\end{figure}




\begin{figure}[p]
    \centering
   
	\begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28355 (WT) and 28367 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28355_vs_28367.eps}
    \end{subfigure}

    \vspace{0.5cm}
    
	\begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28355 (WT) and 28353 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28355_vs_28353.eps}
    \end{subfigure}


    \caption[Correlation of the number of recombination events in shared hotspots for all pairs of mice]
    {\textbf{Correlation of the number of recombination events in shared hotspots for all pairs of mice.}
        % \par The linear regression was significant for the WT mice (slope $=1.03$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 257$) and for the two mutant mice (slope = $0.69$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 241$).
    }
\label{fig:pairwise-RR-shared-BIS2}
\end{figure}







% + liste de tous les hotspots analyses chez les humains (ou pour moi, plutot souris) dans annexe B de Popa.



%%% CHAPITRE 2
%% Si je veux metttre une appendix sur les defects des souris qui sont mutantes pour un gene de la meiose: Bolcun-Filas, E., & Schimenti, J. C. (2012). Genetics of Meiosis and Recombination in Mice. International Review of Cell and Molecular Biology, 179–227. doi:10.1016/b978-0-12-394309-5.00005-5 (appendix)

