\begin{savequote}[8cm]
‘After all, it is a common weakness of young authors to put too much into their papers.’

  \qauthor{--- Ronald Fisher, \textit{\usebibentry{fisher1950contributions}{title}} \citeyearpar{fisher1950contributions}}
\end{savequote}

% \chapter{\label{app:list-hostpots}Human and mouse recombination hotspots individually analysed with sperm-typing studies}
\chapter{\label{app:data-and-figs}Supplementary data and figures}

\minitoc{}

\section{Supplementary data}
\subsection{PRDM9\textsuperscript{Dom2/Cst}-targeted hotspots studied}

The table below gives the list of mouse hotspots targeted by either PRDM9\textsuperscript{Dom2} or PRDM9\textsuperscript{Cst} that have been individually studied.


\begin{table}[h]
    \centering
	\begin{adjustbox}{width = 1\textwidth}
    \begin{tabular}{rrrr}
        \toprule
        \textbf{Name} & \textbf{Target allele} & \textbf{Chromosome} & \textbf{Reference} \\

        % \textbf{category} & \textbf{targets} & \textbf{fragments} & \textbf{events} & \textbf{($\times$ 10\textsuperscript{-6})} \\

        \midrule
        A3 & PRDM9\textsuperscript{Dom2} & 1 & \citet{kelmenson2005torrid, cole2010comprehensive} \\
        G7c & PRDM9\textsuperscript{Dom2} & 17 & \citet{snoek1998molecular} \\
        E\textsubscript{\textgreek{β}} & PRDM9\textsuperscript{Dom2} & 17 & \citet{steinmetz1982molecular} \\ % strain B6
        Esrrg1 & PRDM9\textsuperscript{Cst} & 1 & \citet{billings2013dna} \\
        Hlx1 & PRDM9\textsuperscript{Cst} & 1 & \citet{ng2008quantitative,billings2013dna} \\
        HS9 & PRDM9\textsuperscript{Dom2} & 19 & \citet{bois2007highly,getun2010nucleosome} \\%B6/DBA2 strain
        HS22 & PRDM9\textsuperscript{Dom2} & 19 & \citet{getun2010nucleosome} \\
        HS59.4 & PRDM9\textsuperscript{Dom2} & 19 & \citet{getun2010nucleosome} \\
        HS61.1 & PRDM9\textsuperscript{Dom2} & 19 & \citet{wu2010anatomy,getun2010nucleosome} \\
        Pbx1 & PRDM9\textsuperscript{Dom2} & 1 & \citet{billings2013dna,baker2015multimer} \\
        Psmb9 & PRDM9\textsuperscript{Cst} & 17 & \citet{guillon2002initiation,baudat2007cis} \\
        \bottomrule
    \end{tabular}
	\end{adjustbox}
    \caption[List of PRDM9\textsuperscript{Dom2}- and PRDM9\textsuperscript{Cst}-targeted hotspots individually studied]
    {\textbf{List of PRDM9\textsuperscript{Dom2}- and PRDM9\textsuperscript{Cst}-targeted hotspots individually studied.}
    }
\label{tab:hotspots-studied-sperm-typing}
\end{table}


\subsection{Disclaimer for the resources used}
% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389181/
% The ensuing 25 years saw the number of identified mammalian hot spots increase incrementally, often by chance. The list of recognized hot spots now includes Ath1 (also known as Tnfsf4)9, Scnm1 (REF. 10) and the HS22 region11 in mice, at least four hot spots in the H2 region12,

This work was performed using the computing facilities of the CC LBBE/PRABI\@.


\subsection{Erroneously called W~$\rightarrow$~S and S~$\rightarrow$~W events}

Quantifying gBGC comes back to measuring the $\frac{WS}{WS+SW}$ ratio.
However, since the large majority of pot-NCO-1 events corresponded to FPs, we had to distinguish the (potential) contribution of FPs to this ratio from that of genuine NCO-1 events.
In particular, this ratio may depart from the expected 50\% ratio if (1) a non-negligible proportion of FPs arise from sequencing miscalls and (2) W~$\rightarrow$~S and S~$\rightarrow$~W sequencing errors appear at different frequencies.\\

% \paragraph{Proportion of FPs due to sequencing errors\\}
First, we thus wanted to quantify the proportion of FPs due to sequencing miscalls.
To do this, we estimated the sequencing error rate directly in our sequencing data by monitoring the apparition of \textit{de novo} variants:
given that the mutation rate ($\sim$10\textsuperscript{-8}/bp) is much lower than the sequencing error rate ($\sim$10\textsuperscript{-3}/bp), we assumed that, outside the polymorphic sites identified by variant-calling, any base call that differed from the nucleotide of the reference genome was a sequencing error and counted them to compute the conditional frequency matrix of sequencing errors\footnote{Matrix $M$ was computed based on the analysis of one chromosome (chromosome 10) for all of our 18 samples individually (because the sequencing errors may vary between the biological samples and sequencing runs). This matrix gives the probability of each erroneous base call, given the genuine nucleotide.} ($M$):

\begin{equation*}
M = \begin{bmatrix}
\Pr( A\rightarrow A \mid A) & \Pr( A\rightarrow C \mid A) & \Pr( A\rightarrow G \mid A) & \Pr( A\rightarrow T \mid A) \\
\Pr( C\rightarrow A \mid C) & \Pr( C\rightarrow C \mid C) & \Pr( C\rightarrow G \mid C) & \Pr( C\rightarrow T \mid C) \\
\Pr( G\rightarrow A \mid G) & \Pr( G\rightarrow C \mid G) & \Pr( G\rightarrow G \mid G) & \Pr( G\rightarrow T \mid G) \\
\Pr( T\rightarrow A \mid T) & \Pr( T\rightarrow C \mid T) & \Pr( T\rightarrow G \mid T) & \Pr( T\rightarrow T \mid T) 
\end{bmatrix}
\end{equation*}






$\forall (i,j) \in \{A, C, G, T\}^2$, the number of NCO-1 FPs expected due to sequencing errors involving a genuine base $i$ mistakenly called as a $j$ base ($e_{i\rightarrow j}$) simply equalled the product of the number of central markers (i.e.\ markers \textit{not} located at the extremity of fragments) that were genuinely $i$ in $ij$ polymorphic sites ($g_{i}^{ij}$) by the conditional probability that a genuine $i$ would mistakenly be called a $j$ ($\Pr( i\rightarrow j \mid i )$):

\begin{equation} \label{eq:nb-errors}
	e_{i\rightarrow j} = g_{i}^{ij} \times \Pr( i\rightarrow j \mid i )
\end{equation}


$g_{i}^{ij}$ was not directly accessible from the data because we could not know which base calls were correctly sequenced.
Though, this number was linked to the number of central markers containing an $i$ allele and involved in a polymorphic site $ij$ ($n_{i}^{ij}$) through the following equation:

\begin{equation} \label{eq:genuine-to-called}
	n_{i}^{ig} = g_{i}^{ij} \times ( 1 - \Pr( i\rightarrow j \mid i ) ) + g_{j}^{ij} \times \Pr( j\rightarrow i \mid j )
\end{equation}


When we computed the $M$ matrix, we found that the frequency of sequencing errors was very low ($\simeq 10^{-3}$).
Thus, to approximate $g_{i}^{ij}$, we used the simplifying assumption that the frequency of wrong calls were close to zero and that of good calls close to 1:

\begin{subequations} 
	\begin{alignat}{5}
		\forall &(i,j) &{}\in{}& \{A, C, G, T\}^2 \; \backslash \: i \neq j, &{}\Pr({}& i\rightarrow j \mid i ) \simeq 0,\label{eq:assumption-low-freqs}\\
		\forall &i	   &{}\in{}& \{A, C, G, T\},							 &{}\Pr({}& i\rightarrow i \mid i ) \simeq 1
	\end{alignat}
\end{subequations}


From equation~\ref{eq:assumption-low-freqs}, equation~\ref{eq:genuine-to-called} simplified to:

\begin{equation} \label{eq:genuine-to-called-simplified}
	n_{i}^{ij} \simeq g_{i}^{ij}
\end{equation}

And, by incorporating equation~\ref{eq:genuine-to-called-simplified} into equation~\ref{eq:nb-errors}, we had:

\begin{equation*} \label{eq:nb-errors-with-only-known-parameters}
	e_{i\rightarrow j} = n_{i}^{ij} \times \Pr( i\rightarrow j \mid i )
\end{equation*}


Finally, the total number of FPs that were expected due to sequencing errors ($E$) was the total sum of each type of sequencing error: 

\begin{equation*} \label{eq:sum-all-NCOs-expected}
	E = \underset{i \neq j} {\sum_{(i, j) \in \{A, C, G, T\}^2}} e_{i\rightarrow j}
\end{equation*}




This allowed us to predict that, among the total 287,577,349 fragments overlapping 3 markers or more, 231,905 were expected to be discovered as NCO-1 FPs due to sequencing errors only.
This represented 66.7\% of the 347,652\footnote{The sequencing error estimate was calculated upon all sequenced fragments, i.e.\ before setting the sequencing error filter, and thus had to be compared to the total number of NCO-1 FPs obtained without the filter (Table~\ref{tab:NCO-1-FP-rate-no-filter}).} NCO-1 FPs that we found in pot-NCO-1 events (110,615 in control regions + an estimate of 237,037 in hotspots, Table~\ref{tab:NCO-1-FP-rate-no-filter}).


\begin{table}[t]
    \centering
    % \begin{adjustbox}{width = 1\textwidth}
		\begin{tabular}{rrrrr}
            \toprule
            \textbf{Target} & \textbf{Nb of} & \textbf{Nb of} & \textbf{Nb of} & \textbf{Event rate} \\

            \textbf{category} & \textbf{targets} & \textbf{fragments} & \textbf{events} & \textbf{($\times$ 10\textsuperscript{-6})} \\

            \midrule
            Hotspots & 1,018 & 228,984,512 & 243,390 & 1062.9 \\
            Controls & 500 & 106,850,906 & 110,615 & 1035.2 \\
            \midrule
            \multicolumn{1}{r}{\textbf{FP rate}} & \multicolumn{4}{r}{\textbf{97.4 \%}} \\
            \bottomrule
        \end{tabular}
    % \end{adjustbox}
    \caption[Number of pot-NCO-1 events detected in hotspot and control targets without the sequencing error filter]
    {\textbf{Number of pot-NCO-1 events detected in hotspot and control targets without the sequencing error filter.}
        \par Pot-NCO-1 events were detected without the sequencing error filter controlling that the allele supporting the genotype call with the mapping onto the B6 genome is identical to that based on the mapping onto the CAST genome.
        All fragments or events overlapping at least 1 bp with a given target are counted in this table.
        The event rate corresponds to the ratio of candidate recombination events over the total number of fragments.
        The maximum false positive (FP) rate is the ratio of the event rate in control targets over that in hotspots.
    }
\label{tab:NCO-1-FP-rate-no-filter}
\end{table}



We further evaluated the imprecision on this percentage by calculating, for each sample individually\footnote{With the exception of the four samples which were lowly sequenced}, the ratio between the latter number of FPs expected in the sample due to sequencing errors and the total number of fragments in the sample. 
We sequentially applied the multiplier of each sample to the total number of fragments and finally determined that the proportion of FPs due to sequencing errors capped between 60 and 78\% of all FPs.\\


Therefore, the largest part (66.7\%, CI $= [60\%; 78\%$]) of FPs arose from sequencing errors.
The next step thus consisted in estimating the $\frac{WS}{WS+SW}$ ratio expected because of these sequencing errors.
To do this, we simply computed the total number of FPs containing an erroneous W~$\rightarrow$~S base call ($E_{W\rightarrow S}$) and the number containing an erroneous S~$\rightarrow$~W base call ($E_{S\rightarrow W}$) as follows:

\begin{align}
        E_{W\rightarrow S}&= e_{A\rightarrow C} + e_{A\rightarrow G} + e_{T\rightarrow C} + e_{T\rightarrow G}, \\
        E_{S\rightarrow W}&= e_{C\rightarrow A} + e_{C\rightarrow T} + e_{G\rightarrow A} + e_{G\rightarrow T}
\end{align}

Importantly, we found that $E_{S\rightarrow W}$ was greater than $E_{W\rightarrow S}$, i.e.\ S bases were more oftenly mistakenly sequenced as W bases than the other way round.
More precisely, we found that the $\frac{WS}{WS+SW}$ ratio expected with such FPs (i.e.\ $\frac{E_{W\rightarrow S}}{E_{W\rightarrow S} + E_{S\rightarrow W}}$) equalled 0.39.

We note that this estimate was slightly higher than the $\frac{WS}{WS+SW}$ observed in control regions (0.31), possibly because the non-negligible portion (33.3\%) of FPs that did not originate from these sequencing errors may somehow also bias the ratio.









% \subsection{Source code to reproduce figures}
%
% The source code to reproduce figures will be put online shortly.
%


% SRA et reproductibilite des figures
% mettre aussi la liste des positions genomiques obtenues

\hypersetup{linkcolor=titlepagecolorsection}
\section{Supplementary figures for Chapter~\ref{ch:6-recombination-parameters}}
\hypersetup{linkcolor=black}

\subsection{Recombination rate and DMC1 binding affinity}

\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/Correlation_intensity_DMC1.eps}
    \caption[Proportionality between the recombination rate and DMC1 binding intensity]
    {\textbf{Proportionality between the recombination rate and DMC1 binding intensity.}
        \par All 1,018 hotspots were divided into 10 classes of increasing DMC1 signal (x-axis), i.e.\ the number of DMC1 ChIP-seq tags on each PRDM9 ChIP-seq peak (DMC1 ChIP-seq data on B6xCAST hybrid mice from \citealp{smagulova2016evolutionary}).
        The observed number of recombination events identified per sequenced Mb (left y-axis) was converted into a CO rate (right y-axis) as detailed in Chapter~\ref{ch:6-recombination-parameters}.
        The points and error bars respectively represent the mean number of events (or CO rate) and the standard error on the mean for hotspots of each class.
    }
\label{fig:correlation-DMC1-standard-error}
\end{figure}



\subsection{Hotspot asymmetry and DMC1 binding affinity}


\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/Recombination_rates_2_axes_sym_vs_Asym_DMC1_with_legend_and_outliers-corrected_CO_x2.eps}
    \caption[Asymmetric hotspots display lower recombinational activity than expected by their DMC1 binding affinity]
    {\textbf{Asymmetric hotspots display lower recombinational activity than expected by their DMC1 binding affinity.}
        \par All 1,018 hotspots were divided into 10 classes of increasing DMC1 signal (x-axis), i.e.\ the number of DMC1 ChIP-seq tags on each DMC1 ChIP-seq peak (DMC1 ChIP-seq data on B6xCAST hybrid mice from \citealp{smagulova2016evolutionary}).
        The observed number of recombination events identified per sequenced Mb (left y-axis) was converted into a CO rate (right y-axis) as detailed in Chapter~\ref{ch:6-recombination-parameters}.
        Symmetric hotspots (green, $N = 650$) were distinguished from asymmetric hotspots (orange, $N = 236$) as detailed in Chapter~\ref{ch:7-quantification-BGC}.
        The linear regression model for symmetric (slope~$= 1.8$; intercept~$=0$; \textit{p}-val~$< 2.2 \times 10^{-16}$) and asymmetric (slope~$= 8.6$; intercept~$=0$; \textit{p}-val~$< 2.2 \times 10^{-16}$) hotspots are drawn as dotted lines.
    }
\label{fig:asymmetry-and-DMC1}
\end{figure}



\newpage
\subsection{Distribution of switch points}


\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/density_switch_points.eps}
    \caption[Distribution of switch points along hotspots for Rec-1S and Rec-2S events]
    {\textbf{Distribution of switch points along hotspots for Rec-1S and Rec-2S events.}
}
\label{fig:density-switch-points}
\end{figure}



\newpage
\hypersetup{linkcolor=titlepagecolorsection}
\section{Supplementary figures for Chapters~\ref{ch:7-quantification-BGC} and~\ref{ch:8-HFM1}}
\hypersetup{linkcolor=black}

\subsection{Correlation between expected and observed donor}



\begin{figure}[h!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/appendices/CorrelationDMC1_FINAL_BIS_on_lab_computer_COLOURS.eps}
    \caption[Correlation between the expected and observed proportions of CAST-donor fragments across hotspots displaying at least 5 events, coloured per PRDM9 target]
    {\textbf{Correlation between the expected and observed proportions of CAST-donor fragments across hotspots displaying at least 5 events, coloured per PRDM9 target.}
        \par The expected proportion of CAST-donor fragments (x-axis) was based on the probability that the DSB initiates on the B6 haplotype from DMC1 ssDNA-sequencing (SSDS) data by \citet{smagulova2016evolutionary} (see main text).
        Only the 582 hotspots displaying a minimum of 5 recombination events were reported in this figure.
        The Pearson correlation between the two measures gave: $R^2 = 0.66$; {\textit{p}-val $< 2.2 \times 10^{-16}$}.
}
\label{fig:correl-donor-DMC1-with-colour}
\end{figure}



\subsection{Genetic background of all chromosomes}


% LEFT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-3.4cm
	\rightskip-2.7cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth]{figures/chap8/HFM1_background_28355-DOM.eps}
	\captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28355]
	{\textbf{Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28355.}
		\par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		Each segment corresponds to the position of a target (hotspot or control region) and was coloured in red when the background inferred was BD/BD (homozygous) and in blue when the background inferred was BD/CAST (heterozygous).
	}
\label{fig:mosaic-backgrounds-2}
\end{sidewaysfigure}


% RIGHT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-2.4cm
	\rightskip-2.4cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth]{figures/chap8/HFM1_background_28367-DOM.eps}
	\captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28367]
	{\textbf{Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28367.}
		\par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		Each segment corresponds to the position of a target (hotspot or control region) and was coloured in red when the background inferred was BD/BD (homozygous) and in blue when the background inferred was BD/CAST (heterozygous).
	}
\label{fig:mosaic-backgrounds-3}
\end{sidewaysfigure}


% LEFT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-3.4cm
	\rightskip-2.7cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth]{figures/chap8/HFM1_background_28371-DOM.eps}
	\captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28371]
	{\textbf{Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28371.}
		\par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		Each segment corresponds to the position of a target (hotspot or control region) and was coloured in red when the background inferred was BD/BD (homozygous) and in blue when the background inferred was BD/CAST (heterozygous).
	}
\label{fig:mosaic-backgrounds-4}
\end{sidewaysfigure}





\newpage
% \subsection{Pairwise recombination rates of shared hotspots}
\subsection{Pairwise comparison of the RR in shared hotspots}



\begin{figure}[p]
    \centering
    \begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28371 (WT) and 28353 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28371_vs_28353.eps}
    \end{subfigure}

    \vspace{0.5cm}

    \begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28371 (WT) and 28367 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28371_vs_28367.eps}
    \end{subfigure}

    \caption[Correlation of the number of recombination events in shared hotspots for all pairs of mice]
    {\textbf{Correlation of the number of recombination events in shared hotspots for all pairs of mice.}
        % \par The linear regression was significant for the WT mice (slope $=1.03$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 257$) and for the two mutant mice (slope = $0.69$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 241$).
    }
\label{fig:pairwise-RR-shared-BIS-1}
\end{figure}




\begin{figure}[p]
    \centering
   
	\begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28355 (WT) and 28367 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28355_vs_28367.eps}
    \end{subfigure}

    \vspace{0.5cm}
    
	\begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between 28355 (WT) and 28353 (mutant)}
        \includegraphics[width=\textwidth]{figures/chap8/28355_vs_28353.eps}
    \end{subfigure}


    \caption[Correlation of the number of recombination events in shared hotspots for all pairs of mice]
    {\textbf{Correlation of the number of recombination events in shared hotspots for all pairs of mice.}
        % \par The linear regression was significant for the WT mice (slope $=1.03$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 257$) and for the two mutant mice (slope = $0.69$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots} = 241$).
    }
\label{fig:pairwise-RR-shared-BIS2}
\end{figure}







% + liste de tous les hotspots analyses chez les humains (ou pour moi, plutot souris) dans annexe B de Popa.



%%% CHAPITRE 2
%% Si je veux metttre une appendix sur les defects des souris qui sont mutantes pour un gene de la meiose: Bolcun-Filas, E., & Schimenti, J. C. (2012). Genetics of Meiosis and Recombination in Mice. International Review of Cell and Molecular Biology, 179–227. doi:10.1016/b978-0-12-394309-5.00005-5 (appendix)

