\begin{savequote}[8cm]
	
	% ‘You are always taught to do as much as you can. Always put checks in. Always look for exceptions. Always handle the most general case. Always give the user the best advice. Always print a meaningful error message. Always this. Always that. You have so many things in the background that you're supposed to do, there's no room left to think. I say, forget all that and ask yourself, “What's the simplest thing that could possibly work?”
	% I think the advice got turned into a command: “Do the simplest thing that could possibly work.” That's a little more confusing, because there isn't this notion that as soon as you've done it, we'll evaluate it.’
	
	% "The Simplest Thing that Could Possibly Work: A Conversation with Ward Cunningham, Part V". Interview with Bill Venners, www.artima.com. January 19, 2004.
	% https://www.artima.com/intv/simplest3.html
	
	
	% Talk is cheap. Show me the code. — Linus Torvalds

	‘The assumption is that when something turns out to not be ideal, it will be refactored again. Everything is subject to refactoring.’
	% https://www.artima.com/intv/ownership.html
	\qauthor{--- Ward Cunningham, \textit{\usebibentry{cunningham2003collective}{title}} \citeyearpar{cunningham2003collective} }
	
\end{savequote}

% \chapter{\label{ch:8-HFM1}Impact of HFM1 knock-out on recombination in mice}
% \chapter{\label{ch:8-HFM1}Adaptation to studies of knock-out mutants}
% \chapter{\label{ch:8-HFM1}Adaptation to other studies of knock-out mutants}
% \chapter{\label{ch:8-HFM1}Adaptation of the method to other studies on recombination: Hfm1 knock-out mutants}
% \chapter{\label{ch:8-HFM1}Adaptation of the method to other studies on recombination}
% \chapter{\label{ch:8-HFM1}Methodological adaptations to study recombination-dependent gene knock-out mutants}
% \chapter{\label{ch:8-HFM1}Methodological adaptations to study recombination-dependent genes}
\chapter{\label{ch:8-HFM1}Methodological adaptations to other studies of recombination}
% \chapter{\label{ch:8-HFM1}Methodological adaptations to study \textit{Hfm1} knock-out mutants}

%\otherpagedecoration


\minitoc{}

{\small{} \itshape{}

\paragraph{This chapter in brief —}

The method we previously implemented to detect recombination in single individuals can be used to study the role of genes essential to the process of recombination.
% We previously implemented an approach allowing to detect recombination in single individuals which could be useful to study the role of genes essential to recombination.
This requires the use of individuals homozygous for the mutant version of the gene but nonetheless displaying a high level of heterozygosity for recombination to be detectable.
As this can only be achieved with F2 individuals,
% As such, recombination events should be searched for in the hotspots of F2 individuals fulfilling these criteria.
% The classical approach to understand the specific role of a given recombination-dependent gene consists in studying individuals with an inactivated version of the gene.
% % Even though many of the actors of the recombination process are known, the specific role of some of them remains to be elucidated.
% % Classically, this can be done by studying individuals for which a given recombination-dependent gene has been inactivated.
% Such introgression of a knocked-in (KI) gene into a hybrid can be done by crossing one of its ancestors with a third strain containing the non-functional allele.
% Thus, recombination events should be searched for in the hotspots of a F2 individual containing three distinct genetic backgrounds.
%
we adapted the method we implemented for simple F1 hybrids to such design.
Basically, we had to distinguish the polymorphic sites expressing variation between the two parental genomes from those originating from the third introgressed genome.
% In the end, our method was as powerful as the original one for simple F1 hybrids.
This implementation was as powerful as the original method and we could thus study the role of the interaction between HFM1 and MLH1:
% , a DNA helicase important for CO control: we observed that its inactivation led to increased recombination rate and shortened CO conversion tracts.
we observed that impeding this interaction led to an increased recombination rate and shortened CO conversion tracts.

}

\newpage

% - on a une methode originale qui permet d'etudier des recombinants chez des individus uniques
% - donne possib de regarder la mutation d'un gene donne sur la recombi
% - B et V interesses par HF1 car
% - ont realise capture seq chez des mutatns
% - la difficulte d'analyse vient du fait qu'il faut des individus homoz pour HFM1 et heterozygotes sur le reste du genome pour detecter la recombinaison
% - possible en faisant plusieurs croisements mais implique de modifier le protocole d'analyse


As the method we implemented in Chapter~\ref{ch:5-methodology} allows to detect recombination in single individuals, it can be used to study the individual role of genes involved in the process of recombination.
In particular, Bernard de Massy and Valérie Borde are interested in the specific role of the mouse gene \textit{Hfm1} whose yeast homologue (\textit{MER3}) codes for a meiosis-specific DNA helicase \citep{nakagawa1999saccharomyces,nakagawa2002saccharomyces} that participates in CO control and in DNA heteroduplex extension \citep{mazina2004saccharomyces,nakagawa2002mer3}.
This gene is also essential to CO formation in other fungi \citep{sugawara2009coprinus}, plants \citep{mercier2005two,chen2005arabidopsis}, humans \citep{tanaka2006hfm1} and mice \citep{guiraldelli2013mouse}.

It was recently shown that, in yeasts, Mer3 can connect the MutL\textgreek{b} heterodimer of Mlh1-Mlh2 and that this interaction limits CT lengths genome-wide \citep{duroc2017concerted}.
In mice, the interplay between HFM1 and MLH1 is conserved, but whether or not its role in regulating CT length is also maintained remains a mystery.
% A recent study on yeasts showed that Mer3 can connect the MutL\textgreek{b} heterodimer of Mlh1-Mlh2, that this interaction limits CT lengths genome-wide and that, in mice, the interplay between HFM1 and MLH1 is conserved \citep{duroc2017concerted}.
% In mice, the interaction between HFM1 and MLH1 is conserved \citep{duroc2017concerted} but whether or not its role in regulating CT length also applies to mice remains a mystery.
% Though, even if it has been shown that it diminishes the recombination rate in mice \citep{guiraldelli2013mouse}, its impact on other parameters of recombination, like the length of conversion tracts, has not been established yet.%\paragraphbreak
% To analyse its impact on recombination, it is necessary to have individuals that are homozygous for the inactivated version of the gene (\textit{Hfm1\textsuperscript{-/-}}) but which nonetheless exhibit a high level of heterozygosity (for recombination to be detectable).
% This was achieved in the laboratories of Bernard de Massy and Valérie Borde by introgressing the mutant gene into F2 individuals, as I detail in the first section of this chapter.
To find that out, the laboratories of Valérie Borde and Bernard de Massy introgressed a punctual mutation that impedes the interaction between HFM1 and MLH1 (\textit{Hfm1\textsuperscript{KI}}) into F2 individuals, as I detail in the first section of this chapter.
% exhibiting a high level of heterozygosity.
In this experimental design, the individuals studied contain three genetic backgrounds and thus, our method to detect recombination needs to be refactored.
%, which calls and thus required to refactor our method to study recombination.
% To analyse its impact on recombination, it is necessary to have individuals that are homozygous for the mutant version of the gene but which nonetheless exhibit a high level of heterozygosity (for recombination to be detectable).
% % Thus, the \textit{Hfm1\textsuperscript{KI}} mutation was introgressed into F2 individuals as detailed in the first section of this chapter.
% Though, this novel experimental design required to refactor our method to study recombination in these F2 individuals containing a third genetic background.
I describe in the last two sections of this chapter how we worked this out and what the preliminary results of this analysis were.

% In this chapter, I first detail how this was achieved in the laboratories of Bernard de Massy and Valérie Borde by introgressing the mutant gene into F2 individuals
% Next, I describe how we refactored our method to study recombination in these F2 individuals containing a third genetic background and what the primary results of this analysis were.
%
% This can be achieved by studying F2 individuals, but brings the .
% Thus, Bernard de Massy's and Valérie Borde's laboratories introgressed the inactivated
%
% In the context of a collaboration with their laboratories, we thus
%
% Though, as compared to the


% Even though it is nowadays largely acknowledged that the formation of recombination events (COs and NCOs) depends on a complex network of proteins (see Chapter~\ref{ch:2-recombination-mechanistics}), the specific role and impact of many of these actors has not been elucidated yet.
% In particular, the yeast gene \textit{MER3} (mouse homologue: \textit{Hfm1}) codes for a meiosis-specific DNA helicase \citep{nakagawa1999saccharomyces,nakagawa2002saccharomyces} that participates in CO control and in DNA heteroduplex extension \citep{mazina2004saccharomyces,nakagawa2002mer3}.
% This gene is also essential to CO formation in other fungi \citep{sugawara2009coprinus}, plants \citep{mercier2005two,chen2005arabidopsis}, humans \citep{tanaka2006hfm1} and mice \citep{guiraldelli2013mouse}.
% Though, even if it has been shown that it diminishes the recombination rate in mice \citep{guiraldelli2013mouse}, its impact on other parameters of recombination, like the length of conversion tracts, has not been established.
%
% Because the method that we implemented (see Chapter~\ref{ch:5-methodology}) showed unprecedented power to detect recombination in single individuals, it seemed possible to study the role of \textit{Hfm1} by comparing the recombinational outcome of wild-type and \textit{hfm1} mutant (\textit{Hfm1\textsuperscript{-/-}}) mice.
% In this chapter, I detail how the mutant gene was introgressed into F2 mice descending from B6 and CAST strains, how we refactored our method to study recombination in these F2 individuals containing a third genetic background and what the primary results of this analysis were.
%



\section{Experimental design}
\subsection{Introgression of the mutant \textit{hfm1} allele}
% knock-out (KO) gene}



\begin{table}[p]
	\begin{subtable}[h]{\textwidth}
		
		\subcaption{Ancestry of S28353 and S28355.} 
		\centering
		\begin{adjustbox}{width = 1\textwidth}
			\begin{tabular}{rrrrrrrr}

				\toprule
				\textbf{Mouse ID} & \textbf{Relationship} & \textbf{\% B6} & \textbf{\% DBA2} & \textbf{\% CAST} & \textbf{HFM1} & \textbf{Mother} & \textbf{Father} \\

				\midrule
				39856   & Maternal grandmother  & 0.0   & 0.0   & 100.0 & WT/WT & N/A    & N/A \\
				28130   & Maternal grandfather  & 75.0  & 25.0  & 0.0   & KI/WT & 72205 & N/A \\
				F0\#2 (72205)   & Paternal grandmother  & 50.0  & 50.0  & 0.0   & KI/WT & N/A    & N/A \\
				N/A      & Paternal grandfather  & 100.0 & 0.0   & 0.0   & WT/WT & N/A    & N/A \\
				\midrule
				22228   & Mother                & 37.5  & 12.5  & 50.0  & KI/WT & 39856 & 28130 \\
				28196   & Father                & 75.0  & 25.0  & 0.0   & KI/WT & 72205 & N/A \\
				\midrule
				28353   & Mutant analysed       & 56.25 & 18.75 & 25.0  & KI/KI & 22228 & 28196 \\
				28355   & WT analysed           & 56.25 & 18.75 & 25.0  & WT/WT & 22228 & 28196 \\
				\bottomrule

			\end{tabular}
		\end{adjustbox}
\label{tab:ancestry-28353-28355}
	\end{subtable}
	\vspace{2cm}


	\begin{subtable}[h]{\textwidth}

		\subcaption{Ancestry of S28367.} 
		\centering
		\begin{adjustbox}{width = 1\textwidth}
			\begin{tabular}{rrrrrrrr}

				\toprule
				\textbf{Mouse ID} & \textbf{Relationship} & \textbf{\% B6} & \textbf{\% DBA2} & \textbf{\% CAST} & \textbf{HFM1} & \textbf{Mother} & \textbf{Father} \\

				\midrule
				F0\#3 (72212)  & Maternal grandmother  & 50.0  & 50.0  & 0.0   & KI/WT & N/A    & N/A \\
				N/A      & Maternal grandfather  & 100.0 & 0.0   & 0.0   & WT/WT & N/A    & N/A \\
				28163   & Paternal grandmother  & 75.0  & 25.0  & 0.0   & KI/WT & 72205 & N/A \\
				39978   & Paternal grandfather  & 0.0   & 0.0   & 100.0 & WT/WT & N/A    & N/A \\
				\midrule
				28172   & Mother                & 75.0  & 25.0  & 0.0   & KI/WT & 72212 & N/A \\
				28238   & Father                & 37.5  & 12.5  & 50.0  & KI/WT & 28163 & 39978 \\
				\midrule
				28371   & Mutant analysed       & 56.25 & 18.75 & 25.0  & KI/KI & 28172 & 28238 \\
				\bottomrule

			\end{tabular}
		\end{adjustbox}
\label{tab:ancestry-28367}
	\end{subtable}
	\vspace{2cm}


	\begin{subtable}[h]{\textwidth}

		\subcaption{Ancestry of S28371.}
		\centering
		\begin{adjustbox}{width = 1\textwidth}
			\begin{tabular}{rrrrrrrr}

				\toprule
				\textbf{Mouse ID} & \textbf{Relationship} & \textbf{\% B6} & \textbf{\% DBA2} & \textbf{\% CAST} & \textbf{HFM1} & \textbf{Mother} & \textbf{Father} \\

				\midrule
				39856   & Maternal grandmother  & 0.0   & 0.0   & 100.0 & WT/WT & N/A    & N/A \\
				28130   & Maternal grandfather  & 75.0  & 25.0  & 0.0   & KI/WT & 72205 & N/A \\
				F0\#2 (72205)  & Paternal grandmother  & 50.0  & 50.0  & 0.0   & KI/WT & N/A    & N/A \\
				N/A      & Paternal grandfather  & 100.0 & 0.0   & 0.0   & WT/WT & N/A    & N/A \\
				\midrule
				28250   & Mother                & 37.5  & 12.5  & 50.0  & KI/WT & 39856 & 28130 \\
				28198   & Father                & 75.0  & 25.0  & 0.0   & KI/WT & 72205 & N/A \\
				\midrule
				28371   & WT analysed           & 56.25 & 18.75 & 25.0  & WT/WT & 28250 & 28198 \\
				\toprule

			\end{tabular}
		\end{adjustbox}
\label{tab:ancestry-28371}
	\end{subtable}
	\vspace{1cm}


	\caption[Genealogy of the four mice analysed]
	{\textbf{Genealogy of the four mice analysed.}
		\par The genealogies (parents and grandparents) of each of the two mutant mice (IDs: 28353 and 28371) and of the two wild-type (WT) mice (IDs: 28355 and 28367) analysed in this study, as well as the characteristics (background composition in B6, CAST and DBA2 genomes, and the \textit{Hfm1} alleles carried: either the mutant impeding the interaction between HFM1 and MLH1 (KI) or the wild-type (WT) allele) of all the individuals involved in the ancestry are reported in the subtables above: \textbf{(a)} 28353 and 28353; \textbf{(b)} 28367; \textbf{(c)} 28371.
	}
\label{tab:genealogies}
\end{table}

A mutant \textit{Hfm1} allele (\textit{Hfm1\textsuperscript{KI}}) was introduced in the zygote of a cross between two F1 mice deriving from hybridisations between two \textit{Mus musculus domesticus} strains: strain C57BL/6J, hereafter called B6 and strain DBA/2J, hereafter called DBA2.
The resulting founder mice (F0\#2 and F0\#3) were thus heterozygous for the \textit{Hfm1} gene (\textit{Hfm1\textsuperscript{WT/KI}}) and their genetic backgrounds were composed of 50\% DBA2 and 50\% B6 genomes.
Further crosses with other B6 and \textit{Mus musculus castaneus} (strain CAST/EiJ, hereafter called CAST) mice resulted in individuals carrying either two mutant alleles for \textit{Hfm1} (\textit{Hfm1\textsuperscript{KI/KI}}), two WT alleles (\textit{Hfm1\textsuperscript{WT/WT}}) or one allele of each (\textit{Hfm1\textsuperscript{WT/KI}}).
The genetic backgrounds for these mice were composed of a mixture of B6, DBA2 and CAST genomes (Table~\ref{tab:genealogies}).
Of these, two \textit{hfm1} homozygous mutant (28353 and 28367) and two WT (28355 and 28371) male mice were selected for further analysis: their sperm DNA was extracted and sonicated to produce fragments of a mean size of 450 bp.

\subsection{Target selection, DNA capture and sequencing}
% \subsection{Selection of target hotspots}

Like in Chapter~\ref{ch:5-methodology}, we selected hotspots from the list identified by \citet{baker2015prdm9} on the basis of PRDM9 ChIP-seq peak detection.
We used the same criteria as before: a minimum of 4 SNPs in the 300-bp central region, a strict maximum of 60 sites with low sequence quality in the 1-kb central region and at least 90\% of identity between the B6 and the CAST reference genome on at least 80\% of the selected region.

Though, since the main aim of this analysis was to test for any effect of the \textit{Hfm1} mutation on CO CT length, we extended the width of our selected hotspots to 3 kb.
Thus, the third selection criterium discarded a larger number of candidate hotspots than in Chapter~\ref{ch:5-methodology}, since identity was required on 3 kb instead of 1 kb.
In the end, 890 3-kb long hotspots were retained and, as in Chapter~\ref{ch:5-methodology}, 500 control regions were added to that list of targets.\\



% \subsection{DNA capture, sequencing and mapping}

For the efficiency of DNA capture to be identical in both haplotypes, two baits were designed for each of the 1,390 targets: one corresponding to the CAST haplotype and one to the B6 haplotype.
We then performed two successive rounds of DNA capture on each of the four DNA samples from the four mice.
Libraries were then sequenced by an Illumina device using a 250-bp paired-end protocol, and the sequenced reads were mapped onto the B6 and the CAST reference genomes as described in Chapter~\ref{ch:5-methodology}.
Overall, read mapping statistics and capture efficiency were similar to what was found in Chapter~\ref{ch:5-methodology} (Table~\ref{tab:HFM1-statistics-seq-mapping-capture}).




\begin{table}[t]
    \centering
	\begin{adjustbox}{width = 1\textwidth}
    \begin{tabular}{rrrrrrr}
        \toprule
		\multicolumn{2}{c}{\textbf{Sample}} & \multicolumn{2}{c}{\textbf{Mapping (\%)}} & \multicolumn{3}{c}{\textbf{Capture efficiency}} \\
		\cmidrule(l){1-2} \cmidrule(l){3-4} \cmidrule(l){5-7}
        \textbf{Library} & \textbf{Library} & \textbf{Ref.} & \textbf{Ref.} & \textbf{\# Filtered} & \textbf{\% in} & \textbf{\# in} \\
        \textbf{ID} & \textbf{size} & \textbf{B6} & \textbf{CAST} & \textbf{Fragments} & \textbf{targets} & \textbf{targets} \\
		\midrule
        % \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-12}
        28355 & 164,210,468 & 98.76 & 98.00 & 162,168,344 & 48.62 & 78,851,718 \\
        28371 & 171,930,499 & 98.25 & 98.20 & 170,081,808 & 48.63 & 82,713,025 \\
		\textbf{Total WT} & \textbf{336,140,967} & \textbf{98.84} & \textbf{98.10} & \textbf{332,250,152} & \textbf{48.63} & \textbf{161,564,743} \\
		\\
        28353 & 161,294,272 & 99.15 & 98.35 & 159,920,297 & 48.62 & 78,851,718 \\
        28367 & 227,590,570 & 97.91 & 97.18 & 222,826,196 & 37.11 & 82,713,025 \\
		\textbf{Total mutants} & \textbf{388,884,842} & \textbf{98.42} & \textbf{97.67} & \textbf{382,746,493} & \textbf{48.63} & \textbf{186,150,465} \\
        % \textbf{Total} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{976,826,736} & \textbf{99.68} & \textbf{99.37} & \textbf{468,513,483} & \textbf{71.63} & \textbf{335,605,734} \\
        \bottomrule
    \end{tabular}
	\end{adjustbox}
    \caption[Sequencing, mapping and capture-efficiency summary metrics]
    {\textbf{Sequencing, mapping and capture-efficiency summary metrics.}
        \par Reads were mapped onto the B6 and CAST reference genomes, and fragments were filtered as described in Chapter~\ref{ch:5-methodology}.
		The lines in bold represent the totals for the two WT and the two mutant mice.
    }
\label{tab:HFM1-statistics-seq-mapping-capture}
\end{table}

%
% # TOUT CELA SUREMENT PAS LES BONS CHIFFRES.... (VOIR COMMENT FAIT DANS LE CHAP 5)
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28353*bamtools_counts.txt|paste -sd+|bc
% 161294272
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28355*bamtools_counts.txt|paste -sd+|bc
% 164210468
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28367*bamtools_counts.txt|paste -sd+|bc
% 227590570
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28371*bamtools_counts.txt|paste -sd+|bc
% 171930499
%
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28353*.bamtools_stats.txt|grep Proper
% 'Proper-pairs':    37937381	(99.0255%)
% 'Proper-pairs':    38687806	(98.9575%)
% 'Proper-pairs':    41273169	(99.3141%)
% 'Proper-pairs':    42021941	(99.2724%)
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28355*.bamtools_stats.txt|grep Proper
% 'Proper-pairs':    40670865	(98.9692%)
% 'Proper-pairs':    41412074	(98.9033%)
% 'Proper-pairs':    39693167	(98.6611%)
% 'Proper-pairs':    40392238	(98.4867%)
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28367*.bamtools_stats.txt|grep Proper
% 'Proper-pairs':    65968009	(99.2325%)
% 'Proper-pairs':    66943830	(99.1935%)
% 'Proper-pairs':    44628688	(96.1375%)
% 'Proper-pairs':    45285669	(95.9392%)
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/01_domesticus/Bamtools_stats/S28371*.bamtools_stats.txt|grep Proper
% 'Proper-pairs':    42446254	(98.9801%)
% 'Proper-pairs':    43273542	(98.8626%)
% 'Proper-pairs':    41772008	(98.9598%)
% 'Proper-pairs':    42590004	(98.8984%)
%
% >>> 37937381+38687806+41273169+42021941
% 159920297
% >>> 40670865+41412074+39693167+40392238
% 162168344
% >>> 65968009+66943830+44628688+45285669
% 222826196
% >>> 42446254+43273542+41772008+42590004
% 170081808
% >>>
% >>> 159920297/161294272
% 0.9914815635858414
% >>> 162168344/164210468
% 0.9875639840451584
% >>> 222826196/227590570
% 0.979066030723505
% >>> 170081808/171930499
% 0.9892474516694097
% >>>
% >>> (162168344+170081808)/(164210468+171930499)
% 0.9884250496607871
% >>> (159920297+222826196)/(161294272+227590570)
% 0.9842155097420845
%
% ## CAST
% >>> 37946282+38695093+41289188+42036366
% 159966929
% >>> 40659331+41399188+39667619+40363612
% 162089750
% >>> 66024330+67000106+44576996+45230530
% 222831962
% >>> 42423366+43247282+41767287+42584094
% 170022029
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/02_castaneus//Bamtools_stats/S28353*bamtools_counts.txt|paste -sd+|bc
% 162652742
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/02_castaneus//Bamtools_stats/S28355*bamtools_counts.txt|paste -sd+|bc
% 165395876
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/02_castaneus//Bamtools_stats/S28367*bamtools_counts.txt|paste -sd+|bc
% 229294851
% pbil-deb ➜  3_Working_directory cat $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/1_Mapping/02_castaneus//Bamtools_stats/S28371*bamtools_counts.txt|paste -sd+|bc
% 173131818
%
% >>> 159966929/162652742
% 0.983487440992541
% >>> 162089750/165395876
% 0.9800108317090083
% >>> 222831962/229294851
% 0.9718140683412032
% >>> 170022029/173131818
% 0.9820380272330993
% >>>
% >>>
% >>> (159966929+222831962)/(162652742+229294851)
% 0.9766583539141673
% >>> (162089750+170022029)/(165395876+173131818)
% 0.9810475919290668
%
%
% # IN TARGETS
% pbil-deb ➜  3_Working_directory awk 'NR>1 {print $5}' $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/2_Targetting/Tot_nb_fragments_per_hotspot_MUT_S28353.txt|paste -sd+|bc
% 78851718
% pbil-deb ➜  3_Working_directory awk 'NR>1 {print $5}' $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/2_Targetting/Tot_nb_fragments_per_hotspot_MUT_S28367.txt|paste -sd+|bc
% 82713025
% pbil-deb ➜  3_Working_directory awk 'NR>1 {print $5}' $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/2_Targetting/Tot_nb_fragments_per_hotspot_WT_S28371.txt|paste -sd+|bc
% 82713025
% pbil-deb ➜  3_Working_directory awk 'NR>1 {print $5}' $DATA/5_HFM1/Sequencing_1/BEFORE_05_Analyses_of_Recombinants/03_General_Statistics/02_Bias_checks/2_Targetting/Tot_nb_fragments_per_hotspot_WT_S28355.txt|paste -sd+|bc
% 78851718
%




% \section{Detection of recombination in F2 individuals}
\subsection{Expected genetic background composition}


The point mutation on \textit{Hfm1} originated from B6/DBA2-background founder mice (F0\#2 and F0\#3) and was introgressed into a B6xCAST hybrid \textit{via} two consecutive crosses: 
on the one hand, the founder mice were crossed with B6/B6-background mice, thus yielding one 75\%-B6/25\%-DBA2 parent;
on the other hand, other 75\%-B6/25\%-DBA2 mice were crossed with CAST mice to yield a second parent with a background composed of 37.5\% B6, 12.5\% DBA2 and 50\% CAST genomes (Table~\ref{tab:genealogies}).
Each of the four selected mice (28353, 28355, 28367 and 28371) were then obtained by crossing the two aforementioned parents together.
Thus, their background encompassed 56.25\% B6, 18.75\% DBA2 and 25\% CAST genomes.



\begin{table}[t]
    \centering
    % \begin{adjustbox}{width = 1\textwidth}
    \begin{tabular}{rrrr}

        \toprule
		\multicolumn{2}{c}{\textbf{Detailed}} & \multicolumn{2}{c}{\textbf{Simplified}} \\
		\cmidrule(l){1-2} \cmidrule(l){3-4}
        \textbf{Background} & \textbf{\% expected} & \textbf{Background} & \textbf{\% expected} \\

        % \midrule
		\cmidrule(l){1-2} \cmidrule(l){3-4}
        B6/B6     & 28.125 & \multirow{3}{*}{DOM/DOM} & \multirow{3}{*}{50.0} \\
        B6/DBA2   & 18.750 & & \\
        DBA2/DBA2 & 3.125 & & \\
		% \midrule
		\cmidrule(l){1-2} \cmidrule(l){3-4}
        B6/CAST   & 37.500 & \multirow{2}{*}{DOM/CAST} & \multirow{2}{*}{50.0} \\
        DBA2/CAST & 12.500 & & \\
        % \midrule
		\cmidrule(l){1-2} \cmidrule(l){3-4}
		CAST/CAST & 0.000 & CAST/CAST & 0.0 \\
        \bottomrule

    \end{tabular}
    % \end{adjustbox}
    \caption[Expected distribution of genetic backgrounds in the mice analysed]
	{\textbf{Expected distribution of genetic backgrounds in the mice analysed.}
		\par Because the B6 and DBA2 genomes present high sequence conservation \citep{davis2005genomewide}, we regrouped them under the label ‘DOM’.
		The expected genomic proportion (and thus proportion of targets) in each of the six possible ‘detailed’ backgrounds were reported on the left panel and the expected proportions in each of the three ‘simplified’ backgrounds were reported on the right panel.
	} 
\label{tab:background-expected}
\end{table}




% \begin{table}[t]
%     \centering
%     % \begin{adjustbox}{width = 1\textwidth}
%     \begin{tabular}{rrr}
%
%         \toprule
%         \textbf{Background (simplified)} & \textbf{Background (detailed)} & \textbf{\% expected} \\
%
%         \midrule
%         DOM/DOM       & B6/B6     & 28.125 \\
%         DOM/DOM       & B6/DBA2   & 18.750 \\
%         DOM/DOM       & DBA2/DBA2 & 3.125 \\
%         DOM/CAST     & B6/CAST   & 37.500 \\
%         DOM/CAST     & DBA2/CAST & 12.500 \\
%         CAST/CAST   & CAST/CAST & 0.000 \\
%         \bottomrule
%
%     \end{tabular}
%     % \end{adjustbox}
%     \caption[Expected distribution of genetic backgrounds in the four mice analysed]
%     {\textbf{Expected distribution of genetic backgrounds in the four mice analysed.}
%         \par
%     }
% \label{tab:background-expected}
% \end{table}
%
%Frequence de
%- background B6/B6: 28.125
%- background B6/DBA2: 18.75
%- background DBA2/DBA2: 3.125
%- background B6/CAST: 37.5
%- background CAST/CAST: 0
%- background DBA2/CAST: 12.5





	
More precisely, the expected genomic proportion (and therefore, the expected proportion of targets) of each genetic background were those reported in Table~\ref{tab:background-expected}.
Overall, 68.75\% of the targeted loci were expected to be heterozygous (either B6/DBA2, B6/CAST or DBA2/CAST) and could, in principle, be used to detect recombination events.

However, the power to detect recombination depends on the density of heterozygous sites, and the latter is much lower at B6/DBA2-background targets than at B6/CAST- or DBA2/CAST-background loci.
Indeed, the B6 and the DBA2 genomes present a low sequence divergence of 0.2\% \citep{keane2011mouse} because these two strains derive from the same mouse subspecies (\textit{Mus musculus domesticus}) from which they inherited large genomic regions \citep{davis2005genomewide}.
We note that, since the latter two strains derive from the same subspecies, we will regroup the labels B6 and DBA2 under a more general notation: ‘DOM’.
In comparison, as the DOM (B6 or DBA2) and CAST strains derive from two distinct subspecies which diverged about 350,000 to 500,000 years ago \citep{geraldes2008inferring}, they present a much higher genome-wide divergence of 0.74\% \citep{keane2011mouse}.

Therefore, in order to avoid any spurious fluctuation in detectability between individuals and to thus allow the comparison of recombination rates across samples, we chose to search for recombination events exclusively in one type of heterozygous background.
And, so as to maximise the detectability of recombination events, we focused on the background displaying the highest rate of polymorphism: DOM/CAST-background targets.
The following section will be dedicated to detailing the procedure we implemented to identify them specifically.



% The recombination events of interest in this analysis were those arising between a CAST haplotype and a B6 or a DBA2 haplotype.
% What's more, the B6 and the DBA2 genomes present a low sequence divergence of 0.2\% \citep{keane2011mouse} because they were obtained from the same mouse subspecies (\textit{Mus musculus domesticus}) from which they inherited large genomic regions \citep{davis2005genomewide}, whereas the B6 and CAST strains which diverged about 350,000 to 500,000 years ago \citep{geraldes2008inferring} present a much higher genome-wide divergence of 0.74\% \citep{keane2011mouse}.
% Thus, for the sake of simplicity, we chose \textit{not} to distinguish between the B6 and the DBA2 backgrounds and, instead, grouped these two haplotypes under the label ‘DOM’.
% With this simplified notation, an expected 50\% of the hotspots of the analysed mice should be in a DOM/DOM background, and the other 50\% in a DOM/CAST background (Table~\ref{tab:background-expected}).\\

% The background composition of hotspots made a noteworthy difference with the dataset analysed in Chapter~\ref{ch:5-methodology}, in which all hotspots were heterozygous.
% %because the individual analysed was a F1 hybrid.
% But the most critical difference was the presence of a third genomic origin (DBA2): the polymorphic sites between DBA2 and B6 haplotypes would undoubtedly lead to spurious genotype calls and thus, to false positive discoveries.
% Therefore, prior to searching for recombination events in this dataset, it was necessary to distinguish the B6-DBA2 polymorphic sites from the DOM-CAST polymorphic sites, in order to base the genotyping solely on the latter class of markers.







\section{Detection of recombination in F2 individuals}
\subsection{Inference of the origin of polymorphic sites}

% - dans le cas a trois genomes, on a trois haplotypes possibles B, C, D et donc des marqueurs entre B et C, B et D et C et D (cf figure). Or, ces marqueurs sont presents a des frequences differentes car les taux de divergence varient entre les trois genomes.
% - Donc, a priori, la detectabilite sera differente selon le type de background dans lequel on est (et donc de la densite en SNP)
% - Or, comme on va comparer la recombinaison entre individus (qui n'ont pas forcement le meme background), on ne veut pas avoir de telles differences de detectabilite
% - donc, on choisit de se focaliser uniquement sur les marqueurs les plus requents: ceux DOM/CAST
%
% - il faut donc identifier les marqueurs DOM/CAST
% - comme CA/CA n'existe pas (cf table), tout site variant (soit DOM/CAST soit B6/DBA2) pour lequel au moins un individu est homozygote pour la reference du genome CAST est forcement un marqueur entre B6 et DBA2
% - donc, sur cette base, on peut identifier tous les sites qui sont B6/DBA2, et les exclure.
%
% - ensuite, on va se focaliser specifiquement sur les hotspots DOM/CAST, donc il faut genotyper les hotspots.
% - pour ce faire, on regarde tous les sites DOM/CAST et on regarde le genotype de l'individu a ces marqueurs. Si plus de 90 donc HET ou DOM
% - on verifie en meme temps que Prdm9 est bien heterozygote


\begin{figure}[b!]
    \centering
    \includegraphics[width = 0.8\textwidth]{figures/inkscape/HFM1-origin-markers-with-braces.eps}
	\caption[The three possible types of polymorphic sites]
	{\textbf{The three possible types of polymorphic sites.}
		\par According to the principle of parsimony, any polymorphic site (circle) should result, in most cases, in two of the strains carrying the same allele and one of them carrying a different one.
		In this example, the polymorphic site on the left corresponds to a DOM-CAST marker, where the B6 and the DBA2 haplotypes carry the same allele, different from that of the CAST haplotype.
		The polymorphic sites in the middle and on the right correspond to two B6-DBA2 markers, with either the B6 (middle) or the DBA2 (right) haplotype carrying the same allele as the CAST one.
		Given the divergence between strains (see main text), DOM-CAST markers occur more often than the B6-DBA2 markers.
	}
\label{fig:marker-origin}
\end{figure}



% - dans le cas a trois genomes, on a trois haplotypes possibles B, C, D et donc des marqueurs entre B et C, B et D et C et D (cf figure). Or, ces marqueurs sont presents a des frequences differentes car les taux de divergence varient entre les trois genomes.
% - Donc, a priori, la detectabilite sera differente selon le type de background dans lequel on est (et donc de la densite en SNP)
% - Or, comme on va comparer la recombinaison entre individus (qui n'ont pas forcement le meme background), on ne veut pas avoir de telles differences de detectabilite
% - donc, on choisit de se focaliser uniquement sur les marqueurs les plus requents: ceux DOM/CAST

Distinguishing the targets of interest (DOM/CAST-background targets) from others (DOM/DOM-background targets) comes back to genotyping the DOM-CAST markers (i.e.\ the polymorphic sites for which the CAST strain carries an allele different from that carried by the B6 and the DBA2 strains).
% determine whether hotspots are in a DOM/DOM or in a DOM/CAST genetic background, it is necessary to genotype the DOM-CAST markers (i.e.\ polymorphic sites for which the CAST strain carries an allele different from that carried by the B6 and the DBA2 strains).
Though, given that the F2 individuals carry a mosaic of three genomes, three types of polymorphic sites can occur: either the B6, the DBA2 or the CAST genome carries an allele different from that of the other two (Figure~\ref{fig:marker-origin}).
Therefore, prior to genotyping targets, the DOM-CAST markers must be distinguished from the other (B6-DBA2) markers.

% To do this, DOM-CAST markers must be identified
% This requires to first distinguish between the DOM-CAST markers and other types of markers.
% Indeed, three types of polymorphic sites may occur: either the B6, the DBA2 or the CAST genome carries an allele different from that of the other two (Figure~\ref{fig:marker-origin}).
% Given the divergence between the three strains (0.2\% between B6 and DBA2 and 0.74\% between the two DOM (B6 or DBA2) and CAST), the frequency of DOM-CAST markers is higher than that of B6-DBA2 markers (i.e.\ polymorphic sites for which the B6 and the DBA2 strains carry distinct alleles).

% Since the F2 individuals originate from crosses between three different strains (B6, DBA2 and CAST), three types of polymorphic sites may occur: either the B6, the DBA2 or the CAST genome carries an allele different from that of the other two (Figure~\ref{fig:marker-origin}).
% Though, since the genome-wide divergence between the B6 and the DBA2 strains is much lower (0.2\%) than that between the CAST and one of the two DOM (B6 or DBA2) strains (0.74\%), the frequency of DOM-CAST markers (i.e.\ polymorphic sites for which the CAST strain carries an allele different than that carried by the B6 and the DBA2 strains) is higher than that of B6-DBA2 markers (i.e.\ polymorphic sites for which the B6 and the DBA2 strains carry distinct alleles).
% And, since the detectability of recombination events relies upon marker density, detectability would vary according to the genetic background if both types of markers (DOM/CAST and B6/DBA2) were to be considered.
% Therefore, in order to avoid any spurious fluctuation in detectability between individuals, we chose to restrict our analysis to only one type of markers: the DOM-CAST markers, which are the most frequent ones.

% - il faut donc identifier les marqueurs DOM/CAST
% - comme CA/CA n'existe pas (cf table), tout site variant (soit DOM/CAST soit B6/DBA2) pour lequel au moins un individu est homozygote pour la reference du genome CAST est forcement un marqueur entre B6 et DBA2
% - donc, sur cette base, on peut identifier tous les sites qui sont B6/DBA2, et les exclure.
% To distinguish between DOM-CAST markers and B6-DBA2 markers (i.e.\ polymorphic sites for which the B6 and the DBA2 strains carry distinct alleles), we proceeded as follows.
Given the crosses made, no portion of the genome of the F2 individuals could display a CAST/CAST background (Table~\ref{tab:background-expected}).
Therefore, if, at a given polymorphic site, at least one of the four individuals is homozygous for the allele carried by the CAST strain, the site necessarily corresponds to a B6-DBA2 marker (Figure~\ref{fig:marker-origin}).
We distinguished between B6-DBA2 and DOM-CAST markers on this basis.
% It was therefore on this basis that we distinguished between B6-DBA2 and DOM-CAST markers.



% Because the B6 and the DBA2 genomes were much more highly conserved \citep{davis2005genomewide} than the B6 and the CAST genomes \citep{keane2011mouse}, we used the simplifying assumption that any polymorphic site between the CAST and the B6 genomes would also appear between the CAST and the DBA2 genomes.
% In other words, we assumed that the B6 and the DBA2 strains carried the same allele at DOM-CAST polymorphic sites.
% When mapped against the CAST reference genome, the ‘reference’ (REF) allele at any such DOM-CAST polymorphic site was necessarily the CAST allele and the ‘alternate’ (ALT) allele was the DOM (B6 and DBA2) allele.
% Thus, given that no hotspot could be in a CAST/CAST background (Table~\ref{tab:background-expected}), there was no possible \textit{scenario} for which an individual could be genotyped by the variant-caller as REF/REF at a DOM-CAST polymorphic site.
%
% In contrast, the polymorphic sites between the B6 and the DBA2 genomes could be, in certain cases, genotyped as REF/REF\@.
% Indeed, at such B6-DBA2 polymorphic sites, the allele carried by the B6 strain was, by definition, different than that carried by the DBA2 strain.
% According to the principle of parsimony, one of them (either the B6 or the DBA2 strain) should carry the same allele as the CAST strain.
% If the B6 (resp.\ DBA2) allele was identical to the CAST allele, the B6 (resp.\ DBA2) allele would be the labelled as the reference (REF) allele and the DBA2 (resp.\ B6) allele as the alternate (ALT), when mapped against the CAST reference genome.
% Therefore, an individual with a B6/CAST- or a B6/B6- background (resp.\ DBA2/CAST- or DBA2/DBA2-background) would be genotyped REF/REF at such B6-DBA2 polymorphic sites.
%
% To distinguish between the two types of markers (B6-DBA2 or DOM-CAST), we thus followed the following rule:
% if, in at least one of the four mice, a marker was genotyped REF/REF, we inferred that it was a B6-DBA2 marker.
% Otherwise, we inferred that it was a DOM-CAST marker.
% We note that this led to erroneous calls (a B6-DBA2 marker mistakenly labelled as a DOM-CAST marker) in cases where all four mice had the B6/B6 or the B6/CAST (resp.\ the DBA2/DBA2 or the DBA2/CAST) background. Though, in such scenarii, the DBA2 and the B6 haplotypes were strictly undiscernable from one another and, consequently, no erroneous genotyping error could possibly occur.
% % this would not lead to any erroneous genotyping error.



\subsection{Identification of the genetic background}



% RIGHT PAGE
\begin{sidewaysfigure}[p]
    \centering
    \leftskip-2.4cm
    \rightskip-2.4cm
    \rotfloatpagestyle{empty}
    \includegraphics[width = 1.25\textwidth]{figures/chap8/HFM1_background_28353-DOM.eps}
    \captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
    \caption[Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28353]
    {\textbf{Mosaic of genetic backgrounds inferred at each target along the autosomes of mouse 28353.}
        \par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		Each segment corresponds to the position of a target (hotspot or control region) and was coloured in red when the background inferred was DOM/DOM (homozygous) and in blue when the background inferred was DOM/CAST (heterozygous). 
		The corresponding figures for the three other mice (28355, 28367, 28371) are reported in Appendix~\ref{app:data-and-figs}.        
    }
\label{fig:mosaic-backgrounds}
\end{sidewaysfigure}



% - ensuite, on va se focaliser specifiquement sur les hotspots DOM/CAST, donc il faut genotyper les hotspots.
% - pour ce faire, on regarde tous les sites DOM/CAST et on regarde le genotype de l'individu a ces marqueurs. Si plus de 90 donc HET ou DOM
% - on verifie en meme temps que Prdm9 est bien heterozygote


% To further assess whether the distinction we made between B6-DBA2 and DOM-CAST markers was correct, we wanted to deduce the genetic background of each hotspot on the basis of the marker classification we made (see above) and to verify whether the alternance of DOM/DOM and DOM/CAST genetic backgrounds fitted the expectations.
	
% Since hotspots located in a DOM/DOM genetic background should be heterozygous at all its B6-DBA2 markers and homozygous at all its DOM-CAST markers whereas hotspots located in a DOM/CAST background should be homozygous at all its B6-DBA2 markers and heterozygous at all its DOM-CAST markers (Figure~\ref{fig:marker-origin}), we used the following criteria to infer their backgrounds:
% To further assess whether the distinction we made between B6-DBA2 and DOM-CAST markers was correct, we aimed at genotyping the genetic background of each hotspot on the basis
% wanted to deduce from them the genetic background of each hotspot and verify whether or not the alternance of DOM/DOM and DOM/CAST genetic backgrounds fitted the expectations.
% To do this, we inferred the genetic backgrounds using stringent criteria:

Next, we inferred the genetic backgrounds using the following criteria:
if more than 90\% of the DOM-CAST markers of a given hotspot were genotyped as heterozygous in a given individual, a DOM/CAST background was inferred;
if more than 90\% of the DOM-CAST markers were genotyped as homozygous, a DOM/DOM background was inferred;
in any other case, the background was not inferred.

% Last, we refined the analysis for the targets whose genetic background could not be inferred with the criterium aforementioned: if they were flanked on both the 5’ and the 3’ sides by at least 5 targets with a DOM/DOM (resp.\ DOM/CAST) genetic background, we attributed the DOM/DOM (resp.\ DOM/CAST) background to them.




% After labelling the origin of all the polymorphic sites, we inferred the genetic background in each target for each of the four mice as follows:
% if more than 90\% of the markers classified as DOM/CAST displayed a REF/ALT genotype, a DOM/CAST-background was inferred;
% if more than 90\% of the markers classified as DOM/CAST displayed a REF/ALT genotype, a DOM/DOM-background was inferred;
% in any other case, the background was not inferred.
% Last, we refined the analysis for the targets whose genetic background could not be inferred with the criterium aforementioned: if they were flanked on both the 5’ and the 3’ sides by at least 5 targets with a DOM/DOM (resp.\ DOM/CAST) genetic background, we attributed the DOM/DOM (resp.\ DOM/CAST) background to them.


% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_refined.txt|cut -f1|grep NA|wl
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_refined.txt|cut -f2|grep NA|wl
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_refined.txt|cut -f3|grep NA|wl
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_refined.txt|cut -f4|grep NA|wl
% NA dans 23 hotspots out of $1390 \times 4$ individuals
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_refined.txt|cut -f1|grep NOCOV|wl
% NOCOV dans 168 hotspots de plus

% Refait avec le NOT refined
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_not_refined.txt|cut -f1|grep NA|wl
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_not_refined.txt|cut -f2|grep NA|wl
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_not_refined.txt|cut -f3|grep NA|wl
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_not_refined.txt|cut -f4|grep NA|wl
% NA dans 135 hotspots out of $1390 \times 4$ individuals
% awk -v OFS="\t" 'NR>1 {print $5,$6,$7,$8}' ../HFM1_pour_laurent/List_genotypes_hotspots_not_refined.txt|cut -f1|grep NOCOV|wl
% NOCOV dans 145 de plus
% (5560-280)/5560
% 0.9496402877697842


Out of the 4$\times$1390 targeted loci, 145 (2.6\%) had a read coverage too low for the target to be genotyped.
Aside from those, the aforementioned \textit{modus operandi} allowed us to genotype 97.5\% of all the targets presenting sufficient coverage and ended in a mosaic of DOM/DOM and DOM/CAST genetic backgrounds consistent with 0 or 1 (and sometimes 2) crossing-overs per chromosome (Figure~\ref{fig:mosaic-backgrounds} and Appendix~\ref{app:data-and-figs}). 
This provided strong support that our inference was correct.
% , half of which being in a heterozygous DOM/CAST background, and the other half in a homozygous DOM/DOM background (Table~\ref{tab:proportion-het-backgrounds}).
Among the remaining 2.5\% (135) ambiguous targets, 6 (4\%) were flanked by DOM/DOM-background targets on one side and by DOM/CAST-background targets on the other side: these most likely corresponded to sites where recombination occurred in one of the parents.
All other ambiguous targets (94\%) were flanked on both sides by DOM/DOM-background targets: these were most likely erroneously inferred because some B6-DBA2 markers were erroneously classified as DOM-CAST markers.


% Aside from the 2.6\% (145 out of 4$\times$1390) of hotspots which had a too low coverage to be genotyped, the aforementioned \textit{modus operandi} allowed us to genotype 97.5\% of all hotspots with sufficient coverage and ended in a mosaic of DOM/DOM and DOM/CAST genetic backgrounds consistent with 0 or 1 (and sometimes 2) crossing-overs per chromosome in each individual (Figure~\ref{fig:mosaic-backgrounds} and Appendix~\ref{app:data-and-figs}), which provided strong support that our inference was correct.
% Among the 5\% of hotspots which were ambiguous,
%

% We note that, aside from the fraction of hotspots which could not be labeled because read coverage was too low, the totality of the remaining 3\% ambiguous hotspots were surrounded by DOM/DOM-background hotspots, which suggests that they most probably corresponded to regions encompassing many B6-DBA2 polymorphic sites leading to ambiguous genotype calls.
All in all, across all 1,390 loci of the 4 mice, 7 were incongruent with the surrounding genetic background (either because they were subject to a double crossing-over, or because our inference was incorrect at these sites). We thus chose to remove them from the analysis.
Altogether, the proportion of heterozygous DOM/CAST-background targets (Table~\ref{tab:proportion-het-backgrounds}) was close to the expected 50\% (Table~\ref{tab:background-expected}).
To further verify that these observed proportions fitted what was expected, we simulated a DOM/CAST$\times$DOM/DOM cross in which COs (number given by the sex-averaged genetic length) were drawn randomly along each chromosome. We found that the distribution of the expected proportion of heterozygous targets (data not shown) fitted with the observations (Table~\ref{tab:proportion-het-backgrounds}).

This genotyping map also allowed to control that all four mice were heterozygous for \textit{Prdm9} since this gene was located in a DOM/CAST background in all samples.
% the \textit{Prdm9} was located in a DOM/CAST background for all four mice and thus, that all four mice were heterozygous for \textit{Prdm9}.




% \clearpage
\begin{table}[t]
    \centering
    \begin{tabular}{rrrrrrr}
        \toprule
        \textbf{Category} & \textbf{Sample} & \textbf{\# DOM/CAST} & \textbf{\# DOM/DOM} & \textbf{\% of Het.} \\
        \textbf{} & \textbf{} & \textbf{background} & \textbf{background} & \textbf{targets} \\
	    
		\midrule
		\multirow{2}{*}{WT}      & 28355    & 764   & 561  & 57.66 \\ 
		& 28371    & 845   & 461  & 64.70 \\ 
        \midrule
        \multirow{2}{*}{Mutant}  & 28353    & 663   & 669  & 49.77 \\ 
        & 28367    & 624   & 693  & 47.38 \\ 
        \midrule
		\multicolumn{2}{r}{\textbf{Total}} & \textbf{2896} & \textbf{2384} & \textbf{54.85} \\
        \bottomrule
    \end{tabular}
    \caption[Observed proportion of heterozygous targets in the studied mice]
	{\textbf{Observed proportion of heterozygous targets in the studied mice.}
		\par The background for each hotspot was inferred as described in the main text, for wild-type (WT) mice (top panel) and mutant mice (bottom panel). The line in bold represents the average across all four mice.
	}
\label{tab:proportion-het-backgrounds}
\end{table}






\begin{table}[b!]
    \centering
    \begin{tabular}{rrrrrr}
        \toprule
        \textbf{Target} & & \textbf{Nb of} & \textbf{Nb of} & \textbf{Nb of} & \textbf{Event rate} \\

        \textbf{category} & \textbf{Sample} & \textbf{targets} & \textbf{fragments} & \textbf{events} & \textbf{($\times$ 10\textsuperscript{-6})} \\

        \midrule
        \multirow{7}{*}{\textbf{\textit{Hotspots}}} & 28355    & 485 & 28,181,748  & 1,298 & 46.1 \\ 
		& 28371    & 552 & 34,015,365  & 1,847 & 54.3 \\ 
		& \textbf{Tot.\ WT} &  \textbf{1037} & \textbf{62,197,113}  & \textbf{3,145} & \textbf{50.6} \\ 
		\\
		& 28353 & 429 & 25,598,721 & 3,486 & 136 \\
		& 28367 & 390 & 30,863,121  & 2,082 & 67.4 \\
		& \textbf{Tot.\ mutants} & \textbf{819} & \textbf{56,461,842}  & \textbf{5,568} & \textbf{98.6} \\
		\midrule
        
		\multirow{7}{*}{\textbf{\textit{Controls}}} & 28355 & 279 & 15,206,411  & 34 & 2.24 \\ 
		& 28371 & 293 & 16,997,729  & 58 & 3.41 \\ 
		& \textbf{Tot.\ WT} & \textbf{572} & \textbf{32,204,140}  & \textbf{92} & \textbf{2.86} \\ 
		\\
		& 28353 & 234 & 13,658,994 & 33 & 2.42 \\
		& 28367 & 234 & 17,565,253  & 25 & 1.42 \\
		& \textbf{Tot.\ mutants} & \textbf{468} & \textbf{31,224,247}  & \textbf{58} & \textbf{1.86} \\ 
	    \midrule
        \multicolumn{1}{r}{\textbf{FP rate}} & \multicolumn{5}{r}{\textbf{3.22 \%}} \\
        \bottomrule
    \end{tabular}
    \caption[Number of events detected in hotspot and control targets]
    {\textbf{Number of events detected in hotspot and control targets.}
        \par Events (false positives (FPs) or genuine recombination events) were detected using the unique-molecule genotyping pipeline described in Chapter~\ref{ch:5-methodology}.
        All fragments or events overlapping at least 1 bp with a given target are counted in this table.
        The event rate corresponds to the ratio of candidate recombination events over the total number of fragments.
        The maximum false positive (FP) rate is the ratio of the event rate in control targets over that in hotspots.
		The lines in bold represent the totals for the two WT and the two mutant mice.
    }
\label{tab:HFM1-FP-rate}
\end{table}
% \clearpage
% TOT NB FRAGS HOTSPOTS
% >>> 56461842+62197113
% 118658955
% TOT NB EVENTS HOTSPOTS
% >>> 5568+3145
% 8713
% >>>
% TOT NB FRAGS CONTROL
% >>> 31224247+32204140
% 63428387
% TOT NB EVENTS CONTROL
% >>> 58+92
% 150
% RATE CONTRO
% >>> 150/63428387
% 2.3648717411022922e-06
% RATE FRAGS
% >>> 8713/118658955
% 7.342892915246051e-05
% RATIO (FP RATE)
% >>> 2.3648717411022922e-06/7.342892915246051e-05
% 0.03220626758960502


\subsection{Detection of events in heterozygous hotspots}

Finally, for each individual, we applied the unique-molecule genotyping pipeline described in Chapter~\ref{ch:5-methodology} to all the heterozygous targets and we found that the maximum FP error rate for this re-adaptation of our approach (3.22\%, Table~\ref{tab:HFM1-FP-rate}) was similar to that from Chapter~\ref{ch:5-methodology} (3.73\%, Table~\ref{tab:FP-rate}).

Altogether thus, our procedure was as efficient to detect recombination events in F2 individuals containing three genetic backgrounds as it was for F1 hybrids.
From this point on, we could thus assess the impact of the \textit{hfm1} mutation on several aspects of recombination.







\section{Impact of the mutation on recombination}
\subsection{Impact on the recombination rate (RR)}
\label{chap8:recombination-rate}

We observed that the recombination rate (RR) was, on average, almost twice as high for mutants as for WT mice (Table~\ref{tab:HFM1-FP-rate}).
This finding was unexpected since the only effect of the interaction between Mer3 and Mlh1 that was reported in yeasts concerned the length of gene conversion tracts, but not the recombination rate \citep{duroc2017concerted}.
% Surprisingly, this effect was the opposite of what others had previously found.
% Indeed, another study in mice observed a reduction by 20\% in the number of bivalents in \textit{hfm1} mutants \citep{guiraldelli2013mouse} and the formation of COs was reduced by 38\% in \textit{Arabidopsis} \textit{mer3} mutants \citep{mercier2005two} and by 50--60\% in \textit{Saccharomyces cerevisiae} \textit{mer3} mutants \citep{borner2004crossover,jessop2006meiotic}.

In our case, this modification of the RR was majoritarily driven by the extremely high recombination rate of mouse 28353 (136 events per million of sequenced fragments, Table~\ref{tab:HFM1-FP-rate}), which was over twice that of the other mutant mouse 28367 (67.4 events per million of sequenced fragments).

Though, if, say, the subset of heterozygous hotspots of mouse 28353 were more intense (i.e.\ displayed higher recombinational activity on average than other hotspots), this observation would not correspond to a genuine biological effect.
In the following subsection, I describe how we thus controlled for such technical biases.
% In the following subsection, I describe how we distinguished between such technical effects and genuine biological effects.
% Thus, in the following subsection, I describe how we examined whether this effect was genuine or if it was a technical effect due to the distribution of heterozygous hotspots in each mice.



\subsection{Pairwise comparison of the RR in shared hotspots} 


\begin{figure}[p]
    \centering
    \begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between the two WT mice}
        \includegraphics[width=\textwidth]{figures/chap8/28371_vs_28355.eps}
    \end{subfigure}

	\vspace{0.5cm}

    \begin{subfigure}[b]{0.75\textwidth}
        \subcaption{Between the two mutant mice}
		\includegraphics[width=\textwidth]{figures/chap8/28367_vs_28353.eps}
    \end{subfigure}
    \caption[Correlation of the number of recombination events in shared hotspots for the two WT and the two mutant mice]
	{\textbf{Correlation of the number of recombination events in shared hotspots for the two WT (a) and the two mutant (b) mice.}
		\par The linear regression was significant for the two WT mice (slope $=1.03$; \textit{p}-val~$<~2~\times~10^{-16}$; $n_{hotspots} = 257$) and for the two mutant mice (slope = $0.69$; \textit{p}-val $< 2 \times 10^{-16}$; $n_{hotspots}~=~241$).
		Figures for all other pairwise correlations are reported in Appendix~\ref{app:data-and-figs}.
		Results of the linear correlation were similar for Rec-1S and Rec-2S events, as well as when controlling for the total number of events sequenced at each hotspot (data not shown).
	}
\label{fig:pairwise-RR-shared}
\end{figure}


To test whether the variation in overall recombination rate (RR) across mice was due to the fact that the sets of hotspots analysed (i.e.\ heterozygous hotspots) were different between mice, we performed comparisons of the RR in shared hotspots for all pairs of mice (Figure~\ref{fig:pairwise-RR-shared} and Appendix~\ref{app:data-and-figs}).
We found that the difference in recombination rates between WT and mutant mice was observed even for shared hotspots, which proved that the effect was not due to a differential sampling of heterozygous loci.

% In addition, since \textit{Hfm1} is known to be involved in CO formation \citep{mazina2004saccharomyces} — but not, to our knowledge, in NCO formation, — and because the effect in previous studies was only reported for COs \citep{guiraldelli2013mouse,mercier2005two,borner2004crossover}, we reproduced the pairwise comparisons separately for Rec-1S and Rec-2S events (Appendix~\ref{app:data-and-figs}).
In addition, to see whether the difference in RR applied specifically to one type of recombination products (either COs or NCOs), we reproduced the pairwise comparisons separately for Rec-1S and Rec-2S events (Appendix~\ref{app:data-and-figs}).
We found that the results were similar for both Rec-1S and Rec-2S events, which showed that both COs and NCOs were affected.

All in all, the RRs for the two WT mice were extremely close (Figure~\ref{fig:pairwise-RR-shared}.a.): the slope of the linear regression was almost 1 (slope $=1.03$; \textit{p}-val $<2 \times 10^{-16}$).
However, the recombination rates between the two mutant mice was extremely variable (Table~\ref{tab:HFM1-FP-rate} and Figure~\ref{fig:pairwise-RR-shared}.b.).
What drives such variability among \textit{hfm1} mutants remains, at this stage, unknown: to get more insight into this topic, it would be necessary to analyse the data from additional mutant mice displaying distinct mosaics of genetic backgrounds, and see, for instance, if the increased RR is associated to a given locus in the DOM/DOM or DOM/CAST background.


\subsection{Impact on CO tract length}


\begin{table}[b!]
    \centering
	\begin{adjustbox}{width = 1\textwidth}
    \begin{tabular}{rrrrrrr}


        \toprule
		& \multicolumn{3}{c}{\textbf{WT}} & \multicolumn{3}{c}{\textbf{Mutant}} \\
		\cmidrule(l){2-4} \cmidrule(l){5-7}
        \textbf{Parameter} & \textbf{Both} & \textbf{28355} & \textbf{28371} & \textbf{Both} & \textbf{28353} & \textbf{28367} \\

        \midrule
        \textbf{\textit{CO:NCO ratio}}      & 0.108 [0.009--0.189] & 0.095 & 0.098 & 0.092 [0.0003--0.40] & 0.051 & 0.166 \\
        \textbf{\textit{CO CT length}}\\
        $Mean$                              & 744 [219--2790] & 539 & 654 & 236 [145--478] & 238 & 253 \\
        $Sd$                                & 582 [101--765] & 514 & 759 & 292 [30--397] & 416 & 232 \\
        \textbf{\textit{NCO CT length}}\\
        $Mean$                              & 34 [5--47] & 35 & 31 & 30 [0.75--397] & 49 & 32 \\
        $Sd$                                & 43 [1--101] & 38 & 57 & 108 [13--260] & 130 & 66 \\
        
        \bottomrule

    \end{tabular}
	\end{adjustbox}
	\caption[Recombination parameters inferred from an approximate bayesian computation for WT and mutant mice]
	{\textbf{Recombination parameters inferred from an approximate bayesian computation for WT and mutant mice.} 
		\par Parameters (CO:NCO ratio and CO and NCO conversion tract (CT) length reported in bp) were estimated for the two WT and the two mutant mice.
	95\% confidence intervals were reported between brackets.
	Because the observed recombination rate varied greatly between the two mutants (Subsection~\ref{chap8:recombination-rate}), we also reported than point estimates for all single individuals.
	}
\label{tab:HFM1-ABC-results}
\end{table}


Finally, because the interaction between the HFM1 yeast homologue (Mer3) and the MLH1 yeast homologue (Mlh1) has been shown to play a role in DNA heteroduplex extension \citep{duroc2017concerted}, we wanted to assess whether tract lengths differed between the WT and the \textit{hfm1} mutant mice.

Because CO and NCO CT lengths are not directly observable from the data, we performed an approximate bayesian computation (ABC) similar to what was described in Chapter~\ref{ch:6-recombination-parameters}, based on 50,000 simulations reproducing this experiment (thus, as compared to Chapter~\ref{ch:6-recombination-parameters}, we modified hotspot width, fragment start and stop positions and polymorphic sites to fit with this experiment).\\




Altogether, the CO:NCO ratio and the NCO CT length estimated for WT mice were strikingly close to those measured on the WT mice of Chapter~\ref{ch:6-recombination-parameters} (Table~\ref{tab:ABC-results}).
CO CTs were slightly longer (albeit \textit{not} significantly) than those found in the previous ABC, which likely comes from the fact that the targeted regions were wider in this experiment whereas the maximum distance between DSB sites and CO switch points was limited in the previous one.

Interestingly, we found a clear CO CT length reduction in \textit{hfm1} mutant mice as compared to WT mice (Table~\ref{tab:HFM1-ABC-results}).
Because the observed recombination rates varied greatly between the two mutants (see Subsection~\ref{chap8:recombination-rate}), we checked whether this effect was also visible in single individuals and found that, indeed, the inferred conversion tract lengths were stable, no matter the recombination rate.
% This observation was consistent with the idea that, in mice, the \textit{Hfm1} gene plays a role in extending the DNA heteroduplex.\\
This observation was consistent with the idea that, in mice, like in yeasts \citep{duroc2017concerted}, the interaction between HFM1 and MLH1 plays a role in extending the DNA heteroduplex.\\


In summary, the method we implemented to detect recombination was adaptable to cases where other genomes had been introgressed into the hybrid and allowed to gain new insight into recombination in mice.
Though, as any approach, it had inherent limitations, which I will discuss in the following chapter, together with the scientific implications of the whole work done in the context of this thesis.



