\begin{savequote}[8cm]
	% ‘[…] the process of mutation itself is not adaptive. On the contrary, the mutants which arise are, with rare exceptions, deleterious to their carriers, at least in the environments which the species normally encounters. Some of them are deleterious apparently in all environments. Therefore, the mutation process alone, not corrected and guided by natural selection, would result in degeneration and extinction rather than in improved adaptedness.’
	% ‘[…] the mutants which arise are, with rare exceptions, deleterious to their carriers, at least in the environments which the species normally encounters.’
	% — Theodosius Dobzhansky
	% 'On Methods of Evolutionary Biology and Anthropology', American Scientist, 1957, 45, 385.


	%%%%% KIMURA
	% ‘[…] if the neutral or nearly neutral mutation is being produced in each generation at a much higher rate than has been considered before, then we must recognize the great importance of random genetic drift due to finite population number in forming the genetic structure of biological populations.’
	‘Finally, if my chief conclusion is correct, and if the neutral or nearly neutral mutation is being produced in each generation at a much higher rate than has been considered before, then we must recognize the great importance of random genetic drift due to finite population number in forming the genetic structure of biological populations.’
	\qauthor{--- Motoo Kimura, \textit{\usebibentry{kimura1968evolutionary}{title}} \citeyearpar{kimura1968evolutionary} }
	
	% ‘Of course, Darwinian change is necessary to explain change at the phenotypic level — fish becoming man — but in terms of molecules, the vast majority of them are not like that.’
	%
	% ‘This leads us to an important principle for the neutral theory stating that “the neutral mutants” are not the limit of selectively advantageous mutants but the limit of deleterious mutants when the effect of mutation on fitness becomes indefinitely small. This means that mutational pressure causes evolutionary change whenever the negative-selection barrier is lifted.’
	% ‘[…] “the neutral mutants” are not the limit of selectively advantageous mutants but the limit of deleterious mutants when the effect of mutation on fitness becomes indefinitely small. This means that mutational pressure causes evolutionary change whenever the negative-selection barrier is lifted.’
	% ‘[…] “the neutral mutants” are not the limit of selectively advantageous mutants but the limit of deleterious mutants when the effect of mutation on fitness becomes indefinitely small.’
	% \qauthor{--- Motoo Kimura, \textit{\usebibentry{kimura1983neutral}{title}} \citeyearpar{kimura1983neutral} }

	% FROM
	% https://books.google.fr/books?hl=fr&lr=&id=olIoSumPevYC&oi=fnd&pg=PR9&dq=%22The+neutral+theory+of+molecular+evolution%22+by+Motoo+Kimura&ots=P0R5memrff&sig=5w8VhVKBKCo2IxJIhcSPc-36vnA#v=onepage&q=deleterious&f=false
	% — avec recherche du mot "deleterious" (page 113)


	% DEPUIS L'AUTRE
	% ‘Unlike the Darwinian theory of evolution by natural selection, the neutral theory claims that the overwhelming majority of evolutionary changes at the molecular level are not caused by Darwinian natural selection acting on advantageous mutants, but by random fixation of selectively neutral or very nearly neutral mutants through the cumulative effect of sampling drift (due to finite population number) under continued input of new mutations.’
	% \qauthor{--- Motoo Kimura, \textit{\usebibentry{kimura1991neutral}{title}} \citeyearpar{kimura1991neutral} }

\end{savequote}

\chapter{\label{ch:4-gBGC}GC-biased gene conversion, the designer of genomic lanscapes} 
%\otherpagedecoration

\minitoc{}

% DES TITRES POSSIBLES
% The evolution of genomic GC-content through biased gene conversion
% GC-biased gene conversion and the evolution of genomic landscapes (de Duret et Galtier)
% GC-biased gene conversion, the designer of genomic lanscapes
%
% The Role of GC-Biased Gene Conversion in Shaping the Fastest Evolving Regions of the Human Genome
% The role of GC-biased gene conversion in shaping genomic GC-content
% How GC-biased gene conversion shapes the genomic GC-content
%
%
% Biased gene conversion and GC-content evolution
% The evolution of genomic base composition through GC-biased gene conversion
% GC-biased gene conversion and the evolution of base composition
% Gene conversion and GC-content evolution in mammals
% GC-content evolution and biased gene conversion
% GC-biased gene conversion and the evolution of genomic content
% Biased gene conversion and GC-content evolution
% The genomic landscape of GC-content and biased gene conversion

% GC-biased gene conversion, the fourth force of genome evolution (depuis Lesecque) 




Gene conversion, \textit{i.e.} the process through which one DNA sequence is non-reciprocally copied onto another one (the homologue in the case of allelic gene conversion), leads to the non-Mendelian segregation of genetic information at the locus where it occured. 
If the two alleles are equally likely to be converted, this has no incidence at the population scale: allelic frequencies remain constant over generations. 
If, however, one homologue preferentially converts the other, it is more frequent in the pool of gametes and the transmission of alleles is necessarily biased: the donor has an evolutionary advantage over the acceptor.

Such biased gene conversion (BGC) exists under two forms: DSB-induced BGC (dBGC) when the bias comes from a differential competency for homologues to host the double-strand break (see Chapter~\ref{ch:3-recombination-variation}), and GC-biased gene conversion (gBGC) when it comes from the nature of the nucleotides involved.
Indeed, the repair of the cut homologue involves the formation of heteroduplexed DNA, \textit{i.e.} a stretch of DNA where the two strands bear distinct alleles.
These mismatches are either ‘restored’ if the original allele of the cut sequence is reinstated, or ‘converted’ if it is supplanted by the allele of the homologue.
The position of these events delineate ‘conversion tracts’ (CTs) — which are designated as ‘complex CTs’ when they alternate conversion and restoration events \citep{borts1989length} and ‘simple CTs’ otherwise.

In most species, whether through conversions or restorations, the repair favours GC over AT alleles \citep{pessia2012evidence,glemin2014gc,glemin2015quantification,figuet2014biased,wallberg2015extreme,bolivar2016recombination}, hence the term ‘GC-biased gene conversion’ (gBGC). 
Because its consequences on genome evolution ressemble those of natural selection, the very existence of this recently discovered phenomenon has been questioned by many in the more global context of the controversy opposing selectionists to neutralists (see Chapter~\ref{ch:1-history-genetics}).
I will therefore start this chapter by reviewing the breakthrough of gBGC in the climate of this debate, then explore the similitudes of its implications for genome evolution with those of natural selection and finish by looking into the first studies that characterised it.


% Gene conversion, \textit{i.e.} the process through which one DNA sequence is replaced by another one (the homologue in the case of allelic gene conversion), was first observed through tetrad analyses of fungal products of meiosis (see Chapter~\ref{ch:1-history-genetics}).
% This phenomenon was first interpreted as mutational, hence the term ‘conversion’ \citep{winkler1930konversion} but it was later understood that it was in fact purely recombinational.
% Generally, indeed, the chromosomal sequence that bears the double-strand break (DSB) is converted by its homologue, thus yielding a unidirectional transfer of genetic information.
% If, at the scale of a population of meioses, one homologue preferentially converts the other, it is preferentially transmitted, \textit{i.e.} leads to biased gene conversion (BGC) — in that particular case, DSB-induced BGC (dBGC), as was discussed in the previous chapter.

% Another form of meiotic drive exists: GC-biased gene conversion (gBGC).
% Indeed, the repair of the cut homologue involves the formation of heteroduplexed DNA, \textit{i.e.} a DNA portion where the two strands display distinct alleles at heterozygous loci.
% These mismatches can be repaired either through a ‘restauration’ event, \textit{i.e.} towards the original genotype of the cut sequence, or through a ‘conversion’ event, \textit{i.e.} towards the genotype of the homologous sequence.
% Conversion events are observable all along conversion tracts (CTs), which, when they alternate conversion with restauration events, are termed ‘complex CTs’ \citep{borts1989length}.
% Similar to dBGC, if, at the scale of a population of meioses, one allele has a greater chance of converting the other one, the gene conversion event is biased.

% The discovery of such BGC towards GC alleles (gBGC) was made only very recently and took part into the debate between selectionists and neutralists.
% I will start with reviewing this discovery in the context of this debate, then explore on what grounds the consequences of gBGC are similar to those of natural selection hence the debate on the nature of the causality, and last, look into the first characterisation and quantification of gBGC\@.


\section{Discovery of GC-biased gene conversion (gBGC)}
\subsection{The debated origin of isochores}

\begin{figure}[!b]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap4/isochores-human.eps}
	\caption[Overview of isochores on four human chromosomes]
	{\textbf{Overview of isochores on four human chromosomes.}
		\par Human chromosomes 1, 2, 3 and 4 are divided into 100-kb windows coloured according to their mean GC-content: the spectrum of GC-level was divided into five classes (indicated by broken horizontal lines) from ultra-marine blue (GC-poorest L1 isochores) to scarlet red (GC-richest H3 isochores). 
		Grey vertical lines correspond to gaps present in the sequences and grey vertical regions to centromeres.
		\par This figure was reproduced from \citet{costantini2006isochore} and corresponds to a subsample of the original figure.
	}
\label{fig:isochores-human}
\end{figure}

% The first ‘Chargaff's rule’, which states that the observation that the amount of adenosine (A) equals that of thymine (T) and that the amount of cytosine (C) equals that of guanine (G) \citep[reviewed in \citealp{kresge2005chargaff}][]{kresge2005chargaff}, reflects the
%
% The discovery of the double-helix structure of DNA \citep{franklin1953molecular,watson1953molecular,wilkins1953molecular} was made possible thanks to several advances on the study of DNA\@.
% One of them was the so-called ‘Chargaff's rules’ \citep[reviewed in][]{kresge2005chargaff}, \textit{i.e.} the observation that, on the one hand, the amount of adenosine (A) equals that of thymine (T), and on the other hand the amount of cytosine (C) that of guanine (G) \citep{chargaff1950chemical}, which reflects the pairing of A with T bases, and of C with G bases in double-stranded DNA\@.
%

In double-stranded DNA, adenosine (A) and thymine (T) nucleotides pair up while cytosine (C) nucleotides mate guanine (G) bases \citep[reviewed in \citealp{kresge2005chargaff}]{kresge2005chargaff}.
Therefore, when studying the composition of a stretch of DNA, it is conventional to measure its GC-content.

Originally, this was done \textit{via} the ultra-centrifugation of DNA fragments \citep{meselson1957equilibrium,corneo1968isolation}.
Using this technique, a few studies have characterised GC-content distribution in several eukaryotes \citep{filipski1973analysis,thiery1976analysis,macaya1976approach,macaya1978analysis,cortadas1977analysis} and revealed that mammalian, avian and reptilian genomes — but not amphibians nor fishes \citep{bernardi1990compositional} — display a long-range compositional heterogeneity (Figure~\ref{fig:isochores-human}).
The long regions of 100 kb or more that carry a relatively homogeneous GC-content were later termed ‘isochores’ \citep{cuny1981major}.\\


GC-rich isochores are enriched in genes \citep[reviewed in \citealp{bernardi2005distribution}]{bernardi1985mosaic,mouchiroud1991distribution,lander2001initial} that are shorter and more compact than in GC-poor regions \citep{duret1995statistical} and regional GC-content further correlates with the timing of DNA replication \citep{federico1998generichest,watanabe2002chromosomewide,costantini2008replication}, the density in transposable elements (TEs) \citep{smit1999interspersed,lander2001initial,mousegenomesequencingconsortium2002initial} and the recombinational activity \citep{fullerton2001local,kong2002highresolution}.\\


Since base composition of homologous genomic regions correlate between the three amniotic lineages (mammals, birds and reptiles) \citep{kadi1993compositional,caccio1994singlecopy,hughes1999warmblooded}, it is thought that isochores were inherited from their last common ancestor (LCA).
Since then, certain lineages have undergone additional somehow steep changes.
For instance, the isochore GC-content of mice is less variable than that of other mammals — a pattern that is in the derived state as compared to nonrodents \citep{galtier1998isochore} and which likely reflects one \citep{mouchiroud1988compositional} or two \citep{smith2002compositional} extra ‘murid shifts’ since the LCA\@.

Originally, two main hypotheses had been proposed as for the origin of isochores \citep[reviewed in][]{duret2009biased}.
According to the mutational bias hypothesis, isochores would be caused by a variation along chromosomes in the mutational bias towards either AT or GC nucleotides \citep{filipski1988why,wolfe1989mutation,francino1999isochores,fryxell2000cytosine}.
If this were true, GC~$\rightarrow$~AT and AT~$\rightarrow$~GC mutations should have the same probability of fixation at neutral sites.
The finding that this was not the case \citep{eyre-walker1999evidence, smith2001synonymous, lercher2002evolution, webster2004fixation, spencer2006influence} ruled out this theory.

Another proposition involving natural selection has been thoroughly defended by one of the major discoverers of isochores \citep{bernardi2000isochores,bernardi2007neoselectionist}. 
In his view, the fact that G and C bases are linked \textit{via} three hydrogen bonds (instead of two for A and T bases) would compensate for the purportedly instable nature of DNA in warm-blooded animals.
% allow a better stability of DNA in warm-blooded animals whose body temperature would purportedly make their genetic material more sensitive to denaturation.
However, this does not explain why only a fraction of the genome is affected by higher GC-content \citep{duret2009biased}.
This theory was further invalidated by the facts that no correlation between body temperature and GC-content was found \citep{belle2002analysis,ream2003base} and that this isochore organisation also takes place in cold-blooded animals like reptiles \citep{hughes1999warmblooded,hamada2003presence,costantini2016anolis}.
In addition, a scenario according to which all sites are under selection has theoretical limitations: given the elevated rate of deleterious mutations in their protein-coding sequences \citep{eyre-walker1999high,keightley2000deleterious}, mammalian genomes would probably accumulate a mutation load too high to be coped with \citep{eyre-walker2001evolution}.

An alternative role for natural selection in causing isochore organisation would be its fine-tuning the expression of tissue-specific genes \citep{vinogradov2003isochores,vinogradov2005noncoding}.
This hypothesis may not hold, though, since the correlation between GC-content and gene expression is extremely weak \citep[reviewed in \citealp{duret2009biased}]{semon2005relationship,semon2006no}.\\


Since natural selection thus seems insufficient to explain, on its own, the bias towards the fixation of GC alleles, another track has been considered: GC-biased gene conversion (gBGC).





\subsection{An alternative causation: the gBGC model}

The excess of AT~$\rightarrow$~GC substitutions in a context where GC~$\rightarrow$~AT mutations are preponderant can be explained in two non-mutually exclusive ways: either because of non-stationarity (\textit{i.e.} the GC-content in GC-rich isochores would still be decreasing) or because of GC-biased gene conversion (gBGC) as initially mentioned by \citet{holmquist1992chromosome} and \citet{eyre-walker1993recombination,eyre-walker1999evidence} but only first firmly defended by \citet{galtier2001gccontent}.

The latter model originates from the observation that the mismatch repair (MMR) system — the main repair pathway during recombination \citep[reviewed in \citealp{evans2000roles} and \citealp{spies2015mismatch}]{alani1994interaction,nicolas1994polarity} which is also involved in the mending of base misincorporations during DNA replication \citep{surtees2004mismatch} — may favour G and C alleles \citep{brown1988different}. (Figure~\ref{fig:gBGC-galtier}). \\


\begin{figure}[t]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap4/gBGC-galtier.eps}
	\caption[Gene conversion during a recombination event involving a strong (G or C) \textit{versus} a  weak (A or T) base mismatch]
	{\textbf{Gene conversion during a recombination event involving a strong (G or C) \textit{versus} a  weak (A or T) base mismatch.}
		\par A pair of homologous chromosomes displaying a heterozygous site with a G:C pair represented as a black rectangle and a A:T pair as a white rectangle (\textbf{a}) undergoes a recombination event which materialises as a heteroduplex (\textbf{b}) containing a T:G mismatch (\textbf{c}).
		The T:G mismatch is repaired and results either in a G:C (\textbf{d}) or a A:T (\textbf{d'}) pair which yield a non-Mendelian segregation of alleles (\textbf{e} and \textbf{e'}).
		This has an incidence at the population-scale if the repair towards G:C (\textbf{d}) or A:T (\textbf{d'}) is more frequent than the other one. 
		It is called GC-biased gene conversion (gBGC) in the particular case where the repair towards G:C (\textbf{d}) occurs more often.
		\par This figure was reproduced from \citet{galtier2001gccontent}.
	}
\label{fig:gBGC-galtier}
\end{figure}


At the time, one major plea against gBGC was that there was only a narrow range within which gBGC can significantly alter base composition\footnote{If $N_e \times w \ll 1$ (where $N_e$ is the effective population size and $w$ is the per nucleotide conversion rate times the average conversion bias), gBGC has little effect on base composition.} without making it extreme\footnote{If $N_e \times w \gg 1$ (where $N_e$ is the effective population size and $w$ is the per nucleotide conversion rate times the average conversion bias), gBGC leads to extreme base composition.}.
This was adressed by \citet{duret2008impact} who found that the gBGC model explains well the relationship between recombination and substitution rates.
Indeed, considering that gBGC acts only at recombination hotspots, the substitution rate increases greatly at these loci, but stops before their GC-content reaches 100\%, because hotspots generally have a short lifespan \citep{ptak2005finescale,winckler2005comparison}. 
%Indeed, considering that gBGC acts only at recombination hotspots which have a short lifespan \citep{ptak2005finescale,winckler2005comparison}, the substitution rate increases greatly, but only transiently — before the GC-content reaches 100\%. 
% Nonetheless, very active and longer-lived hotspots are expected to become strong substitution hotspots \citep{duret2008impact}, as observed in the human genome \citep[reviewed in \citealp{duret2009biased}]{dreszer2007biased}.

% Because of its occuring solely at hotspots,
gBGC also provides an explanation for the higher heterogeneity of GC-rich isochores \citep{clay2001compositional,clay2001compositionala}: GC-content would vary depending on the intensity and age of hotspots.
In particular, as soon as a hotspot gets inactivated, its GC-content should start decreasing, consistently with what has been observed in the GC-rich regions\footnote{According to the gBGC hypothesis, GC-rich regions are those that host the hotspots.} of primates \citep{duret2002vanishing,belle2004decline,meunier2004recombination,duret2006gc}.
In contrast, in avian genomes that have evolutionarily stable recombination hotspots, GC-content continues increasing \citep{webster2006strong,capra2011substitution,mugal2013twisted}.\\


Another objection to gBGC \citep{eyre-walker1999evidence} came with the observation that GC-content at the synonymous third position of codons (GC\textsubscript{3}) is generally greater than intronic GC-content \citep{clay1996human}.
But \citet{duret2001elevated} provided an explanation compatible with gBGC to this observation: assuming that transposons are GC-poorer than the GC-rich regions of the genome, their accumulation within introns (but not exons) would justify such difference between intronic GC-content and GC\textsubscript{3}.\\

Altogether, the presence of isochores seems to fit theoretically with gBGC \citep{duret2006new}.
But, under the gBGC hypothesis, a number of other consequences are expected and their footprints can be researched in genomes.



\subsection{Footprints of gBGC in mammalian genomes}

One strong prediction of the gBGC model is that highly recombining regions should be GC-rich, which happens to be the case in several instances.

For example, components of the genome that undergo ectopic gene conversion (\textit{i.e.} conversion between copies of a gene family) — like tranfer RNAs (tRNAs), introns of ribosomal RNAs (rRNAs) \citep{galtier2001gccontent}, human and mouse major histocompatibility complex (MHC) regions \citep{hogstrand1999gene} and other gene families \citep{backstrom2005gene,galtier2003gene,kudla2004gene} — are all GC-richer than the rest of the genome.

The human pseudoautosomal region (PAR) of X and Y chromosomes — the only portion of male sexual chromosomes which has homology and therefore recombines — provides another example of the association between recombination and GC-content.
Indeed, given its short size, the per-nucleotide recombination rate (RR) of the PAR is much higher than that of autosomes \citep{soriano1987high}, while the non-PAR sections of sex chromosomes recombine even less (X chromosome) or not at all (Y chromosome).
Under the gBGC model, the average GC\textsubscript{3} of these four genomic domains is expected to increase with their RR — which, as a matter of fact, is the case \citep{galtier2001gccontent}.

This relationship between recombination and GC-content is really impressive in the \textit{Fxy} gene that has been translocated onto the boundary of the mouse PAR a few million years ago: as compared to its X-linked portion, the PAR-side part of \textit{Fxy} has undergone an acceleration in substitution rates \citep{perry1999evolutionary} together with a strong increase in GC-content at both coding and non-coding positions \citep{montoya-burgos2003recombination,galtier2007adaptation} — a finding that is consistent with gBGC occuring at the highly recombining PAR-side of the gene.
Surprisingly however, the \textit{XG} gene overlapping the PAR boundary of primates does not show the same pattern \citep{yi2004recombination}.
Nevertheless, this observation does not necessarily conflict with the gBGC model: if \textit{XG} was wholly located within the PAR before displacing onto its boundary, — or rather, before the PAR boundary displaces onto the gene, since the mammalian PAR gradually erodes \citep{lahn1999four,marais2003sex}, — it would have accumulated a high GC-content and would now be undergoing a slow, mutation-driven decrease in GC-content that would not be detectable yet \citep{galtier2004recombination}.\\


% Another strong prediction of the gBGC model is the influence of recombination on substitution patterns.
At the genome-wide scale, GC-content correlates positively with recombination rate in yeasts \citep{gerton2000global,birdsell2002integrating}, nematodes and flies \citep{marais2001does,marais2003neutral,marais2002hillrobertson}, birds \citep{internationalchickengenomesequencingconsortium2004sequence}, turtles \citep{kuraku2006cdnabased}, paramecia \citep{duret2008analysis}, algae \citep{jancek2008clues}, plants \citep{glemin2006impact} and humans \citep{fullerton2001local,yu2001comparison,meunier2004recombination,khelifi2006gc,duret2008impact}.
%, though with a low coefficient of determination ($R\textsuperscript{2} = 0.15$).

But, since the evolution of GC-content is relatively slow as compared to that of recombination rates in mammalian clades, it has been claimed that these estimates should be measured on similar time scales to be correctly compared \citep{duret2009biased}.
To do this, the stationary GC-content (GC\textsuperscript{*}), \textit{i.e.} the GC-content that sequences would reach at equilibrium if patterns of substitution remained constant over time, is generally used.
Under the assumption that all sites evolve independently from one another \citep{sueoka1962genetic}, this statistic can be calculated as:

\begin{equation*}
	GC\textsuperscript{*} = \frac{u}{u+v}
\end{equation*}

where $u$ and $v$ represent respectively the AT~$\rightarrow$~GC and the GC~$\rightarrow$~AT substitution rates.
But, because the latter assumption is not valid in vertebrates where the mutation rate of a given base depends on the nature of the neighbouring bases\footnote{For instance, CpG sites (\textit{i.e.} CG dinucleotides) are hypermutable \citep{arndt2003distinct}.}, \citet{duret2008impact} used a maximum likelihood approach to improve the estimation of GC\textsuperscript{*} and showed that it correlated better with recombination rate than with the observed GC-content (Figure~\ref{fig:correlation-recombi-stationary-GC}), which further suggests that recombination acts upon GC-content, and not the other way round, as was proposed by \citet{gerton2000global,blat2002physical} and \citet{petes2002context}.

In past primate lineages, GC\textsuperscript{*} also correlates well with the historical recombination rate \citep{munch2014finescale,lesecque2014red}.\\


% These correlations are in accordance with the gBGC model.

These correlations between GC-content and recombination appear to be greater in males than in females for several species including mice, dogs and sheeps \citep{popa2012sexspecific} as well as humans \citep{webster2005maledriven, dreszer2007biased, duret2008impact}.
% Also, GC\textsuperscript{*} is better correlated with male recombination rate (RR) than female RR in mice, dogs and sheeps \citep{popa2012sexspecific} as well as in human Alu repeats \citep{webster2005maledriven} and hotspots \citep{dreszer2007biased} and in 1-Mb windows of autosomes \citep{duret2008impact}.
Since female DSBs are repaired many years later than male DSBs \citep{coop2007evolutionary}, it is possible that the repair of mismatches proceeds differently in the two sexes, which could explain the seemingly male-specific gBGC \citep{duret2009biased}.
Alternatively, the sex-specific strategies for the distribution of recombination events along chromosomes (and more specifically, as a distance to telomeres) seem to account for this difference between males and females \citep{popa2012sexspecific}.


\begin{figure}[h]
	\centering
	\includegraphics[width = 0.7\textwidth]{figures/chap4/correlation-recombi-stationary-GC.eps}
	\caption[Correlation between the stationary GC-content (GC\textsuperscript{*}) and the crossover rate (cM/Mb) in human autosomes]
	{\textbf{Correlation between the stationary GC-content (GC\textsuperscript{*}) and the crossover rate (cM/Mb) in human autosomes.}
		\par Each dot corresponds to a 1-Mb-long genomic region. Green dots correspond to the predictions of the gBGC model. Data was extracted from the HapMap project.
		\par This figure was reproduced from \citet{duret2009biased} and originally adapted from \citet{duret2008impact}.
	}
\label{fig:correlation-recombi-stationary-GC}
\end{figure}





% Dans recombination: 
% longer in females humans (the map)
% A second-generation combined linkage physical map of the human genome.
% Matise TC, Chen F, Chen W, De La Vega FM, Hansen M, He C, Hyland FC, Kennedy GC, Kong X, Murray SS, Ziegle JS, Stewart WC, Buyske S
% Genome Res. 2007 Dec; 17(12):1783-6.
%
% longer in dogs
% A comprehensive linkage map of the dog genome.
% Wong AK, Ruhe AL, Dumont BL, Robertson KR, Guerrero G, Shull SM, Ziegle JS, Millon LV, Broman KW, Payseur BA, Neff MW
% Genetics. 2010 Feb; 184(2):595-605.







\section{Interference with natural selection}

Several of the aforementioned observations supporting gBGC would also be predicted under a natural selection model.
For instance, since linkage reduces the efficacy of selection \citep{hill1966effect}, a correlation between GC-content and recombination rate would be expected if there was a very high selection coefficient in favour of GC alleles \citep{galtier2001gccontent}.
More generally, the dynamics of the fixation process for one locus is identical no matter which of the two forces (biased gene conversion or natural selection) is responsible for it \citep{nagylaki1983evolution}, and this explains why the first observations were interpreted as resulting from natural selection \citep{eyre-walker1999evidence}.
In this subsection, I review a few case studies in which such confounding patterns between gBGC and natural selection exist.


\subsection{The case of codon usage bias (CUB)}

Codon usage bias (CUB) corresponds to the observation that the frequency of use of synonymous codons (\textit{i.e.} sequences of three nucleotides coding for the same amino acid (AA)) can vary across or within species \citep{fitch1976there}.
Both adaptative (natural selection) and non-adaptative (mutation \citep{marais2001synonymous} or biased gene conversion) forces account for CUB \citep{bulmer1991selectionmutationdrift,sharp1993codon,akashi1998translational}, but there remains a controversy about the quantitative contribution of each of these mechanisms to CUB \citep{pouyet2016etude}.\\

% It was early understood that codon usage responds to natural selection in microbial genomes \citep{ikemura1985codon} through what is called ‘translational selection’, \textit{i.e.} the coevolution of tRNA content and codon usage in a way that optimises translation.
In \textit{Drosophila}, the CUB of each gene is correlated to transfer RNA (tRNA) content \citep{akashi1994synonymous,duret1999expression,bierne2006variation,behura2011coadaptation}, particularly for genes that are highly expressed \citep{chavancy1979adaptation,shields1988silent,moriyama1997codon}.
This association between CUB and gene expression also holds true in \textit{Caenorhabditis elegans} \citep{duret1999expression}, \textit{Daphnia pulex} \citep{lynch2017population} and in single-celled organisms \citep{bennetzen1982codon,gouy1982codon,ikemura1985codon,sharp1987rate,sharp1989codon} and has been interpreted as ‘translational selection’: the coevolution of tRNA content with codon usage would increase either the accuracy or the efficiency of translation \citep{sharp1995dna,duret2002evolution}.
Though, other processes, like messenger RNA (mRNA) stability, protein folding, splicing regulation and robustness to translational errors could also play a role \citep[reviewed in \citealp{clement2017evolutionary}]{chamary2006hearing,cusack2011preventing,plotkin2011synonymous}.\\

In contrast, in lowly recombining regions of \textit{Drosophila} \citep{kliman1993reduced} and in species with small effective population size ($Ne$) \citep{subramanian2008nearly,galtier2018codon}, like mammals (but see \citealp{chamary2006hearing}), selection for codon usage is weak.
Instead, in mammals, codon usage is primarily governed by variations in GC-content \citep[but see \citealp{doherty2013translational}]{semon2006no,rudolph2016codondriven,pouyet2017recombination}, which implies that gBGC could be one of the drivers of CUB in that clade.

In \textit{Drosophila}, gBGC could also participate to CUB\@.
Indeed, one peculiar feature of codon usage in this species is that, for all 20 amino acids (AAs), the preferred codon systematically ends with a G or a C nucleotide \citep[reviewed in][]{duret2009biased}.
Even if the reason for this remains unknown, the finding that the base composition of the third position of 4-fold degenerate\footnote{A codon is said to be \textit{n}-fold degenerate if \textit{n} distinct three-nucleotide sequences result in the same amino acid (AA).} codons is similar to that of non-coding regions \citep{clay2011gc3} indicates that the patterns of CUB could (at least partly) come from evolutionary processes influencing base composition irrespectively of translational selection — such as gBGC \citep{duret2002evolution}. 
A similar observation made in plants was also interpreted as the consequence of gBGC \citep{clement2017evolutionary}.

% VOIR SI PARLE DES QUESTIONS METHODO (et donc, surement avec un transition sur HAR)
% METHODO (Codon Usage Bias in Animals: Disentangling the Effects of Natural Selection, Effective Population Size, and GC-Biased Gene Conversion  Nicolas Galtier)
% Disentangling the effect of natural selection from that of neutral forces such as gBGC is a difficult task (Ratnakumar et al. 2010). Clément et al. (2017) introduced a model of codon usage accounting both for gBGC, which is assumed to affect AT↔GC mutations, and selection, which is assumed to affect mutations between preferred and non preferred codons.
 % Here, we faced the additional problem that, in a substantial number of species, gene GC content was correlated to gene expression irrespective of codon usage, so that even defining preferred codons was challenging. We therefore addressed the problem by restricting our analysis of selection on codon usage to GC-conservative pairs of synonymous codons, which are supposedly unaffected by gBGC.
 % https://academic.oup.com/mbe/article/35/5/1092/4829954#115998508




\subsection{The case of human accelerated regions (HAR)}

gBGC has also been mistaken for positive selection in fast-evolving regions specific to the human genome \citep[reviewed in][]{duret2009biased}.
Such regions, — named human accelerated regions (HAR) when they code for proteins, and human accelerated conserved non-coding sequences (HACNS) otherwise, — have been searched for by several independent groups \citep{pollard2006forces,pollard2006rna,prabhakar2006accelerated,bird2007fastevolving,bush2008genomewide,lindblad-toh2011highresolution} in a quest to find the molecular adaptations that make the human genome distinct from other mammals.

HARs and HACNSs have first been interpreted as resulting from positive selection \citep[reviewed in][]{hubisz2014exploring} but, because they harbour an excess of AT~$\rightarrow$~GC substitutions, gBGC has been proposed as an alternative origin for these accelerated sequences \citep{galtier2007adaptation,berglund2009hotspots,duret2009comment,katzman2010gcbiased,ratnakumar2010detecting}.
And indeed, about one fifth of HARs seem to have evolved under gBGC alone \citep{kostka2012role}.\\


Altogether, gBGC mimics natural selection in terms of consequences on the nucleotidic sequence, and this is likely to bring biases to molecular evolution analyses \citep{berglund2009hotspots,ratnakumar2010detecting,webster2012direct,romiguier2017analytical,bolivar2018biased,bolivar2019gcbiased}.
Consequently, prior to concluding that positive selection explains a given observation, one should check that the extended null hypothesis of molecular evolution (\textit{i.e.} both the neutral and the gBGC models) has been rejected \citep{galtier2007adaptation, duret2009biased}.
To check for this, three observations should be taken into consideration: first, whether AT~$\rightarrow$~GC substitutions are preponderant; second, whether the studied locus is in a highly recombining region; and third, whether both functional and non-functional sites are affected.
Whenever all three criteria are met, gBGC remains a likely explanation for any observed acceleration in substitution rates.

But, if gBGC interferes with natural selection, what happens when both forces drive evolution in the opposite direction? 
% Can gBGC overcome natural selection?




\subsection{The deleterious effects of gBGC}


duret galtier 2009 major effect on slightly deleterious	and neutral
galtier 2009
higher frequency when close to recombination hostpots (necsu)

gBGC maintains deleterious mutations associated to human diseases \citep{necsulea2011meiotic,capra2013modelbased,lachance2014biased,xue2016basebiased}



depuis romiguier
When a mutation toward GC is deleterious, gBGC can counteract positive selection and maintain or fix deleterious alleles. High fixation rates of non-synonymous mutations at a locus should thus not be systematically interpreted as being beneficial for the fitness of the individual, particularly when considering that gBGC has been proved to be able to maintain deleterious mutations associated to human diseases (Necşulea et al., 2011; Capra et al., 2013; Lachance and Tishkoff, 2014).




+ explication evolutive pour reduire le bias vers AT



Plus loin: \citep{marais2003biased} pour dire que  GC-biased DNA repair has been
demonstrated in yeasts and vertebrates and conjectured
in most organisms, possibly reflecting an adaptation to
% frequent GC ! AT mutations
and importance of neutral side-effects
associated with recombination on genome evolution
(depuis meunier et Duret)

% De plus, Hershberg & Petrov (2010) ont aussi montré qu’il existe un biais universel vers AT chez les procaryotes qui impacte leur BUC : les mutations de G vers A ou de C vers T sont plus fréquentes que les autres mutations et notamment plus fréquentes que A vers G ou T vers C



gBGC is generally speaking a deleterious process in that it promotes G and C alleles irrespective of their effect on fitness (Galtier et al. 2009; Glémin 2010; Necşulea et al. 2011; Lachance and Tishkoff 2014). It might be that the molecular machinery involved in recombination is more efficiently selected to minimize b in large Ne species. A formal model would be required to validate this verbal hypothesis, though. 



\section{Characterisation and quantification of gBGC}
% \subsection{Theoretical insights}
\subsection{Characterisation \textit{via} theoretical studies}
- parler aussi de Ne ce que ca veut dire et equation de Nagylaki
- modeles d'evolution
\subsection{Quantification \textit{via} site frequency spectra (SFS)}
- donnees de polymorphisme
% \subsection{Empirical evidence/characterisation for gBGC}
\subsection{Characterisation \textit{via} empirical studies}
- caracterisation avec Lesecque (et d'autres?)
- Direct proofs


AUSSI A DIRE (Galtier 2018 codon) (deja dit dans l'intro, MAIS IL FAUT RAJOUTER QUE PAS DANS LA DROSO)
gBGC has been identified as the main driver of GC-content evolution in vertebrates (Duret and Galtier 2009; Figuet et al. 2014; Glémin et al. 2015; Bolívar et al. 2016) and several other taxa (Pessia et al. 2012; Glémin et al. 2014; Wallberg et al. 2015).

Et pas de gBGC dans la droso
% Robinson MC, Stone EA, Singh ND (2014) Population genomic analysis reveals no evidence for GC- biased gene conversion in Drosophila melanogaster. Molecular Biology & Evolution 31: 425–433.



\textbf{VOir entre recombining et recombinant region molecule fragment
corriger les two times higher
Reprendre Lesecque pour mieux expliquer l'eovlution des isochores.}


% faudrait citer http://pbil.univ-lyon1.fr/members/duret/publications/PDF/2006-Duret-Gene.pdf
%
% A SAVOIR POUR MOI
% duree de vie d'un hotspot?
% pourquoi plutot sur les bords?
% Est-ce que des cas avec AT favorise par rapport a GC

% \begin{figure}[!b]
%     \centering
%     \includegraphics[width = 1\textwidth, trim = 1.4cm 0cm 1.5cm 0cm]{figures/chap4/outcome-heteroduplex.eps}
%     \caption[Possible outcomes of mismatch repair in heteroduplex DNA for crossing-overs (COs)]
%     {\textbf{Possible outcomes of mismatch repair in heteroduplex DNA for crossing-overs (COs).}
%         \par ATTENTION, IL FAUDRAIT REFAIRE LA LEGENDE (si je garde cette figure de duret et galtier 2009) — cette legence est celle de Arbeithuber 2015.
%         During the repair of DSBs, mismatches in intermediate heteroduplex tracts at polymorphic sites (triangles) can be either resolved restoring the original allele or can lead to gene conversion (gBGC) favoring GC alleles (red) versus AT alleles (blue). In the case of gBGC, more COs will have breakpoints with GC alleles distal to the DSB than proximal, distorting the segregation ratio of alleles between reciprocals.
%         \par This figure was reproduced from \citet{duret2009biased}.
%     }
% \label{fig:outcome-heteroduplex}
% \end{figure}




Intensity of gBGC in species (Clement 2017) — B estimated.
We detected gBGC in all but one species but its intensity is rather weak (Tables 4 and 5 and S4 and S5 Tables), of the same order to what was estimated in humans [38] but lower than in other mammals [39], maize [72], and particularly honey bee [41].
72. Rodgers-Melnick E, Vera DL, Bass HW, Buckler ES (2016) Open Chromatin Reveals the Functional Maize Genome. Proceeding of the National Academy of Science USA in press.
% 38. Gle ́min S, Arndt PF, Messer PW, Petrov D, Galtier N, et al. (2015) Quantification of GC-biased gene conversion in the human genome. Genome Research 25: 1215–1228. https://doi.org/10.1101/gr. 185488.114 PMID: 25995268
% 39. Lartillot N (2013) Phylogenetic patterns of GC-biased gene conversion in placental mammals and the evolutionary dynamics of recombination landscapes. Molecular Biology & Evolution 30: 489–502.
% 40. Clement Y, Arndt PF (2013) Meiotic Recombination Strongly Influences GC-Content Evolution in Short Regions in the Mouse Genome. Molecular Biology and Evolution.
% 41. Wallberg A, Gle ́ min S, Webster MT (2015) Extreme recombination frequencies shape genome variation and evolution in the honeybee, Apis mellifera. PLoS Genetics 11: e1005189. https://doi.org/10.1371/ journal.pgen.1005189 PMID: 25902173
%
gBGC has been experi- mentally demonstrated in yeast [15,16], humans [17,18], birds [19] and rice [20]
 Many indi- rect genomic evidences also supported gBGC in eukaryotes [21,22] and even recently in some prokaryotes [23], although it seems to be weak or absent in some species as Drosophila [24] where selection on codon usage predominates [25,26,27,28].


Clement 2017
% Gle ́min et al. [33] also proposed that changes in the steepness of the recombination/gBGC gradient could explain variation in GC content distributions among species, from unimodal GC-poor to bimodal GC-rich distribution


Vu dans les especes (Codon Usage Bias in Animals: Disentangling the Effects of Natural Selection, Effective Population Size, and GC-Biased Gene Conversion  Nicolas Galtier)
In animals, gBGC had so far been identified in vertebrates (Figuet et al. 2014), bees, and ants (Kent et al. 2012; Wallberg et al. 2015), and Daphnia (Keith et al. 2016), but not in D. melanogaster (Robinson et al. 2014), albeit on the X chromosome (Galtier et al. 2006; Haddrill and Charlesworth 2008).
We here considerably expand the range of species and taxa in which gBGC is documented, adding annelids, echinoderms, tunicates, nemertians, cnidarians, lepidopterans, gastropod and bivalve molluscs, decapod and isopod crustaceans. 
This study adds to the growing evidence that gBGC is a nearly universal process affecting a wide range of organisms (Pessia et al. 2012; Long et al. 2018).


Relation avec Ne (Galtier)
In contrast, no relationship was detected between the intensity of gBGC and Ne in our analysis. 
% A similar pattern was recently reported in plants, based on a data set of eleven species (Clément et al. 2017). This result is somewhat surprising in that, just like selection, gBGC should only be effective if of magnitude well above that of drift. The intensity of the signal for gBGC is expected to be determined by the product of four parameters, namely Ne, the effective population size, r, the per base recombination rate, l, the length of gene conversion tracts, and b0, the repair bias in favor of GC. The rlb0 product is often denoted as b (Glémin et al. 2015). Our results rule out the hypothesis that b is constant—or a Ne effect should be detected. r, l and/or b0 must therefore vary substantially across species, and/or be inversely related to Ne.


Why b scales inversely with Ne
We can think of two possible reasons why b0 would scale inversely with Ne. First, gBGC is generally speaking a deleterious process in that it promotes G and C alleles irrespective of their effect on fitness (Galtier et al. 2009; Glémin 2010; Necşulea et al. 2011; Lachance and Tishkoff 2014). It might be that the molecular machinery involved in recombination is more efficiently selected to minimize b in large Ne species. A formal model would be required to validate this verbal hypothesis, though. Secondly, Lesecque et al. (2013) demonstrated that in yeast, when several SNPs are part of the same conversion tract, these are most often converted in the same direction—same donor and same recipient chromosomes—the direction only being influenced by SNPs located at the extremities of tracts. This implies a mechanical decay of the average GC bias as the number of SNPs per tract increases, since AT versus GC SNPs located in the middle of a conversion tract are converted in either direction with probability 0.5. This mechanism, if effective in animals too, might contribute to explaining the lack of a Ne effect on gBGC intensity, SNP density being positively correlated with Ne. Of note, the evolution of genomic GC-content has been associated with traits related to Ne in mammals and birds (Romiguier et al. 2010; Weber et al. 2014). This might be explained by b being fairly homogeneous within groups, but much more variable across distantly related taxa, so that B would only respond to Ne at a relatively small time scale. It might also be the case that the relationship between GC-content dynamics and life-history traits in mammals and birds is not (entirely) mediated by Ne—but rather by, for example, the mutation rate in a nonequilibrium situation (Romiguier et al. 2010; Bolívar et al. 2016).






Mettre plutot dans les caracteristiques
gBGC par MMR in yeasts (Lesecque)
et peut etre aussi chez humains (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4510005/)

+ dire que associe a des fins de tract de conversion

des pensees perso:
BER et MMR interagissent dans le cadre de la demethylation (https://www-ncbi-nlm-nih-gov.inee.bib.cnrs.fr/pmc/articles/PMC4856981/pdf/gkw059.pdf) donc pourraient aussi pendant la recombi

Explication sur BER\@: voir dans Duret 2009

Pessia: toutes les espces

%%% SAUVETAGE DES ONGLETS






% Nicolas and Petes — https://sci-hub.tw/10.1007/bf01924007
%
%
% gBGC
% Ere_walker — https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1460637/pdf/10353909.pdf
% Duret GC vanishing — https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1462357/pdf/12524353.pdf
% Lercher — http://www.lifesci.sussex.ac.uk/home/Adam_Eyre-Walker/Website/Publications_files/LercherGenetics02.pdf
% Meunier Duret — http://dev.bx.psu.edu/old/courses/bx-fall07/duret2.pdf
% Duret Eyre-Walker — http://pbil.univ-lyon1.fr/members/duret/publications/PDF/2006-Duret-Gene.pdf
% Galter Duret — https://www.sciencedirect.com/science/article/pii/S0168952507001138
% Muyle — https://academic.oup.com/mbe/article/28/9/2695/1014245
% https://sci-hub.tw/10.1002/9780470015902.a0020834.pub2
%
%
% ISOCHORES (surement inutile)
% Bernardi, G. (2000). Isochores and the evolutionary genomics of vertebrates. Gene, — https://sci-hub.tw/10.1016/s0378-1119(99)00485-0
% The genome: an isochore ensemble and its evolution — http://content.ebscohost.com.inee.bib.cnrs.fr/ContentServer.asp?EbscoContent=dGJyMMvl7ESep7E4y9f3OLCmr1Gep69Ssq24SbGWxWXS&ContentCustomer=dGJyMOzpsE21p7JOuePfgeyx9Yvf5ucA&T=P&P=AN&S=R&D=a9h&K=79720846
% Rise and fall of theory — https://en.wikipedia.org/wiki/Isochore_(genetics)
%
%
% CHAP3
% COA in yeast — https://www.nature.com/articles/ng.83
% DNA methylation state accessibility — https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3721348/
% https://www.google.com/search?ei=oILUXPiCHsSLlwSrwbeQDw&q=dna+methylation+chromatin+state+accessibility&oq=dna+methylation+chromatin+state+access&gs_l=psy-ab.1.0.33i22i29i30.683638.688043..689065...0.0..0.155.1655.4j10......0....1..gws-wiz.......0i7i30i19j0i19j0i22i30i19j33i21j33i160.9TVqd6gil2c
%
%
% Mouse (female recombination rate)
% http://www.informatics.jax.org/silver/frames/frame7-2.shtml
%
%
% Paper https://tex.stackexchange.com/questions/146911/alternatives-to-asterisk-and-star-for-superscripts
%
%
%
% gBGC in avian
% https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1613-z
%
% recherche papiers laurent
% https://scholar.google.fr/citations?user=Jhya3psAAAAJ&hl=fr&oi=sra
%








% Intro chap4: la recombinaison est mutagenique, et en particulier deamination des cytsines. Comment se fait-il donc que pas d'enrichissment en AT? Cela explique par l'existence du gBGC. 1. MMR BER etc (i.e. comment fonctionne) et isochores. 2. Ressemble à la selection naturelle. 3. Preuves directes et indirects
% CHAPITRE 4:
% BGC suppose pour contrecarrer le mutational load (dans intro?)
%
%
% VOIR SI RAJOUTER PLUS HAUT DANS MUTAGENIC (de capilla)
% LIEN AVEC GENET DIVERSITY
% Genetic diversity has been considered a good predictor of recombination rates; that is, levels of DNA sequence variation are normally reduced in genomic regions with low recombination rates [Begun and Aquadro, 1992; Nachman, 2001; Stevison et al., 2016].
% Four different causes have been considered to explain this correlation [Begun and Aquadro, 1992; Aquadro, 1997]: (1) the mutagenic effect of the recombination process itself, (2) functional constraints, (3) adaptive evolution (such as selective sweeps — the reduction of nucleotide variation due to positive selection), and (4) background selection (i.e., loss of genetic diversity at a non-deleterious locus due to negative selection).
% Among all of these possibilities, the correlation between recombination and genetic divergence (scored as the ratio of rates of substitution at non-synonymous and synonymous nucleotide sites, dN/dS) is, however, more controversial, given that conflicting results have been reported in different organisms [for a review, see Smukowski and Noor, 2011]. According to the Hill-Robertson effect [Hill and Robertson, 1966], natural selection can be less effective in regions of low recombination rates, affecting, as a result, rates of adaptation. Whereas this correlation has been detected neither in Drosophila nor in mice [Begun and Aquadro, 1992], contrasting results have been obtained in great apes [Nachman, 2001; Bussell et al., 2006; Stevison et al., 2016].
%





%% DONNER L'INFORMATION SUR LES CO COMPLEXES (deja un peu dit a la toute fin du chaptire 2)

% Dans chapitre 2 (ou 4?) — les evenements complexes identifies chez la levure
% and exchanges are not always simple, some exchange events are complex, containing both crossover and gene conversion events, interrupted by unconverted mark- ers (Borts and Haber 1989)

% Notons, de plus, que le tract est dit "simple" si, pour un même évènement de recombinaison (CO ou NCO) tous les mésappariements des hétéroduplexes sont convertis dans le même sens (i.e. pas d’alternance conversion / réparation). Dans le cas contraire on parle de tract de conversion "complexe" [Mancera et al., 2008].

% De plus, chez Saccharomyces cerevisiae , environ 11% des CO sont associés à des tracts complexes contre seulement 3,4% des NCO [Mancera et al., 2008]. Ceci est en accord avec le fait que le SDSA, principale voie de formation des NCO, ne donne que rare- ment des tracts complexes. On ignore encore l’origine des tracts complexes. Comme évoqué plus haut, ils pourraient venir d’une réparation des hétérodu- plexes par patches, alternant entre conversion et restauration. Ils pourraient aussi provenir de la résolution de dHj complexes comportant plusieurs hété- roduplexes comme suggéré par [Mancera et al., 2008].






%% INFO MISMATCH REPAIR DANS CHAPITRE 4
% % POUR APRES SI SAUTE:
% MISMATCH REPAIR
% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC86394/
% https://academic-oup-com.inee.bib.cnrs.fr/hmg/article/11/15/1697/636769
% https://cshperspectives-cshlp-org.inee.bib.cnrs.fr/content/7/3/a022657
% https://www-ncbi-nlm-nih-gov.inee.bib.cnrs.fr/pmc/articles/PMC4856981/pdf/gkw059.pdf
%
% DNA mismatch repair in mammals: role in disease and meiosis
% Norman Arnheim* and Darryl Shibatat
%
% http://content.ebscohost.com.inee.bib.cnrs.fr/ContentServer.asp?EbscoContent=dGJyMMTo50SeprM4v%2BbwOLCmr1Gep7NSsKu4TLWWxWXS&ContentCustomer=dGJyMOzpsE21p7JOuePfgeyx9Yvf5ucA&T=P&P=AN&S=R&D=a9h&K=2800408
% Andrew B. Buermeyer1, Suzanne M. Deschenes ˆ 1,Sean M. Baker2, and R. Michael Liskay
% MAMMALIAN DNA MISMATCH REPAIR
%
% Voir Hinch 2019 pour des references en biblio
%
% SAIS PLUS POURQUOI
% https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1558-5646.1981.tb04864.x
% https://www-sciencedirect-com.inee.bib.cnrs.fr/science/article/pii/S0169534715003286#bib0065
% https://www-annualreviews-org.inee.bib.cnrs.fr/doi/pdf/10.1146/annurev.genet.41.110306.130301





% %% DEPUIS LES Notes_for_cahp4.txt (deleted then)
% Sur le BGC: parler de la restauration/conservation des alleles.
% Hastings and colleagues in S. cerevisiae (46, 125) and in A. immersus(45)totestthehypothesisthatmismatchrepair cancorrecteithertowardstheinformationontheinvading strand(togiveaconversionevent)ortowardstheinforma- tionontherecipientstrand(togivearestorationevent).
% (depuis Orr-weaver and Szostak 1985, fungal recombination).
%
% BGC against load: voir Bengtsson 1985
% Biased conversion as the primary function of recombination
%
% MMR avec role dans BGC
% It is hard to find a single mechanism that is consistent
% with all of the genetic and physical data obtained in
% fungi. In S. cerevisiae, at the ARG4 locus, there is a
% double-strand break located near the high end of the
% conversion gradient 5~ and there is a good correlation
% between the level of gene conversion and the amount of
% single-strand excision 51. Although these results suggest
% that the polarity gradient of ARG4 might reflect only
% heteroduplex formation, E. Alani, R. Reenan and R.
% Kolodner (pers. commun.) have found that the steepness of the polarity gradient at ARG4 is substantially
% reduced by a mutation affecting mismatch repair. At the
% HIS4 locus, the conversion gradient for high PMS alleles is much less steep than that observed for low PMS
% alleles, and the gradient is nearly eliminated by one of
% the mutations affecting mismatch repair ~a. These results
% indicate that mismatch repair strongly influences the
% shape of the conversion gradient at this locus.
% %%%% de conclu de Nicolas and Pete Polarity of meiotic gene conversion in fungi: contrasting views (1994)
%




%%%% NOTES CHAPITRE PRECEDENT


% CHAPITRE 4
% chapitre 4: BGC comme conseqeucne de la recombinaison
% Isochores
% (+ BGC comme sel nat)
% (+ preuves directes et indirectes du BGC)



%%% DANS GENE CONVERSION (CHAPITRE 4)
% Parler de gene conversion + conversion/restauration
% Parler de tract
% Parler de heteroduplex
% MMR et BER


%%%% CHAPITRE 4
% %% QUAND PARLERAI DU MISMATCH REPAIR
% % https://en.wikipedia.org/wiki/Holliday_junction
% Robin Holliday proposed the junction structure that now bears his name as part of his model of homologous recombination in 1964, based on his research on the organisms Ustilago maydis and Saccharomyces cerevisiae. The model provided a molecular mechanism that explained both gene conversion and chromosomal crossover. Holliday realized that the proposed pathway would create heteroduplex DNA segments with base mismatches between different versions of a single gene. He predicted that the cell would have a mechanism for mismatch repair, which was later discovered.[3] Prior to Holliday's model, the accepted model involved a copy-choice mechanism[26] where the new strand is synthesized directly from parts of the different parent strands.[27]
% 3.  Liu Y, West S (2004). "Happy Hollidays: 40th anniversary of the Holliday junction". Nature Reviews Molecular Cell Biology. 5 (11): 937–44. doi:10.1038/nrm1502. PMID 15520813.
% 27.  Advances in genetics. Academic Press. 1971. ISBN 9780080568027.
%


%% CHAPITRE 4
%% GENE CONVERSION DES NCO (SDSA)
% Wikipedia https://en.wikipedia.org/wiki/Synthesis-dependent_strand_annealing
%SDSA is unique in that D-loop translocation results in conservative rather than semiconservative replication, as the first extended strand is displaced from its template strand, leaving the homologous duplex intact. Therefore, although SDSA produces non-crossover products because flanking markers of heteroduplex DNA are not exchanged, gene conversion does occur, wherein nonreciprocal genetic transfer takes place between two homologous sequences.[10]

% CHAPITRE 4
% % https://books.google.fr/books?id=7V0N6Tt8fUwC&pg=PA43&lpg=PA43&dq=murray+1960+polarity&source=bl&ots=mtj-qfJ1ZM&sig=ACfU3U1rKTqzCqEtcJkNw4ex96F_KPI87Q&hl=fr&sa=X&ved=2ahUKEwiG0b39-tfgAhUJ0RoKHRn4CWsQ6AEwB3oECAkQAQ#v=onepage&q=murray%201960%20polarity&f=false
% Sur la polarité des gene conversion DONC des sites precis ou la recombinaison demarre (a mettre dans les points chauds de recombinaison).
%
% N. Saitou, Introduction to Evolutionary Genomics, Computational Biology 17,
% % DOI 10.1007/978-1-4471-5304-7_2, © Springer-Verlag London 2013
% file:///Users/maudgautier/Downloads/9781447153030-c2.pdf
%
% %% GENE CONVERSION (pargraphe de Whitehouse ou Saitou ou autre??)
% Early studies on gene conversion were mostly restricted to fungal genetics. As
% molecular evolutionary studies of multigene family started, unexpected similarity
% of tandemly arrayed rRNA genes was found [ 15  ]. This phenomenon was termed
% ‘concerted evolution,’ and gene conversion or unequal crossing-over was proposed
% to explain this characteristic of some multigene families (e.g., [ 16  ]). New statistical
% methods were developed to detect gene conversion between homologous
% sequences [ 17, 18  ]. Program GENECONV developed by Sawyer [ 19  ] became the
% standard tool for analyzing gene conversions. We now know that conversion can
% occur in any genomic region irrespective of genes (DNA regions having function)
% or nongenic regions (e.g., [ 20  ]). However, ‘gene conversion’ as technical jargon is
% currently widely accepted, and I follow this nomenclature. Gene conversion can be
% classifi ed into two types: intragenic or between alleles and intergenic or between
% duplicated genes.
%
%
% mismatch repair
% https://fr.wikipedia.org/wiki/Mismatch_repair


% Question à moi-même (pour Laurent): si DSB, pourquoi la partie cassée du chromosome ne part pas ailleurs dans le cytoplasme?



% Biblio souris - meiose
% O’Bryan, M. K. & Kretser, D. Mouse models for genes involved in impaired spermatogenesis. Int. J. Androl. 29, 76–89 (2006).


%%% CHAPITRE 4
%% DANS LES ISOCHORES: 
%The genomes of many eukaryotes, including Saccharomyces cerevisiae, are mosaics of regions with high- and low-GC base composition.
%The Isochores as a Fundamental Level of Genome Structure and Organization: A General Overview. Costantini M, Musto H J Mol Evol. 2017 Mar; 84(2-3):93-103.

%DANS BGC: des gens qui disent que BGC pourrait etre une adaptation pour repondre au mutational load de la mutation (dans recombi). Eg: Crossovers are associated with mutation and biased gene conversion at recombination hotspots (Barbara Arbeithuber, Andrea J. Betancourt, Thomas Ebner, and Irene Tiemann-Boege 2015)
% Crossovers are associated with mutation and biased gene conversion at recombination hotspots (Barbara Arbeithuber, Andrea J. Betancourt, Thomas Ebner, and Irene Tiemann-Boege 2015) — ce papier important poir toutes les references sur BGC au debut.

%%% CHAPITRE 4: et notamment, distinguer BGC de mutation
% Dit dans COOP PRZEWORSKi: These observations raise the possibility that recombination might be mutagenic, as reported for mitotic recombination in yeast24. However, a recent analysis suggests that the broad-scale associations are not causal, but instead arise from covariates such as GC content25
% 25. Biased gene conversion plutot que mutagenique. Spencer, C. et al. The influence of recombination on human genetic diversity. PLoS Genet. 2, 1375–1385 (2006).
% A careful examination of the associations between diversity, divergence, genomic features and finescale recombination (inferred from LD patterns) on chromosome 20. The authors find evidence for biased gene conversion in recombination hotspots







% % SISTER
% Hinch: Nevertheless, it appears that some fraction of programmed meiotic DSBs are repaired using the sister chromatid [Hyppa and Smith, 2010].
% sex chromosomes: no homology donc repaired via sister chromatids at late prophase. Silenced (cf Altemose)
%
% % SYNCHRONISATION CELL CYCLE
% Meiotic success also hinges on theability to synchronize the meiotic transcriptional programwith cell cycle progression and cell growth. This isachieved in yeast by coupling double strand break for-mation with progression of the replication fork
% % Borde V, Goldman AS, Lichten M:Direct coupling betweenmeiotic DNA replication and recombination initiation.Science2000,290:806-809.
%
% % REGULATION
% This step regulated: Pachytene checkpoint: avoid defects (Handel Schimenti)
% If error, meiotic silencing (https://en.wikipedia.org/wiki/Synapsis)
%
% % DIFF MEIOSE MITOSE
% attachement des chromosomes par les kinetochores differe de la meiose (https://cshperspectives.cshlp.org/content/7/5/a015859.long)
% Autre difference avec la mitose: le spindle qui peut etre asymetrique.




%%% AUTRES CHAPITRES


%%  A GARDER POUR CHAPITRE 5 ET LES BIAIS DE LA METHODE SI ERREUR DE GENOTYPAGE
% Artefacts for sperm-typing (by Venn??)
% Cross-over results in the transmission of mosaic parental haplotypes to the next gen- eration. Consequently, we can view sibling genomes as independent samples of the parental haplotypes and the action of recombination on those haplotypes. Hence, to detect cross-over in pedigrees the task is to infer the inheritance pattern underlying the genotypes observed in family members (c.f., Thompson, 2000). Note that arte- fact (genotyping errors) and biological process (de novo mutation, which are worthy of their own study) can mimic the action of recombination at local-scales.









%% SOME POSSIBLE QUOTES
% CHAP 5
% “Truth has nothing to do with the conclusion, and everything to do with the methodology.”
% ― Stefan Molyneux

% “As to methods there may be a million and then some, but principles are few. The man who grasps principles can successfully select his own methods. The man who tries methods, ignoring principles, is sure to have trouble.”
% ― Harrington Emerson

% CHAP 
% If the facts do not fit the theory, change the facts! — Einstein.
% “Everything must be taken into account. If the fact will not fit the theory---let the theory go.” 
% ― Agatha Christie, The Mysterious Affair at Styles


% CHAP ABC (6)
% “So my antagonist said, "Is it impossible that there are flying saucers? Can you prove that it's impossible?" "No", I said, "I can't prove it's impossible. It's just very unlikely". At that he said, "You are very unscientific. If you can't prove it impossible then how can you say that it's unlikely?" But that is the way that is scientific. It is scientific only to say what is more likely and what less likely, and not to be proving all the time the possible and impossible.”
% ― Richard P. Feynman

% CHAP 7 (theorie de BGC qui evolue)
% “The whole [scientific] process resembles biological evolution. A problem is like an ecological niche, and a theory is like a gene or a species which is being tested for viability in that niche.”
% ― David Deutsch, The Fabric of Reality: The Science of Parallel Universes--and Its Implications







%%%% LISTE DE QUOTES UTILISEES COMME TEST POUR CHAPITRE 4

% Motoo Kimura
% "This neutral theory claims that the overwhelming majority of evolutionary changes at the molecular level are not caused by selection acting on advantageous mutants, but by random fixation of selectively neutral or very nearly neutral mutants through the cumulative effect of sampling drift (due to finite population number) under continued input of new mutations" (Kimura, 1991)

% Servira pour la partie 3 du chaptire 1
% KIMURA
 % "Looking back, I think that it is a curious human nature, that if a certain doctrine is constantly being spoken of favorably by the majority, endorsed by top authorities in their books and taught in classes, then a belief is gradually built up in one's mind, eventually becoming the guiding principle and the basis of value judgement. At any rate, this was the time when the panselectionist or 'neo-Darwinian' position was most secure in the history of biology: the heyday of the traditional 'synthetic theory' of evolution."
 % http://wasdarwinwrong.com/kortho37.htm


% % QUOTES POSSIBLES
% RONALD FISHER
% I believe that no one who is familiar, either with mathematical advances in other fields, or with the range of special biological conditions to be considered, would ever conceive that everything could be summed up in a single mathematical formula, however complex.
% The evolutionary modification of genetic phenomena. Proceedings of the 6th International Congress of Genetics 1, 165-72, 1932.
%
%
% % CHAP 7?? Fisher
% In relation to any experiment we may speak of this hypothesis as the “null hypothesis,” and it should be noted that the null hypothesis is never proved or established, but is possibly disproved, in the course of experimentation. Every experiment may be said to exist only in order to give the facts a chance of disproving the null hypothesis.
% The Design of Experiments, Edinburgh: Oliver and Boyd, 1935, p. 18
%
% % Stephen Jay gould
% Results rarely specify their causes unambiguously. If we have no direct evidence of fossils or human chronicles, if we are forced to infer a process only from its modern results, then we are usually stymied or reduced to speculation about probabilities. For many roads lead to almost any Rome.
% "Senseless Signs of History", p. 34
%
% % Pour chapitre 3 si je change
% But all evolutionary biologists know that variation itself is nature's only irreducible essence. Variation is the hard reality, not a set of imperfect measures for a central tendency. Means and medians are the abstractions.
%
%
% Just as in a book misprints are more likely to produce nonsense than better sense, so mutations will almost always be deleterious, almost always, in fact, they will kill the organism or the cell, often at so early a stage in its existence that we do not even realize it ever came into being at all. {John C. Kendrew,the Cambridge scientist who is a Nobel laureate for his discovery of the structure of the protein myoglobin, THE THREAD OF LIFE, 1966, pp.106-107}
%
% It is a considerable strain on one's credulity to assume that finely balanced systems such as certain sense organs (the eye of vertebrates, or the bird's feather) could be improved by random mutations. This is even more true of some ecological chain relationships. However, objectors to random mutations have so far been unable to advance any alternative explanation that was supported by substantial evidence. {Harvard biologist Ernst Mayr, SYSTEMATICS & THE ORIGIN OF SPECIES, 1942, p.296}
%
%
%
% Mutations and chromosomal changes arise in every sufficiently studied organism with a certain finite frequency, and thus constantly and unremittingly supply the raw materials for evolution. But evolution involves something more than origin of mutations. Mutations and chromosomal changes are only the first stage, or level, of the evolutionary process, governed entirely by the laws of the physiology of individuals. Once produced, mutations are injected in the genetic composition of the population, where their further fate is determined by the dynamic regularities of the physiology of populations. A mutation may be lost or increased in frequency in generations immediately following its origin, and this (in the case of recessive mutations) without regard to the beneficial or deleterious effects of the mutation. The influences of selection, migration, and geographical isolation then mold the genetic structure of populations into new shapes, in conformity with the secular environment and the ecology, especially the breeding habits, of the species. This is the second level of the evolutionary process, on which the impact of the environment produces historical changes in the living population.
% — Theodosius Dobzhansky
% Genetics and Origin of Species (1937), 13.
%
%
% Mutation is random; natural selection is the very opposite of random
% — Richard Dawkins
% The Blind Watchmaker (1996), 41
%
%
%
%
% The process of mutation is the only known source of the raw materials of genetic variability, and hence of evolution. It is subject to experimental study, and considerable progress has been accomplished in this study in recent years. An apparent paradox has been disclosed. Although the living matter becomes adapted to its environment through formation of superior genetic patterns from mutational components, the process of mutation itself is not adaptive. On the contrary, the mutants which arise are, with rare exceptions, deleterious to their carriers, at least in the environments which the species normally encounters. Some of them are deleterious apparently in all environments. Therefore, the mutation process alone, not corrected and guided by natural selection, would result in degeneration and extinction rather than in improved adaptedness.
% — Theodosius Dobzhansky
% % 'On Methods of Evolutionary Biology and Anthropology', American Scientist, 1957, 45, 385.
%
%
%http://wasdarwinwrong.com/kortho37.htm#Notes
%https://www.nature.com/scitable/topicpage/neutral-theory-the-null-hypothesis-of-molecular-839

% Citation de Dawkins a regarder

