\begin{savequote}[8cm]
‘Intense selection favours a variable response to the environment… Were this not so, the world would be much duller than is actually the case.’

\qauthor{--- John B. S. Haldane, \textit{\usebibentry{haldane1990causes}{title}} \citeyearpar{haldane1990causes} }
% ‘Now, \textit{here}, you see, it takes all the running \textit{you} can do, to keep in the same place. If you want to get somewhere else, you must run at least twice as fast as that!’
% \qauthor{--- Lewis Carroll, \textit{\usebibentry{carroll1871lookingglass}{title}} \citeyearpar{carroll1871lookingglass} }
\end{savequote}

\chapter{\label{ch:3-recombination-variation}Causes and consequences of recombination rate evolution} 
%\otherpagedecoration

\minitoc{}

% INTRO precendente: pourquoi une reproduction sexuee: c'est couteux, mais on le fait — reponse: favorisé par la selection naturelle car elimine les mutations deleteres et amene des combinaisons avantageuse
%
% INTRO ici: la recombi est interessante parce qu'elle promeut deux types de brassage (CO+segregation) ca fait de la diversite genetique.
% Formation de CO donc implique d'echanger des pans entiers des chromosomes.
% ca se traduit par des rearrangements chromosomiques.
% +variation environnement?
%
% AUTRE INTRO
% la recombi est interessante parce qu'elle promeut deux types de brassage (CO+segregation) ca fait de la diversite genetique, avec les CO qui echangent des pans entiers de chromosomes.
% Diversite genetique, sur laquelle la seleciton naturelle peut agir.
% Mais, selection naturelle, depend de l'environnement / par rapport a un milieu donne.
% Ce qui veut dire que va creer de la divergence entre les especes.
% la speciation se fait a partir d'addition d'effets locaux dans les especes (cf darwin)
% Pour pouvoir observer cela, il faut deja detecter la recombinaison.
% Ensuite, on peut etudier comment ca varie au sein des genomes
% Puis, on peut comprendre les causes et consequences (PRDM9 et RedQueen) des variations entre especes, et la speciation avec.
%
% https://www.evolutionmontpellier2018.org/program-second-joint-congress-evolutionary-biology



The very mechanism of meiosis introduces genetic mixing in two separate ways.
On the one hand, the paternal and maternal chromosomes are independently re-assorted during the first meiotic division.
On the second hand, genetic content is exhanged during recombination at the points where homologues cross over (a.k.a.\ chiasmata).

This genetic diversity instilled by meiosis forms the substratum upon which, according to Charles Darwin (1809--1882) himself, new species are ultimately generated — provided that a few conditions are met:

\begin{quote}
	\textit{‘The principle, which I have designated by this term [ed.\ divergence of character], is of high importance on my theory, and explains, as I believe, several important facts. […] according to my view, varieties are species in the process of formation, or are, as I have called them, incipient species. How, then, does the lesser difference between varieties become augmented into the greater difference between species?’}
	\qauthor{--- Charles Darwin, \textit{\usebibentry{charles1859origin}{title}} \citeyearpar{charles1859origin} }
\end{quote}

As enunciated by his theory, the prequisites to the variety-to-species transition also entail ‘a severe struggle for life [which] certainly cannot be disputed’, the occurrence of ‘variations useful to any organic being’ and ‘the strong principle of inheritance’ through which ‘they will tend to produce offspring similarly characterised’.
This, — which he calls ‘natural selection’, — is therefore tightly linked to the process of meiotic recombination.

Furthermore, the notion of biological species itself, formally defined by Ernst Mayr (1904--2005) as ‘groups of interbreeding natural populations that are reproductively (genetically) isolated from other such groups’ \citep{mayr1999systematics}, rests on the ability to sexually reproduce and thus, to meiotically recombine.

The relationship between these two concepts (further developed in \citealp{felsenstein1981skepticism} and \citealp{butlin2005recombination}) is such that, in the mammalian clade, the only speciation gene discovered so far is the one that controls the localisation of double-strand breaks (DSBs) — the starting points of recombination — on the genome.

I will come back to this essential gene and to its impact on the evolution of recombination rate in the third section of this chapter. 
But prior to that, I will review the existing methods to detect recombination genome-wide, and the observations on the multiple levels of recombination rate (RR) variation along genomes and across species.




%
%
%
% INTRO
% On comprend maintenant les mecanismes de la meiose et de la recombinaison.
% % Le principe meme de meiose implique deux types de brassage genetique: la segregation independante des homologues + la recombinaison, et la formation de CO. Donc ca cree de nouvelles combinaisons alleliques, considerees comme l'avantage du sexe par rapport a la reproduction non sexuee.
% En effet, sur ces combinaisons alleliques, il peut y avoir un phenomene de selection naturelle: on genere de la diversite avec la mutation et la selection, et on selectionne les interessantes avec la selection naturelle.
% D'apres Darwin, qui a invente le concept, ce sont ces additions elementaires de diversite qui forment ensuite des especes (car on voit de la continuuite).
% La speciation se fait effectivement lorsqu'une population s'isole et cree de la diversite. Quand elle est suffisamment differente des autres groupes, on va pouvoir la nommer comme une espece (et generalement, espece si descendant sterile). (i.e.ne recombine pas?)
% % et de fait, le gene de "speciation" des mammiferes est implique dans le controle de la position de la recombinaison sur le genome (on en parlera plus longuement en section 3).
% Avant de s'attarder sur ce gene qui explique donc la position de la recombinaison, il faut pouvoir detecter la recombinaison dans les genomes. En premiere partie, les methodes pour la detecter
% Puis, on va observer comment les taux varient au sein des genomes et entre les especes
% Ensuite, on parlera plus longuement de ce gene de speciation (PRDM9) et on verra quelle est une de ses consequences sur l'evolution du taux de recombinaison.
%
% CHAP 3: 1) comment mesurer et detecter la recombi. 2) les variations dans l'espace et le temps. 3) les positions des hotspots par PRDM9 + le dBGC




%
% Therefore during the modification of the descendants of any one species, and during the incessant struggle of all species to increase in numbers, the more diversified these descendants become, the better will be their chance of succeeding in the battle of life. Thus the small differences distinguishing varieties of the same species, will steadily tend to increase till they come to equal the greater differences between species of the same genus, or even of distinct genera.
%
%
%
%


\section{Genome-wide detection of recombination}
\subsection{Genetic maps and pedigree analyses}% deCODE (et historique Sturtevant, et definition des markers) Et ChIPs}
\subsection{Analysis of linkage disequilibrium (LD)}%(et HapMap et biais)}
\subsection{High-resolution sperm-typing studies}%genome-wide genotyping: high-resolution studies}

Voir Baudat de Massy 2013
Voir McVenn
Voir Lesecque
Voir Popa
Voir Hinch

% https://www.ncbi.nlm.nih.gov/books/NBK21986/ — tetrad analysis utilisee pour mapper les doubles CO
% MAPS LINKAGE
Quand parle des maps de linkage: voir citation Muller %1920:98-101
‘[I]t has never been claimed, in the theory of linear linkage, that the per cents of crossing over are actually proportional to the map distances: what has been stated is that the per cents of crossing overs are calculable from the map distances…’
% Spatial distribution of DSBs: Lam and Keeney 2015
% CHAPITRE 3: Revue de Massy initiation recombinaison parle des cartes de CO chez plein d'especes + les differences de recombinaison selon les regions genomiques
% POUR CHAPITRE 3: DONC au total pour observer, on peut regarder ATM, Spo11. Egalement generalement RAD51 et DMC1. mais donc plus large car dans processing. 200 a 400 foci dans les souris.


% \section{Methods}
% Genetic maps
% sperm assays
% linage of alleles and detection in pedigrees
% genetic maps using LD
% immunoprecip followed by sequencing
% Genetic marker
% Genetc maps (ordering markers, genetic distance to calculate, sex average VS sex-specific)
% LD (quantify LD, hapmap, biases with LD)
% sperm-typing
% Identify recombination events (pedigree + DeCODE, LD, count, models of recombination (approximate likelihood, approximate genealogy, african american admixture), typing in single gametes)
%

\section{The landscape of recombination}
\subsection{Positive crossing-over (CO) interference}%%% PARLER DE PAR
\subsection{Intragenomic patterns of variation}
\subsection{Inter-individual effects of sex and age}

Parler aussi de l'effet sur la diversite genetique
CO assurance and interference
Le long des genomes (compartiments genomiques — telomeres, centromeres, intergenique) + local chromatin accessibility + parler des hotspot?
Male femelle
%%%%%% chap3: DSBs: In humans and mice, DSBs tend to avoid centromeres and heterochromatic acrocentric short arms, although they are heavily enriched near telomeres in human males [17]. % ALTEMOSE

% CO interference + obligatory CO: lien avec CO interferents par Mlh1 et CO non interferents par Mus81 + au moins un dans le PAR
% DANS CROSSING-OVER: generate genetic diversity along with independent assortment (http://www.macmillanhighered.com/BrainHoney/Resource/6716/digital_first_content/trunk/test/hillis2e/hillis2e_ch07_5.html)


% CHAPITRE 3
% BAUDAT MASSY 2013
% In addition, the number of SPO11-dependent DSBs formed in every meiocyte is regulated by several factors. In contrast to crossovers, meiotic DSBs are difficult to quantify, especially because they are transient. Counting the foci formed by the recombinases RAD51 and DMC1 (see section below) has been widely used as the best available estimate of the number of DSBs in individual cells (oocytes or spermatocytes). It gives a count of 200–400 foci per cell in mice and humans3. A method for comparing the relative level of meiotic DSBs between mice of different genotypes was recently developed by quantifying the global level of oligonucleotides that are covalently attached to SPO11 (REF. 65). The role of the ataxia telangiectasia mutated (ATM) kinase as a nega- tive regulator of meiotic DSBs has been demonstrated using this method66. Situations in which SPO11 is unable to generate a wild-type number of DSBs correlate with defects in synapsis, suggesting that a minimum number of DSBs is needed to allow proper interactions between homologues67,68.
%


% Aussi des hotspots (chapitre 3) - depuis Baudat et de Massy CO
% CO are not randomly distrib- uted, but occur in multiple specific regions of the genome called CO hotspots (de Massy 2003, Kauppi et al. 2004). A hotspot is a region 1Y2 kb wide where CO are clustered, as a result of localized initiation events (see below). The average spacing between hotspots is 50Y100 kb and hotspot activities vary over three to four orders of magnitude, from 0.9 to 0.0005 cM as determined in the human genome. In the human genome the number of hotspots is estimated around 25 000Y50 000 (Myers et al. 2005). The variations of hotspot density and activity along chromosomes result in domains with high (jungles) or low (desert) recombination activity. Most sub-telomeric regions are recombination jun- gles in male meiosis whereas centromeric regions are recombination deserts. However, the determinants of CO variation along chromosomes are not known, even though some factors correlated with CO density are beginning to be analyzed (Buard & de Massy 2007, TIG in revision).
%


%% INTERFERENCE
% Cf lesecque revue + POPA Un paragraphe tres complet
% + dans Baudat 2007 et 2013: interfering par dHJ, non-interfering Ms81.
% Pas d'interference (ou peu? Baudat CO) dans les NCO

%% INTERFERENCE (CAPILLO)
% Since only a small fraction of DSBs are eventually processed as COs, a highly regulated genetic control determines both CO homeostasis and chromosomal distribution. In this way, if a CO occurs in a certain position, the probability for a new CO to take place nearby increases with chromosomal length. As a consequence, COs tend to follow an evenly spaced distribution across chromosome axes [Jones, 1967; Kleckner et al., 2003; Wang et al., 2015]. Importantly, this CO interference is influenced by the physical distance along the chromosomal axes (micrometers) rather than the genomic (Mb) or genetic distance (cM) [Wang et al., 2015]. However, not all COs are subject to interference, leading to recognition of interfering (class I) and non-interfering (class II) COs in different organisms [Hollingsworth and Brill, 2004; Phadnis et al., 2011]. Non-interfering COs are Mus81-Mms4 dependent and distribute themselves randomly along the chromosomes independent of each other, whereas interfering COs have been found to be distributed according to a gamma distribution. In mice, most COs manifest interference and are controlled by proteins Msh4-Msh5 [Berchowitz and Copenhaver, 2010], although some Mus81 activity has been detected during meiosis [Holloway et al., 2008]. Despite the evolutionary rationale of CO interference is still unknown, spaced COs ensure faithful chromosomal segregation and might facilitate linkage of functionally related genes [Wang et al., 2015 and references therein].

% OBLIGATORY CO
% These cytological analyses also show that at least one CO is formed per chromosome arm both in humans and mice, apart from short heterochromatic arms from acrocentric chromosomes. This regulation of CO frequency is a manifestation of the rule of the obligatory CO. Interestingly, mouse strains carrying Robertsonian translocations have two CO per chro- mosome, one per euchromatic arm (Dumas & Britton- Davidian 2002). The rule of the obligatory CO is also observed between the X and Y, where one CO is always observed in the pseudoautosomal region. A second level of CO regulation is shown by the
% measures of distances between chiasmata or Mlh1 foci, or by genetic distances. These show that, both in mouse and human, CO are not randomly distributed and are more evenly spaced than expected if they occurred independently, a phenomenon defined as positive interference (Lawrie et al. 1995, Laurie & Hulten 1985, Anderson et al. 1999, Broman & Weber 2000, Broman et al. 2002).
%

% Voir aussi Neil Hunter: CO control

% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003294/ : CO control (Yanowitz)

% CHAPITRE 3
% An additional checkpoint that monitors whether each chromosome has received a crossover is thought to exist, although molecular insight into this checkpoint is lacking[84].
% Depuis Yanowitz
% % 84. Mehrotra S, McKim KS. Temporal analysis of meiotic DNA double-strand break formation and repair in Drosophila females. PLoS Genet. 2006;2:e200. [PMC free article] [PubMed] [Google Scholar]
%
%
% Aussi: il faudra parler du fait que, chez les femmes, il y a d'autres DSB qui se forment de facon non programmee. Et sont repares differemments (en particulier, plus longs + plus complex? voir chapitre 3). Plus, des problemes de trisomie plus souvent chez les femmes a cause du dictyate arrest.
%







% - CO SC independent: Kohl and de los Santos
% A mettre plus loin quand on parle des CO ET DE LA POSITIVE INTERFERENCE??
% Loidl
% which stabilizes the connection between partner chromosomes (151). It is also crucial
% for the maturation of a certain class of COs, the number and distribution of which are controlled
% by mutual local exclusion (positive interference).
% This CO class is predominant in most eukaryotes studied so far, but a small proportion of COs are SC independent and noninterfering (see 29, 66).
% 66. Kohl KP, Sekelsky J. 2013. Meiotic and mitotic recombination in meiosis. Genetics 194:327–34
% 29. de los Santos T, Hunter N, Lee C, Larkin B, Loidl J, Hollingsworth NM. 2003. The Mus81/Mms4
% endonuclease acts independently of double-Holliday junction resolution to promote a distinct subset of
% crossovers during meiosis in budding yeast. Genetics 164:81–94

% SUR LES CROSSOVERS (CLASSES)
% Hunter - synaptonemal complexities
% Recent data suggest that two classes of crossovers form in S. cerevisiae (and probably plants and mammals; e.g., de Los Santos et al., 2003). The majority, Class-I, is subject to distribution controls and dependent on Zip1 and at least five other meiosis-specific proteins (Börner, V., Kleckner, N. and N. Hunter, unpublished data). Loss of Class-I crossovers causes chromosomes to missegregate in most meioses, inferring a critical function for these events, and thus SC, in S. cerevisiae. In contrast, Class-II crossovers are randomly distributed and SC independent. Loss of these events does not affect SCs or cause chromosomes to missegregate.




%
% \section{Variation}
% isochores
% reparition variation des CO
% differences within species
% differences between male and females
% genomic and environmental
% differences between species
% evolutionary effects of recombination rate variation on structure of genome (ducleotide diversity and divergence, distinguish scenarii, codon bias, other)
% broad scale rate and PAR
% CO assurance + interference
% intragenomic variation in recombination and natural selection
% Distribution of meiotic recombination (broad scale, fine scale, PRDM9)
% role of genetic variation
%
%

\section{Evolvability of recombination rates (RR)}
\subsection{Inter- and intra-species comparison of fine-scale RR}
\subsection{PRDM9, the fast-evolving mammalian speciation gene}
\subsection{The Red Queen dynamics of hotspot evolution}
hotspot paradox et dBGC

PRDM9 (role hybrid sterility speciation)
positive sel prdm9
%exemple de l'un qui a perdu PRDM9 mais qui n'est pas stérile pour autant.

% \section{Evolution}
% PRDM9 (role hybrid sterility speciation)
% hotspots and evolution
% positive sel prdm9
% Rapid evolution of the landscape







% GENERAL

% Troisieme chapitre:
% Methodological approaches to study recombination
% Variation of recombination rates wtithin genomes and among species
% evolvability of recombination rates
%
% Plan Mammalian Meiotic Recombination: A Toolbox for Genome Evolution (https://www.karger.com/Article/FullText/452822):
% \begin{itemize}
%     \item Recombination and he repair of DSBs (Organization of meiotic chromosomes: importance of chromosomal axes, molecular events involved in he formation and repair of DSBs)
%     \item Methodological approaches to he study of recombination
%     \item Genetic and epigenetic marks of DSBs and recombination hotspots
%     \item Variation of recombination rates within genomes and among species (Variability at the chromosomal level, variation of fine-scale recombination maps)
%     \item Evolvability of recombination rates (Chromosomal rearrangements as recombination modifiers)
% \end{itemize}
%
% Plan de Hotposts for initiation of meiotic recombination (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6237102/)
% \begin{itemize}
%     \item Defining DSB hotspots
%     \item Chromatin shapes the meiotic DSB landscape (Nucleosome occupancy, meiotic chromosome architecture)
%     \item Meiotic DSB and crossover distributions
%     \item PRDM9 and H3K4me3
%     \item The hotspot paradox
%     \item Recombination initiation in repetitive sequences
%     \item Byond hotspots: DSB-dependent spatial regulation
% \end{itemize}
%
% Autre:
% \begin{itemize}
%     \item Recombining without hotspots (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4684701/)
%     \item Knockout of PRDM9 (http://science.sciencemag.org.inee.bib.cnrs.fr/content/352/6284/474)
% \end{itemize}








%%% THESES DES AUTRES
% these darrier - partie recombinaison et DL
% methodes detection
% repartition et variation des CO
% points chauds de recombi
% evolution des points chaud
% DL
%
% 2 theses droso rate variation Fine scale recombination variation in Drosophila melanogaster
% many levels of varation
% diff species (between within)
% sex-biased recombi
% genomic and environmental variation
% +
% evolutionary effects of recombination rate variation on structure of genome (ducleotide diversity and divergence, distinguish scenarii, codon bias, other)
%
%
% odenthal:
% rapid evolution of the landscape in human genomes
%
%
% these prdm9 bovins
% diversite de prdm9 (altemose)
%
%
% Hinch: Detect and measure
% sperm assays
% linage of alleles and detection in pedigrees
% genetic maps using LD
% immunoprecip followed by sequencing
%
% Hinch: localisation, control and evolution of recombination
% CO assurance + interference
% hotspots of recombination
% evolution of hotspots
% role of genetic variation
% differences between male and females
% broad scale rate and PAR
%
% Hussin:
% intragenomic variation in recombination and natural selection
% detecting
% patterns of variation in humans
%
% Lesecque: dynamique spatiale et temporelle de recombi et BGC
% mesurer recombi
% taux de recombinaison le long des genomes
% dynamique temporelle des points chauds
%
% POPA
% Genetic marker
% Genetc maps (ordering markers, genetic distance to calculate, sex average VS sex-specific)
% LD (quantify LD, hapmap, biases with LD)
% sperm-typing
% gene conversion rate
%
% VENN
% Distribution of meiotic recombination (broad scale, fine scale, PRDM9)
% Identify recombination events (pedigree + DeCODE, LD, count, models of recombination (approximate likelihood, approximate genealogy, african american admixture), typing in single gametes)
%
%





















%%%% NOTES CHAPITRE PRECEDENT




% CHAPITRE 4
% chapitre 4: BGC comme conseqeucne de la recombinaison
% Isochores
% (+ BGC comme sel nat)
% (+ preuves directes et indirectes du BGC)



%%% DANS GENE CONVERSION (CHAPITRE 4)
% Parler de gene conversion + conversion/restauration
% Parler de tract
% Parler de heteroduplex
% MMR et BER


%%%% CHAPITRE 4
% %% QUAND PARLERAI DU MISMATCH REPAIR
% % https://en.wikipedia.org/wiki/Holliday_junction
% Robin Holliday proposed the junction structure that now bears his name as part of his model of homologous recombination in 1964, based on his research on the organisms Ustilago maydis and Saccharomyces cerevisiae. The model provided a molecular mechanism that explained both gene conversion and chromosomal crossover. Holliday realized that the proposed pathway would create heteroduplex DNA segments with base mismatches between different versions of a single gene. He predicted that the cell would have a mechanism for mismatch repair, which was later discovered.[3] Prior to Holliday's model, the accepted model involved a copy-choice mechanism[26] where the new strand is synthesized directly from parts of the different parent strands.[27]
% 3.  Liu Y, West S (2004). "Happy Hollidays: 40th anniversary of the Holliday junction". Nature Reviews Molecular Cell Biology. 5 (11): 937–44. doi:10.1038/nrm1502. PMID 15520813.
% 27.  Advances in genetics. Academic Press. 1971. ISBN 9780080568027.
%


%% CHAPITRE 4
%% GENE CONVERSION DES NCO (SDSA)
% Wikipedia https://en.wikipedia.org/wiki/Synthesis-dependent_strand_annealing
%SDSA is unique in that D-loop translocation results in conservative rather than semiconservative replication, as the first extended strand is displaced from its template strand, leaving the homologous duplex intact. Therefore, although SDSA produces non-crossover products because flanking markers of heteroduplex DNA are not exchanged, gene conversion does occur, wherein nonreciprocal genetic transfer takes place between two homologous sequences.[10]

% CHAPITRE 4
% % https://books.google.fr/books?id=7V0N6Tt8fUwC&pg=PA43&lpg=PA43&dq=murray+1960+polarity&source=bl&ots=mtj-qfJ1ZM&sig=ACfU3U1rKTqzCqEtcJkNw4ex96F_KPI87Q&hl=fr&sa=X&ved=2ahUKEwiG0b39-tfgAhUJ0RoKHRn4CWsQ6AEwB3oECAkQAQ#v=onepage&q=murray%201960%20polarity&f=false
% Sur la polarité des gene conversion DONC des sites precis ou la recombinaison demarre (a mettre dans les points chauds de recombinaison).
%
% N. Saitou, Introduction to Evolutionary Genomics, Computational Biology 17,
% % DOI 10.1007/978-1-4471-5304-7_2, © Springer-Verlag London 2013
% file:///Users/maudgautier/Downloads/9781447153030-c2.pdf
%
% %% GENE CONVERSION (pargraphe de Whitehouse ou Saitou ou autre??)
% Early studies on gene conversion were mostly restricted to fungal genetics. As
% molecular evolutionary studies of multigene family started, unexpected similarity
% of tandemly arrayed rRNA genes was found [ 15  ]. This phenomenon was termed
% ‘concerted evolution,’ and gene conversion or unequal crossing-over was proposed
% to explain this characteristic of some multigene families (e.g., [ 16  ]). New statistical
% methods were developed to detect gene conversion between homologous
% sequences [ 17, 18  ]. Program GENECONV developed by Sawyer [ 19  ] became the
% standard tool for analyzing gene conversions. We now know that conversion can
% occur in any genomic region irrespective of genes (DNA regions having function)
% or nongenic regions (e.g., [ 20  ]). However, ‘gene conversion’ as technical jargon is
% currently widely accepted, and I follow this nomenclature. Gene conversion can be
% classifi ed into two types: intragenic or between alleles and intergenic or between
% duplicated genes.
%
%
% mismatch repair
% https://fr.wikipedia.org/wiki/Mismatch_repair


% Question à moi-même (pour Laurent): si DSB, pourquoi la partie cassée du chromosome ne part pas ailleurs dans le cytoplasme?



% Biblio souris - meiose
% O’Bryan, M. K. & Kretser, D. Mouse models for genes involved in impaired spermatogenesis. Int. J. Androl. 29, 76–89 (2006).







% % SISTER
% Hinch: Nevertheless, it appears that some fraction of programmed meiotic DSBs are repaired using the sister chromatid [Hyppa and Smith, 2010].
% sex chromosomes: no homology donc repaired via sister chromatids at late prophase. Silenced (cf Altemose)
%
% % SYNCHRONISATION CELL CYCLE
% Meiotic success also hinges on theability to synchronize the meiotic transcriptional programwith cell cycle progression and cell growth. This isachieved in yeast by coupling double strand break for-mation with progression of the replication fork
% % Borde V, Goldman AS, Lichten M:Direct coupling betweenmeiotic DNA replication and recombination initiation.Science2000,290:806-809.
%
% % REGULATION
% This step regulated: Pachytene checkpoint: avoid defects (Handel Schimenti)
% If error, meiotic silencing (https://en.wikipedia.org/wiki/Synapsis)
%
% % DIFF MEIOSE MITOSE
% attachement des chromosomes par les kinetochores differe de la meiose (https://cshperspectives.cshlp.org/content/7/5/a015859.long)
% Autre difference avec la mitose: le spindle qui peut etre asymetrique.







