\begin{savequote}[8cm]
	
	‘So my antagonist said, “Is it impossible that there are flying saucers? Can you prove that it's impossible?” “No”, I said, “I can't prove it's impossible. It's just very unlikely”. At that he said, “You are very unscientific. If you can't prove it impossible then how can you say that it's unlikely?” But that is the way that is scientific. It is scientific only to say what is more likely and what less likely, and not to be proving all the time the possible and impossible.’
	
	\qauthor{--- Richard Feynman, \textit{\usebibentry{feynman1964character}{title}} \citeyearpar{feynman1964character} }
	
\end{savequote}

% \chapter{\label{ch:6-recombination-parameters}Estimation of mouse recombination parameters}
% \chapter{\label{ch:6-recombination-parameters}Characterisation of mouse recombination hotspots}
% \chapter{\label{ch:6-recombination-parameters}Characterisation of mouse meiotic recombination}
\chapter{\label{ch:6-recombination-parameters}Characterisation of recombination in mouse autosomal hotspots}
% \chapter{\label{ch:6-recombination-parameters}Characterisation of recombination in autosomal hotspots of mice}
%\otherpagedecoration


\minitoc{}

{\small{} \itshape{}

\paragraph{This chapter in brief —}

Even if recombinational activity is known to vary by orders of magnitude across individual hotspots, the properties determining this variation are still poorly understood.
Further progress in comprehending the basis of these fluctuations can only arise with the thorough examination of individual hotspots but, in mammals, only a handful of these have been directly characterised at high resolution.
% after the characterisation of individual hotspots.
% However, in mammals, only a handful of these have been directly characterised at high resolution.
Here, thanks to the 18,821 events that we detected with the approach developed in Chapter~\ref{ch:5-methodology}, we identified the main factors governing the recombinational activity of individual hotspots, we precisely described recombination in over a thousand hotspots and we estimated the hidden biological parameters of recombination through an inferential approach.
%we precisely described recombination in over a thousand mouse autosomal hotspots thanks to the 18,821 events that we detected with the approach developed in Chapter~\ref{ch:5-methodology} and we used an inferential approach to accurately estimate the biological parameters of recombination in mice.
% We first identified the main factors governing the recombinational activity of individual hotspots and then inferred the real recombination parameters by indirect approaches.
% We found that PRDM9 binding affinity predicted accurately the recombinational activity and that, overall, the relative proportion of COs and NCOs and the lengths of their conversion tracts do not differ much across autosomal hotspots.
Overall, this study provides the first global picture of recombination patterns in mouse autosomal hotspots.

% Pourquoi on caracterise la recombi: pour l'instant, caracterise sur qqs hotspots selectionnes (voir la liste en annexe) et sur le chromosome 1. Mais pas effectue en detail sur un grand nombre de hotspots. Nous, on veut voir si les parametres mesures sur les hotspots etudies sont representatifs de l'ensemble des hotspots et ce qui explique la variation de recombinaison entre hotspots (les determinants).
% Pour ce faire, on utilise les 18,821 evenements detectes avec l'approche developpee au chapitre~\ref{ch:5-methodology}.
% Premierement, les evenements detectes semblents etre appeles avec confiance, car sont correles a plusieurs determinants de la recombinaison.
% On peut donc caracteriser les recombinants tranquillement: position et longuer des tracts, donneurs, proportion de CO/NCO par hotspot…
% Mais, ce ne sont pas les rels parametres de recombi qui, eux, sont caches.
% Donc, il faut passer par une approche inferentielle pour extrapoler les parametres reels.
% On trouve des parametres prfaitement compatibles avec ce qui a ete observe chez qqs hotspots.

}

\newpage

Although many of the molecular details of the recombination process have been dissected (see Chapter~\ref{ch:2-recombination-mechanistics}) and the recombinational activity is known to vary by orders of magnitude across mouse individual hotspots \citep{paigen2008recombinational}, the properties determining this variation are still poorly understood.
So far, in mice, only a handful of recombination hotspots have been directly characterised \citep{yauk2003highresolution,bois2007highly,baudat2007cis,ng2008quantitative,cole2010comprehensive,cole2014mouse}.
But, thanks to the recent development of techniques allowing to precisely map recombination hotspots, — like ChIP-seq against DMC1 and RAD51 \citep{smagulova2011genomewide}, sequencing-based detection of DMC1-bound ssDNA \citep{khil2012sensitive, brick2012genetic} and sequencing-based detection of SPO11 oligos \citep{lange2016landscape}, — high-resolution genome-wide maps of recombination initiation hotspots have been obtained in several mouse strains and F1 hybrids \citep{smagulova2016evolutionary}.

% Recent techniques like ChIP-seq against DMC1 and RAD51 \citep{smagulova2011genomewide} and sequencing-based detection of DMC1-bound ssDNA \citep{khil2012sensitive, brick2012genetic} and of SPO11 oligos \citep{lange2016landscape} allow to precisely map recombination hotspots.
% Thanks to these, high-resolution genome-wide maps of recombination initiation hotspots have been obtained in several strains and F1 hybrid mice \citep{smagulova2016evolutionary}.
%
% But only a handful of mouse recombination hotspots have been directly characterised \citep{yauk2003highresolution,bois2007highly,baudat2007cis,ng2008quantitative,cole2010comprehensive,cole2014mouse}
% But the recombination activity can vary by orders of magnitude \citep{paigen2008recombinational}.
% This calls for understanding the parameters that modify this magnitude.

Here, to better understand the extent of the variation in recombinational activity and the factors governing it, we precisely characterised recombination in 1,018 hotspots of a mouse F1 hybrid descended from a cross between \textit{Mus musculus domesticus} (strain C57BL/6J, hereafter called B6) and \textit{Mus musculus castaneus} (strain CAST/EiJ, hereafter called CAST).
% we propose to account for this lack by precisely recombination activity in over 1,000 mouse hotspots.
In this chapter, I show how the set of recombination events detected with the approach developed in Chapter~\ref{ch:5-methodology} allowed us to describe some of the determinants of recombinational activity, better characterise recombination and infer its hidden parameters \textit{via} inferrential approaches.
% describe what the determinants of recombinational activity are
% show that the set of recombination events that we found with the approach developed in Chapter~\ref{ch:5-methodology} is reliable to characterise recombination and infer its hidden parameters \textit{via} inferrential approaches.
%





% \section{Confidence in the detected events}
% \section{A high-confidence set of recombination events}
\section{Determinants of recombinational activity}





% \subsection{Consistency with observations on one chromosome}
\subsection{A high-confidence set of recombination events}

\citet{paigen2008recombinational} have examined in detail the recombinational activity of one B6xCAST chromosome (chromosome 1), with a particular focus on the telomere-proximal 24.7 Mb.
33 of the intervals of the latter region contained hotspots that were all included in our study. 
We thus compared the recombination rates they found on these intervals to the total number of recombination events we observed in the 37 hotspots comprised in these intervals (Figure~\ref{fig:correlation-paigen}).

\begin{figure}[t]
    \centering
    \includegraphics[width = 1\textwidth]{figures/chap6/Correl-paigen-log-events-vs-RR-BIS.eps}
	\caption[Correlation between the expected recombination rate and the observed number of events on the 33 intervals analysed by \citet{paigen2008recombinational}]
	{\textbf{Correlation between the expected recombination rate and the observed number of events on the 33 intervals analysed by \citet{paigen2008recombinational}.}
		\par We compared the recombination rate of 33 intervals defined by \citet{paigen2008recombinational} to the total number of events we detected on these intervals, brought back to the length of each interval (see main text).
		The intervals selected were those which exclusively encompassed hotspots that were analysed in our study.
		The adequation between the two measures was extremely high (Pearson correlation: $R^2 = 0.974$; \textit{p}-val $< 2.2 \times 10^{-16}$).
    }
\label{fig:correlation-paigen}
\end{figure}


% cut -f10 $DATA/2_dBGC/6_ThirdSequencing/08_Compare_with_Paigen/01_Original_Paigen_Data/PositiveControls.txt|sed '1,1d' | sed 's/,/./'|Rscript -e 'summary(as.numeric(readLines("stdin")))'
%   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.
%    1.8   185.9   205.7   251.8   372.4   877.2
Though, the intervals they defined are much wider ($median = 205$ kb) than the ones we analysed (1 kb).
Thus, the recombination rate they measured equals:

\begin{equation*}
	R_i = \frac{\sum\limits_{h=1}^{n_{h}^{i}} ( R_h \times L_h)}{L_i}
\end{equation*}

where $R$ and $L$ respectively represent the recombination rate and the length of the region considered, the subscripts $i$ and $h$ respectively stand for the interval defined by \citet{paigen2008recombinational} and the 1-kb hotspots we defined, and $n_h^i$ corresponds to the number of hotspots in interval $i$.

We thus brought back the total number of events we detected to the length of the interval and found that it correlated extremely well with their data (Pearson correlation: $R^2 = 0.974$; \textit{p}-val $< 2.2 \times 10^{-16}$).
Therefore, the recombination events we detected are highly reliable since they concord exceptionally well with those identified by an independent study.



\begin{figure}[b!]
    \centering
    \includegraphics[width = 1\textwidth]{figures/chap6/Correlation_intensity_PRDM9.eps}
	\caption[Proportionality between the recombination rate and PRDM9 binding intensity]
	{\textbf{Proportionality between the recombination rate and PRDM9 binding intensity.}
		\par All 1,018 hotspots were divided into 10 classes of increasing PRDM9 signal (x-axis), i.e.\ the number of PRDM9 ChIP-seq tags on each PRDM9 ChIP-seq peak, brought back to the width of the peak (PRDM9 ChIP-seq data on B6xCAST hybrid mice from \citealp{baker2015prdm9}).
		The observed number of recombination events identified per sequenced Mb (left y-axis) was converted into a CO rate (right y-axis) as detailed in Subsection~\ref{chap6:extrapolation-CO-rate}.
		The points and error bars respectively represent the mean number of events (or CO rate) and the standard error on the mean for hotspots of each class.
    }
\label{fig:correlation-PRDM9-standard-error}
\end{figure}


\subsection{PRDM9 binding, a predictor of hotspot intensity}%Correlation with PRDM9 hotspot intensity}
% \subsection{PRDM9 binding and recombinational activity}%Correlation with PRDM9 hotspot intensity}
% \subsection{PRDM9 hotspot intensity: a good predictor of activity}%Correlation with PRDM9 hotspot intensity}

Next, we analysed the relationship between PRDM9 binding and the recombinational activity of the hotspots we selected: we found that the more intensely bound by PRDM9 a hotspot was, the greater its recombinational activity (Figure~\ref{fig:correlation-PRDM9-standard-error}).
A similar relationship was found with DMC1 binding intensity, using DMC1 ChIP-seq data from \citet{smagulova2016evolutionary} (data not shown).

Thus, PRDM9 binding intensity as well as DMC1 binding intensity (both proxies of the propensity for a hotspot to form DSBs) are accurate predictors of recombinational activity.





\subsection{The lower recombination rate of asymmetric hotspots}
% \subsection{DMC1 hostpot intensity: a good predictor of activity}%Correlation with DMC1 hotspot intensity}
% $DATA/2_dBGC/6_ThirdSequencing/05_Analyses_of_Recombinants/03_General_Statistics/05_Distribution_of_recombinants/Intensity_and_counts_good_classes_and_DMC1.txt



In a B6xCAST hybrid, the PRDM9 target motif located on the homologous chromosome originating from the B6 parent (hereafter called B6 haplotype) may differ from that located on the homologue originating from the CAST parent (hereafter called CAST haplotype) because of the rapid mutational accumulation along the separate lineages (B6 or CAST) since their common ancestor \citep{davies2016reengineering,smagulova2016evolutionary}.
Consequently, in certain hotspots, PRDM9 may bind preferentially one of the two haplotypes while, in other hotspots, it may bind both haplotypes equally.
The former class of hotspots is referred to as ‘asymmetric’ and the latter as ‘symmetric’.

\citet{li2018highresolution} previously identified that such variations in hotspot asymmetry explain part of the variations in recombination rates for a given DMC1 signal.
To reproduce their finding, we distinguished between symmetric and asymmetric hotspots based on the strand-specific PRDM9 ChIP-seq data from \citet{baker2015prdm9} (see Chapter~\ref{ch:7-quantification-BGC}).
We found that, for a given PRDM9 signal, the number of recombination events was greater for symmetric than for asymmetric hotspots (two to four times greater) (Figure~\ref{fig:asymmetry-and-PRDM9}).
We found a similar relationship with strand-specific DMC1 signal (data not shown).

As hypothesised by \citet{li2018highresolution}, this relationship can be explained if these asymmetric hotspots are repaired using the sister chromatid instead of the homologue.
Since one haplotype is not bound by PRDM9 in such asymmetric hotspots, it is possible that the presence of PRDM9 on both homologues may play a role in homology search.
Yet, hotspot asymmetry does not account for all the variation (see the width of boxplots in Figure~\ref{fig:asymmetry-and-PRDM9}): instead, the random sampling of hotspots, or another (unknown) source of variability may explain the residual variation.
% Yet, there remains a residual variation that cannot be explained by hotspot asymmetry, and may alternatively come from the random sampling of hotspots or from a large genuine variability between hotspots.
% De plus, on l'observe directement avec PRDM9 alros que eux DMC1, ce qui veut dire qu'il ne s'agit pas d'un effet uniquement du temps de reparation, mais bien un effet des hotspots asymetriques qui sont moins forts.
% Pourquoi seraient moins forts?


\begin{figure}[t]
    \centering
    \includegraphics[width = 1\textwidth]{figures/chap6/Recombination_rates_2_axes_sym_vs_Asym_PRDM9_with_legend_and_outliers.eps}
    \caption[Asymmetric hotspots display lower recombinational activity than expected by their PRDM9 binding affinity]
    {\textbf{Asymmetric hotspots display lower recombinational activity than expected by their PRDM9 binding affinity.}
		\par All 1,018 hotspots were divided into 10 classes of increasing PRDM9 signal (x-axis), i.e.\ the number of PRDM9 ChIP-seq tags on each PRDM9 ChIP-seq peak, brought back to the width of the peak (PRDM9 ChIP-seq data on B6xCAST hybrid mice from \citealp{baker2015prdm9}).
		The observed number of recombination events identified per sequenced Mb (left y-axis) was converted into a CO rate (right y-axis) as detailed in Subsection~\ref{chap6:extrapolation-CO-rate}.
		Symmetric hotspots (green) are distinguished from asymmetric hotspots (orange) as detailed in Chapter~\ref{ch:7-quantification-BGC}.
		The linear regression model for symmetric (slope~$= 18$; intercept~$=0$; \textit{p}-val~$< 2.2 \times 10^{-16}$) and asymmetric (slope~$= 7.9$; intercept~$=0$; \textit{p}-val~$= 9.21 \times 10^{-7}$) hotspots are drawn as dotted lines.
    }
\label{fig:asymmetry-and-PRDM9}
\end{figure}









\section{Apparent recombination parameters}
\subsection{Terminology used to characterise recombination}

\begin{figure}[b!]
	\centering
	\includegraphics[width = 1\textwidth, trim = 0cm 0.7cm 0cm 0cm, clip]{figures/chap6/Terminology_plot_red_yellow_small.eps}
	\caption[Terminology used to characterise recombination events]
	{\textbf{Terminology used to characterise recombination events.}
		\par The read coverage of the PRDM9 ChIP-seq peak (data from \citealp{baker2015prdm9}) is drawn in the top panel (PRDM9 ChIP-seq).
		\textit{B6} (red) and \textit{CAST} (yellow) reference alleles are reported in the middle panel (Markers).
		Examples of sequenced fragments that are or are not recombination events are drawn in the bottom panel (Fragments).
		See main text for the description of each annotation.
	}
\label{fig:terminology}
\end{figure}



The terms used to describe recombination events are illustrated in Figure~\ref{fig:terminology}.

An ‘interval’ (i.e.\ a sequence delineated by two consecutive markers) is either delimited by two \textit{B6} variants, two \textit{CAST} variants or one \textit{B6} and one \textit{CAST} variant.
On the one hand, intervals positioned in between two \textit{B6} (resp. \textit{CAST}) markers are considered to originate from the B6 (resp. CAST) haplotype.
On the other hand, intervals positioned in between one \textit{B6} variant and one \textit{CAST} variant are not assignable to either haplotype but necessarily comprise one end of the conversion tract (CT).
The latter types of intervals are called ‘switch intervals’ (grey segments in Figure~\ref{fig:terminology}).

We use a proxy for the genuine position of the CT edge: the ‘switch point’, defined as the midpoint of the switch interval.
Recombination events including strictly one switch point are called ‘single-switch recombination events’ (‘Rec-1S’); those including strictly two switch points are called ‘double-switch recombination events’ (‘Rec-2S’) and those including more than two switch points are called ‘multiple-switch recombination events’ (‘Rec-MS’).

For Rec-1S events, one edge of the CT is known: the position of the only switch point.
We approximate that its second edge corresponds to the inferred position of the DSB site, i.e.\ the summit of the PRDM9 ChIP-seq peak (data from \citealp{baker2015prdm9}).
Thus, the inferred (or observable) CT (noted CT\textsuperscript{$\star$}) for Rec-1S events corresponds to the region between the switch point and the inferred DSB position.
As for Rec-2S events, the CT\textsuperscript{$\star$} corresponds to the region between the two switch points and, for Rec-MS events, the position of the CT\textsuperscript{$\star$} cannot be inferred.



% Last, we infer that the haplotype from which the variants included in the CT\textsuperscript{$\star$} originate correspond to the donor in the gene conversion event. When the CT\textsuperscript{$\star$} includes no variant (which is the case for a portion of the Rec-1S) and when the CT\textsuperscript{$\star$} is not defined (case for Rec-MS), we cannot infer the donor.



\subsection{Identification of the gene-conversion donor}

After a gene conversion event, the receiver (cold) haplotype is converted into the donor (hot) haplotype all along the CT\@.
Thus, when all markers located within the CT\textsuperscript{$\star$} were typed \textit{B6} (resp. \textit{CAST}), we infered that the donor was the B6 (resp. CAST) haplotype.
If, however, the CT\textsuperscript{$\star$} included no variant (which was the case for a portion of Rec-1S events) or if it was not defined (which was the case for Rec-MS events), we could not infer which haplotype was the donor.\\

For a given hotspot, the fraction of CAST-donor (resp. B6-donor) recombination events should be proportional to the probability that the DSB initiates on the B6 (resp. CAST) haplotype.
Thus, to assess the accuracy of our inference, we used DMC1 ssDNA-sequencing (SSDS) data from a B6\textsubscript{f}xCAST\textsubscript{m} hybrid \citep{smagulova2016evolutionary} to estimate the relative proportion of DSB initiation on each haplotype: 
we remapped the raw data (SRR2961459, SRR2961460) on both the B6 and the CAST reference genomes and we genotyped all the variants of each fragment. 
When the totality of its variants were typed \textit{B6}, we assigned the B6 haplotype to the fragment and, respectively, we assigned the CAST haplotype to it when the totality of its variants were typed \textit{CAST}.
We then compared this probability for each hotspot to initiate a DSB on the B6 haplotype (i.e.\ the expected proportion of CAST-donor events) to the actual fraction of CAST-donor events inferred with the approach described above (Figure~\ref{fig:correl-donor-DMC1}).


\begin{figure}[t]
    \centering
    \includegraphics[width = 1\textwidth]{figures/chap6/CorrelationDMC1_FINAL_BIS_on_lab_computer.eps}
	\caption[Correlation between the expected and the observed proportions of CAST-donor fragments across hotspots]
	{\textbf{Correlation between the expected and the observed proportions of CAST-donor fragments across hotspots.}
		\par 
    % \par The expected proportion of CAST-donor fragments (x-axis) is computed based on the probability of the DSB to initiate on the B6 haplotype (see \ref{par:MM-DMC1-analysis} for details on the analysis of DMC1 ssDNA-sequencing (SSDS) data by \citep{smagulova2016evolutionary}). The observed proportion of CAST-donor fragments (y-axis) is calculated as described in the main text (see above). Solely the 582 hotspots displaying a minimum of 5 recombination events are reported in this graph. The coefficient of correlation between the expected and observed proportions of CAST-donor fragments is $r = 0.81$ ($p-value < 2.2\e{-16}$).}
}
\label{fig:correl-donor-DMC1}
\end{figure}




Overall, we found a strong positive correlation between the expected and the observed proportions of CAST-donor fragments (Pearson correlation: $R^2 = 0.66$; {\textit{p}-val $< 2.2 \times 10^{-16}$}).
Therefore, our donor-inference method is sufficient to explain, at the hotspot scale, most of the variance due to DSB initiation bias. This is a strong proof that our very simple and intuitive way of assigning its donor to each fragment is trustworthy.



\subsection{Description of the recombination events} 

Among the 18,821 recombination events detected across 898 hotspots (median~$= 10$; max $= 327$ events per hotspot), 11,665 corresponded to Rec-1S events, 5,932 to Rec-2S events and 1,224 to Rec-MS events.
The CTs\textsuperscript{$\star$} of Rec-1S events (median~$= 97$~bp; mean~$= 142$~bp) were longer — and, consequently, somewhat more spread (Figure~\ref{fig:example-recombinants} and Appendix~\ref{app:list-hostpots}) — than the CTs\textsuperscript{$\star$} of Rec-2S events (median~$= 78$ bp; mean~$= 90$ bp).
% awk '$26=="NCO" {print}' $DATA/2_dBGC/6_ThirdSequencing/05_Analyses_of_Recombinants/02_Recombinants_and_False_Positives_Dataset/Recombinants_dataset.txt|cut -f24|Rscript -e 'summary(as.numeric(readLines("stdin")))'
%    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.
%    9.00   54.00   78.00   90.36  113.00  475.00
% awk '$26=="CO" {print}' $DATA/2_dBGC/6_ThirdSequencing/05_Analyses_of_Recombinants/02_Recombinants_and_False_Positives_Dataset/Recombinants_dataset.txt|cut -f24|Rscript -e 'summary(as.numeric(readLines("stdin")))'
%    Min. 1st Qu.  Median    Mean 3rd Qu.    Max.
%     1.0    36.0    97.0   142.7   216.0  1142.0
These features of Rec-1S and Rec-2S are respectively reminiscent of those of COs and NCOs.

% However, the ratio of Rec-1S:Rec-2S (1.97:1) was much different from that expected for CO:NCO (1:9).
% This arises from two limitations of the direct observations.
% First, Rec-1S events may not comprise exclusively COs, but also NCOs.
% Indeed, everytime one edge of a sequenced fragment falls into the middle of a NCO CT, this event will necessarily be detected as a Rec-1S.
% Second, the detection of events is limited by the density in polymorphic sites and this effect is much greater for NCOs since their CTs are generally short.
% some NCOs are intrinsically undetectable due to
%

However, our data revealed about twice as many Rec-1S as Rec-2S — an observation much different from the expected CO:NCO ratio.
Indeed, in mice, among the 200--300 DSBs formed per meiosis, 20 are expected to be repaired as COs and the remaining 180--280 as NCOs \citep{baudat2007regulating, martinez-perez2009distribution}. 
Since NCOs affect only one of four chromatids (while COs affect two), one would \textit{a priori} expect to identify only one quarter of NCOs (i.e.\ 45--70) and half of COs ($\approx$10), hence a CO:NCO ratio ranging between 1:4.5 and 1:7. 

Two non-mutually exclusive reasons justify the gap between the observed and expected ratios.
%Rec-1S:Rec-2S and the expected CO:NCO ratio. 
On the one hand, the Rec-1S:Rec-2S ratio does not directly reflect the CO:NCO ratio.
Indeed, everytime one edge of a sequenced fragment falls into the middle of a NCO CT, this event is necessarily detected as a Rec-1S. 
Thus, Rec-1S events do not exclusively comprise COs: a portion of them correspond to NCOs.
On the other hand, for NCOs to be detected with our approach, their CTs must be long enough to overlap at least two markers (see Chapter~\ref{ch:5-methodology}).
Though, since NCO CTs are only a few base pairs to a few tens of base pairs long \citep{cole2014mouse}, one would \textit{a priori} expect a non-negligible (but unknown at this stage) proportion of them to be intrinsically undetectable, especially in regions with low SNP density.\\

Therefore, to characterise recombination regardless of these two limitations of direct observations, it appears necessary to use inferential approaches to uncover the true recombination parameters, as developed in the following section.



% RIGHT PAGE
\begin{sidewaysfigure}[p]
    \centering
    \leftskip-2.4cm
    \rightskip-2.4cm
    \rotfloatpagestyle{empty}
    \includegraphics[width = 1.25\textwidth, trim = 0cm 0cm 0cm 3.94cm, clip]{figures/chap6/P9peak_chr11_10175985_10177504_10177504.eps}
    \captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\abovecaptionskip-0.1cm
    \caption[Recombination events in a PRDM9\textsuperscript{CAST}-targeted hotspot located on chromosome 11 (chr11:10175985--10177504)]
	{\textbf{Recombination events in a PRDM9\textsuperscript{CAST}-targeted hotspot located on chromosome 11 (chr11:10175985--10177504).}
		\par The figure is centred on the PRDM9 ChIP-seq peak summit.
		The top panel depicts the maximum number of detectable Rec-1S (light grey) and Rec-2S (dark grey) switch intervals.
		%, which partially accounts for variation in recombination rate along and across hotspots.
		The two middle panel indicate variations in read coverage and the positions of markers (filled circles).
		The bottom panel pictures the detected Rec-1S (upper board) and Rec-2S (bottom board) events.
		\textit{B6}-typed markers and intervals are coloured in red, \textit{CAST}-typed markers and intervals in yellow, and switch intervals in grey.
	}
\label{fig:example-recombinants}
\end{sidewaysfigure}




% \subsection{CO:NCO ratios and polymorphism}
% Mettre une figure en lien, facon de montrer que juste lie a la detectabilite (ou si lien existe, pas detectable).
% Et donc la detectabilite a un role majeur.





\section{Inferred recombination parameters}
\subsection{Approximate bayesian computation (ABC)}

In order to discover which range of values of the biological parameters were compatible with our observations, we implemented an approximate bayesian computation (ABC) approach \citep{csillery2010approximate,sunnaker2013approximate}.
In short, an ABC consists in creating a simulator that reproduces at best the biological experiment, to perform a large number of simulations with variable input parameters and to assess which range of values are biologically relevant, by confronting the summary statistics representative of the output of the simulations to the biological observations.


% (here: the recombination parameters given in Table~\ref{tab:ABC-results}).
% Summary statistics, representative of the output, are calculated to be confronted with the observations (here: the distributions of Rec-1S and Rec-2S observed CT (CT\textsuperscript{$\star$}) lengths and the observed Rec-1S:Rec-2S ratio).
% In the end, the recombination parameters are estimated by selecting the simulated sets of parameters that end in the identification of recombination events displaying characteristics close to the ones obtained with the experimental dataset of recombination events.



\subsubsection{Implementation of the simulator}
We built a simulator that mimicked the formation of recombination events, their sequencing and their genotyping.
Briefly, all simulated recombination events were distributed across the 1,018 hotspots proportionately to their predicted propensity to form DSBs, which we approximated by their PRDM9 signal (i.e.\ the number of tags on each PRDM9 ChIP-seq peak from \citet{baker2015prdm9}, brought back to the width of the peak).
For each simulated hotspot, the ratio of CO over NCO recombination events was $r_{CO:NCO}$.

SNPs, insertions and deletions were positioned along each hotspot at the exact locations were they were found by variant-calling on the real data (see Chapter~\ref{ch:5-methodology}). 
CO CT lengths were drawn from a normal distribution of mean $m_{CO}$ and standard deviation $sd_{CO}$, and NCO CT lengths were drawn from a gamma distribution of alpha $\frac{m_{NCO}^{2}}{sd_{NCO}^{2}}$ and beta $\frac{sd_{NCO}}{m_{NCO}}$ (i.e.\ a distribution with mean $m_{NCO}$ and standard deviation $sd_{NCO}$). 
The middle point of the CT for both COs and NCOs was positioned at the inferred DSB site (i.e.\ the summit of the PRDM9 ChIP-seq peak) and each recombination event was assigned a donor (either B6 or CAST) under a binomial distribution with probability 0.5.

For each simulated recombination event, we randomly selected one of the two gametes involved in the recombination event and simulated $n_{fragments}$ sequenced fragment, whose start and end positions were drawn from the real positions of the fragments in the experimental dataset. 
We ran our unique-molecule genotyping pipeline (see Chapter~\ref{ch:5-methodology}) on all the sequenced fragments to identify, among all the simulated ones, those that would be detected as recombination events.



\subsubsection{Selection of the simulations compatible with the experimental data}
In total, we simulated 100,000 datasets $\mathcal{D^{*}}$ by assigning to each of the input parameters a value taken from the following prior distributions: 
\begin{itemize}
    \item $m_{CO} \hookrightarrow \mathcal{U}([100, 1000])$, 
    \item $sd_{CO} \hookrightarrow \mathcal{U}([50, 300])$,
    \item $m_{NCO} \hookrightarrow \mathcal{U}([1, 300])$, 
    \item $sd_{NCO} \hookrightarrow \mathcal{U}([1, 100])$,
    \item and $r_{CO:NCO} = 10^{r}$ with $r \hookrightarrow \mathcal{U}([-2, 1])$,
\end{itemize}
where $\mathcal{U}$ represents the uniform distribution.\\

For each simulated dataset as well as for the experimental dataset, we summarised the results of the recombination events found with the following summary statistics: the observed Rec-1S:Rec-2S ratio $r_{Rec-1S:Rec-2S}^{obs}$, the observed mean and quartiles of Rec-1S CT\textsuperscript{$\star$} lengths ($l_{Rec-1S}^{mean}$, $l_{Rec-1S}^{0.25}$, $l_{Rec-1S}^{0.5}$, $l_{Rec-1S}^{0.75}$) and the observed mean and quartiles of Rec-2S CT\textsuperscript{$\star$} lengths ($l_{Rec-2S}^{mean}$, $l_{Rec-2S}^{0.25}$, $l_{Rec-2S}^{0.5}$, $l_{Rec-2S}^{0.75}$). 

We then used the R package ‘abc’ \citep{csillery2012abc} to select the simulated datasets $\mathcal{D^{*}}$ that ended in summary statistics $\mathcal{S^{*}}$ close to the summary statistics $\mathcal{S}$ of the experimental dataset $\mathcal{D}$, with a tolerance threshold ($\epsilon$) of 5\% (i.e.\ $\mathcal{D^{*}}$ was retained if $d(\mathcal{S^{*}}, \mathcal{S}) \leq \epsilon$). 





\subsection{Comparison with direct observations}


\begin{table}[b!]
	\centering
	\begin{adjustbox}{width = 1\textwidth}
		\begin{tabular}{rrr}

			\toprule
			\textbf{Parameter} & \textbf{Literature} & \textbf{ABC approach} \\

			\midrule
			\textbf{\textit{CO:NCO ratio}}      & 0.1\textsuperscript{[1]}  & 0.119 [0.014--0.20] \\
			\textbf{\textit{CO CT length (bp)}}\\
			$Mean$                              & 566\textsuperscript{[2]}  & 447 [245--874] \\
			$Sd$                                & 277\textsuperscript{[2]}  & 363 [92--471] \\
			\textbf{\textit{Detectable NCO markers CT\textsuperscript{$\star$} length (bp)}}\\
			$Mean$                              &  94\textsuperscript{[2]}  & 95 [74--110] \\
			$Sd$                                &  62\textsuperscript{[2]}  & 49 [30--60] \\

			\textbf{\textit{Real NCO CT length (bp)}}\\
			$Mean$                  &  \textit{-}  & 36 [4--54] \\
			$Sd$                    &  \textit{-}  & 45 [3--86] \\

			\bottomrule

		\end{tabular}
	\end{adjustbox}
	\caption[Consistency between the recombination parameters inferred \textit{via} our ABC approach and those directly measured by independent studies]
	{\textbf{Consistency between the recombination parameters inferred \textit{via} our ABC approach and those directly measured by independent studies.}
		\par References from which the values were extracted are given inside superscript brackets: [1] corresponds to \citet{cole2010comprehensive} and [2] corresponds to \citet{cole2014mouse}. 
		CT stands for ‘conversion tract’ and CT\textsuperscript{$\star$} for ‘inferred (or observable) conversion tract’. 
		Only a portion of NCOs are detectable by tetrad analyses (those whose CT\textsuperscript{$\star$} overlaps at least 1 marker). 
Thus, we report the CT\textsuperscript{$\star$} length of both this subset of detectable NCOs that have been analysed by tetrad analyses, but also report the mean CT length of all NCOs (both detectable and undetectable). 
		For the ABC, the 95\% confidence intervals are reported between brackets.
	}
\label{tab:ABC-results}
\end{table}




CO and NCO CT lengths had previously been measured in two hotspots \textit{via} the analysis of mouse tetrads \citep{cole2014mouse} and the mouse CO:NCO ratio had been determined \textit{via} cytological estimates of DSB numbers: among about 250 DSBs arising in each meiosis, around 23 are repaired as COs \citep{baudat2007regulating, martinez-perez2009distribution}, which, with the assumption that the remaining ones are repaired as NCOs, leads to a rough estimate for the CO:NCO ratio of 0.1.\\

To assess the correctness of the parameter ranges identified by the ABC, we compared them to the aforementioned estimates (Table~\ref{tab:ABC-results}).
Altogether, we found that these two sets of parameter ranges were strikingly close.
This adequacy was particularly impressive regarding the CO:NCO ratio, considering the fact that we did not set any prior constraint on any of the simulated parameters.

Similarly, the length of the CTs\textsuperscript{$\star$} of detectable NCOs (i.e.\ those with a CT overlapping at least one marker) estimated by the ABC was almost identical to that directly observed by tetrad analyses.
However, this reported CT\textsuperscript{$\star$} length did not take into account that of undetectable NCO CTs (i.e.\ those too short to overlap any marker). As such, the actual mean CT length for all NCO events is necessarily shorter than that reported by direct observations and can only be provided by the ABC\@: we estimated it to be around 36 bp (Table~\ref{tab:ABC-results}).

As for COs, even if the 95\% confidence interval from the ABC included it, the value reported in the literature was slightly higher than the punctual estimate from the ABC\@.
This was likely due to the fact that the summary statistics were compared to the observations of CTs spreading onto a maximum of 500 bp (as the 1-kb hotspots were centred on the DSB site). 
If, instead, observations had been extended to larger regions, the estimated CTs would surely have been longer (as in Chapter~\ref{ch:8-HFM1}).\\

All in all thus, the ABC allowed to estimate the mean recombination parameters for the 1,018 hotspots we had selected and thus provided a broad insight of recombination patterns in mice.





\subsection{Extrapolation of recombination parameters}
\label{chap6:extrapolation-CO-rate}

Next, we used the results of the ABC to extrapolate other pieces of information on recombination: the CO rate and the composition in COs and NCOs of the observed Rec-1S events. 

\subsubsection{Estimation of the average CO rate}
Applying our unique-molecule genotyping pipeline on simulated recombination events (as was done with the ABC) allowed us to estimate the proportion of events that are detectable.

We defined the detectability ($d$) as the ratio of detected recombination events ($n$) over the total number of recombining gametes that were simulated ($N_r$):

\begin{equation} \label{eq:detectability}
    d = \frac{n}{N_r}
\end{equation}

As for the recombination rate ($R$), it corresponds to the proportion of recombining gametes ($N_r$) among all the gametes analysed ($N_g$):

\begin{equation} \label{eq:recombination-rate}
    R = \frac{N_r}{N_g}
\end{equation}

% Only a part ($n$) of these recombining gametes are detected as recombination events. We will refer to the proportion that we discovered as the detectability ($d$):
%
% \begin{equation} \label{eq:detectability}
%     d = \frac{n}{N_r}
% \end{equation}

Combining equations~\ref{eq:recombination-rate} and~\ref{eq:detectability}, we get:

\begin{equation*}
    R = \frac{n}{d \times N_g}
\end{equation*}

In the 4,997 simulations selected by the ABC, 6.688\% of simulated recombination events were discovered, which gave us a direct estimate for $d$. 
As for $n$ and $N_g$, we observed 18,821 recombination events out of 228,984,512 fragments analysed. 
Using these values, we found that the recombination rate in 1-kb long hotspots was $1.23 \times 10^{-3}$.
Since 0.119 of all recombination events corresponded to COs (see Table~\ref{tab:ABC-results}), the CO rate in 1-kb long hotspots was $1.46 \times 10^{-4}$, i.e.\ an average recombination rate of 14.6 cM/Mb across all analysed hotspots.

\begin{figure}[p]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap6/Correl_CO_rate_extrapolation_small.eps}
	\caption[Adequation between two independent manners of extrapolating the CO rate]
	{\textbf{Adequation between two independent manners of extrapolating the CO rate.}
		\par For each of the 1,018 hotspots, the CO rate was extrapolated based on the detectability inferred by the ABC (x-axis) and compared to the CO rate extrapolated from the number of informative fragments (y-axis).
		The calculus for the extrapolation of the CO rate \textit{via} the ABC was the following: $R = \frac{n}{d \times N_g} \times f_{CO}$, where $n$ is the number of recombination events detected, $d$ the detectability (inferred by the ABC), $N_g$ the total number of fragments analysed and $f_{CO}$ the proportion of COs in all the recombination events.
		This extrapolated CO rate was converted into cM/Mb by multiplying it by $10^2 \times \frac{10^{-3}}{10^{-6}}$ (the $10^{-3}$ multiplying factor comes from the fact that the CO rate was measured on 1-kb long hotspots).
		As for the extrapolation from the number of informative fragments, our calculus was the following: $R = \frac{n_{Rec-1S}}{L^{i}_{seq} \times N^{i}_{f}}$, where $n_{Rec-1S}$ represents the number of Rec-1S events, $L^{i}_{seq}$ the length sequenced on each fragment and $N^{i}_{f}$ the number of informative fragments (i.e.\ those overlapping a minimum of 4 markers).
		This rate was then converted into cM/Mb by multiplying it by $10^2 \times 10^6$.
		The two measures correlated extremely well (Pearson correlation: $R^2 = 0.84$; \textit{p}-val $< 2.2 \times 10^{-16}$) and the slope of the linear regression equalled 0.9342 (\textit{p}-val $< 2 \times 10^{-16}$).
	}
\label{fig:extrapolation-CO-rate}
\end{figure}



Using the simplifying assumption that the CO:NCO ratio is similar in all hotspots, we used the process just described to transform the number of events in a given hotspot to its recombination rate: this is how the right y-axis (CO rate in cM/Mb) was calculated for Figures~\ref{fig:asymmetry-and-PRDM9} and~\ref{fig:correlation-PRDM9-standard-error}.\\
 % extrapolation of CO rates based on the total number of events directly detected and the detectability inferred \textit{via} the ABC

Alternatively, the CO rate could be extrapolated independently of the results of the ABC\@: assuming that Rec-1S events mainly correspond to COs, the CO rate would equal the fraction of Rec-1S events per sequenced base pair where events are detectable.
Indeed, recombination events can only be detected in ‘informative’ fragments, i.e.\ fragments overlapping at least 4 markers, and this irregularity should not be counted into the CO rate.
Thus, when based on the number of informative fragments ($N^{i}_{f}$), their sequenced length ($L^{i}_{seq}$) and the number of Rec-1S detected ($n_{Rec-1S}$), the recombination rate $R$ equals:

\begin{equation*}
    R = \frac{n_{Rec-1S}}{L^{i}_{seq} \times N^{i}_{f}}
\end{equation*}\\



We found a remarkable adequation between those two independent ways of extrapolating the CO rate (Figure~\ref{fig:extrapolation-CO-rate}): the slope of the linear regression between the two was extremely close to 1 (slope $ =0.9342$; \textit{p}-val $< 2 \times 10^{-16}$).


% However, when comparing either of these extrapolated CO rates to the CO rate measured by \citet{paigen2008recombinational} on the 33 intervals of chromosome 1,
However, even if these two independent extrapolated estimates of CO rates concorded well, they were 20 times lower than those measured by \citet{paigen2008recombinational} on chromosome 1 (data not shown). 
As such, our extrapolation may underestimate the actual CO rate by a factor 20. Though, if it is indeed the case, we do not know where the gap comes from.






\subsubsection{CO:NCO composition of Rec-1S events}

As estimated by the ABC (Table~\ref{tab:ABC-results}), for every 1,000 recombination events repaired as NCOs, 119 are repaired as COs and, since NCOs affect only one chromatid when COs affect two, only 500 NCOs are expected to be seen. 
In the simulations selected by the ABC, the detectability for COs equalled 0.105 while that for NCOs equalled 0.0548. 
Based on these estimates, one would expect to detect 27.4 NCOs (out of the 500 chromatids affected) and 12.5 COs (out of the 119 chromatids affected).

Because they encompass only one switch point, all COs should be detected as Rec-1S events.
NCOs, however, could be detected as either Rec-1S or Rec-2S events. 
In the simulations selected by the ABC, 49.9\% of all the NCOs detected were detected as Rec-1S. 
Thus, among the 27.4 NCOs expected, 13.7 should be detected as Rec-1S events and 13.7 as Rec-2S events. 

All in all thus, we would expect to detect 26.2 Rec-1S events (13.7 (52.3\%) NCOs + 12.5 (47.7\%) COs) and 13.7 Rec-2S (all NCOs), i.e.\ a Rec-1S:Rec-2S ratio of 1.91, thus very close the ratio found experimentally (1.96).




% As a matter of fact, the results of the ABC evidence that Rec-2S all correspond to NCOs while Rec-1S represent a mixture of 52.3\% NCOs and 47.7\% COs (see \ref{par:MM-CO-NCO-estimates-in_Recs}).
% NCOs are less detectable than COs (3 times less according to the results of the ABC), since, to be detected, NCO CTs (which are only a few tens of base pairs long) must overlap at least two markers.
% Donc si on fait le total des deux, on revient bien dans un ratio de 1:6 comme attendu (1:4.5 a 1:7).










%OK % FIN CHAP 6
% CHAP 7: au moins motifs+ hitchhinking
% ce soir: abstract en francais.
% demain: fin chap 7 + chap8 (au moins design + adaptation methode)
% Samedi: fin chap 8 + chap9 en entier
% Dimanche: chap10 (au moins 1 section) + conclusion + preambule
% Lundi: fin chapitre 10
% Mardi + mercredi + jeudi: chap1 section 3
% Vendredi + dimanche : figures sur Inkscape
% Lundi + mardi: resume etendu + abbreviations + definitions + verif les references
% Mercredi + Jeudi: relecture totale.
% Vendredi: remerciements.
% + Preparation de SMBE

% \textbf{NOTE a Laurent: Je me demande s'il est pertinent de comparer ces deux mesures car, dans le cas de l'ABC, on extrapole le taux de recombinaison reel (i.e.\ nb de COs en cM/Mb génomique) alors que dans le cas des fragments informatifs, on obtient un taux de COs en cM/Mb séquencée.}







% chap 6: (plan a revoir)
% High-confidence and determinants (correl DMC1 + correl Paigen) + avant parler des cartes DMC1 Spo11 et correlation PRDM9 donc expliquer le meilleur descripteur (sachant que Prdm9 moins precis)
% Inferrence des CT et donneur (avec analyse de ssDNA DMC1) + description du jeu de donnees (Rec-1)
% Inference des parametres reels de recombinaison par ABC (simul + selection + comparaison litterature et extrapolation du taux de CO et de la proportion de CO/NCO dans Rec-1S).
% + qq part la relation entre CO/NCO et le nbre de SNPs

% intro: souris avec les infos connues dedans sur la recombi (DCM1, les cartes PRDM9 ChIP-seq Baker, Spo11…) — differences entre tous ces descripteurs de la recombinaison et ce qui est su chez la souris
% high confidence + determinants (trouves dans les hotspots intenses en relation avec DMC1) et inferrence des CT + donneur et description du jeu de donnees
% ABC et inference des parametres reels de recombinaison chez la souris (mise au point des simulations + selection des simul + resultats sur les param de recombi)

% chap7:
% clasification des hotspots cibles par B6 ou CAST — motifs + validation
% population structuree hitchhinking (et quantification du dBGC)
% quantification du BGC (g et d)
% + qqpart le fait que BGC regarde sur les bouts de tracts et rien de vu.
% + extraction des NCO1 pour mesurer gBGC
% Furthermore, the dBGC coefficent ($d$) for each hotspot can directly be extrapolated from Figure \ref{fig:correl-donor-DMC1}. Indeed, $d$ can be estimated based on the mean frequency of the \textit{CAST} allele in the pool of gametes ($x$) (\textit{i.e.} the observed proportion of CAST-donor fragments) through the following relationship: $x = \frac{1}{2} \times (1 + d)$ \citep{nagylaki1983evolutionA}.
% As the observed per-hotspot proportions of CAST-donor fragments span the whole spectrum of values (from 0\% to 100\%), the dBGC coefficient also spans its whole spectrum across hotspots (from -1 to 1). In particular, hotspots that were most eroded in one parental lineage are those for which the absolute dBGC coefficient is the greatest, while hotspots displaying a quasi-null dBGC coefficient correspond to symmetric hotspots (\textit{i.e.} hotspots where both homologues are bound by PRDM9 with equal affinity).
%
% Hotspot centres, defined as the summits of PRDM9 ChIP-seq peaks, coincided with the positions of PRDM9 binding motifs (see \ref{par:MM-motifs-close-to-hotspot-centres}) and most (77\%) Spo-11 ChIP-seq peak centres previously detected in B6 \citep{lange2016landscape} were located closer than 50-bp away from our PRDM9 binding motifs (see \ref{par:MM-motifs-close-to-DSBs}). Altogether, this suggests that hotspot centres approximate accurately the genuine locations of DSB sites.

%
% chap8:
% design experimental avec l'introgression (schema croisement, selection des hotspots, identification du background genetique des F2)
% adaptation de la methode + ABC ()
% premiers resultats sur la recombinaison (taux de recombinaison variables dus a reelle difference (inexpliquee), plus longs COs, 

% Design experimental (schema croisement avec introgression, selection des hotspots + design baits et sequencing + infos sur mapping etc, expected background)
% Detection d'evenements quand F2 (identification du background, validation via simulations de Laurent, adaptation du pipeline de genotypage de chapitre 5 i.e.\ juste faire le truc sur les hotspots heterozygotes)
% Resultats (taux de recombinaison variables dus a reelle difference (inexpliquee donc besoin de sequencer plus), ABC qui donne des resultats stables entre samples malgre les differences de recombination rates, ABC donne COs plus longs — possible plus longs que precedent projet car on analyse sur 5kb au lieu de 3)




% chap9:
% methode (powerful et adaptable a d'autres etudes de la recombinaison + mais limite majeure de detectabilite, en particulier, on rate les NCO1 qui semblent cruciaux pour l'etude du BGC (seulement mesure indirecte) + qqch en lien avec chap8? ou rare events a chercher change completement la donne avec l'utilisation des outils: tous les biais qui peuvent exister ou les imperfections deviennent critiques. e.g. misalignments par des INDELs, les erreurs de sequencage peu nombreuses mais trop elevees quand meme, error rate degenotypage…)
% recombinants et dBGC (hotspot asymmetry et favoured direction dBGC + dBGC hitchhiking chez populations structurees + parametres identifies proches de ce que vu chez la souris et si on compare avec l'humain on trouve que CO/NCO de 1/10) + etude recombinaison a partir des mutants?
% gBGC (confidence b + comparaison CO/NCO + comparaison humain/souris avec possibilite que evolution de la machinerie molec de formation BGC et transition epistemologie)
% Confidence b
% pour NCO
% $R^2 = 0.5630626$; \textit{p}-val $< 2.2 \times 10^{-16}$
% pour CO
% $R^2 = 0.4183843$; \textit{p}-val $< 2.2 \times 10^{-16}$



% chap10:
% epistemologie (proprietes emergentes ou pas + toute la fin de mes notes Notes_for_discussion_personnal.txt sur microevol/macroevol et fonctionnel vs mecanismes)
% comment la science avance (role des differents scientifiques qui ont apporte des nouvelles theories sont mieux connus que ceux qui font les decouvertes + analyse de l'evolution des recherches en evolution notamment avec apport de techniques comme genetique et ordis pour bioinfo + share knowledge gnomics.io)
% bioinformaticien (regarder les donnees (importance sur les emthodes, en aprticulier vu que realignement et teste des choses comme filtres, jusqu'a ce que validation par des simul) + seulement des tendances, jamais reels biologiques donc faut passer par inferences — en particulier, important dans le cas des evolutionnistes, car on interprete le passe sans pouvoir le demontrer (cf Jay Gould) et ses imperfections sont utiles poour comprendre de nouvelles choses).




% Annexes (erreur du jeune est d'en mettre trop)
%% pour chap6
% mettre les figures DMC1 (les deux — correlation par groupe de 10 plus relation asymetrie)
% + position des switch points
%% data availability: mettre lien vers le github pour reproduire les figure et avoir les tableaux d'entree + numero accession SRA
%% pour chap8: mettre les autres images de l'identification du background + image des correlations sur les memes hotspots
%% les listes de hotspots etudies

% Annexe des permissions
% La meilleure:
% [It] is not the nature of things for any one man to make a sudden, violent discovery; science goes step by step and every man depends on the work of his predecessors. When you hear of a sudden unexpected discovery—a bolt from the blue—you can always be sure that it has grown up by the influence of one man or another, and it is the mutual influence which makes the enormous possibility of scientific advance. Scientists are not dependent on the ideas of a single man, but on the combined wisdom of thousands of men, all thinking of the same problem and each doing his little bit to add to the great structure of knowledge which is gradually being erected. 
% — Sir Ernest Rutherford
% Concluding remark in Lecture ii (1936) on 'Forty Years of Physics', revised and prepared for publication by J.A. Ratcliffe, collected in Needham and Pagel (eds.), Background to Modern Science: Ten Lectures at Cambridge Arranged by the History of Science Committee, (1938), 73-74. Note that the words as prepared for publication may not be verbatim as spoken in the original lecture by the then late Lord Rutherford.




% [I]f texts are unified by a central logic of argument, then their pictorial illustrations are integral to the ensemble, not pretty little trifles included only for aesthetic or commercial value. Primates are visual animals, and (particularly in science) illustration has a language and set of conventions all its own.
% De Stephen Jay Gould
% de Jay Gould encore
% God bless all the precious little examples and all their cascading implications; without these gems, these tiny acorns bearing the blueprints of oak trees, essayists would be out of business.
% Questioning the Millennium (second edition, Harmony, 1999), p. 42
% de Einstein 
% When a man after long years of searching chances on a thought which discloses something of the beauty of this mysterious universe, he should not therefore be personally celebrated. He is already sufficiently paid by his experience of seeking and finding. In science, moreover, the work of the individual is so bound up with that of his scientific predecessors and contemporaries that it appears almost as an impersonal product of his generation.
% From the story "The Progress of Science" in The Scientific Monthly edited by J. McKeen Cattell (June 1921), Vol. XII, No. 6. The story says that the comments were made at the annual meeting of the National Academy of Sciences at the National Museum in Washington on April 25, 26, and 27. Einstein's comments appear on p. 579, though the story may be paraphrasing rather than directly quoting since it says "In reply Professor Einstein in substance said" the quote above.
% 
% In science men have discovered an activity of the very highest value in which they are no longer, as in art, dependent for progress upon the appearance of continually greater genius, for in science the successors stand upon the shoulders of their predecessors; where one man of supreme genius has invented a method, a thousand lesser men can apply it. … In art nothing worth doing can be done without genius; in science even a very moderate capacity can contribute to a supreme achievement. 
% — Bertrand Russell
% Essay, 'The Place Of Science In A Liberal Education.' In Mysticism and Logic: and Other Essays (1919), 41.

% It is a wrong business when the younger cultivators of science put out of sight and deprecate what their predecessors have done; but obviously that is the tendency of Huxley and his friends … It is very true that Huxley was bitter against the Bishop of Oxford, but I was not present at the debate. Perhaps the Bishop was not prudent to venture into a field where no eloquence can supersede the need for precise knowledge. The young naturalists declared themselves in favour of Darwin’s views which tendency I saw already at Leeds two years ago. I am sorry for it, for I reckon Darwin’s book to be an utterly unphilosophical one. 
% — William Whewell
% Letter to James D, Forbes (24 Jul 1860). Trinity College Cambridge, Whewell Manuscripts.

% Very few people, including authors willing to commit to paper, ever really read primary sources–certainly not in necessary depth and contemplation, and often not at all ... When writers close themselves off to the documents of scholarship, and then rely only on seeing or asking, they become conduits and sieves rather than thinkers. When, on the other hand, you study the great works of predecessors engaged in the same struggle, you enter a dialogue with human history and the rich variety of our own intellectual traditions. You insert yourself, and your own organizing powers, into this history–and you become an active agent, not merely a ‘reporter.’ 
% — Stephen Jay Gould

% [It] is not the nature of things for any one man to make a sudden, violent discovery; science goes step by step and every man depends on the work of his predecessors. When you hear of a sudden unexpected discovery—a bolt from the blue—you can always be sure that it has grown up by the influence of one man or another, and it is the mutual influence which makes the enormous possibility of scientific advance. Scientists are not dependent on the ideas of a single man, but on the combined wisdom of thousands of men, all thinking of the same problem and each doing his little bit to add to the great structure of knowledge which is gradually being erected. 
% — Sir Ernest Rutherford
% Concluding remark in Lecture ii (1936) on 'Forty Years of Physics', revised and prepared for publication by J.A. Ratcliffe, collected in Needham and Pagel (eds.), Background to Modern Science: Ten Lectures at Cambridge Arranged by the History of Science Committee, (1938), 73-74. Note that the words as prepared for publication may not be verbatim as spoken in the original lecture by the then late Lord Rutherford.

% It is strange that only extraordinary men make the discoveries, which later appear so easy and simple.
% GEORG C. LICHTENBERG, 1742 TO 1799




% REMERCIEMENTS
% You have … been told that science grows like an organism. You have been told that, if we today see further than our predecessors, it is only because we stand on their shoulders. But this [Nobel Prize Presentation] is an occasion on which I should prefer to remember, not the giants upon whose shoulders we stood, but the friends with whom we stood arm in arm … colleagues in so much of my work. 
% — Sir Peter B. Medawar
% From Nobel Banquet speech (10 Dec 1960).



