\begin{savequote}[8cm]
‘Intense selection favours a variable response to the environment… Were this not so, the world would be much duller than is actually the case.’

\qauthor{--- John B. S. Haldane, \textit{\usebibentry{haldane1932causes}{title}} \citeyearpar{haldane1932causes} }
\end{savequote}

\chapter{\label{ch:3-recombination-variation}Causes and consequences of recombination rate evolution} 
%\otherpagedecoration

\minitoc{}


The very mechanism of meiosis introduces genetic mixing in two separate ways.
On the one hand, the paternal and maternal chromosomes are independently re-assorted during the first meiotic division.
On the second hand, genetic content is exchanged during recombination at the points where homologues cross over (a.k.a.\ chiasmata).

Even if this phenomenon was not known in Charles Darwin's time, he had the intuition that genetic diversity — which meiosis participates in instilling — was essential to the formation of new species:

\begin{quote}
	\textit{‘The principle, which I have designated by this term [ed.\ divergence of character], is of high importance on my theory, and explains, as I believe, several important facts. […] according to my view, varieties are species in the process of formation, or are, as I have called them, incipient species. How, then, does the lesser difference between varieties become augmented into the greater difference between species?’}
	\qauthor{--- Charles Darwin, \textit{\usebibentry{darwin1859origin}{title}} \citeyearpar{darwin1859origin} }
\end{quote}

As enunciated by his theory, the transition from varieties to species requires ‘a severe struggle for life [which] certainly cannot be disputed’ (natural selection), the occurrence of ‘variations useful to any organic being’ (mutations) and ‘the strong principle of inheritance’ through which ‘they will tend to produce offspring similarly characterised’ (heredity).
As such, the emergence of new species is tightly linked to the process of meiotic recombination since it is a major vector of genetic variation at the heart of the process of heredity.


Furthermore, the notion of biological species itself, formally defined by Ernst Mayr (1904--2005) as ‘groups of interbreeding natural populations that are reproductively (genetically) isolated from other such groups’ \citep{mayr1999systematics}, rests on the ability to sexually reproduce and thus, to meiotically recombine.

The relationship between these two concepts (further developed in \citealp{felsenstein1981skepticism} and \citealp{butlin2005recombination}) is such that, in the mammalian clade, the only speciation gene discovered so far (PRDM9) is the one that controls the localisation of double-strand breaks (DSBs) on the genome \citep{baudat2010prdm9,myers2010drive,parvanov2010prdm9}.

I will come back to this essential gene and to its impact on the evolution of recombination rate in the third section of this chapter. 
But prior to that, I will review the existing methods to detect recombination genome-wide, and the multiple layers of recombination rate (RR) variation that have been observed along genomes and across species.


\section{Genome-wide detection of recombination}

\subsection{Linkage maps \textit{via} the analysis of crosses or pedigrees}
\label{chap3:linkage-maps}

The comprehension of genetic linkage by the group of Thomas Hunt Morgan (see Chapter~\ref{ch:1-history-genetics}) was the inaugural step towards the establishment of the first genetic map (a.k.a.\ linkage map) \citep{sturtevant1913linear}.
Basically, these maps abstractly represent the proportion of crossing-overs (COs) occurring between pairs of ‘genetic markers’, i.e.\ polymorphic\footnote{Which presents several forms. In other words: subject to inter-individual variability.} DNA sequences located at fixed genomic positions.

Initially, genetic markers exclusively comprised genes coding for visually discernable phenotypes.
Since their relatively wide genomic spacing granted a poor resolution to detect recombination, they were eventually supplanted by other types of markers: restriction fragment length polymorphisms (RFLPs) i.e.\ sequences enzymatically shortenable first used for linkage analysis by \citet{botstein1980construction}; minisatellites and microsatellites \citep{hamada1982potential} i.e.\ tandem repeats of short motifs highly variable in length \citep{ellegren2004microsatellites} and widely spread in eukaryotes \citep{hamada1982novel}; and single-nucleotide polymorphisms (SNPs) i.e.\ one-base sequence variations.\\

When the two parental chromosomes carry distinct alleles at these loci\footnote{Fixed position of a genetic marker on a chromosome (from the Latin word \textit{locus}: ‘place’)}, one can track their transmission by genotyping the markers in the descendants.
As such, the mosaic of paternal and maternal haplotypes — and thus, the positions of recombination exchange points — can be reconstituted using various statistical methods \citep[reviewed in \citealp{backstrom2009gene}]{haldane1919combination, kosambi1943estimation}.

These kindred individuals are generally obtained by crossing members of highly divergent inbred populations \citep[e.g.][]{rowe1994maps,dietrich1996comprehensive}, one of which being, if possible, homozygous for the recessive alleles (‘test cross’) so as to disentangle the genotypes of the descendants \citep[reviewed in][]{brown2002mapping}.
Alternatively, in species that have long generation time or that cannot be manipulated genetically for ethical considerations, successive generations of existing families (a.k.a.\ pedigrees) can be examined \citep[e.g.][]{kong2002highresolution,kong2010finescale,cox2009new}.\\

Examining large numbers of individuals allows to estimate the genetic distance (measured in ‘morgans’ (M) as a tribute to its designer) between pairs of markers: one centimorgan (cM) expresses a frequency of 1 CO every 100 meioses.
However, for high recombination frequencies (i.e.\ long distances), some experiments \citep[e.g.][]{morgan1911random,morgan1912data} showed exceptions to additivity: the genetic distance between two polymorphic sites could be smaller than the sum of their distances with an in-between marker.
Indeed, in cases of ‘double crossing-overs’ (i.e.\ two COs occurring within a given interval — which is more likely in wider stretches), the two loci are inherited together.
Thus, the CO event is not detectable and, in the end, the recombination frequency is underestimated.

In addition, genetic distances are not proportional to physical remoteness, as stated by Hermann Muller (1890--1967) \citep{muller1920are} in a response to William Castle (1867--1962) who disputed the graphical representation of these maps \citep[reviewed in \citealp{vorms2013models}]{castle1919are,castle1919arrangement}:

\begin{quote}
\textit{‘[I]t has never been claimed, in the theory of linear linkage, that the per cents of crossing over are actually proportional to the map distances [ed.\ physical distances]:  what has been stated is that the per cents of crossing overs are calculable from the map distances — or, to put the matter in more mathematical terms, that the per cents of crossing over are functions of the distances of points from each other along a straight line.’}
\end{quote}


Decades later, the complete sequencing of the \textit{Saccharomyces cerevisiae} chromosome III \citep{oliver1992complete} confirmed this statement by enabling the first direct comparison between linkage and physical maps. 
The discrepancies between the two distances legitimised the introduction of a new measurement: the estimation of recombination rates (RRs) per physical distance (expressed in cM/Mb), useful to compare RRs across genomic regions, individuals or species.\\

Altogether, linkage maps directly measure recombination occurring in the offspring and thus allow to observe differences between sexes \citep[e.g.][]{cheung2007polymorphic,coop2008highresolution} or among individuals \citep[e.g.][]{broman1998comprehensive}.
However, the resolution of these maps is restrained by the position of polymorphic sites and the number of meioses analysed. 
Consequently, in mammals, except for one very recent study \citep{halldorsson2019characterizing}, the resolution has remained capped at tens to hundreds of kilo base pairs (kb) \citep{shifman2006highresolution,billings2010patterns,kong2010finescale}.
This limitation motivated the development of a population-genetic method to learn about RRs at a finer-scale: the linkage disequilibrium (LD) analysis.


\subsection{Linkage disequilibrium (LD) analysis}%(et HapMap et biais)}
\label{chap3:LD}

Populations of unrelated beings can be analysed in a fashion similar to family members since kinship (or non-kinship) only conveys a \textit{relative} sense: unrelated individuals are merely more distantly akin than traditional pedigrees \citep{nordborg2002linkage}.

Therefore, the principle remains the same for populations of unrelated individuals as for families: recombination breaks down linkage disequilibrium (LD) \citep{lewontin1960evolutionary}, i.e.\ non-random associations between loci (materialised by non-random segregations of alleles), which results in the fragmentation of LD into blocks.
Reciprocally, analysing patterns of LD (i.e.\ the positions of LD blocks) will allow to trace back the underlying recombination process.\\

Concretely, LD can be quantified using statistics of association between allelic states at pairs of loci \citep{lewontin1964interaction,hill1968linkage} and the recombination rates (RRs) further estimated through a myriad of methods \citep[reviewed in][]{stumpf2003estimating} which basically consist in using the allelic diversity of each LD block to reconstruct the genealogy \citep[reviewed in][]{hinch2013landscape}.
Indeed, patterns of LD do not account for recombination only \citep[reviewed in][]{venn2013inferring}: they are also shaped by other forces such as population history \citep{golding1984sampling}, mutation \citep{calafell2001haplotype} (though easily distinguishable from recombination \citep{hudson1985statistical}), natural selection \citep{barton2000genetic} and drift \citep{charlesworth1997effects}. 
Modelling the underlying genealogical history of the population therefore allows to take the latter effects into account and thus, to estimate RR accurately from LD patterns \citep{stumpf2003estimating}.\\

Recombination events have been inferred by LD analysis in a plethora of mammalian orders including Artiodactyla \citep{farnir2000extensive,mcrae2002linkage,nsengimana2004linkage}, Carnivora \citep{menotti-raymond1999genetic,sutter2004extensive,verardi2006detecting}, Lagomorpha \citep{carneiro2011genetic}, Rodentia \citep{brunschwig2012finescale}, Perissodactyla \citep{corbin2010linkage, mccue2012high} and Primates \citep{auton2012finescale}.
Though, the resolution of recombination events is greatest in humans, where it has reached 1 to 2 kb \citep{theinternationalhapmapconsortium2007seconda, hinch2011landscape, the1000genomesprojectconsortium2015global}.
Such precision arises from the fact that there have had many oppportunities for recombination to take place between the last common ancestor (LCA) of a population of unrelated beings and its studied descendants.
Since recombination decreases LD at every generation \citep{slatkin2008linkage}, the more ancient the LCA, the shorter the LD blocks and thus, the higher the resolution.

However, the recombination events identified with LD analysis sum up the whole recombination process that has occurred since the LCA\@: historical recombination, rather than current recombination, is uncovered.
In addition, LD studies give a population average of recombination, with no possibility to extricate sex-specific nor individual recombination events.
Third, both LD studies and linkage maps allow the detection of COs, but not NCOs.

Another method, — sperm-typing, — solves the three aforementioned caveats: it provides fine-scale mapping of current CO and NCO recombination events in separate individuals.



\subsection{High-resolution sperm-typing studies}%genome-wide genotyping: high-resolution studies}

Sperm-typing consists in analysing the transmission of recombination events directly in the sperm of an individual.
This was made possible by the development of a polymerase chain reaction\footnote{Molecular biology method used to make copies of a specific DNA fragment.} (PCR) method allowing to genotype single diploid and haploid cells \citep{li1988amplification}.
Since PCR only allows the copy of size-limited DNA sequences and cannot be performed automatically, sperm-typing cannot be applied genome-wide \citep{coop2008highresolution}, unless a microfluidic device is used \citep{fan2011wholegenome,wang2012genomewide}.
Instead, sperm-typing is generally restricted to regions of high recombinational activity inferred from linkage or LD maps (see Subsections~\ref{chap3:linkage-maps}~and~\ref{chap3:LD}).\\

It can be applied either to single gametes or to total-sperm DNA \citep[reviewed in][]{arnheim2003hot}.
In single-sperm typing, the PCR is performed on the lysed sperm of an individual gamete with the use of pairs of primers\footnote{Short single-stranded nucleic acid used to initiate DNA synthesis.} flanking two polymorphic markers at the extremities of the locus of interest \citep{cui1989singlesperm,lien1993simple}.
This \textit{modus operandi} has soon been used to construct linkage maps on highly recombining regions \citep{schmitt1994multipoint,lien2000evidence,cullen2002highresolution} while others \citep{tusie-luna1995gene, jeffreys1998highresolution, jeffreys2001intensely, guillon2002initiation} have used the alternative approach with total-sperm DNA which requires allele-specific PCR to capture and amplify recombinant molecules \citep{wu1989allelespecific}.

In both cases, the precise CO exchange point can be mapped using the genetic markers internal to the selected locus.
Sperm-typing thus offers the best resolution for recombination exchange points since it is only limited by SNP density — a resolution even sufficient to detect the difficult-to-access NCOs that only affect a few markers \citep{hellenthal2006insights}, as in \citet{tusie-luna1995gene} and \citet{guillon2002initiation}.

However, even though some authors have managed allele-specific PCR in pooled ovaries \citep{guillon2005crossover, baudat2007cis} and single oocytes \citep{cole2014mouse}, it has almost exclusively been used for the study of male products of meiosis.\\

The three methods described so far allow to detect the outcome of the recombination process: COs (and NCOs in the case of sperm-typing).
To get insights into other stages of the recombination process, one can use chromatin-immunoprecipitation (ChIP) of proteins involved in a given recombination stage (see Chapter~\ref{ch:2-recombination-mechanistics}) to crosslink them on their DNA binding sites, followed by the identification of bound DNA sequences either with a microarray (ChIP-chip) or by direct sequencing of the fragments (ChIP-seq) \citep[reviewed in][]{park2009chipseq}. 
The sites of recombination initiation have been identified by using this technique with Spo11 proteins in yeasts \citep{gerton2000global,mieczkowski2007loss,pan2011hierarchical} and mice \citep{lange2016landscape} and the repair sites with RPA proteins in yeasts \citep{borde2009histone} and RAD51 and DMC1 proteins in mice \citep{smagulova2011genomewide,brick2012genetic}.
Alternatively, sites of recombination initiation have been mapped by analysing the enrichment of single-stranded DNA (ssDNA) in yeasts \citep{blitzblau2007mapping,buhler2007mapping} and mice \citep{khil2012sensitive}.

These methods do not rely on the existence of polymorphic markers and, therefore, only depend on the size of the region bound by the protein.
As such, the resolution reaches up to $\sim$500 bp for DMC1, $\sim$50 bp for PRDM9 and a few base pairs for SPO11.\\

All these approaches have contributed to a better understanding of recombination genome-wide.
In particular, it was soon understood that COs do not appear at random locations on the genome.
The reasons for this particular distribution became the object of many research works.




\section{The landscape of recombination}

\subsection{The non-random distribution of crossing-overs (COs)}

The number and distribution of crossing-overs (COs) along the genome are subject to a tight regulation \citep[reviewed in][]{jones1984control, jones2006meiotic}: a minimum number of COs (‘CO assurance’), evenly spaced (‘CO interference’) — including when few DSBs are generated (‘CO homeostasis’) — are formed preferentially with the homologous chromosome.

\subsubsection{Crossing-over assurance (COA), or the ‘obligatory crossing-over’}
Together with sister chromatid cohesion, COs hold the homologous chromosomes joint until anaphase I \citep[reviewed in][]{roeder1997meiotic} and are therefore essential to the proper disjunction of bivalents.
Accordingly, in most sexually-reproducing organisms, the total number of COs ranges between one per chromosome and one per chromosome arm\footnote{With the notable exceptions of honey bees \citep{beye2006exceptionally} and birds \citep{groenen2009highdensity} which display higher numbers of COs per chromosome, and of \textit{Drosophila melanogaster} males who do not display any CO throughout their genome \citep{mckee1998pairing}.}, irrespective of chromosome length \citep{dutrillaux1986role,pardo-manueldevillena2001recombination,dumas2002chromosomal,hillers2003chromosomewide,hassold2004cytological,dumont2017variation}.
As such, mammalian genetic map lengths (which are proportional to CO numbers) can be predicted with the haploid number of chromosome arms (Figure~\ref{fig:correlation-CO-nb-chromosomes}).

The sexual chromosomes also comply to this phenomenon: they systematically have one CO on their pseudoautosomal region (PAR), a feature likely facilitated by the much higher DSB rate on the PAR than on the autosomes \citep{kauppi2011distinct}.
However, this ‘obligatory CO’ rule suffers exceptions: \textit{Drosophila melanogaster} females do not display any CO on their tiny 4\textsuperscript{th} chromosome nor, in certain cases, on their X chromosome \citep{orr-weaver1995meiosis, koehler1998human} and neither do marsupial sex chromosomes \citep{sharp1982sex}.\\

\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap3/correlation-CO-nb-chromosomes.eps}
	\caption[Correlation between genetic map lengths and the number of chromosomal arms in mammals]
	{\textbf{Correlation between genetic map lengths and the number of chromosomal arms in mammals.}
		\par The y-axis represents genetic map lengths from which the number of crossing-overs (COs) can be extrapolated.
		The x-axis represents the total number of chromosomal arms per species, excluding the small arms of acrocentric chromosomes and the sex chromosomes of baboon and rhesus macaques.
		The black line corresponds to the best fit between these two measures.
		\par This figure was reproduced from \citet{coop2007evolutionary} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:correlation-CO-nb-chromosomes}
\end{figure}

In \textit{Caenorhabditis elegans}, crossing-over assurance (COA) is so strong that only one DSB per pair of chromosome suffices to guarantee a CO \citep{rosu2011robust}.
Nevertheless, chromosome pairs holding only one DSB may be uncommon since the number and position of DSBs is also under tight control, at least in yeasts \citep{wu1995factors,fan1997competition,robine2007genomewide,anderson2015reduced}: the formation of a DSB reduces the likelihood for another to form nearby \citep{garcia2015tel1}. 
This phenomenon, called ‘interference’, applies to DSBs and another one, also called interference but applying this time to COs \textit{via} a distinct mechanism, has also been reported, as reviewed in the upcoming paragraph.

\subsubsection{Crossing-over interference (COI)}
Early studies on recombination \citep{sturtevant1915behavior,muller1916mechanism} have shown that, when more than one CO appears on a given chromosome, the chiasmata they form tend to be evenly spaced \citep{jones1967control, jones1974correlated, jones1984control,jones2006meiotic}.
Indeed, the occurrence of a CO hampers the coincident formation of another one in the same pair of chromosomes \citep{vanveen2003meiosis,hillers2004crossover} — the physical length of prophase chromosomes, rather than the genomic (bp) or genetic (cM) distance, being the primary parameter \citep{zhang2014crossover,wang2015meiotic}.
So far, COI has been noted in several species including \textit{Arabidopsis thaliana} \citep{drouaud2007sexspecific}, \textit{Saccharomyces cerevisiae} \citep{shinohara2003crossover}, \textit{Homo sapiens} \citep{laurie1985further,broman2000characterization} and \textit{Mus musculus} \citep{lawrie1995chiasma,anderson1999distribution,broman2002crossover}.\\

The mechanism of COI remains unclear but several models have been proposed \citep[reviewed in][]{youds2011choice}.
One early hypothesis, — the polymerisation model, — posits that the completion of a CO triggers the polymerisation of an inhibitor of recombination, thus preventing the formation of adjacent COs \citep{maguire1988crossover,king1990polymerization}.
According to another one, — the stress model, — axis buckling converts the recombination intermediate into a CO, and this mechanical tension is released in the vicinity of established COs, thus making neighbouring DSBs repair into NCOs instead \citep{borner2004crossover,kleckner2004mechanical}.
The most recent pieces of evidence point that, in mice, COI may operate in two consecutive steps: at late zygotene and at pachytene \citep{boer2006two}.

Correlations between the length of the synaptonemal complex (SC) and interference have been reported \citep{sym1994crossover,lynn2002covariation,petkov2007crossover}, but others have found that COI does not depend on the SC \citep{deboer2007meiotic,shodhan2014msh4}, which suggests that COI operates before SC formation: either prior to single-end invasion (SEI) \citep{hunter2001singleend, bishop2004early} or during the stabilisation of the SEI \citep{shinohara2008crossover}.\\

Whatever the mechanism at play, it may have a role in controlling the outcome of the repair
(e.g.\ by preferentially recruiting the MUS81 repair machinery).
Indeed, the COs formed \textit{via} the DSBR pathway comply to COI whereas those repaired \textit{via} the MUS81 pathway do not \citep{santos2003mus81,kohl2013meiotic}.
In particular, neither \textit{Schizosaccharomyces pombe} for which all COs depend on the Mus81 pathway \citep{munz1994analysis,hollingsworth2004mus81,cromie2006single} nor \textit{Aspergillus nidulans} which lacks SC \citep[reviewed in \citealp{shaw1998meiosis} and \citealp{egel1995synaptonemal}]{strickland1958analysis} show CO interference.

As for NCOs, their formation is undoubtedly promoted by COI to downregulate the number of COs \citep{rockmill2003sgs1,youds2010rtel1,crismani2012fancm,seguela-arnaud2015multiple}.


\subsubsection{Crossing-over homeostasis (COH)}
Even though it has been disputed \citep{shinohara2008crossover}, the mechanism that ensures COI may be responsible for another level of regulation: crossing-over homeostasis (COH) \citep[reviewed in \citealp{youds2011choice}]{joshi2009pch2,zanders2009pch2delta}.
COH promotes the formation of COs at the expense of NCOs when fewer DSBs than the wild-type level are generated.
This phenomenon was initially observed in \textit{Saccharomyces cerevisiae} \citep{martini2006crossover,chen2008global}, but also exists in \textit{Caenorhabditis elegans} \citep{yokoo2012cosa1,globus2012joy}, \textit{Drosophila melanogaster} \citep{mehrotra2006temporal} and \textit{Mus musculus} \citep{cole2012homeostatic}.



\subsubsection{Preference for the homologue over the sister chromatid in DSB repair}
So that the homologous chromosomes disjoin properly, a fourth regulatory level applies to the repair of DSBs into COs: the promotion of interhomologue repair over intersister mending.
Template choice must be regulated differently in mitosis and meiosis \citep{andersen2010meiotic}.
Indeed, in mitosis, the sister chromatid is always favoured \citep{kadyk1992sister,bzymek2010double}, whereas evidence in \textit{Saccharomyces cerevisiae} suggests that, in meiosis, two thirds \citep{goldfarb2010frequent} to nearly all \citep{pan2011hierarchical} DSBs are repaired using the homologue.\\

Cohesins and components of the SC seem to be implicated in template choice \citep[reviewed in \citealp{pradillo2011template}]{couteau2004component,kim2010sister} but the proteins that play a role in homology search are also adequate candidates for this endeavour \citep[reviewed in][]{youds2011choice}.
Indeed, in \textit{Saccharomyces cerevisiae}, the phosphorylation of Hop1 (mouse homologue: HORMAD1) triggers a mechanism that prevents intersister repair of DSBs \citep{niu2005partner}: it inhibits Rad51 \citep{niu2009regulation}, thus leaving homology search to Dmc1 which promotes interhomologue recombination more efficiently than Rad51 \citep{schwacha1997interhomolog}.

Elucidating these four layers of control on the formation and genome-wide distribution of COs was largely fostered by the immunodetection of the MLH1 protein (which is a marker of CO events) on meiotic chromosome spreads. Such maps have been obtained in multiple clades including primates \citep[e.g.][]{sun2005variation,codina-pascual2006crossover,garcia-cruz2011comparative,gruhn2013cytological,munoz-fuentes2015strong}, rodents \citep[e.g.][]{froenicke2002male,dumont2011genetic}, ruminants \citep[e.g.][]{vozdova2013comparative,sebestova2016effect} and other eutherians \citep[e.g.][reviewed in \citealp{capilla2016mammalian}]{borodin2008recombination,segura2013evolution,mary2014meiotic}.

Further analysis of maps like those has allowed to uncover both the large-scale and fine-scale patterns of recombination rate (RR) variation along the genomes, which are reviewed in the forthcoming subsection.




\subsection{Intragenomic patterns of variation}
\subsubsection{Large-scale variations across genomic regions}

When compared over the scale of megabases (Mb), recombination rates (RRs) vary by an order of magnitude in both humans (Figure~\ref{fig:recombination-rate-variation}.a.) \citep{nachman2002variation,myers2005finescale} and mice \citep{billings2010patterns, morgan2017structural}.

\pagebreak
These large-scale variations associate with certain elements of the genome (reviewed in \citealp{demassy2013initiation} and \citealp{lam2015mechanism}).
Centromeric regions, for instance, are generally associated with little or no recombination, like in mammals \citep{qiao2012interplay} and yeasts: in \textit{Schizosaccharomyces pombe}, components of the RNA interference (RNAi) pathway repress DSB formation around centromeres \citep{ellermeier2010rnai} and in \textit{Saccharomyces cerevisiae}, Spo11 relocalises onto chromosome arms at prophase, thus preventing the formation of DSBs adjacent to centromeres \citep{kugou2009rec8}.
This feature likely aids in the proper disjunction of homologues, since centromere-proximal COs result in aneuploidy in yeasts \citep{rockmill2006centromereproximal}, humans \citep{hassold2001err} and flies \citep{koehler1996recombination}.

A similar suppression is also observed at telomeric regions in yeasts \citep{blitzblau2007mapping,buhler2007mapping}, possibly because DSBs in repetitive sequences are likely to be repaired through the non-allelic homologous recombination (NAHR) pathway which can alter genome architecture \textit{via} chromosomal rearrangements \citep{sasaki2010genome}.
However, recombination seems increased in the neighbouring (subtelomeric) regions of yeasts \citep{chen2008global,barton2008meiotic} albeit this was not observed in other genome-wide studies \citep{buhler2001dna,pan2011hierarchical}.
High RRs are also observed in the subtelomeric regions of mammals \citep{kong2002highresolution,jensen-seaman2004comparative,pratto2014recombination} and plants \citep{giraut2011genomewide}.\\



In lieu of occurring at centromeres and telomeres, recombination primarily localises within interstitial regions, themselves fragmented into DSB-rich and DSB-poor domains — of about 100 kb in \textit{Saccharomyces cerevisiae} \citep{baudat1997clustering,borde1999use}.
The DSB-rich domains are associated with higher GC-content in yeasts \citep{gerton2000global,petes2001meiotic,marsolier-kergoat2009gc}, rodents \citep{jensen-seaman2004comparative} and mammals \citep{eyre-walker1993recombination,fullerton2001local}.
In humans and chimpanzees, these domains are further enriched in 5’ and 3’ untranslated regions (UTRs) and CpG islands \citep{kong2002highresolution, auton2012finescale}.

It was suggested early that these highly-recombinant regions may correspond to structural genes \citep{thuriaux1977recombination}, which is indeed the case in maize \citep[reviewed in \citealp{okagaki2018critical}]{nelson1959intracistron, nelson1962waxy, nelson1975waxy, dooner1997recombination, dooner2008maize}.
Notwithstandingly, neither \textit{Arabidopsis thaliana} \citep{kim2007recombination,horton2012genomewide}, \textit{Schizosaccharomyces pombe} \citep{cromie2007discrete} nor mammals \citep{mcvean2004finescale,myers2005finescale,brick2012genetic} share this characteristic: in humans and mice, recombination correlates negatively with both gene content \citep{kong2002highresolution,jensen-seaman2004comparative} and gene transcription rate \citep{mcvicker2010genomic,pouyet2017recombination}.

Recently, \citet{halldorsson2019characterizing} argued that the mechanism guiding recombination away from genes may have emerged through evolution in order to reduce the deleterious effect of its inherent \textit{de novo} mutations (DNMs) on coding sequences. 
The mutagenicity of recombination was indeed demonstrated in yeasts \citep{strathern1995dna,rattray2015elevated} and humans \citep{arbeithuber2015crossovers,halldorsson2019characterizing} and explained, — together with Hill-Robertson effects, — the correlations found between recombination and genetic diversity in humans \citep{nachman2001single,lercher2002human,hellmann2003neutral,hellmann2005why,spencer2006influence,montgomery2013origin,smith2018large} and other species \citep{begun1992levels,aquadro1997insights,webster2012direct,cutter2013genomic}.\\

\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap3/recombination-rate-variation.eps}
	\caption[Heterogeneity in recombination rates along the human genome]
	{\textbf{Heterogeneity in recombination rates along the human genome.}
		\par \textbf{a |} The shape of the distribution of recombination rates (RRs) depends on the level of resolution.
		\textbf{b |} Most recombination events cluster in a small proportion of the total genomic sequence.
		\par This figure was reproduced from \citet{coop2007evolutionary} and originally adapted from \citep{myers2005finescale} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:recombination-rate-variation}
\end{figure}


More generally, sites of recombination initiation seem to correspond to regions of open chromatin: highly active sites present trimethylation of the 4\textsuperscript{th} lysine of histone H3 (H3K4me3) marks in yeasts \citep{borde2009histone} and mice \citep{buard2009distinct} and DNA hypomethylation in plants \citep{maloisel1998suppression,melamed-bessudo2012deficiency,mirouze2012loss}.
Curiously though, in mammals, long-range recombination rates seem to be associated to DNA hypermethylation rather than hypomethylation \citep{sigurdsson2009hapmap,zeng2014specific}.

Nucleosome-depleted regions (NDRs) are another typical feature of open chromatin and recombinational activity is stronger at these sites in mammals \citep[reviewed in \citealp{jabbari2019common}]{getun2010nucleosome,lange2016landscape,yamada2017genomic} as well as in \textit{Schizosaccharomyces pombe} \citep{decastro2012nucleosomal} and \textit{Saccharomyces cerevisiae} \citep{wu1994meiosisinduced,berchowitz2009positive} for which NDRs host most DSBs.
More precisely, recombination is found near transcription start sites (TSSs) of gene promoters in budding yeasts \citep{baudat1997clustering, petes2001meiotic, mancera2008highresolution}, dogs \citep{auton2013genetic,campbell2016pedigreebased}, plants \citep{hellsten2013finescale,choi2018nucleosomes} and birds \citep{singhal2015stable}.



\subsubsection{Recombination hotspots}

The level of resolution matters tremendously when analysing patterns of RR variation \citep[reviewed in][]{smukowski2011recombination}.
Indeed, at finer genomic scales of 1–10~kb, recombination rates considerably vary (Figure~\ref{fig:recombination-rate-variation}.a.): in humans \citep{mcvean2004finescale,the1000genomesprojectconsortium2010map} and other eukaryotes \citep{mezard2015where}, 80\% of recombination events gather in only 20\% of the genome (Figure~\ref{fig:recombination-rate-variation}.b.), primarily into 1—2-kb\footnote{In mammals. But, in yeasts, recombination hotspots span several kilo base pairs.} regions called ‘recombination hotspots’ \citep{myers2005finescale}.

Hotspots are generally defined as sequences that show a recombinational activity several times greater than the background rate \citep{crawford2004evidence,stapley2017variation}. 
However, the activity of adjacent regions and the genome-wide average are alternately used as the comparative criterium \citep{demassy2013initiation}, which renders the delimitation and the number of hotspots slightly imprecise.

Nevertheless, apart from \textit{Drosophila melanogaster} \citep{comeron2012many,manzano-winkler2013how}, \textit{Caenorhabditis elegans} \citep{kaur2014crossover} and \textit{Apis mellifera} \citep{mougel2014highresolution,wallberg2015extreme} which lack them, recombination hotspots have been identified in a myriad of eukaryotes, including \textit{Saccharomyces cerevisiae} \citep{sun1989doublestrand,lichten1995meiotic}, \textit{Schizosaccharomyces pombe} \citep{steiner2005natural,cromie2007discrete}, \textit{Arabidopsis thaliana} \citep{drouaud2006variation}, \textit{Zea mays} \citep{brown1991recombination,dooner1997recombination,yao2002molecular,fu2002recombination}, \textit{Triticum aestivum} \citep{saintenac2011variation} and other plants \citep{mezard2006meiotic}, \textit{Canis lupus} \citep{axelsson2012death}, \textit{Mus musculus} \citep{guillon2002initiation,kauppi2007meiotic,smagulova2011genomewide}, \textit{Pan troglodytes} \citep{winckler2005comparison,auton2012finescale} and \textit{Homo sapiens} \citep{jeffreys2001intensely,myers2005finescale}.\\


The first experimental evidence for hotspots was found serendipitously in the H2 region (i.e.\ major histocompatibility complex, MHC) of mouse chromosome 17 \citep{steinmetz1982molecular}.
The first human hotspots were later identified in \textgreek{β}-globin and insulin regions \citep{chakravarti1984nonuniform,chakravarti1986evidence}. 
Since then, the list of recognised hotspots has grown extensively \citep[reviewed in][]{arnheim2007mammalian,paigen2010mammalian} and many have been studied individually \textit{via} sperm-typing studies \citep[e.g.\ ][]{hubert1994high,jeffreys2001intensely,schneider2002direct} (see Appendix~\ref{app:data-and-figs}).

Later, genome-wide lists of hotspots — concordant with sperm-typing analyses \citep[e.g.][]{tiemann-boege2006highresolution} — have been achieved by analysing linkage disequilibrium in pedigrees or populations (see Subsections~\ref{chap3:linkage-maps}~and~\ref{chap3:LD}): about 30,000 have been uncovered in humans \citep{myers2005finescale,theinternationalhapmapconsortium2007seconda} and 47,000 in mice \citep{brunschwig2012finescale}.\\


Two additional layers of RR variation exist at the hotspot level in mammals.
First, the recombinational activity of individual hotspots varies over orders of magnitude \citep{jeffreys2001intensely,kauppi2004where,paigen2008recombinational}, with the number of hotspots per class of intensity following a negative exponential relationship \citep{paigen2010mammalian}.
Second, the apparent\footnote{The density of polymorphic markers (which can vary across hotspots) affects the ability to detect NCOs. As such, the apparent CO:NCO ratio may differ from the genuine CO:NCO ratio.} relative ratio of CO to NCO outcomes also varies between hotspots in flies \citep{singh2012classical}, yeasts \citep{mancera2008highresolution}, mice \citep{paigen2008recombinational} and humans \citep{jeffreys2004intense}.

These relative differences in hotspot activity come from their both \textit{cis-} and \textit{trans-} regulations \citep[reviewed in][]{paigen2010mammalian} which also account for the differences in hotspot usage among individuals.



\subsection{Inter-individual differences in hotspot usage}

\subsubsection{Sexual dimorphism}

Sex differences in recombination were discovered over a century ago with the first linkage studies in \textit{Drosophila melanogaster} \citep{morgan1912complete,morgan1914no}, \textit{Bombyx mori} \citep{takanaytianzhong1914sexualWITHJAP} and \textit{Gammarus chevreuxi} \citep{huxley1928sexual}. 
Since then, several levels of sexual dimorphism have been unveiled.

First, as compared to males, the overall recombinational activity is greater in females\footnote{This feature (a species with different RRs in both sexes) is termed ‘heterochiasmy’.} for most mammals \citep{dunn1967sex} including mice \citep{shifman2006highresolution} and humans \citep{donis-keller1987genetic,broman1998comprehensive} — a result consistent with the fact that the genetic maps are longer in females than in males in these two species \citep{lynn2004variation,cox2009new} as well as in pigs \citep{mikawa1999linkage}, dogs \citep{neff1999secondgeneration} and thale cresses \citep{drouaud2007sexspecific}. 
In mammals, this observation could be partly due to the fact that female meiosis entails a dictyate arrest which can last for decades (from the fetal age to ovulation), thus leaving time for spontaneous DSBs to arise and to be repaired as complex COs (see Chapter~\ref{ch:2-recombination-mechanistics}).
But it has also been argued that the synaptonemal complex (SC) length \textit{per se} could play a major role in determining recombination rate differences, since the SC is much longer — and the DNA loops much shorter — in occytes than in spermatocytes \citep{tease2004intersex}.
Of note, this effect is reversed in sheeps \citep{maddox2001enhanced}, flycatchers \citep{backstrom2008genebased} and most marsupials \citep{bennett1986novel,hayman1988further,hayman1990meiosis} and it not visible in one marsupial \citep{hayman1990comparative} nor cattle \citep{kappes1997secondgeneration}.

Second, sexual differences are regionalised: CO rates in men are several times lower near centromeres and higher near telomeres than in women \citep[reviewed in][]{buard2007playing}, arguably because the SC is shorter in males \citep{tease2004intersex} and their synapsis preferentially initiates at subtelomeric regions \citep{brown2005meiotic}.
Contrariwise, females display more numerous interstitial initiation sites and their recombination landscape is thus generally flatter \citep{paigen2008recombinational}.

Despite these sexual differences in hotspot usage — which can be so strong that a few hotspots are sometimes perceived as entirely sex-specific \citep{shiroishi1990recombinational,shiroishi1991genetic}, — nearly all hotspots are shared by both males and females \citep{bherer2017refined}.


Altogether, this sexual dimorphism mainly results from disparities in hotspot usage \citep{brick2018extensive} possibly coming from haploid selection \citep{lenormand2005recombination}, imprinting \citep{lercher2003imprinted} or sex-based differences in chromatin structure \citep{gerton2005homologous} and SC length \citep{petkov2007crossover}.


\subsubsection{Heterogeneity between individuals}

Hotspot usage is also variable between individuals of the same sex (reviewed in \citealp{popa2011evolution} and \citealp{capilla2016mammalian}).

In humans, fluctuations in recombination rates are greater between women than between men, but both sexes show inter-individual variation \citep{cheung2007polymorphic}.
For instance, the major histocompatibility complex (MHC) shows a 2-fold difference among 5 men \citep{yu1996individual}, some hotspots are active in only a few men \citep{neumann2006polymorphism} and the CO:NCO ratio shows inter-individual disparities \citep{jeffreys2005factors,sarbajna2012major}.

As for mice, an inter-individual effect was also found in one strain \citep{koehler2002genetic}, but not in others.
Thus, RRs vary not only between chromosomal regions and individuals, but also across populations and species, which indicates that they evolve with time, as reviewed in the following section.





\section{Evolvability of recombination rates (RRs)}

\subsection{Intra- and inter-species comparison of fine-scale RRs}

The comparison of human linkage disequilibrium (LD) maps has shown that LD blocks are highly correlated among populations \citep{gabriel2002structure}, but the positions of the historical recombination hotspots they uncover are not entirely concordant with the one-generation recombination of genetic maps \citep{tapper2005map}. 
This non-concordance between historical and actual recombination was also observed independently at specific regions \citep{jeffreys2005human,kauppi2005localized} and suggests that the set of hotspots reorganises through time.
Thus, discrepancies in the fine-scale RR should be found both within and among species.\\

On the one hand, recombination rates exhibit intra-species disparity.
In mice, for instance, the number of MLH1 foci (a proxy for the number of COs) differs between strains \citep{koehler2002genetic,paigen2008recombinational,baier2014variation} and, in humans, the use of recombination hostpots vary across populations \citep{berg2011variants,hinch2011landscape}.

On the other hand, even though closely related species show similar average recombination rates (RRs) \citep{dumont2008evolution,hassold2009cytological,garcia-cruz2011comparative,auton2012finescale} when compared over the scale of megabases (Mb), dissimilarities appear at finer scales, as was shown between humans and macaques \citep{wall2003comparative}, between humans and chimpanzees \citep{ptak2004absence,ptak2005finescale,winckler2005comparison} and between humans and great apes \citep{stevison2016time}.\\

The reasons for such a rapid turnover of recombination hotspots were understood about a decade ago with the discovery of the protein that determines the position of recombination hotspots in mammals: PRDM9.


\subsection{\textit{Prdm9}, the fast-evolving mammalian speciation gene}

\subsubsection{Discovery of the \textit{Prdm9} gene}
Positive regulatory (PR) domain zinc finger protein 9 (PRDM9) — encoded by a gene originally named \textit{Meisetz} (for ‘meiosis-induced factor containing PR/SET domain and zinc-finger motif’) — was discovered in mouse germ cells as a histone H3 lysine 4 methyltransferase protein essential to the progression through meiotic prophase \citep{hayashi2005histone,hayashi2006meisetz}.
In 2010, three groups simultaneously identified it as responsible for the positioning of recombination hotspots in mice and humans \citep[reviewed in \citealp{cheung2010genetic} and \citealp{hochwagen2010meiosis}]{baudat2010prdm9,myers2010drive,parvanov2010prdm9}.

One of these groups had previously identified a degenerate 13-bp GC-rich motif \citep{myers2005finescale} implicated in the activity of 40\% of human hotspots \citep{myers2008common,webb2008sperm} and had predicted that it was likely bound by a zinc finger protein of at least 12 units \citep{myers2008common}.
Later, the computational analysis of all predicted zinc-finger DNA-binding proteins in the human genome yielded PRDM9 as both the only binding partner compatible with the observed degeneracy of the motif and the only candidate consistent with the lack of activity in chimpanzees \citep{myers2010drive}.

The other two groups had previously independently identified a $\sim$5-Mb region on mouse chromosome 17 containing a \textit{trans-}acting locus controlling the activation of specific hotspots \citep{grey2009genomewide,parvanov2009transregulation}, respectively named \textit{Dsbc1} and \textit{Rcr1} at the time.
\citet{parvanov2010prdm9} used a mouse cross to narrow the interval down to 181 kb and argued that, among the four genes it comprised, \textit{Prdm9} was the only relevant candidate that could explain the differences in hotspot usage.

\citet{baudat2010prdm9} also reduced the interval with additional crosses to identify \textit{Prdm9} as a relevant candidate. 
They further sequenced several human variants and found that the human \textit{Prdm9} alleles were associated with hotspot usage, thus providing convincing evidence that it plays a major role in hotspot positioning, and demonstrated its sequence-specific binding to the 13-bp motif \textit{in vitro}.

The dots were later reconnected with two past studies: one had found a haplotype associated with the control of recombination \citep{shiroishi1982new} — this haplotype actually contained \textit{Prdm9}; and in another, a protein binding a minisatellite motif had been partially purified \citep{wahls1991two} — this protein turned out to be PRDM9 \citep{wahls2011dna}.\\

Since then, the role of PRDM9 in regulating the position of recombination hotspots has been confirmed multiple times in humans \citep{berg2010prdm9,pratto2014recombination} and observed in other primates \citep{groeneveld2012high,heerschop2016pioneering,schwartz2014primate}, rodents \citep{buard2014diversity,capilla2014genetic,kono2014prdm9}, ruminants \citep{sandor2012genetic,ahlawat2016zinc,ahlawat2016evidence,ahlawat2017evolutionary} and equids \citep{steiner2013characterization}.

Nevertheless, PRDM9 does not bind solely its specific binding motifs \citep{grey2017vivo} and, in PRDM9-lacking mice, DSBs are located at functional sites \citep{brick2012genetic}.
It has been proposed that DSB repair at such sites is inefficient and leads to sterilty \citep{brick2012genetic} but a recent study proved that PRDM9 is \textit{not} essential to fertility in male mice \citep{mihola2019histone}. 
As for humans, a woman lacking a functional \textit{Prdm9} allele was found to be fertile \citep{narasimhan2016health}.
\pagebreak
Hotspots are also defined independently of PRDM9 in canids \citep{axelsson2012death,munoz-fuentes2011prdm9,auton2013genetic} and birds \citep{singhal2015stable} in which they instead locate at transcription start sites (TSSs) and are stable over evolutionary times.


\subsubsection{Structure of the protein}

PRDM9 determines the precise localisation of hotspots thanks to its carboxy-terminal tandem array of 8 to over 20 Cys\textsubscript{2}-His\textsubscript{2} (C2H2) zinc fingers (Znf) \citep[reviewed in][]{paigen2018prdm9}: the residues -1, +3 and +6 (relative to the alpha helix) of each Znf specify the DNA trinucleotide to bind and thus, altogether, the sequence target of the Znf array \citep{neale2010prdm9}.
A few fingers contribute preponderantly to the principal motif recognised (Figure~\ref{fig:PRDM9-structure}.B.) and one Znf is separated from the rest of the array and closer to the central region (Figure~\ref{fig:PRDM9-structure}.A.).

The central region also contains the histone methyltransferase PR/SET domain which is distantly related to the family of Suppressor of variegation 3–9, Enhancer of Zeste and Trithorax (SET) domains \citep[reviewed in][]{grey2018prdm9}.
Thanks to this domain required for DSB formation \citep{diagouraga2018prdm9}, PRDM9 can catalyse the mono-, di- and trimethylation of H3K4 and H3K36\footnote{H3K4, H3K36: Lysine 4 (resp.\ 36) of histone H3.} \citep{wu2013molecular,powers2016meiotic} but also its own authomethylation \citep{koh-stenta2017discovery} which may help to regulate its activity by modulating the folding of the PR/SET domain.

The N-terminus hosts the Kr\"uppel-associated box (KRAB)–related domain involved in protein:protein interactions \citep{parvanov2016prdm9,parvanov2017prdm9,imai2017prdm9}, and a synovial sarcoma X repression domain (SSXRD).
These two domains are also known to be involved in transcriptional repression \citep{margolin1994kruppelassociated,lim1998krabrelated} but no such activity was identified in human PRDM9 \citep{born2014bsubdomain}, and they both seem essential to the hotspot-targeting role of PRDM9 \citep{baker2017repeated,thibault-sennett2018interrogating}.

\begin{figure}[t]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap3/PRDM9-structure.eps}
	\caption[Molecular structure of PRDM9]
	{\textbf{Molecular structure of PRDM9.}
		\par \textbf{A |} The PRDM9 protein consists of a KRAB-like, a SSXRD, a PR/SET and a zinc finger (Znf) array domains. 
		The three residues (located at positions -1, +3 and +6 relative to the alpha helix) that are explicitely lettered specify the DNA target of each Znf. 
		Human \textit{A} and mouse \textit{Dom2} alleles are shown.
		\textbf{B |} The composition of the tandem Znf array of the major human and mouse \textit{Prdm9} alleles are represented as a sequence of squares, coloured based on the composition of residues at positions -1, +3 and +6.
		The boxes frame the fingers that contribute most to the principal motif of each allele.
		\par This figure was reproduced from \citet{paigen2018prdm9} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:PRDM9-structure}
\end{figure}



\subsubsection{Multimerisation and hybrid sterility}

PRDM9 has been proposed to act as a multimer \citep{baker2015multimer,altemose2017map,schwarz2019prdm9} which may explain the dominance of certain alleles reported for human \textit{C} over \textit{A} \citep{pratto2014recombination} and \textit{I} over \textit{A} alleles \citep{baudat2010prdm9}, as well as mouse \textit{13R} over \textit{9R} \citep{brick2012genetic} and \textit{Cst} over \textit{Dom2} alleles \citep{smagulova2011genomewide, baker2015prdm9, baker2015multimer}.

Multimer formation certainly may play a role in PRDM9-mediated homologue pairing \citep{davies2016reengineering} and dominance may affect the dosage sensitivity of PRDM9 \citep{flachs2012interallelic, segurel2011case} and thus participate in both hybrid infertility — which was observed long before the known implication of \textit{Prdm9} \citep{forejt1974genetic} — and in speciation \citep{mihola2009mouse}.

Given its critical role in fertility, one might expect PRDM9 to be under strong purifying selection and thus to be highly conserved. 
But, counterintuitively, it seems to evolve rapidly.


\subsubsection{The rapid evolution of \textit{Prdm9}}

The Znf array forms a vast reservoir of variability since it may differ both in length (number of fingers) and composition, thus yielding extensive allelic possibilities for \textit{Prdm9}.

Indeed, a large number of \textit{Prdm9} alleles have been uncovered in primates \citep{groeneveld2012high,heerschop2016pioneering} and ruminants \citep{ahlawat2016zinc}.
As for mice, over 100 distinct alleles have been detected thus far \citep{buard2014diversity, kono2014prdm9}.
Most laboratory inbred strains derived from the \textit{Mus musculus domesticus} subspecies carry either the \textit{Dom2} or \textit{Dom3} allele while those derived from \textit{Mus musculus musculus} carry the \textit{Msc} allele and those derived from \textit{Mus musculus castaneus} the \textit{Cst} allele (Figure~\ref{fig:PRDM9-structure}.B.).

Human populations also vary in their PRDM9 allelic composition \citep{berg2010prdm9, berg2011variants, fledel-alon2011variation}: African populations have $\sim$50\% of allele \textit{A}, 13\% of allele \textit{C} and the rest composed of other minor alleles \citep{berg2011variants}; non-African populations mainly encompass allele \textit{A} and, to a smaller extent, allele \textit{B} \citep{baudat2010prdm9,berg2010prdm9,hinch2011landscape}; and the Neanderthal and Denisovan samples studied so far exhibit yet other alleles \citep{schwartz2014primate,lesecque2014red}.\\

Such great allelic diversity, which is associated with diversity in hotspot usage, is made possible by the high mutation rate of \textit{Prdm9} \citep{jeffreys2013recombination} and by the strong positive selection exerted on its decisive Znf residues \citep{oliver2009accelerated,thomas2009extraordinary,ponting2011what}.




\subsection{The Red Queen dynamics of hotspot evolution}

\subsubsection{DSB-induced biased gene conversion (dBGC) and the erosion of targets}

Once PRDM9 has bound its allele-specific target, a DSB is initiated and subsequently repaired as a CO or a NCO (see Chapter~\ref{ch:2-recombination-mechanistics}).
In most hotspots studied, the distribution of CO exchange points — which likely reflect the position of the resolution of the transient Holliday junction rather than the DSB initiation site \citep{smith2001homologous} — decreases identically on the two sides (5’ and 3’) of the DSB \citep{arnheim2007mammalian}.
However, a skewed CO exchange point distribution appeared in a few hotspots \citep{jeffreys2002reciprocal,jeffreys2005factors,yauk2003highresolution,neumann2006polymorphism} and was interpreted as a visible corollary of the differential DSB initiation on the two homologues \citep{baudat2007cis}.

Indeed, PRDM9 can \textit{a priori} bind its target on either homologue (‘haplotype’ henceforth). 
However, if one haplotype has a higher PRDM9-binding affinity, it hosts more DSBs and is thus ‘hotter’ \citep{zelazowski2016marks}.
Therefore, the other, ‘colder’ haplotype is used as a template to repair the DSB, which results in the hot haplotype being frequently converted by the cold one. 
This meiotic initiation bias thus yields biased gene conversion (BGC) recombination events and, since this phenomenon is induced by the preferential placement of DSBs on one haplotype, I will henceforth call it ‘DSB-induced BGC’ (dBGC), as others before \citep{lesecque2014conversion, grey2018prdm9}.

A differential binding affinity between the two haplotypes arises when one target motif acquires mutations:
the more affinity-disruptive mutations the targeted motif gains (i.e.\ the more eroded the hotspot), the more asymmetrically the DSBs initiate (i.e.\ the more asymmetric the hotspot), and the stronger the dBGC effect \citep[reviewed in][]{tiemann-boege2017consequences}.



\subsubsection{The hotspot paradox}


As just stated, during the repair of the DSB, the hot (recombination-activating) haplotype is converted into the cold (recombination-suppressing) haplotype from the other chromosome \citep{gutz1971site,schuchert1988ade6m26,jeffreys2009rise} and therefore suffers a meiotic drive against itself.
Consequently, in the long-term, the very mechanism of recombination is expected to lead to the self-destruction of hotspots — a prediction that seems antipodal with the observation that hotspots are abundant in sexually active eukaryotes.
This dilemma has been called the ‘hotspot paradox’ \citep{boulton1997hotspot}: individually, hotspots are suicidal but, collectively, they are maintained.\\

Over the decade following the discovery of that paradox, several theoretical studies have been conducted to try and understand how hotspots are maintained despite their self-destruction \citep{boulton1997hotspot,pineda-krch2005persistence,coop2007live}.
Three main hypotheses were put forward by these studies to justify the maintenance of hotspots.

First, all three studies have proposed that recombination-activating back-mutations could arise in hotspots to counteract their extinction by dBGC\@. Though, all three conclude that the mutation rate required in face of the intensity of gene conversion would need to be unfeasibly large for them to be likely to be observed.

Second, the authors suggested that, given the benefits of recombination on fertility and viability, there could be a selective force opposing the spread of recombination-suppressing haplotypes: to ensure the correct segregation of alleles, recombination hotspot alleles could be directly selected for. However, for such a selective force to be strong enough to counterbalance hotspot extinction, DSBs would have to resolve into COs with a much higher probability than is observed in reality.

The third main hypothesis put forward was arguably the most plausible one: hotspots appear to compete for a finite amount of recombination with other adjacent hotspots.
As such, it may be possible for them to increase their activity — and thus to start experiencing drive — only when nearby ones have been lost.
This inter-hotspot competition drastically slowed down the expected rate of extinction. Still, it did not allow hotspots to persist indefinitely.
As such, at that time, the mystery remained complete as to the way the paradox could be solved. 








\subsubsection{Determinants of the Red Queen dynamics}

\begin{figure}[b!]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap3/Alice-red-queen-BW.eps}
	\caption[Original drawing of Alice and the Red Queen by John Tenniel]
	{\textbf{Original drawing of Alice and the Red Queen by John Tenniel.}
		\par The ‘Red Queen dynamics’ term is derived from a statement of the Red Queen in Lewis Carroll's \textit{\usebibentry{carroll1871lookingglass}{title}} \citeyearpar{carroll1871lookingglass} about the nature of her world: \textit{‘Now, \textit{here}, you see, it takes all the running \textit{you} can do, to keep in the same place. If you want to get somewhere else, you must run at least twice as fast as that!’.}
		\par This figure is free of rights and was reproduced from \citet{carroll1871lookingglass}.
	}
\label{fig:Alice-red-queen-BW}
\end{figure}


Further progress in solving the hotspot paradox came in 2010 with the discovery of PRDM9 as the determinant of hotspot localisation \citep{baudat2010prdm9,myers2010drive,parvanov2010prdm9}.
Indeed, it had been mentionned two years before that the hotspot paradox could theroretically be resolved if a \textit{trans}-acting modifier (thus escaping gene conversion) had the ability to activate or inactivate the hotspots \citep{peters2008combination,friberg2008cut}.


\citet{ubeda2011red} formally formulated the model involving PRDM9 as the \textit{trans}-acting protein solving the paradox under the form of a race for evolution termed a ‘Red Queen dynamics’ \citep{vanvalen1973new}, after the words of the Red Queen in the \textit{\usebibentry{carroll1871lookingglass}{title}} book by Lewis Caroll \citeyearpar{carroll1871lookingglass} (Figure~\ref{fig:Alice-red-queen-BW}).

In their model (Figure~\ref{fig:hotspot-paradox}), owing to dBGC, the target loci lose their propensity to be bound by PRDM9, thereby reducing the overall recombination rate and creating a selective pressure for PRDM9 alleles to evolve and target a new set of binding sites.
This intragenomic conflict leads to a never-ending situation where recombinogenic PRDM9 alleles continually chase their target motifs and evolve into other allelic variants as soon as their targeted sites are sufficiently eroded.


More recently, \citet{latrille2017red} formalised a quantitative population-genetic model accounting for all possible actors of the Red Queen model.
Their mathematical developments led to the identification that both an extremely high mutation rate of PRDM9 and a strong dBGC eroding its target motifs are required for the model to be valid.\\



However, \citet{ponting2011what} questioned this theory on the basis that the number of recombination hotspots ($\sim$25,000 in humans) far exceeds the number of chromosome arms ($\sim$40) and proposed four explanations justifying the strong and sustained positive selection on the DNA-binding determinant sites of PRDM9.

First, it could be that only a portion of the hotspots are bound by PRDM9 with strong affinity and that PRDM9 could evolve to keep a high binding affinity with a maximum number of these strong sites.

Second, since the PAR of sexual chromosomes is very short and is the only region where COs can form between these chromosomes, PRDM9 may be driven to evolve rapidly to ensure their correct segregation.

Third, if multiple weakly deleterious alleles accumulate in a non-recombining region, PRDM9 may be driven to evolve and target this particular region to break down the detrimental linkage in it.

Last, PRDM9 may evolve so as to prevent diseases, since increased CO rates in certain regions can lead individuals to certain diseases.


\begin{figure}[p]
    \centering
    \includegraphics[width = 0.8\textwidth]{figures/inkscape/hotspot-paradox.eps}
    \caption[The Red Queen model of recombination hotspots]
    {\textbf{The Red Queen model of recombination hotspots.}
        \par In mice, the position of recombination hotspots, defined as regions of elevated recombination rate, is determined by PRDM9.
		At a given generation (top panel), one allelic variant of this protein, PRDM9\textsuperscript{allele\_1}, targets specifically its target motif (yellow square) thanks to its sequence-specific zing finger array (yellow triangles).
		Over time, because of double-strand break induced biased gene conversion (dBGC), the recombination-activating haplotypes carrying the target motif get eroded (crossed yellow square), which directly leads to a deprivation of hotspots as fewer sites are targeted by the PRDM9 allele present in the individual (middle panel).
		According to the Red Queen model of recombination hotspots, this creates a selective pressure for PRDM9 to evolve rapidly into a new allele, PRDM9\textsuperscript{allele\_2}, carrying a distinct zinc finger array (red triangles) targeting a new set of motifs (red square).
		As such, the recombination landscape with this new allele (bottom panel) is completely different from the one with the original allele (top panel).
    }
\label{fig:hotspot-paradox}
\end{figure}


\subsubsection{Experimental proofs of the Red Queen model}

Whichever the reason driving PRDM9 to evolve, all hypotheses rest on the following assumption of the Red Queen model: that the destruction of PRDM9 targets \textit{via} dBGC is at the origin of the raise in frequency of new PRDM9 variants.
Though, for this model to be plausible, dBGC must be strong enough to lead to a significant loss of hotspots genome-wide.
Therefore, \citet{lesecque2014red} empirically quantified the dynamics of hotspot turnover by estimating the age and life expectancy of human hotspots.
Their estimates showed that human hotspots were both much younger and much shorter-lived than had previsouly been suggested, and that dBGC was extremely high in certain hotspots.
As such, they showed that dBGC was indeed sufficiently strong to explain the rapid loss of hotspots.

Further experimental testings of PRDM9 driving the evolutionary erosion of hotspots were carried in mice by \citet{baker2015prdm9}.
They indeed compared the activity of a \textit{Prdm9} allele originating from the \textit{Mus musculus castaneus} subspecies (\textit{Prdm9\textsuperscript{Cst}}) in both \textit{Mus musculus castaneus} and \textit{Mus musculus domesticus}.
They found that most variants affecting PRDM9\textsuperscript{Cst} binding had arisen specifically in the \textit{Mus musculus castaneus} subspecies in which it had evolved and that hotspots had thus been greatly eroded in that lineage, which confirmed experimentally the predictions of the Red Queen model.

As a consequence of this haplotype difference, F1 hybrids between the two subspecies showed large haplotype biases in PRDM9 binding.
The latter were sometimes so large that novel hotspots appeared in the hybrid, as a result of the interplay between one parent's \textit{Prdm9} allele and the other parent's chromosome (for the hotspot on the ‘self’ chromosome had eroded).

\citet{smagulova2016evolutionary} further analysed the consequences of such sequence divergence generated by hotspot turnover in mouse hybrids and suggested that, because COs are disfavoured at the hotspots showing large haplotype biases, this may lead to reduced fertility and, ultimately, to speciation.
The precise reasons why a shortage of symmetric hotspots can cause asynapsis remain to be elucidated, but it has been proposed that it may be due to a concomittant asymmetry in PRDM9-dependent chromatin remodelling \citep{davies2016reengineering} or to an excessively high level of heterozygosity impeding recombination \citep{gregorova2018modulation}.\\




Altogether, DSB-induced biased gene conversion (dBGC) is an important driver for the evolution of the recombination landscape.
Though, it is not the only one: another form of meiotic drive (GC-biased gene conversion, gBGC) also shapes the genome around recombination hotspots. 
I will review it in the following chapter.



