\begin{savequote}[8cm]
	
	‘In relation to any experiment we may speak of this hypothesis as the “null hypothesis,” and it should be noted that the null hypothesis is never proved or established, but is possibly disproved, in the course of experimentation. Every experiment may be said to exist only in order to give the facts a chance of disproving the null hypothesis.’
	
	\qauthor{--- Ronald Fisher, \textit{\usebibentry{fisher1935design}{title}} \citeyearpar{fisher1935design} }
	
\end{savequote}

\chapter{\label{ch:7-quantification-BGC}Quantification of biased gene conversion in mouse hotspots}
% \chapter{\label{ch:7-quantification-BGC}Quantification of biased gene conversion in mouse F1 hybrids}
% \chapter{\label{ch:7-quantification-BGC}Quantification of DSB-induced and GC-biased gene conversion}
%\otherpagedecoration


\minitoc{}

{\small{} \itshape{}

\paragraph{This chapter in brief —}

In order to shed new light into the relationship between the intensity of GC-biased gene conversion (gBGC) and the effective population size ($N_e$), we wanted to precisely quantify the transmission bias ($b_0$) in a mammalian species with low $N_e$: mice.
We first quantified DSB-induced biased gene conversion (dBGC) in autosomal hotspots and observed that, in our B6xCAST F1 hybrid, dBGC hitchhiked the past gBGC that had occurred in the parental lineages.
We then controlled for this confounding effect to quantify gBGC in both COs and NCOs.
We found that the transmission bias ($b_0$) was null for COs and very weak for multiple-marker NCOs. 
In contrast, single-marker NCOs showed large $b_0$.
% suggere mecanisme OU pression de selection quand bcp de SNPs (a discuter a la fin de ce chpaitre + dans chapitre 9)

% Resume
% BGC avec Ne variable
% On meusre chez souris
% On a mesure le dBGC sur 1000 hotspots
% On observe que, dans des populations structures comme dans nos hybrides F1, il existe un hitchhiking du dBGC sur le gBGC passe.
% donc il nous a fallu controller pour cet effet pour quantifier le gBGC
% On observe que l'intensite du gBGC est nul pour les CO, faible pour les NCO longs et plus eleve chez les NCO a 1 SNP.
% Mais pas d'effet sur les bordures de SNPs dans les fins de tracts.
% Cela suggere qu'il y a un mecanisme different qui met en place le gBGC sur els NCO courts et longs. Ou qu'il y a une pression de selection qui limite l'effet du gBGC sur les longs tracts.


}

\newpage

Considering that the intensity of GC-biased gene conversion (gBGC) at the population-scale ($B$) is the product of the effective population size ($N_e$) by the gBGC coefficient ($b$) (see Chapter~\ref{ch:4-gBGC}), the finding that $B$ confines into a very small range of values — even across animals with considerably disparate $N_e$ — was puzzling \citep{galtier2018codon}.
Logically thus, one or several of the parameters on which $b$ depends — among which the transmission bias $b_0$ — should vary inversely with $N_e$.

Though, among mammals, the transmission bias ($b_0$) has only been measured in humans \citep{williams2015noncrossover, halldorsson2016rate} and consequently, the interplay between $b$ and $N_e$ remains unexplained.
% To shed new insight into this relationship, we thus quantified gBGC in another mammalian species with larger $N_e$ (mice).
In this chapter, I describe how we managed to shed new insight into this relationship by quantifying gBGC in another mammalian species with larger $N_e$ (mice).
Since this quantification required to classify hotspots according to their PRDM9 target so as to control for the confounding effect of DSB-induced biased gene conversion (dBGC), I will start this chapter with two sections presenting this process.
% But, before that, it appeared necessary to classify hotspots according to their PRDM9 target so as to control for the confounding effect of DSB-induced biased gene conversion (dBGC).
%, as I report in the last section of this chapter.
% But, before that, it appeared necessary to classify hotspots according to their PRDM9 target so as to control for the confounding effect of DSB-induced biased gene conversion (dBGC).
% I describe this process in the first two sections of this chapter.



% \section{Classification of hotspots}
% \subsection{Identification of the PRDM9 target}

\section{Identification of the PRDM9 target}
\subsection{Methodology to classify hotspots}
\label{chap7:PRDM9-target-inference}

In a B6xCAST F1 hybrid, hotspots are either activated by the \textit{Prdm9} allele originating from the B6 lineage (\textit{Prdm9\textsuperscript{Dom2}}) or by that originating from the CAST lineage (\textit{Prdm9\textsuperscript{Cst}}).
To discriminate between these two scenarii, we classified all 1,018 hotspots based on two criteria (Table~\ref{tab:classification-hotspots}).

First, we used PRDM9 ChIP-seq data in the parental B6 and CAST strains from \citet{baker2015prdm9}: 
when a peak was detected in one parental strain, the hotspot was necessarily targeted by the allele present in that strain, i.e.\ PRDM9\textsuperscript{Dom2} (resp.\ PRDM9\textsuperscript{Cst}) when the peak was found in the B6 (resp.\ CAST) lineage.
 % and PRDM9\textsuperscript{Cst} when it was found in the CAST lineage.

% When, however, no PRDM9 ChIP-seq peak was detected in either parent, knowing which allele targeted the hotspot was not straightforward.
% % As a substitute, we used information from the strand-specific detection of PRDM9 ChIP-seq reads in the B6xCAST cross from \citet{baker2015prdm9} to predict which haplotype was hot, i.e.\ targeted by PRDM9, and which was cold, i.e.\ donor in the subsequent gene conversion event.
% As a substitute, we used information from the strand-specific detection of PRDM9 ChIP-seq reads in the B6xCAST cross from \citet{baker2015prdm9} to predict which haplotype was cold, i.e.\ the donor in the subsequent gene conversion event.
% % Indeed, if the majority of PRDM9 ChIP-seq tags map onto one haplotype, it means that this haplotype is preferentially bound by PRDM9 and thus, that it hosts the DSB more often than the alternate (thus, cold) haplotype.
% Indeed, the haplotype onto which the majority of PRDM9 ChIP-seq tags map is necessarily that preferentially bound by PRDM9 and thus, the hot haplotype.
% Like \citet{baker2015prdm9}, we set the threshold at 75\%: if more than 75\% of ChIP-seq reads mapped onto the B6 (resp.\ CAST) haplotype, we inferred that the cold haplotype was CAST (resp.\ B6).
%

When, however, no PRDM9 ChIP-seq peak was detected in either parent, knowing which allele targeted the hotspot was not straightforward.
% As a substitute, we used information from the strand-specific detection of PRDM9 ChIP-seq reads in the B6xCAST cross from \citet{baker2015prdm9} to predict which haplotype was hot, i.e.\ targeted by PRDM9, and which was cold, i.e.\ donor in the subsequent gene conversion event.
% As a substitute, we used information from the strand-specific detection of PRDM9 ChIP-seq reads in the B6xCAST cross from \citet{baker2015prdm9}.
As a substitute, we used information from the strand-specific detection of PRDM9 ChIP-seq reads \citep{baker2015prdm9}.
Indeed, the proportion of PRDM9 ChIP-seq tags mapping onto one haplotype directly reflects its propensity to be bound by PRDM9 (relatively to that of the other haplotype).
Using the assumption that the least bound haplotype had a lower affinity because it had co-evolved with the \textit{Prdm9} allele targeting the hotspot and had thus undergone erosion in the parental lineage, we inferred that \textit{Prdm9\textsuperscript{Dom2}} (resp.\ \textit{Prdm9\textsuperscript{Cst}}) was the target when at least 75\% of PRDM9 ChIP-seq tags mapped preferentially onto the CAST (resp.\ B6) haplotype.
 % and \textit{Prdm9\textsuperscript{Cst}} when they mapped preferentially onto the B6 haplotype.


% Using the assumption that the differential PRDM9 affinity arose from the erosion of the least bound haplotype in the parental lineage, we inferred that the \textit{Prdm9} allele targeting the hotspot was the one which co-evolved with the eroded haplotype.
% We assumed that the least bound haplotype was least bound because it had co-evolved with eroded in the parental lineage where
%
% We assumed that the differential PRDM9 affinity was a manifestation of the erosion of the least bound haplotype during its coevolution with the \textit{Prdm9} allele targeting the hotspot in the parental lineage.
% As such, we could infer that the \textit{Prdm9} allele targeting the hotspot was the one that had coevolved with the least bound haplotype.
%
% We assumed that the haplotype that had co-evolved with the \textit{Prdm9} allele targeting the hotspot had undergone erosion due to biased gene conversion in the parental lineage.
%
% We made the assumption that the differential PRDM9 affinity was the result of the erosion of the least bound haplotype during its co-evolution with the \textit{Prdm9} allele targeting the hotspot in the parental lineage.
% we could determine that the
% We then infered that the \textit{Prdm9} allele targeting the hotspot was the one that had co-evolved with the least bound haplotype
%
%
%
% We assumed that the least bound haplotype was eroded because it had co-evolved with the \textit{Prdm9} allele targeting the hotspot in the parental lineage.
%
%
% And thus, we could
%
% % its motif in the haplotype that co-evolved with the \textit{Prdm9} allele targeting it, we can infer that the
% % \textit{Prdm9} allele targeting the hotspot is the one which co-evolved with the eroded haplotype.
%
%
% La proportion de tags sur un haplo reflete directement sa propensite (relativement a l'autre) a etre cible par PRDM9.
% En supposant que dans la majorite des cas, la difference de ciblage est due a l'erosion dans l'autre haplo, et que cette erosion provient du fait qu'il etait cible dans la lignee parentale par l'llele PrdM9, on peut inferer que l'allele Prdm9 est celui qui a coevolue avec l'haplotype erode (non cible).
%
% - l'haplotype sur lequel les tags mappent est celui cible par PRDM9 (propensite a etre cible). Donc s'i haplotype bias, alors fort bias de formation de DSBs, donc c'est un hot haplotype.
% - on dit que si plus de 75 alors biais fort et c'est l'aplot chaud.
% Or, dans le cas d'une hybride ou les deux alleles sont presents,
%
%  to predict which haplotype was cold, i.e.\ the donor in the subsequent gene conversion event.
% % Indeed, if the majority of PRDM9 ChIP-seq tags map onto one haplotype, it means that this haplotype is preferentially bound by PRDM9 and thus, that it hosts the DSB more often than the alternate (thus, cold) haplotype.
% Indeed, the haplotype onto which the majority of PRDM9 ChIP-seq tags map is necessarily that preferentially bound by PRDM9 and thus, the hot haplotype.
% Like \citet{baker2015prdm9}, we set the threshold at 75\%: if more than 75\% of ChIP-seq reads mapped onto the B6 (resp.\ CAST) haplotype, we inferred that the cold haplotype was CAST (resp.\ B6).
%


\subsection{Symmetric \textit{versus} asymmetric hotspots}

\begin{table}[b!]
	\centering
	\begin{adjustbox}{width = 1\textwidth}
		\begin{tabular}{rrrrrrr}

			\toprule
			& & \multicolumn{2}{c}{\textbf{Selected hotspots}} & &  \multicolumn{2}{c}{\textbf{All hotspots}} \\
			\cmidrule{3-4}
			\cmidrule{6-7}
			\textbf{Hotspot category} & & \textbf{Number} & \textbf{Percentage (\%)} & & \textbf{Number} & \textbf{Percentage (\%)} \\

			\cmidrule{1-1}
			\cmidrule{3-4}
			\cmidrule{6-7}

			\textbf{\textit{PRDM9\textsuperscript{Dom2}-targeted}}\\
			\textit{tB.sym}     & & 24    & 2.36  & & 181   & 2.68 \\ 
			\textit{tB.chB}     & & 63    & 6.19  & & 267   & 3.95 \\ 
			\textit{NOV.tB.chB} & & 80    & 7.86  & & 245   & 3.63 \\ 
			\\

			\textbf{\textit{PRDM9\textsuperscript{Cst}-targeted}}\\
			\textit{tC.sym}     & & 322   & 31.63 & & 2,775 & 41.06 \\
			\textit{tC.chC}     & & 241   & 23.67 & & 1,370 & 20.27 \\ 
			\textit{NOV.tC.chC} & & 156   & 15.32 & & 659   & 9.75 \\ 
			\\

			\textbf{\textit{Unclassified}} & & 132 & 12.9 & & 1,261 & 18.2 \\
			\\

			\cmidrule{1-1}
			\cmidrule{3-4}
			\cmidrule{6-7}
			\textbf{Total}      & & \textbf{1,018} & \textbf{100}   & & \textbf{6,758} & \textbf{100} \\
			\bottomrule

		\end{tabular}
	\end{adjustbox}
	\caption[Distribution of hotspots into each category of our classification]
	{\textbf{Distribution of hotspots into each category of our classification.}
		\par All 6,758 PRDM9 ChIP-seq-defined hotspots identified by \citet{baker2015prdm9} and the subset of 1,018 that we selected were classified into 6 categories of hotspots, as described in the main text.
		Hotspot categories were labeled as follows.
		‘tB’ (resp.\ ‘tC’) stands for a PRDM9 allele originating from the B6 (resp.\ CAST) strain.
		‘chB’ (resp.\ ‘chC’) stands for the B6 (resp.\ CAST) haplotype being the cold one.
		‘sym’ stands for symmetric hotspots, i.e.\ those having both haplotypes equally targeted by the two PRDM9 alleles.
		‘NOV’ stands for novel hotspots, i.e.\ those for which no PRDM9 ChIP-seq peak was detected in either parent.
		Thus, the target (tB or tC) for ‘NOV’ hotspots was exclusively determined based on the strand-specific mapping of PRDM9 ChIP-seq tags (see main text).
	}
\label{tab:classification-hotspots}
\end{table}



We named the aforementioned class of hotspots displaying large haplotype biases (i.e.\ those with over 75\% of PRDM9 ChIP-seq tags mapping onto one haplotype) ‘asymmetric’ hotspots, and all the others ‘symmetric’ hotspots.
% (i.e.\ those displaying no large haplotype bias)

As mentioned above, such asymmetry materialises the erosion of the target motif in the parental lineage. 
% We further subdivided the group of asymmetric hotspots into two subgroups, based on the extent of their erosion:
% either they had undergone full erosion in the parental lineage and
We further subdivided the group of asymmetric hotspots into two subgroups, based on the presence (or not) of a ChIP-seq peak in the parental strain:
either no PRDM9 ChIP-seq peak was detected in either parental strain — in that case, the hotspot had undergone full erosion in the parental lineage and we classified it as a ‘novel’ hotspot; or a PRDM9 ChIP-seq peak was detected in one of the two parents — in that case, the hotspot had only been partially eroded in the parental strain.\\

Altogether thus, we could infer both the \textit{Prdm9} allele (\textit{Prdm9\textsuperscript{Dom2}} or \textit{Prdm9\textsuperscript{Cst}}) and the level of asymmetry (symmetric, asymmetric or novel hotspot) and we used these two pieces of information to classify hotspots into six categories (Table~\ref{tab:classification-hotspots}).

All in all, 87\% (886) of our 1,018 hotspots fell into one of these categories. 
Of these, 81\% (719) were inferred to be targeted by PRDM9\textsuperscript{Cst} (322 symmetric, 241 partially eroded and 156 fully eroded) and 19\% (167) by PRDM9\textsuperscript{Dom2} (24 symmetric, 63 partially eroded and 80 fully eroded).

We note that, as most polymorphic sites in F1 hybrid hotspots result from hotspot erosion in one parental lineage \citep{smagulova2016evolutionary}, the requirement of a minimum of 4 markers in the 300-bp central region that we set to select hotspots (see Chapter~\ref{ch:5-methodology}) led to a greater proportion of asymmetric hotspots in our selection (61\%) than in the total list of 6,758 hotspots identified by \citet{baker2015prdm9} (35\%).\\




\subsection{Validation by detection of the target motifs}

% Identifying the hotspot-activating \textit{Prdm9} allele at fully eroded hotspots (i.e.\ those for which no PRDM9 ChIP-seq peak was detected in the parental strains) rested on the assumption that the differential PRDM9 binding affinity originated from motif erosion in one lineage (see Subsection~\ref{chap7:PRDM9-target-inference}).
Identifying the hotspot-activating \textit{Prdm9} allele at fully eroded hotspots (i.e.\ those for which no PRDM9 ChIP-seq peak was detected in the parental strains) was done by deduction in lieu of direct observations (see Subsection~\ref{chap7:PRDM9-target-inference}) and may thus entail errors.
Since the accuracy of the inferred \textit{Prdm9} target was critical to the posterior quantification of biased gene conversion, we wanted to make sure that our predictions were correct by verifying that occurrences of the PRDM9\textsuperscript{Dom2} (resp.\ PRDM9\textsuperscript{Cst}) target motif were found in hotspots predicted to be bound by it.

\subsubsection{Discovery of the consensus target motifs}
To do this, we first had to discover the motif targeted by PRDM9\textsuperscript{Dom2} and PRDM9\textsuperscript{Cst}.
%— which we inferred to correspond to the position of the PRDM9 ChIP-seq peak summit (see Chapter~\ref{ch:6-recombination-parameters}), 
Since the motifs targeted by PRDM9 are known to be located in the vicinity of the DSB site \citep{brick2012genetic, baker2014prdm9}, we used the 300-bp central regions of the hotspots undoubtedly targeted by each of the \textit{Prdm9} alleles to discover their consensus motif.
For the PRDM9\textsuperscript{Dom2} consensus motif, the hotspots we used were those for which a PRDM9 ChIP-seq peak had been found in the B6 strain:
the B6 and CAST haplotypes of the symmetric PRDM9\textsuperscript{Dom2}-targeted hotspots (tB.sym) and the CAST haplotype of the partially eroded PRDM9\textsuperscript{Dom2}-targeted hotspots (tB.chB).
Respectively, for the PRDM9\textsuperscript{Cst} consensus motif, the hotspots we used were those for which a PRDM9 ChIP-seq peak had been found in the CAST strain:
the B6 and CAST haplotypes of the symmetric PRDM9\textsuperscript{Cst}-targeted hotspots (tC.sym) and the B6 haplotype of the partially eroded PRDM9\textsuperscript{Cst}-targeted hotspots (tC.chC).

% The motifs targeted by PRDM9 are known to be located in the vicinity of the DSB site \citep{brick2012genetic, baker2014prdm9}, which we inferred to correspond to the position of the PRDM9 ChIP-seq peak summit (see Chapter~\ref{ch:6-recombination-parameters}).
%
% Therefore, to learn the PRDM9\textsuperscript{Dom2} consensus motif, we used the 300-bp central regions of the hotspots undoubtedly targeted by PRDM9\textsuperscript{Dom2}, i.e.\ hotspots for which a PRDM9 ChIP-seq peak had been found in the B6 strain:
% the B6 and CAST haplotypes of the symmetric PRDM9\textsuperscript{Dom2}-targeted hotspots (tB.sym) and the CAST haplotype of the partially eroded PRDM9\textsuperscript{Dom2}-targeted hotspots (tB.chB).
% Respectively, to learn the PRDM9\textsuperscript{Cst} consensus motif, we used the 300-bp central regions of the hotspots undoubtedly targeted by PRDM9\textsuperscript{Cst}, i.e.\ hotspots for which a PRDM9 ChIP-seq peak had been found in the CAST strain:
% the B6 and CAST haplotypes of the symmetric PRDM9\textsuperscript{Cst}-targeted hotspots (tC.sym) and the B6 haplotype of the partially eroded PRDM9\textsuperscript{Cst}-targeted hotspots (tC.chC).
%


\begin{figure}[b!]
	\centering
	\begin{subfigure}[b]{0.495\textwidth}
		\includegraphics[width=\textwidth]{figures/chap7/Baker_B6_motif.eps}
		% \caption{PRDM9\textsuperscript{Dom2} \citep{baker2015prdm9}}
		\caption{\textit{Prdm9\textsuperscript{Dom2}} \citep{baker2015prdm9}}
		% \label{fig:baker_B6_motif}
	\end{subfigure}
	\begin{subfigure}[b]{0.495\textwidth}
		\includegraphics[width=\textwidth]{figures/chap7/Baker_CAST_motif.eps}
		% \caption{PRDM9\textsuperscript{Cst} \citep{baker2015prdm9}}
		\caption{\textit{Prdm9\textsuperscript{Cst}} \citep{baker2015prdm9}}
		% \label{fig:baker_CAST_motif}
	\end{subfigure}

	\begin{subfigure}[b]{0.49\textwidth}
		\includegraphics[width=\textwidth, trim=0.848cm 0.485cm 0cm 0cm,clip]{figures/chap7/Motif_B6.eps}
		% \caption{PRDM9\textsuperscript{Dom2} (this study)}
		\caption{\textit{Prdm9\textsuperscript{Dom2}} (this study)}
		% \label{fig:B6_motif}
	\end{subfigure}
	\begin{subfigure}[b]{0.49\textwidth}
		\includegraphics[width=\textwidth, trim=0.835cm 0.478cm 0cm 0cm,clip]{figures/chap7/Motif_CAST.eps}
		% \caption{PRDM9\textsuperscript{Cst} (this study)}
		\caption{\textit{Prdm9\textsuperscript{Cst}} (this study)}
		% \label{fig:CAST_motif}
	\end{subfigure}
	\caption[Comparison of consensus motifs for \textit{Prdm9\textsuperscript{Dom2}} and \textit{Prdm9\textsuperscript{Cst}}]
	{\textbf{Comparison of consensus motifs for \textit{Prdm9\textsuperscript{Dom2}} and \textit{Prdm9\textsuperscript{Cst}}.}
		\par The consensus motifs for \textit{Prdm9\textsuperscript{Dom2}} (left) and \textit{Prdm9\textsuperscript{Cst}} (right) alleles found by \citet{baker2015prdm9} are reported at the top and those found in our study at the bottom.
	}
\label{fig:consensus-motifs}
\end{figure}


In practice, to search for the consensus motifs, we used the MEME motif discovery tool \citep{bailey2006meme} from the MEME Suite (version 4.11.2) \citep{bailey2009meme}, in the any-number-of-repetitions mode and allowing up to 10 motifs of width comprised between 10 and 30 bp.
For each \textit{Prm9} allele, the consensus motif we retained was the one with the lowest E-value.
We found that, in both cases, they were either identical to or the complement reverse of the ones published by \citet{baker2015prdm9} (Figure~\ref{fig:consensus-motifs}).
% We were confident in the correctness of the consensus motifs we identified for both PRDM9\textsuperscript{Dom2} and PRDM9\textsuperscript{Cst} since they were either identical to or the complement reverse of the ones published by \citet{baker2015prdm9} (Figure~\ref{fig:consensus-motifs}).
We also verified that these consensus motifs were specific to the sequences we selected: 
we searched for them in control regions defined as sequences located 5-kb downstream of those used to discover the motifs and found that the consensus motif for \textit{Prdm9\textsuperscript{Dom2}} (resp.\ \textit{Prdm9\textsuperscript{Cst}}) appeared 10 (resp.\ 7) times less in these control sequences than in the training set.
% by searching for them in control regions defined as sequences located 5-kb downstream of those used to discover the motifs: we found the consensus motif for \textit{Prdm9\textsuperscript{Dom2}} (resp.\ \textit{Prdm9\textsuperscript{Cst}}) in 10 (resp.\ 7) times more training sequences than control sequences.


\subsubsection{Occurrences of consensus motifs in the predicted hotspots}

\begin{figure}[p]
	\centering
	\begin{subfigure}[b]{0.495\textwidth}
		\includegraphics[width=\textwidth]{figures/chap7/dom_tB.eps}
		% \caption{Predicted \textit{Prdm9\textsuperscript{Dom2}}-activated.}
		\caption{Predicted PRDM9\textsuperscript{Dom2}-targeted.}
		% \label{fig:distrib_B6_motif_tB}
	\end{subfigure}
	\begin{subfigure}[b]{0.495\textwidth}
		\includegraphics[width=\textwidth]{figures/chap7/dom_tC.eps}
		% \caption{Predicted \textit{Prdm9\textsuperscript{Cst}}-activated.}
		\caption{Predicted PRDM9\textsuperscript{Cst}-targeted.}
		% \label{fig:distrib_B6_motif_tC}
	\end{subfigure}

	\begin{subfigure}[b]{0.495\textwidth}
		\includegraphics[width=\textwidth]{figures/chap7/cas_tB.eps}
		% \caption{Predicted \textit{Prdm9\textsuperscript{Dom2}}-activated.}
		\caption{Predicted PRDM9\textsuperscript{Dom2}-targeted.}
		% \label{fig:distrib_CAST_motif_tB}
	\end{subfigure}
	\begin{subfigure}[b]{0.495\textwidth}
		\includegraphics[width=\textwidth]{figures/chap7/cas_tC.eps}
		% \caption{Predicted \textit{Prdm9\textsuperscript{Cst}}-activated.}
		\caption{Predicted PRDM9\textsuperscript{Cst}-targeted.}
		% \label{fig:distrib_CAST_motif_tC}
	\end{subfigure}
	\caption[Occurrences of \textit{Prdm9\textsuperscript{Dom2}} and \textit{Prdm9\textsuperscript{Cst}} consensus motifs along hotspots predicted to be targeted by these alleles]
	{\textbf{Occurrences of \textit{Prdm9\textsuperscript{Dom2}} and \textit{Prdm9\textsuperscript{Cst}} consensus motifs along hotspots predicted to be targeted by these alleles.}
		\par Occurrences of the consensus motifs for \textit{Prdm9\textsuperscript{Dom2}} (top) and \textit{Prdm9\textsuperscript{Cst}} (bottom) were searched in the B6 (red) and CAST (yellow) haplotypes of hotspots predicted to be targeted by \textit{Prdm9\textsuperscript{Dom2}} (left) or by \textit{Prdm9\textsuperscript{Cst}} (right).
		The numbers of hotspots for which the searched motif was found in at least one haplotype were: \textbf{(a)} $N=158$, \textbf{(b)} $N=242$, \textbf{(c)} $N=78$, \textbf{(d)} $N=698$.
	}
\label{fig:motifs-in-hotspots}
\end{figure}





Next, we searched for occurrences of both these consensus motifs in the two haplotypes of each 1-kb long hotspot, using the FIMO tool \citep{grant2011fimo} with default parameters. 
When more than one occurrence of the motif was found in a given hotspot, we retained solely the motif with the highest log-likelihood ratio score.

Altogether, we found that, in hotspots predicted to be targeted by PRDM9\textsuperscript{Dom2}, the majority (88.75\%) of the haplotypes predicted to be cold (i.e.\ not targeted by PRDM9) on the basis of the strand-specific detection of PRDM9 ChIP-seq reads from \citet{baker2015prdm9} (see Subsection~\ref{chap7:PRDM9-target-inference}) indeed contained a \textit{Prdm9\textsuperscript{Dom2}} motif. 
Conversely in hotspots predicted to be targeted by PRDM9\textsuperscript{Cst}, most (87.82\%) haplotypes predicted to be cold contained a \textit{Prdm9\textsuperscript{Cst}} motif.

More precisely, the distribution of motif occurrences along both haplotypes of the hotspots predicted to be targeted by either one of the two \textit{Prdm9} alleles are reported in Figure~\ref{fig:motifs-in-hotspots}.
As expected, occurrences of the \textit{Prdm9\textsuperscript{Dom2}} consensus motif were specific to hotspots predicted to be targeted by PRDM9\textsuperscript{Dom2} and, conversely, occurrences of the \textit{Prdm9\textsuperscript{Cst}} consensus motif were specific to hotspots predicted to be targeted by PRDM9\textsuperscript{Cst}.
In particular, motifs occurred more often in the ‘nonself’ haplotype (i.e.\ the B6 haplotype for PRDM9\textsuperscript{Cst}-targeted hotspots and the CAST haplotype for PRDM9\textsuperscript{Dom2}-targeted hotspots), most assuredly because the motif had undergone erosion in its ‘self’ lineage.
Also, these motifs gathered in the close vicinity of the inferred DSB sites (i.e.\ the summits of the PRDM9 ChIP-seq peaks): 52\% of them were located closer than 60 bp away from the DSBs.




\addtocontents{toc}{\protect\pagebreak}
\section{dBGC hitchhiking of past gBGC}
% \section{Quantification of DSB-induced biased gene conversion (dBGC)}


\subsection{Direct quantification of dBGC} 


\begin{figure}[p]
    \centering
    \includegraphics[width = 1\textwidth]{figures/chap7/dBGC_distribution.eps}
    \caption[Distribution of the dBGC coefficient across categories of hotspots]
	{\textbf{Distribution of the dBGC coefficient across categories of hotspots.}
		\par The dBGC coefficient ($b_{dBGC}$) was directly extrapolated from the observed frequency of CAST-donor fragments in the pool of gametes ($x$) as follows: $b_{dBGC} = 2 \times x - 1$ \citep{nagylaki1983evolution} (see main text).
		The distribution was reported for four groups of hotspots: those targeted by PRDM9\textsuperscript{Dom2} which were either completely eroded (NOV.tB.chB, dark grey) or not (tB.chB and tB.sym, red) in the B6 lineage, and those targeted by PRDM9\textsuperscript{Cst} which were either completely eroded (NOV.tC.chC, light grey) or not (tC.chC and tC.sym, yellow) in the CAST lineage.
		The frequencies were normalised to the total number of hotspots in each category.
    }
\label{fig:dBGC-distribution}
\end{figure}

Next, we aimed at quantifying DSB-induced biased gene conversion (dBGC) for each hotspot.
Thus, we directly extrapolated the dBGC coefficient ($b_{dBGC}$) from the observed frequency of CAST-donor fragments ($x$) which we measured in Chapter~\ref{ch:6-recombination-parameters}, from the equation of \citet{nagylaki1983evolution}:

\begin{equation*}
	x = \frac{1}{2} \times (1+b_{dBGC})
	% b_{dBGC} = 2 \times x - 1
\end{equation*}


We looked at the distribution of the dBGC coefficient across four categories of hotspots (Figure~\ref{fig:dBGC-distribution}): 
on the one hand, the PRDM9\textsuperscript{Dom2}-targeted hotspots which were either fully eroded or still present in the B6 lineage; and, on the other hand, the PRDM9\textsuperscript{Cst}-targeted hotspots completely eroded or still present in the CAST lineage.
As expected, we observed that hotspots that were most eroded in the parental lineages were those for which the absolute dBGC coefficient was the greatest, while hotspots displaying a quasi-null dBGC coefficient corresponded to symmetric hotspots, i.e.\ targeted equally by PRDM9.


\subsection{dBGC and the overtransmission of \textit{GC} alleles}

After quantifying the intensity of dBGC, we examined the allelic composition of conversion tracts (CTs) to measure that of gBGC\@.
Among the 30,627 AT/GC (WS) polymorphic sites involved in the CTs of the recombination events detected, 17,876 ($58.3\%$, $CI=[57.8\%; 58.9\%]$) carried the \textit{S} (\textit{G} or \textit{C}) allele.
This proportion was slightly lower for Rec-1S ($54.8\%$, $CI=[54.1\%; 55.5\%]$) than for Rec-2S ($64.0\%$, $CI=[63.2\%; 64.9\%]$) events.


\begin{figure}[t]
    \centering
    \includegraphics[width = 1\textwidth]{figures/chap7/Proportion_GC_alleles_BIS.eps}
    \caption[GC-profiles at AT/GC (WS) polymorphic sites of PRDM9\textsuperscript{Dom2}- and PRDM9\textsuperscript{Cst}-targeted hotspots]
	{\textbf{GC-profiles at AT/GC (WS) polymorphic sites of PRDM9\textsuperscript{Dom2}- and PRDM9\textsuperscript{Cst}-targeted hotspots.}
		\par The proportion of \textit{S} (\textit{G} or \textit{C}) alleles originating from the B6 lineage for PRDM9\textsuperscript{Dom2}-targeted hotspots ($N_{hotspots} = 167$) (red curve) or from the CAST lineage for PRDM9\textsuperscript{Cst}-targeted hotspots ($N_{hotspots} = 719$) (yellow curve) was computed over 300-bp sliding windows.
    }
\label{fig:prop-GC-alleles}
\end{figure}





However, this observed transmission bias was not solely due to gBGC, but could — in part — come from dBGC\@.
Indeed, on the one hand, at PRDM9\textsuperscript{Dom2}-targeted (resp.\ PRDM9\textsuperscript{Cst}-targeted) hotspots, the B6 (resp.\ CAST) haplotype was GC-enriched in the vicinity of the DSB (Figure~\ref{fig:prop-GC-alleles}). 
Such local increase in GC-content was a clear signature of the past gBGC that occurred in the parental lineages.
Interestingly, we note that this effect was stronger in PRDM9\textsuperscript{Dom2}-targeted than in PRDM9\textsuperscript{Cst}-targeted hotspots, which could be explained by two independent reasons.
First, as suggested by \citet{smagulova2016evolutionary}, the \textit{Prdm9\textsuperscript{Cst}} allele may be younger than the \textit{Prdm9\textsuperscript{Dom2}} one and, consequently, PRDM9\textsuperscript{Dom2}-targeted hotspots may have undergone more gBGC than PRDM9\textsuperscript{Cst}-targeted hotspots.
Alternatively, considering that PRDM9\textsuperscript{Dom2}-targeted hotspots are less numerous than PRDM9\textsuperscript{Cst}-targeted ones in the B6xCAST hybrid (likely because of dominance effects), the subset of PRDM9\textsuperscript{Dom2}-targeted hotspots active in the hybrid may correspond to hotspots with particularly high PRDM9-affinity, which would thus have undergone stronger gBGC in the past lineage.




\begin{table}[b!]
	\centering
	% \begin{adjustbox}{width = 1\textwidth}
	\begin{tabular}{rrrr}


		\toprule
		& \multicolumn{3}{c}{\textbf{Donor haplotype}} \\
		\cmidrule{2-4}
		\textbf{Hotspot category} & \textbf{B6} & \textbf{CAST} & \textbf{NA} \\

		\midrule

		\textbf{\textit{PRDM9\textsuperscript{Dom2}-targeted}}\\
		\textit{tB.sym}     & 177   & 164   & 68 \\ 
		\textit{tB.chB}     & 553   & 165   & 142 \\ 
		\textit{NOV.tB.chB} & 515   & 87    & 170 \\ 

		\\

		\textbf{\textit{PRDM9\textsuperscript{Cst}-targeted}}\\
		\textit{tC.sym}     & 2400  & 3075  & 1868 \\ 
		\textit{tC.chC}     & 1404  & 4329  & 1226 \\ 
		\textit{NOV.tC.chC} & 248   & 664   & 284 \\

		\bottomrule

	\end{tabular}
	% \end{adjustbox}
	\caption[Number of B6- and CAST-donor fragments per category of hotspots]
	{\textbf{Number of B6- and CAST-donor fragments per category of hotspots.}
		\par The donor haplotype for each fragment was identified as described in Chapter~\ref{ch:6-recombination-parameters} in each of the six categories of hotspots reported in Table~\ref{tab:classification-hotspots}.
	}
\label{tab:count-donors}
\end{table}


On the other hand, for the majority (81\%, resp.\ 75\%) of conversion events occuring at PRDM9\textsuperscript{Dom2}-targeted (resp.\ PRDM9\textsuperscript{Cst}-targeted) hotspots, the B6 (resp.\ CAST) haplotype was the donor (Table~\ref{tab:count-donors}).

In summary, the haplotype which was most often the donor (due to dBGC) was also the GC-richer (due to past gBGC). 
In other words, dBGC occuring in the hybrid somehow hitchhiked the gBGC that occurred in the past lineages, thus creating a confounding effect to estimate the intensity of gBGC at a single meiotic generation.

Consequently, at this point, cancelling the the action of dBGC was absolutely critical to quantify gBGC precisely.







\subsection{Controlling for dBGC to quantify gBGC}

\begin{table}[b!]
    \centering
	\begin{adjustbox}{width = 1\textwidth}
		\begin{tabular}{rrrrrrr}


			\toprule
			\textbf{Category} & \textbf{\# S} & \textbf{\# W} & \textbf{\% S} & \textbf{CI (min-max)} & \textbf{\textit{p}-val} & \textbf{$b_0$} \\


			\midrule
			\textbf{\textit{Inside CTs\textsuperscript{$\star$}}}\\
			$Rec$-$1S$                 & 5408 & 5179 & 0.5108 & 0.5012--0.5204 & 0.0267 & 0.0216 \\
			$Rec$-$2S$                 & 2261 & 2078 & 0.5211 & 0.5061--0.5360 & 0.0057 & 0.0422 \\
			\textbf{\textit{Total}}    & \textbf{7669} & \textbf{7257} & \textbf{0.5138} & \textbf{0.5057--0.5218} & \textbf{0.0007} & \textbf{0.0276} \\
			\\
			\textbf{\textit{Outside CTs\textsuperscript{$\star$}}}\\
			$Rec$-$1S$                 & 9355 & 9433 & 0.4979 & 0.4907--0.5051 & 0.5743 & \textit{-} \\
			$Rec$-$2S$                 & 5051 & 5028 & 0.5011 & 0.4913--0.5109 & 0.8265 & \textit{-} \\
			\textbf{\textit{Total}}    & \textbf{14406} & \textbf{14461} & \textbf{0.4990} & \textbf{0.4933--0.5048} & \textbf{0.7506} & \textbf{-} \\
			\bottomrule

		\end{tabular}
	\end{adjustbox}
	\caption[Transmission of the \textit{S} alleles inside (upper board) and outside (lower board) observed conversion tracts (CTs\textsuperscript{$\star$}) after controlling for dBGC]
	{\textbf{Transmission of the \textit{S} alleles inside (upper board) and outside (lower board) observed conversion tracts (CTs\textsuperscript{$\star$}) after controlling for dBGC.}
		\par Controlling for dBGC was operated by subsampling B6-donor and CAST-donor fragments in individual hotspots (see main text). 
		The values reported in this table correspond to the results obtained after one round of random sampling epresentative of all sampling combinations.
		\# S\@: Number of \textit{S} (\textit{G} or \textit{C}) alleles in the fragments sampled. 
		\# W\@: Number of \textit{W} (\textit{A} or \textit{T}) alleles in the fragments sampled.
		\% S\@: Proportion of \textit{S} alleles in the fragments sampled ($\frac{\# S}{\# S + \# W}$).
		CI\@: 95\%-confidence interval (test of proportions).
		$b_0$: Transmission bias, calculated as $b_0 = 2 \times x - 1$, where $x$ is the mean frequency of a \textit{S} allele within a pool of gametes coming from a WS heterozygous context.
	}
\label{tab:gBGC-frequencies}
\end{table}




To control for the impact of dBGC onto the transmission bias, we equalised the number of fragments coming from B6-donor and from CAST-donor conversion events.
Concretely, we counted, for each hotspot, the total number of B6-donor ($n\textsubscript{B6}$) and of CAST-donor ($n\textsubscript{CAST}$) fragments. 
If there were fewer B6- than CAST-donor fragments (resp.\ fewer CAST- than B6-donor fragments), all B6-donor (resp.\ CAST-donor) fragments as well as a random selection of $n\textsubscript{B6}$ fragments among the $n\textsubscript{CAST}$ CAST-donor (resp.\ $n\textsubscript{CAST}$ among the $n\textsubscript{B6}$ B6-donor) fragments were retained. 
% If there were fewer B6- than CAST-donor fragments ($n\textsubscript{B6} < n\textsubscript{CAST}$), all B6-donor fragments as well as a random selection of $n\textsubscript{B6}$ fragments among the $n\textsubscript{CAST}$ CAST-donor fragments were retained.
% Conversely, if there were fewer CAST- than B6-donor fragments ($n\textsubscript{CAST} < n\textsubscript{B6}$), all CAST-donor fragments as well as a random selection of $n\textsubscript{CAST}$ fragments among the $n\textsubscript{B6}$ B6-donor fragments were retained.

To check if this simple method functioned properly, we examined the portions of the fragments located outside the observed CTs (CTs\textsuperscript{$\star$}).
By definition, gene conversion does not occur in these DNA chunks and, thus, the allelic frequencies are expected \textit{not} to depart from a 1:1 transmission ratio.
We found that the transmission of \textit{S} and \textit{W} alleles indeed abode by the Mendelian transmission of alleles (Table~\ref{tab:gBGC-frequencies}), which confirmed that our per-hotspot equalisation procedure allowed to efficiently control for the dBGC effect. 





\section{Quantification of GC-biased gene conversion}
% \subsection{Controling for dBGC to quantify gBGC}
\subsection{Null $b_0$ in COs and weak $b_0$ in multiple-marker NCOs}

After controlling for dBGC, it became possible to measure the intensity of gBGC\@.
Indeed, for both Rec-1S and Rec-2S events, the proportion of \textit{S} alleles inside CTs\textsuperscript{$\star$} ($x$) was significantly — but weakly — above 50\% (Table~\ref{tab:gBGC-frequencies}).
The transmission bias ($b_0$) could then be calculated directly for both Rec-1S and Rec-2S as ${b_0 = 2 \times x - 1}$ \citep{nagylaki1983evolution}.

Though, these estimates were not directly representative of the transmission bias in COs and NCOs, since we previously showed that, contrary to Rec-2S which exclusively comprised NCOs, Rec-1S events were composed of about 52\% of NCOs and 48\% of COs (see Chapter~\ref{ch:6-recombination-parameters}). 
Assuming that the $b_0$ for the NCOs observed as Rec-2S events was representative of the $b_0$ for the NCOs identified as Rec-1S events, we could decompose the transmission bias as such: 

\begin{equation*}
	b_0^{Rec2S} = (b_0^{Rec1S} \times 0.52) + (b_0^{CO} \times 0.48)
\end{equation*}

Using the $b_0$ values for Rec-1S and Rec-2S reported in Table~\ref{tab:gBGC-frequencies}, the latter formula resulted in $b_0^{CO}$ equalling 0. 
Therefore, COs do not contribute to gBGC in mice.

As for NCOs, their contribution to gBGC could be directly extracted from that measured on Rec-2S events: $b_0^{NCO} = 0.0422$, i.e.\ a 52.11\% transmission of \textit{S} alleles.\\

All in all thus, the transmission bias was null (or too weak to be detectable) for COs and extremely weak — albeit significant — for the NCOs we detected, i.e.\ those whose CT overlapped at least two markers.
Nevertheless, since the CTs of NCOs are extremely short (36 bp on average, see Chapter~\ref{ch:6-recombination-parameters}), a significant proportion of them should overlap only 1 marker.
Thus, we aimed at quantifying the transmission bias in single-marker NCOs as well, this time through an indirect approach that I describe in the following subsection.

\subsection{Strong $b_0$ in single-marker NCOs}

Briefly, estimating the contribution of single-marker NCO (NCO-1) events consisted in three main steps.
First, we adapted the procedure developed in Chapter~\ref{ch:5-methodology} to fish out all events which were likely to be NCO-1 events (pot-NCO-1, for ‘potential NCO-1 events’).
Second, since the large majority of them (90.0\%, Table~\ref{tab:NCO-1-FP-rate}) were false positives (FPs) and most of these FPs (66.7\%) arose from sequencing errors, we measured the sequencing error rate directly in our sequencing data to estimate, among the pot-NCO-1 events, the relative proportion of FPs displaying W\textrightarrow{} S and S\textrightarrow{} W erroneous conversion events.
With this, we could finally deconvolute the fraction of the observed ratio of $\frac{SW}{WS+SW}$ due to FPs and to genuine NCO-1 events, and thus, extrapolate the transmission bias on single-marker NCOs.

I detail these three steps in this subsection.


% - 1) on extrait les NCO-1 (qui ont de nombreux FP principalement du a erreur de seq)
% - 2) on estime le taux de WS du a erreur de sequencage seulement
% - 3) on calcule la difference avec ce qui est observe pour extrapoler le BGC dans les NCO-1

\subsubsection{Extraction of potential single-marker NCOs (pot-NCO-1 events)}

\begin{table}[b!]
	\centering
	\begin{adjustbox}{width = 1\textwidth}
		\begin{tabular}{rrrrrrrr}
			\toprule
			\multicolumn{3}{r}{} & \multicolumn{2}{c}{\textbf{Filter off}} & & \multicolumn{2}{c}{\textbf{Filter on}} \\
			\cmidrule{4-5} 
			\cmidrule{7-8} 
			\textbf{Target} & \textbf{Nb of} & \textbf{Nb of} & \textbf{Nb of} & \textbf{Event rate} & & \textbf{Nb of} & \textbf{Event rate} \\

			\textbf{category} & \textbf{targets} & \textbf{fragments} & \textbf{events} & \textbf{($\times$ 10\textsuperscript{-6})} & & \textbf{events} & \textbf{($\times$ 10\textsuperscript{-6})} \\

			\midrule
			Hotspots & 1,018 & 228,984,512 & 243,390 & 1062.9 & & 147,792 & 645.4 \\
			Controls & 500 & 106,850,906 & 110,615 & 1035.2 & & 62,074 & 580.9 \\
			\midrule
			\multicolumn{1}{r}{\textbf{FP rate}} & \multicolumn{4}{r}{\textbf{97.4 \%}} & & \multicolumn{2}{r}{\textbf{90.0 \%}} \\
			\bottomrule
		\end{tabular}
	\end{adjustbox}
	\caption[Number of pot-NCO-1 events detected in hotspot and control targets with and without the sequencing error filter]
	{\textbf{Number of pot-NCO-1 events detected in hotspot and control targets with and without the sequencing error filter.}
        \par Pot-NCO-1 events were detected with or without a sequencing error filter (see main text).
        All fragments or events overlapping at least 1 bp with a given target are counted in this table.
        The event rate corresponds to the ratio of candidate recombination events over the total number of fragments.
        The maximum false positive (FP) rate is the ratio of the event rate in control targets over that in hotspots.
    }
\label{tab:NCO-1-FP-rate}
\end{table}



The procedure we implemented in Chapter~\ref{ch:5-methodology} to identify recombination events discarded (1) genotyping errors coming from alignment defects by mapping the reads on both the B6 and the CAST reference genomes and (2) genotyping errors coming from sequencing miscalls by requiring a minimum of 2 markers.
% to confirm the genotype. 
To uncover pot-NCO-1 events, we removed this second filter and replaced it with an alternative, less efficient filter to limit the number of sequencing miscalls: we checked that the allele supporting the genotype call with the mapping onto the B6 genome was identical to that based on the mapping onto the CAST genome.
When setting this filter on, the NCO-1 FP error rate was lower than without the filter, but remained as high as 90.0\% (Table~\ref{tab:NCO-1-FP-rate}), which meant that as few as 14,766 of the pot-NCO-1 events detected within hotspots corresponded to genuine NCO-1 events.
% The NCO-1 FP error rate was as high as 90.0\% when this second filter whereas, without it, the error rate was even higher (Table~\ref{tab:NCO-1-FP-rate}).



\subsubsection{Estimation of erroneous W\textrightarrow{} S and S\textrightarrow{} W conversion events}

Quantifying gBGC comes back to measuring the $\frac{WS}{WS+SW}$ ratio.
However, since the large majority of pot-NCO-1 events corresponded to FPs, we had to distinguish the (potential) contribution of FPs to this ratio from that of genuine NCO-1 events.
In particular, this ratio may depart from the expected 50\% ratio if (1) a non-negligible proportion of FPs arise from sequencing miscalls and (2) W\textrightarrow{} S and S\textrightarrow{} W sequencing errors appear at different frequencies.\\

% \paragraph{Proportion of FPs due to sequencing errors\\}
First, we thus wanted to quantify the proportion of FPs due to sequencing miscalls.
To do this, we estimated the sequencing error rate directly in our sequencing data by monitoring the apparition of \textit{de novo} variants:
we assumed that, outside the polymorphic sites identified by variant-calling, any base call that differed from the nucleotide of the reference genome was a sequencing error and counted them to compute the conditional frequency matrix of sequencing errors\footnote{Matrix $M$ was computed based on the analysis of one chromosome (chromosome 10) for all of our 18 samples individually (because the sequencing errors may vary between the biological samples and sequencing runs). This matrix gives the probability of each erroneous base call, given the genuine nucleotide.} ($M$):

\begin{equation*}
M = \begin{bmatrix}
\Pr( A\rightarrow A \mid A) & \Pr( A\rightarrow C \mid A) & \Pr( A\rightarrow G \mid A) & \Pr( A\rightarrow T \mid A) \\
\Pr( C\rightarrow A \mid C) & \Pr( C\rightarrow C \mid C) & \Pr( C\rightarrow G \mid C) & \Pr( C\rightarrow T \mid C) \\
\Pr( G\rightarrow A \mid G) & \Pr( G\rightarrow C \mid G) & \Pr( G\rightarrow G \mid G) & \Pr( G\rightarrow T \mid G) \\
\Pr( T\rightarrow A \mid T) & \Pr( T\rightarrow C \mid T) & \Pr( T\rightarrow G \mid T) & \Pr( T\rightarrow T \mid T) 
\end{bmatrix}
\end{equation*}




\newpage


$\forall (i,j) \in \{A, C, G, T\}^2$, the number of NCO-1 FPs expected due to sequencing errors involving a genuine base $i$ mistakenly called as a $j$ base ($e_{i\rightarrow j}$) simply equalled the product of the number of central markers (i.e.\ markers \textit{not} located at the extremity of fragments) that were genuinely $i$ in $ij$ polymorphic sites ($g_{i}^{ij}$) by the conditional probability that a genuine $i$ would mistakenly be called a $j$ ($\Pr( i\rightarrow j \mid i )$):

\begin{equation} \label{eq:nb-errors}
	e_{i\rightarrow j} = g_{i}^{ij} \times \Pr( i\rightarrow j \mid i )
\end{equation}


$g_{i}^{ij}$ was not directly accessible from the data because we could not know which base calls were correctly sequenced.
Though, this number was linked to the number of central markers containing an $i$ allele and involved in a polymorphic site $ij$ ($n_{i}^{ij}$) through the following equation:

\begin{equation} \label{eq:genuine-to-called}
	n_{i}^{ig} = g_{i}^{ij} \times ( 1 - \Pr( i\rightarrow j \mid i ) ) + g_{j}^{ij} \times \Pr( j\rightarrow i \mid j )
\end{equation}


When we computed the $M$ matrix, we found that the frequency of sequencing errors was very low ($\simeq 10^{-3}$).
Thus, to approximate $g_{i}^{ij}$, we used the simplifying assumption that the frequency of wrong calls were close to zero and that of good calls close to 1:

\begin{subequations} 
	\begin{alignat}{5}
		\forall &(i,j) &{}\in{}& \{A, C, G, T\}^2 \; \backslash \: i \neq j, &{}\Pr({}& i\rightarrow j \mid i ) \simeq 0,\label{eq:assumption-low-freqs}\\
		\forall &i	   &{}\in{}& \{A, C, G, T\},							 &{}\Pr({}& i\rightarrow i \mid i ) \simeq 1
	\end{alignat}
\end{subequations}


From equation~\ref{eq:assumption-low-freqs}, equation~\ref{eq:genuine-to-called} simplified to:

\begin{equation} \label{eq:genuine-to-called-simplified}
	n_{i}^{ij} \simeq g_{i}^{ij}
\end{equation}

And, by incorporating equation~\ref{eq:genuine-to-called-simplified} into equation~\ref{eq:nb-errors}, we had:

\begin{equation*} \label{eq:nb-errors-with-only-known-parameters}
	e_{i\rightarrow j} = n_{i}^{ij} \times \Pr( i\rightarrow j \mid i )
\end{equation*}


Finally, the total number of FPs that were expected due to sequencing errors ($E$) was the total sum of each type of sequencing error: 

\begin{equation*} \label{eq:sum-all-NCOs-expected}
	E = \underset{i \neq j} {\sum_{(i, j) \in \{A, C, G, T\}^2}} e_{i\rightarrow j}
\end{equation*}




This allowed us to predict that, among the total 287,577,349 fragments overlapping 3 markers or more, 231,905 were expected to be discovered as NCO-1 FPs due to sequencing errors only.
This represented 66.7\% of the 347,652\footnote{The sequencing error estimate was calculated upon all sequenced fragments, i.e.\ before setting the sequencing error filter, and thus had to be compared to the total number of NCO-1 FPs obtained without the filter (Table~\ref{tab:NCO-1-FP-rate}).} NCO-1 FPs that we found in pot-NCO-1 events (110,615 in control regions + an estimate of 237,037 in hotspots, Table~\ref{tab:NCO-1-FP-rate}).

We further evaluated the imprecision on this percentage by calculating, for each sample individually\footnote{With the exception of the four samples which were lowly sequenced}, the ratio between the latter number of FPs expected in the sample due to sequencing errors and the total number of fragments in the sample. 
We sequentially applied the multiplier of each sample to the total number of fragments and finally determined that the proportion of FPs due to sequencing errors capped between 60 and 78\% of all FPs.\\


Therefore, the largest part (66.7\%, CI $= [60\%; 78\%$]) of FPs arose from sequencing errors.
The next step thus consisted in estimating the $\frac{WS}{WS+SW}$ ratio expected because of these sequencing errors.
To do this, we simply computed the total number of FPs containing an erroneous W\textrightarrow{} S base call ($E_{W\rightarrow S}$) and the number containing an erroneousS\textrightarrow{} W base call ($E_{S\rightarrow W}$) as follows:

\begin{align}
        E_{W\rightarrow S}&= e_{A\rightarrow C} + e_{A\rightarrow G} + e_{T\rightarrow C} + e_{T\rightarrow G}, \\
        E_{S\rightarrow W}&= e_{C\rightarrow A} + e_{C\rightarrow T} + e_{G\rightarrow A} + e_{G\rightarrow T}
\end{align}

Importantly, we found that $E_{S\rightarrow W}$ was greater than $E_{W\rightarrow S}$, i.e.\ S bases were more oftenly mistakenly sequenced as W bases than the other way round.
More precisely, we found that the $\frac{WS}{WS+SW}$ ratio expected with such FPs (i.e.\ $\frac{E_{W\rightarrow S}}{E_{W\rightarrow S} + E_{S\rightarrow W}}$) equalled 0.39.

We note that this estimate was slightly higher than the $\frac{WS}{WS+SW}$ observed in control regions (0.31), possibly because the non-negligible portion (33.3\%) of FPs that did not originate from these sequencing errors may somehow also bias the ratio.




\subsubsection{Extrapolation of the transmission bias for single-marker NCOs}

In spite of the high FP error rate, we found that the transmission of S alleles was significantly higher in hotspot regions (0.356, CI\textsubscript{95\%} $=[0.3532, 0.3594]$) than in control regions (0.317, CI\textsubscript{95\%} $=[0.3129,0.3220]$). 
Assuming that the 133,026 FPs found in hotspot regions contributed to the $\frac{SW}{WS+SW}$ ratio in the same manner as those found in control regions, and that the contribution of the remaining 14,766 genuine NCO-1 events reflected the transmission bias of NCO-1 events, we could deconvolute both phenomena as such:

\begin{equation*}
	n_{hotspots}^{NCO_1 + FP} \times r_{hotspots}^{NCO_1 + FP}  = n_{hotspots}^{NCO_1} \times r_{hotspots}^{NCO_1} + n_{hotspots}^{FP} \times r_{control}^{FP}
\end{equation*}
where $n_{j}^{i}$ corresponds to the counts of events $i$ in regions $j$, and $r_{j}^{i}$ to the observed ratio of $\frac{SW}{WS+SW}$ due to events $i$ in regions $j$.


Using this formula, we predicted that $r_{hotspots}^{NCO_1}$ equalled 0.70 (i.e.\ that $b_0^{NCO_1}$ equalled 0.40). 
This estimate was much higher than what we found for both Rec-1S and Rec-2S, but, interestingly, it was strikingly close to what had previously been found in humans \citep{halldorsson2016rate} and concorded with recent findings in mice \citep{li2018highresolution}.\\


Altogether thus, we found that the transmission bias differed tremendously between multiple-marker NCOs (NCO-2+) for which $b_0$ was extremely weak, and single-marker NCOs (NCO-1) for which $b_0$ was as high as that of humans.
Thus, the overall contribution of NCOs to gBGC depends on the relative proportion of NCO-1 and NCO-2+ events, and these must thus be estimated to finally quantify gBGC\@.








\subsection{Global estimation of $b_0$ for NCOs}

\begin{figure}[b!]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap7/prop_NCO1_according_to_divergence_real_mutational_process.eps}
	\caption[Relationship between the proportion of NCO CT markers involved in NCO-1 events and marker density]
	{\textbf{Relationship between the proportion of NCO CT markers involved in NCO-1 events and marker density.}
		\par We performed simulations to estimate the proportion of NCO-2+ and NCO-1 events given marker density, by distributing a given number of markers (x-axis) along each hotpot and counting the proportion of NCO-2+ and NCO-1 events, i.e.\ the number of NCO events whose CT overlapped at least two or strictly one marker, respectively.
	}
\label{fig:density-NCO}
\end{figure}



To estimate the overall contribution of NCOs to gBGC, we deconvoluted $b_0^{NCO}$ as the sum of the intensity of gene conversion bias in NCO-1 ($b_{0}^{NCO_{1}}$) and NCO-2+ ($b_{0}^{NCO_{2+}}$) events, weighted by the chance for a given NCO CT marker to be involved in a NCO-1 ($f^{NCO_{1}}$) or in a NCO-2+ ($f^{NCO_{2+}}$) event: 
\begin{equation*} 
	b_{0}^{NCO} = b_{0}^{NCO_{1}} \times f^{NCO_{1}} + b_{0}^{NCO_{2+}} \times f^{NCO_{2+}} 
\end{equation*}

Though, the relative proportion of NCO-2+ events increases with marker density (Figure~\ref{fig:density-NCO}), and our experimental design required a high level of heterozygosity (see Chapter~\ref{ch:5-methodology}), not representative of the real heterozygosity in mouse populations.
However, other studies have estimated the genome-wide polymorphism in wild \textit{Mus musculus domesticus} mice: using a 95\% sensitivity for SNP discovery, it was estimated to be around 0.47\% \citep{davies2015factors}, a result similar to the 0.55\% value previously found on a subset of the genome \citep{frazer2007sequencebased}.

With such SNP density, we would expect 74.75\% of NCO CT markers to come from NCO-1 events (representing 86.23\% of the NCOs overlapping at least one marker) and the remaining 25.25\% from NCO-2+ events (representing 13.77\% of the NCOs overlapping at least one marker) (Figure~\ref{fig:density-NCO}), which would result in an overall $b_0^{NCO}$ of 0.310.







\textbf{NOTES — voir avec Laurent comment quantifier le dBGC sur les hotspots, en tenant compte de l'erosion?}

