\begin{savequote}[8cm]
‘Intense selection favours a variable response to the environment… Were this not so, the world would be much duller than is actually the case.’

\qauthor{--- John B. S. Haldane, \textit{\usebibentry{haldane1990causes}{title}} \citeyearpar{haldane1990causes} }
% ‘Now, \textit{here}, you see, it takes all the running \textit{you} can do, to keep in the same place. If you want to get somewhere else, you must run at least twice as fast as that!’
% \qauthor{--- Lewis Carroll, \textit{\usebibentry{carroll1871lookingglass}{title}} \citeyearpar{carroll1871lookingglass} }
\end{savequote}

\chapter{\label{ch:3-recombination-variation}Causes and consequences of recombination rate evolution} 
%\otherpagedecoration

\minitoc{}

% INTRO precendente: pourquoi une reproduction sexuee: c'est couteux, mais on le fait — reponse: favorisé par la selection naturelle car elimine les mutations deleteres et amene des combinaisons avantageuse
%
% INTRO ici: la recombi est interessante parce qu'elle promeut deux types de brassage (CO+segregation) ca fait de la diversite genetique.
% Formation de CO donc implique d'echanger des pans entiers des chromosomes.
% ca se traduit par des rearrangements chromosomiques.
% +variation environnement?
%
% AUTRE INTRO
% la recombi est interessante parce qu'elle promeut deux types de brassage (CO+segregation) ca fait de la diversite genetique, avec les CO qui echangent des pans entiers de chromosomes.
% Diversite genetique, sur laquelle la seleciton naturelle peut agir.
% Mais, selection naturelle, depend de l'environnement / par rapport a un milieu donne.
% Ce qui veut dire que va creer de la divergence entre les especes.
% la speciation se fait a partir d'addition d'effets locaux dans les especes (cf darwin)
% Pour pouvoir observer cela, il faut deja detecter la recombinaison.
% Ensuite, on peut etudier comment ca varie au sein des genomes
% Puis, on peut comprendre les causes et consequences (PRDM9 et RedQueen) des variations entre especes, et la speciation avec.
%
% https://www.evolutionmontpellier2018.org/program-second-joint-congress-evolutionary-biology



The very mechanism of meiosis introduces genetic mixing in two separate ways.
On the one hand, the paternal and maternal chromosomes are independently re-assorted during the first meiotic division.
On the second hand, genetic content is exhanged during recombination at the points where homologues cross over (a.k.a.\ chiasmata).

This genetic diversity instilled by meiosis forms the substratum upon which, according to Charles Darwin (1809--1882) himself, new species are ultimately generated — provided that a few conditions are met:

\begin{quote}
	\textit{‘The principle, which I have designated by this term [ed.\ divergence of character], is of high importance on my theory, and explains, as I believe, several important facts. […] according to my view, varieties are species in the process of formation, or are, as I have called them, incipient species. How, then, does the lesser difference between varieties become augmented into the greater difference between species?’}
	\qauthor{--- Charles Darwin, \textit{\usebibentry{charles1859origin}{title}} \citeyearpar{charles1859origin} }
\end{quote}

As enunciated by his theory, the prequisites to the variety-to-species transition also entail ‘a severe struggle for life [which] certainly cannot be disputed’, the occurrence of ‘variations useful to any organic being’ and ‘the strong principle of inheritance’ through which ‘they will tend to produce offspring similarly characterised’.
This, — which he calls ‘natural selection’, — is therefore tightly linked to the process of meiotic recombination.

Furthermore, the notion of biological species itself, formally defined by Ernst Mayr (1904--2005) as ‘groups of interbreeding natural populations that are reproductively (genetically) isolated from other such groups’ \citep{mayr1999systematics}, rests on the ability to sexually reproduce and thus, to meiotically recombine.

The relationship between these two concepts (further developed in \citealp{felsenstein1981skepticism} and \citealp{butlin2005recombination}) is such that, in the mammalian clade, the only speciation gene discovered so far is the one that controls the localisation of double-strand breaks (DSBs) on the genome.

I will come back to this essential gene and to its impact on the evolution of recombination rate in the third section of this chapter. 
But prior to that, I will review the existing methods to detect recombination genome-wide, and the multiple layers of recombination rate (RR) variation that have been observed along genomes and across species.

% CHAP 3: 1) comment mesurer et detecter la recombi. 2) les variations dans l'espace et le temps. 3) les positions des hotspots par PRDM9 + le dBGC

% Therefore during the modification of the descendants of any one species, and during the incessant struggle of all species to increase in numbers, the more diversified these descendants become, the better will be their chance of succeeding in the battle of life. Thus the small differences distinguishing varieties of the same species, will steadily tend to increase till they come to equal the greater differences between species of the same genus, or even of distinct genera.




\section{Genome-wide detection of recombination}

\subsection{Linkage maps \textit{via} the analysis of crosses or pedigrees}
\label{chap3:linkage-maps}

The comprehension of genetic linkage by the group of Thomas Hunt Morgan (see Chapter~\ref{ch:1-history-genetics}) was the inaugural step towards the establishment of the first genetic map (a.k.a.\ linkage map) \citep{sturtevant1913linear}.
Basically, these maps abstractly represent the proportion of crossing-overs (COs) occurring between pairs of ‘genetic markers’, \textit{i.e.} polymorphic\footnote{Which presents several forms. In other words: subject to inter-individual variability.} DNA sequences located at fixed genomic positions.

Initially, genetic markers exclusively comprised genes coding for visually discernable phenotypes.
Since their relatively wide genomic spacing granted a poor resolution to detect recombination, they were eventually supplanted by other types of markers: restriction fragment length polymorphisms (RFLPs) \textit{i.e.} sequences enzymatically shortenable first used for linkage analysis by \citet{botstein1980construction}; minisatellites and microsatellites \citep{hamada1982potential} \textit{i.e.} tandem repeats of short motifs highly variable in length \citep{ellegren2004microsatellites} and widely spread in eukaryotes \citep{hamada1982novel}; and single-nucleotide polymorphisms (SNPs) \textit{i.e.} one-base sequence variations carried by at least 1\% of the population.\\

When the two parental chromosomes carry distinct alleles at these loci\footnote{Fixed position of a genetic marker on a chromosome (from the Latin word \textit{locus}: “place”)}, one can track their transmission by genotyping the markers in the descendants.
As such, the mosaic of paternal and maternal haplotypes — and thus, the positions of recombination exchange points — can be reconstituted using various statistical methods \citep[reviewed in \citealp{backstrom2009gene}]{haldane1919combination, kosambi1943estimation}.

These kindred individuals are generally obtained by crossing members of highly divergent inbred populations \citep[\textit{e.g.}][]{rowe1994maps,dietrich1996comprehensive}, one of which being, if possible, homozygous for the recessive alleles (‘test cross’) so as to disentangle the genotypes of the descendants \citep[reviewed in][]{brown2002mapping}.
Alternatively, in species that have long generation time or that cannot be manipulated genetically for ethical considerations, successive generations of existing families (a.k.a.\ pedigrees) can be examined \citep[\textit{e.g.}][]{kong2002highresolution,kong2010finescale,cox2009new}.\\

Examining large numbers of individuals allows to estimate the genetic distance (measured in ‘morgans’ (M) as a tribute to its designer) between pairs of markers: one centimorgan (cM) expresses a frequency of 1 CO every 100 meioses.
However, for high recombination frequencies (\textit{i.e.} long distances), some experiments \citep[\textit{e.g.}][]{morgan1911random,morgan1912data} showed exceptions to additivity: the genetic distance between two polymorphic sites could be smaller than the sum of their distances with an in-between marker.
Indeed, in cases of ‘double crossing-overs’ (\textit{i.e.} two COs occurring within a given interval — which is more likely in wider stretches), the two loci are inherited together.
Thus, the CO event is not detectable and, in the end, the recombination frequency is underestimated.

In addition, genetic distances are not proportional to physical remoteness, as stated by Hermann Muller (1890--1967) \citep{muller1920are} in a response to William Castle (1867--1962) who disputed the graphical representation of these maps \citep[reviewed in \citealp{vorms2013models}]{castle1919are,castle1919arrangement}:

\begin{quote}
\textit{‘[I]t has never been claimed, in the theory of linear linkage, that the per cents of crossing over are actually proportional to the map distances [ed.\ physical distances]:  what has been stated is that the per cents of crossing overs are calculable from the map distances — or, to put the matter in more mathematical terms, that the per cents of crossing over are functions of the distances of points from each other along a straight line.’}
\end{quote}

% HONNEUR PAR HALDANE
% https://books.google.fr/books?id=7J_Xjhoev7AC&pg=PA99&lpg=PA99&dq=centimorgan+honor+sturtevant+haldane&source=bl&ots=Uuy-7Jsbfa&sig=ACfU3U2HzOzVNVp3-Jj5PT3qL3Nx7WcPDw&hl=fr&sa=X&ved=2ahUKEwi_2eXTjsjhAhWRDxQKHc_DCLIQ6AEwCHoECAcQAQ#v=onepage&q=centimorgan%20honor%20sturtevant%20haldane&f=false
% https://www.sizes.com/units/centimorgan.htm
% Quote Haldane 1919
%The unit of distance is thus 100 times Morgan's unit. [page 303]
%…
%It is suggested that the unit of distance in a chromosome as defined above be termed a “morgan,” on the analogy of the ohm, volt, etc. Morgan's unit of distance is therefore a centimorgan. [page 305]
% https://www.ias.ac.in/article/fulltext/reso/016/06/0540-0550

Decades later, the complete sequencing of the \textit{Saccharomyces cerevisiae} chromosome III \citep{oliver1992complete} confirmed this statement by enabling the first direct comparison between linkage and physical maps. 
The discrepancies between the two distances legitimised the introduction of a new measurement: the estimation of recombination rates (RRs) per physical distance (expressed in cM/Mb), useful to compare RRs across genomic regions, individuals or species.\\

Altogether, linkage maps directly measure recombination occurring in the offspring and thus allow to observe differences between sexes \citep[\textit{e.g.}][]{cheung2007polymorphic,coop2008highresolution} or among individuals \citep[\textit{e.g.}][]{broman1998comprehensive}.
However, the resolution of these maps is restrained by the position of polymorphic sites and the number of meioses analysed. 
Consequently, in mammals, except for one very recent study \citep{halldorsson2019characterizing}, the resolution has remained capped at tens to hundreds of kilobases (kb) \citep{shifman2006highresolution,billings2010patterns,kong2010finescale}.
This limitation motivated the development of a population-genetic method to learn about RRs at a finer-scale: the linkage disequilibrium (LD) analysis.





\subsection{Linkage disequilibrium (LD) analysis}%(et HapMap et biais)}
\label{chap3:LD}

Populations of unrelated beings can be analysed in a fashion similar to family members since kinship (or non-kinship) only conveys a \textit{relative} sense: unrelated individuals are merely more distantly akin than traditional pedigrees \citep{nordborg2002linkage}.

Therefore, the principle remains the same for populations of unrelated individuals as for families: recombination breaks down linkage disequilibrium (LD) \citep{lewontin1960evolutionary}, \textit{i.e.} non-random associations between loci (materialised by non-random segregations of alleles), which results in the fragmentation of LD into blocks.
Reciprocally, analysing patterns of LD (\textit{i.e.} the positions of LD blocks) will allow to trace back the underlying recombination process.\\
% The underlying recombination process can be traced back by analysing patterns of LD\@.\\
% the non-random association between loci — called linkage disequilibrium (LD) \citep{lewontin1960evolutionary} — is broken down by recombination.
% Thus, analysing patterns of LD can provide insights into the recombination events that have occurred since the last common ancestor (LCA) of the population.\\

Concretely, LD can be quantified using statistics of association between allelic states at pairs of loci \citep{lewontin1964interaction,hill1968linkage} and the recombination rates (RRs) further estimated through a myriad of methods \citep[reviewed in][]{stumpf2003estimating} which basically consist in using the allelic diversity of each LD block to reconstruct the genealogy \citep[reviewed in][]{hinch2013landscape}.
Indeed, patterns of LD do not account for recombination only \citep[reviewed in][]{venn2013inferring}: they are also shaped by other forces such as population history \citep{golding1984sampling}, mutation \citep{calafell2001haplotype} (though easily distinguishable from recombination \citep{hudson1985statistical}), natural selection \citep{barton2000genetic} and drift \citep{charlesworth1997effects}. 
% To estimate RR accuratley from LD, it is therefore necessary to take the latter effects into account by modelling the underlying genealogical history of the population \citep{stumpf2003estimating}.\\
% If the latter effects are not taken into account, RR cannot be estimated accurately from LD\@.
% Therefore, it is necessary to model the underlying genealogical history of the population to correct for the latter effects \citep{stumpf2003estimating}.\\
% A too simplistic model of the relationship between LD and RR would fail to estimate RR accurately.
% For RR to be estimated accurately, these effects must be distinguished from recombination by additional tests \citep{hudson1985statistical} or taken into account when modelling the genealogy.\\
% — even if some of these forces are distinguishable from recombination through additional tests \citep{hudson1985statistical}.
% Such modelling of the underlying tree is necessary to avoid caveats with a simplistic model of the relationship between LD and RR\@.
Modelling the underlying genealogical history of the population therefore allows to take the latter effects into account and thus, to estimate RR accurately from LD patterns \citep{stumpf2003estimating}.\\

Recombination events have been inferred by LD analysis in a plethora of mammalian orders including Artiodactyla \citep{farnir2000extensive,mcrae2002linkage,nsengimana2004linkage}, Carnivora \citep{menotti-raymond1999genetic,sutter2004extensive,verardi2006detecting}, Lagomorpha \citep{carneiro2011genetic}, Rodentia \citep{brunschwig2012finescale}, Perissodactyla \citep{corbin2010linkage, mccue2012high} and Primates \citep{auton2012finescale}.
Though, the resolution of recombination events is greatest in humans, where it has reached 1 to 2 kb \citep{theinternationalhapmapconsortium2007seconda, hinch2011landscape, the1000genomesprojectconsortium2015global}.
Such precision arises from the fact that there have had many oppportunities for recombination to take place between the last common ancestor (LCA) of a population of unrelated beings and its studied descendants.
Since recombination decreases LD at every generation \citep{slatkin2008linkage}, the more ancient the LCA, the shorter the LD blocks and thus, the higher the resolution.

However, the recombination events identified with LD analysis sum up the whole recombination process that has occurred since the LCA\@: historic recombination, rather than current recombination, is uncovered.
In addition, LD studies give a population average of recombination, with no possibility to extricate sex-specific nor individual recombination events.
Third, both LD studies and linkage maps allow the detection of COs, but not NCOs.

Another method, — sperm-typing, — solves the three aforementioned caveats: it provides fine-scale mapping of current CO and NCO recombination events in separate individuals.





\subsection{High-resolution sperm-typing studies}%genome-wide genotyping: high-resolution studies}

Sperm-typing consists in analysing the transmission of recombination events directly in the sperm of an individual.
This was made possible by the development of a polymerase chain reaction\footnote{Molecular biology method used to make copies of a specific DNA fragment.} (PCR) method allowing to genotype single diploid and haploid cells \citep{li1988amplification}.
Since PCR only allows the copy of size-limited DNA sequences and cannot be performed automatically, sperm-typing cannot be applied genome-wide \citep{coop2008highresolutiona}, unless a microfluidic device is used \citep{fan2011wholegenome,wang2012genomewide}.
Instead, sperm-typing is generally restricted to regions of high recombinational activity inferred from linkage or LD maps (see Sections~\ref{chap3:linkage-maps}~and~\ref{chap3:LD}).\\

It can be applied either to single gametes or to total-sperm DNA \citep[reviewed in][]{arnheim2003hot}.
In single-sperm typing, the PCR is performed on the lysed sperm of an individual gamete with the use of pairs of primers\footnote{Short single-stranded nucleic acid used to initiate DNA synthesis.} flanking two polymorphic markers at the extremities of the locus of interest \citep{cui1989singlesperm,lien1993simple}.
This \textit{modus operandi} has soon been used to construct linkage maps on highly recombining regions \citep{schmitt1994multipoint,lien2000evidence,cullen2002highresolution} while others \citep{tusie-luna1995gene, jeffreys1998highresolution, jeffreys2001intensely, guillon2002initiation} have used the alternative approach with total-sperm DNA that requires allele-specific PCR to capture and amplify recombinant molecules \citep{wu1989allelespecific}.

In both cases, the precise CO exchange point can be mapped using the genetic markers internal to the selected locus.
%% NOTE A MOI-MEME: voir la figure donnee dans these de Hinch — figure 1.10 (qui vient de May 2007) pour m'en inspirer quand ferai la figure sur toutes ces methodes.
Sperm-typing thus offers the best resolution for recombination exchange points since it is only limited by SNP density — a resolution even sufficient to detect the difficult to access NCOs that only affect a few markers \citep{hellenthal2006insights}, as in \citet{tusie-luna1995gene} and \citet{guillon2002initiation}.

However, even though some authors have managed allele-specific PCR in pooled ovaries \citep{guillon2005crossover, baudat2007cis} and single oocytes \citep{cole2014mouse}, it is almost exclusively used to the study of male products of meiosis.\\

The three methods described so far allow to detect the outcome of the recombination process: COs (and NCOs in the case of sperm-typing).
To get insights into other stages of the recombination process, one can use chromatin-immunoprecipitation (ChIP) of proteins involved in a given recombination stage (see Chapter~\ref{ch:2-recombination-mechanistics}) to crosslink them on their DNA binding sites, followed by the identification of bound DNA sequences either with a microarray (ChIP-chip) or by direct sequencing of the fragments (ChIP-seq) \citep[reviewed in][]{park2009chipseq}. 
The sites of recombination initiation have been identified by using this technique with SPO11 proteins in yeasts \citep{gerton2000global,mieczkowski2007loss,pan2011hierarchical} and mice \citep{lange2016landscape} and the repair sites with RPA proteins in yeasts \citep{borde2009histone} and RAD51 and DMC1 proteins in mice \citep{smagulova2011genomewide,brick2012genetic}.
Alternatively, sites of recombination initiation have been mapped by analysing the enrichment of single-stranded DNA (ssDNA) in yeasts \citep{blitzblau2007mapping,buhler2007mapping} and mice \citep{khil2012sensitive}.
These methods do not rely on the existence of polymorphic markers and, therefore, allow a resolution of a few hundreds of kb (\textit{i.e.} the length of the sequenced fragments).\\


All these approaches have contributed to a better understanding of recombination genome-wide.
In particular, it was soon understood that recombination events cluster into 1—2-kb\footnote{In mammals. But, in yeasts, recombination hotspots span several kilobases.} regions called ‘recombination hotspots’.%, \textit{i.e.} regions highly active in recombination flanked by cold sequences suppressed for recombination.
The first mammalian hotspots was discovered in the H2 region of mouse chromosome 17 \citep{steinmetz1982molecular} and the first human hotspots in \textgreek{β}-globin \citep{chakravarti1984nonuniform} and insulin \citep{chakravarti1986evidence} regions. Since then, the list of recognised hotspots have grown extensively \citep[reviewed in][]{arnheim2007mammalian,paigen2010mammalian} \textbf{mettre dans un supp la liste des hotspots humains et souris etudies + ref si temps de le faire en juin.}
% Transition: dire que ces hotspots permettent d'etudier la difference entre les differentes especes ou individus



% + liste de tous les hotspots analyses chez les humains (ou pour moi, plutot souris) dans annexe B de Popa.

% POUR APRES QUAND PARLE DES RECOMBINATION HOTSPOTS
% single-sperm dans recombination hotspots \citep{hubert1994high,schneider2002direct}.





\section{The landscape of recombination}
\subsection{The non-random distribution of crossing-overs (COs)}

The number and the distribution of crossing-overs (COs) along the genome are subject to a tight regulation \citep[reviewed in][]{jones1984control, jones2006meiotic}: setting a minimum number of COs (‘CO assurance’), evenly spacing the COs (‘CO interference’), maintaining the proper number of COs when that of DSBs decreases (‘CO homeostasis’) \textbf{and using the homologue rather than the sister chromatid}.

% On distingue trois types d’interférence (Figure I.8) :
% (i) L’interférence des DSB : l’induction artificielle d’un DSB semble réduire le taux de DSB aux loci voisins [Robine et al., 2007].
% (ii) L’interférence des CO : la formation d’un CO dans une région réduit la probabilité d’apparition d’un autre CO dans un voisinage pouvant excéder 100 Mb chez les Mammifères par exemple [Muller, 1925, de Boer et al., 2007]. Cette interférence est présente chez la plupart des espèces eucaryotes analysées jusqu’à maintenant. [Székvölgyi & Nicolas, 2010].
% (iii) L’homéostasie des CO : elle permet un baisse du nombre de DSB en tout en garantissant la présence de CO qui se fait alors aux dépends des NCO dans une région donnée [Martini et al., 2006].
%






\subsubsection{Crossing-over assurance (COA), or the ‘obligatory crossing-over’}
Together with sister chromatid cohesion, COs hold the homologous chromosomes joint until anaphase I \citep[reviewed in][]{roeder1997meiotic} and are therefore essential to the proper disjunction of bivalents.
Accordingly, in most sexually-reproducing organisms, the total number of COs ranges between one per chromosome and one per chromosome arm, irrespective of chromosome length \citep{pardo-manueldevillena2001recombination,dumas2002chromosomal,hillers2003chromosomewide,dumont2017variation}. 
The sexual chromosomes also comply to this phenomenon since they systematically have one CO on their pseudoautosomal region (PAR).
However, this ‘obligatory CO’ rule suffers an exception: \textit{Drosophila melanogaster} does not display any CO on its tiny 4\textsuperscript{th} chromosome nor, in certain cases, on its X chromosome \citep{orr-weaver1995meiosis, koehler1998human}.

In \textit{Caenorhabditis elegans}, crossing-over assurance (COA) is so strong that only one DSB per pair of chromosome suffices to guarantee a CO \citep{rosu2011robust}.
Nevertheless, forming only one DSB may be uncommon since their number and position is also under tight control, at least in yeasts \citep{wu1995factors,fan1997competition,robine2007genomewide,anderson2015reduced}: the formation of a DSB reduces the probability for another one to form nearby. 
This phenomenon is called ‘interference’ and also applies to COs.



\subsubsection{Crossing-over interference (COI)}

Early studies on recombination \citep{sturtevant1915behavior,muller1916mechanism} have shown that, when more than one CO appears on a given chromosome, the chiasmata they form tend to be evenly spaced \citep{jones1967control, jones1974correlated, jones1984control,jones2006meiotic}.
Indeed, the formation of a CO in a given region decreases the probability for another one to occur nearby \citep{vanveen2003meiosis,hillers2004crossover} — the physical length of prophase chromosomes, rather than the genomic (bp) or genetic (cM) distance, being the primary parameter \citep{zhang2014crossover,wang2015meiotic}.
So far, COI has been observed in \textit{Arabidopsis thaliana} \citep{drouaud2007sexspecific}, \textit{Homo sapiens} \citep{laurie1985further,broman2000characterization} and \textit{Mus musculus} \citep{lawrie1995chiasma,anderson1999distribution,broman2002crossover}.\\

% This so-called ‘CO interference’ (COI) ensures the proper disjunction of homologous chromosomes \citep{bishop2004early}.
The mechanism of COI remains unclear but several models have been proposed \citep[reviewed in][]{youds2011choice}.
One early hypothesis, — the polymerisation model, — posits that the completion of a CO induced the polymerisation of an inhibitor of recombination in its vicinity, thus preventing the formation of adjacent COs \citep{maguire1988crossover,king1990polymerization}.
According to another one, — the stress model, — axis buckling converts the recombination intermediate into a CO, and this mechanical tension is released nearby established COs, thus making neighbouring DSBs repair into NCOs \citep{borner2004crossover,kleckner2004mechanical}.

The most recent pieces of evidence point that, in mice, COI may operate in two consecutive steps: at late zygotene and at pachytene \citep{boer2006two}.
And, even though correlations between the length of the synaptonemal complex (SC) and interference have been reported \citep{sym1994crossover,lynn2002covariation,petkov2007crossover}, others have found that COI does not depend on the SC \citep{deboer2007meiotic,shodhan2014msh4}, which suggests that COI operates after SC formation: either prior to single-end invasion (SEI) \citep{hunter2001singleend, bishop2004early} or during the stabilisation of the SEI \citep{shinohara2008crossover}.\\
% Since COI does not depend on the synaptonemal complex (SC) \citep{deboer2007meiotic}, it certainly operates after its formation: either prior to single-end invasion (SEI) \citep{hunter2001singleend, bishop2004early} or during the stabilisation of the SEI \citep{shinohara2008crossover}.

Whatever the mechanism at play, it may have a role in controlling the outcome of the repair\footnote{This is a non-documented, personnal suggestion. Most (if not all) papers that discuss COI point that COs formed \textit{via} the MUS81 pathway are non-interferent while those formed \textit{via} the DSBR pathway are interferent, which implies that interference is a property emerging from the repair pathway. I propose to think the other way round: the mechanism that ensures COI by checking for the proximity with other COs could arbitrate which repair machinery gets recruited.} (e.g.\ by preferentially recruiting the MUS81 repair machinery).
Indeed, the COs formed \textit{via} the DSBR pathway comply to COI whereas those repaired \textit{via} the MUS81 pathway do not \citep{santos2003mus81,kohl2013meiotic}.
In particular, neither \textit{Saccharomyces pombe} for which all COs depend on the MUS81 pathway \citep{munz1994analysis,hollingsworth2004mus81,cromie2006single} nor \textit{Aspergillus nidulans} which lacks SC \citep[reviewed in \citealp{shaw1998meiosis} and \citealp{egel1995synaptonemal}]{strickland1958analysis} show CO interference.
As for NCOs, they are not subject to interference either \citep{malkova2004gene,baudat2007regulating,miller2016wholegenome} but their formation is undoubtedly promoted by COI to downregulate the number of COs \citep{rockmill2003sgs1,youds2010rtel1,crismani2012fancm,seguela-arnaud2015multiple}.

% Besoin de dire les roles ?
% Two, not necessarily mutually exclusive, roles have been proposed for crossover interference: it may be necessary for the correct formation of the SC [Bo ̈rner et al.,
% 2004] and/or to ensure proper disjunction by causing chiasmata to be placed such that the homologs are adequately held together prior to anaphase I, i.e., enough cohesive force between sister chromatids is transmitted through to the homologs [van Veen and Hawley, 2003]. It is notable that species that do not form SC, such as Aspergillus nidulans and S. pombe, also do not exhibit crossover interference [Egel, 1995].



\subsubsection{Crossing-over homeostasis (COH)}
Even though it has been disputed \citep{shinohara2008crossover}, the mechanism that ensures COI may be responsible for another level of regulation: crossing-over homeostasis (COH) \citep[reviewed in \citealp{youds2011choice}]{joshi2009pch2,zanders2009pch2delta}.
COH promotes the formation of COs at the expense of NCOs when fewer DSBs than the wild-type level are generated.
This phenomenon was first observed in \textit{Saccharomyces cerevisiae} \citep{martini2006crossover,chen2008global}, but also exists in \textit{Caenorhabditis elegans} \citep{yokoo2012cosa1,globus2012joy}, \textit{Drosophila melanogaster} \citep{mehrotra2006temporal} and \textit{Mus musculus} \citep{cole2012homeostatic}.




\subsubsection{Preference for the homologue over the sister chromatid in DSB repair}

So that the homologous chromosomes disjoin properly, a fourth regulatory level applies to the repair of DSBs into COs: the promotion of interhomologue CO over intersister repair.
In mitosis, the sister chromatid is always favoured \citep{kadyk1992sister,bzymek2010double}, whereas evidence in \textit{Saccharomyces cerevisiae} suggests that two thirds \citep{goldfarb2010frequent} or nearly all \citep{pan2011hierarchical} DSBs are repaired using the homologue.
Therefore, the template choice is regulated differently in these two processes \citep{andersen2010meiotic}.

The proteins that play a role in homology search seem to be the adequate candidates for this endeavour \citep[reviewed in][]{youds2011choice}.
Indeed, in \textit{Saccharomyces cerevisiae}, the phosphorylation of HOP1 (mouse homologue: HORMAD1) triggers a mechanism that prevents the repair of DSBs using the sister chromatid \citep{niu2005partner} and inhibits RAD51, thus leaving the homology search to DMC1 \citep{niu2009regulation}, which promotes interhomologue recombination more efficiently than RAD51 \citep{schwacha1997interhomolog}.
Any remaining DSBs can be repaired with the sister chromatid, using the indispensable BRCA1 protein \citep{adamo2008brc}.
Cohesins and components of the SC are also seemingly implicated in template choice \citep[reviewed in \citealp{pradillo2011template}]{couteau2004component,kim2010sister}.

Interestingly, the proportion of interhomologue and intersister COs can vary: in regions dense in DSBs, intersister repair is favoured whereas in less DSB-dense regions, interhomologue repair is favoured \citep{hyppa2010crossover}.





\subsection{Intragenomic patterns of variation}

% CAPILLA — variations — https://www.karger.com/Article/FullText/452822
% Mammalian Meiotic Recombination: A Toolbox for Genome Evolution
% Capilla L.a · Garcia Caldés M.a, b · Ruiz-Herrera A.a, b



% Spatial distribution of DSBs: Lam and Keeney 2015
% CHAPITRE 3: Revue de Massy initiation recombinaison parle des cartes de CO chez plein d'especes + les differences de recombinaison selon les regions genomiques

%%% Noor: dans partie variation entre les especes
%%% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3242630/

% CHAPITRE 3
% BAUDAT MASSY 2013
% Aussi des hotspots (chapitre 3) - depuis Baudat et de Massy CO
% CO are not randomly distrib- uted, but occur in multiple specific regions of the genome called CO hotspots (de Massy 2003, Kauppi et al. 2004). A hotspot is a region 1Y2 kb wide where CO are clustered, as a result of localized initiation events (see below). The average spacing between hotspots is 50Y100 kb and hotspot activities vary over three to four orders of magnitude, from 0.9 to 0.0005 cM as determined in the human genome. In the human genome the number of hotspots is estimated around 25 000Y50 000 (Myers et al. 2005). The variations of hotspot density and activity along chromosomes result in domains with high (jungles) or low (desert) recombination activity. Most sub-telomeric regions are recombination jun- gles in male meiosis whereas centromeric regions are recombination deserts. However, the determinants of CO variation along chromosomes are not known, even though some factors correlated with CO density are beginning to be analyzed (Buard & de Massy 2007, TIG in revision).
%



%%%%%% chap3: DSBs: In humans and mice, DSBs tend to avoid centromeres and heterochromatic acrocentric short arms, although they are heavily enriched near telomeres in human males [17]. % ALTEMOSE

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3242630/
Determinants and correlates of RR variation and how maintained within and among species


parler aussi des consequences sur la genetic diversity avec le DNMs.

% More generally, knowing how recombination in uences DNA sequence diversity is essential for understanding the history of human populations and the dynamics of genome evolution [Pa ̈a ̈bo, 2003; Lohmueller et al., 2009]. These applications require high resolution mapping of recombination events and genome-wide maps of recombination frequencies, known as genetic maps.



\subsection{Inter-individual effects of sex and age}

% McVenn???
% Utile pour correlation MALE/FEMELLE (cf partie 2.3)
deCODE genetic map (one pedigree)
% A pedigree study of particular interest in this thesis is the deCODE pedigree genetic map. Kong et al. (2010) genotyped a large cohort of Icelandic parent-offspring pairs (8850 maternal meioses, 6507 paternal meioses), resulting in sex-specific maps esti
% mated at a resolution of tens of kilobases. 
In order to analyse such a large sample, recombination events were identified by phasing both parent and offspring individually based on extensive IBD-sharing within the Icelandic population and then identifying the signal of recombination events where the offspring switched between paternal and maternal haplotypes (later formalised by (Palin et al., 2011)). 
The high resolution of the map enabled the first fine-scale analysis of the structure of sex-specific variation at genome features. 
Male-specific and female-specific maps had a correlation of 0.659 (10 kb scale), but there were clear sex differences. Hotspots were defined as intervals where the estimated level of recombination was greater than 10 times the genome average. Of the 4762 male-specific hotspots identified, 704 were estimated to have a rate less than 3 in female (relative to the genome average across bins). Similarly, 14.7% of hotspots in females were observed only in females, however the locations of hotspots with moderate intensity overlapped on average. The most dramatic sex- specific differences were observed at genes, especially for intronic rates (male - female = 0.270, p = 2.2 × 10−7; Figure 1.11). The deCODE data confirmed previous ob- servations from sex-averaged LD-based maps (Myers et al., 2005), in particular rates
% were suppressed within transcribed regions.



https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3242630/
Determinants and correlates of RR variation and how maintained within and among species

Parler aussi de l'effet sur la diversite genetique
CO assurance and interference
Le long des genomes (compartiments genomiques — telomeres, centromeres, intergenique) + local chromatin accessibility + parler des hotspot?
Male femelle
%%%%%% chap3: DSBs: In humans and mice, DSBs tend to avoid centromeres and heterochromatic acrocentric short arms, although they are heavily enriched near telomeres in human males [17]. % ALTEMOSE

% CO interference + obligatory CO: lien avec CO interferents par Mlh1 et CO non interferents par Mus81 + au moins un dans le PAR
% DANS CROSSING-OVER: generate genetic diversity along with independent assortment (http://www.macmillanhighered.com/BrainHoney/Resource/6716/digital_first_content/trunk/test/hillis2e/hillis2e_ch07_5.html)


% CHAPITRE 3
% BAUDAT MASSY 2013
% In addition, the number of SPO11-dependent DSBs formed in every meiocyte is regulated by several factors. In contrast to crossovers, meiotic DSBs are difficult to quantify, especially because they are transient. Counting the foci formed by the recombinases RAD51 and DMC1 (see section below) has been widely used as the best available estimate of the number of DSBs in individual cells (oocytes or spermatocytes). It gives a count of 200–400 foci per cell in mice and humans3. A method for comparing the relative level of meiotic DSBs between mice of different genotypes was recently developed by quantifying the global level of oligonucleotides that are covalently attached to SPO11 (REF. 65). The role of the ataxia telangiectasia mutated (ATM) kinase as a nega- tive regulator of meiotic DSBs has been demonstrated using this method66. Situations in which SPO11 is unable to generate a wild-type number of DSBs correlate with defects in synapsis, suggesting that a minimum number of DSBs is needed to allow proper interactions between homologues67,68.
%


% Aussi des hotspots (chapitre 3) - depuis Baudat et de Massy CO
% CO are not randomly distrib- uted, but occur in multiple specific regions of the genome called CO hotspots (de Massy 2003, Kauppi et al. 2004). A hotspot is a region 1Y2 kb wide where CO are clustered, as a result of localized initiation events (see below). The average spacing between hotspots is 50Y100 kb and hotspot activities vary over three to four orders of magnitude, from 0.9 to 0.0005 cM as determined in the human genome. In the human genome the number of hotspots is estimated around 25 000Y50 000 (Myers et al. 2005). The variations of hotspot density and activity along chromosomes result in domains with high (jungles) or low (desert) recombination activity. Most sub-telomeric regions are recombination jun- gles in male meiosis whereas centromeric regions are recombination deserts. However, the determinants of CO variation along chromosomes are not known, even though some factors correlated with CO density are beginning to be analyzed (Buard & de Massy 2007, TIG in revision).
%


%% INTERFERENCE
% Cf lesecque revue + POPA Un paragraphe tres complet
% + dans Baudat 2007 et 2013: interfering par dHJ, non-interfering Ms81.
% Pas d'interference (ou peu? Baudat CO) dans les NCO

%% INTERFERENCE (CAPILLO)
% Since only a small fraction of DSBs are eventually processed as COs, a highly regulated genetic control determines both CO homeostasis and chromosomal distribution. In this way, if a CO occurs in a certain position, the probability for a new CO to take place nearby increases with chromosomal length. As a consequence, COs tend to follow an evenly spaced distribution across chromosome axes [Jones, 1967; Kleckner et al., 2003; Wang et al., 2015]. Importantly, this CO interference is influenced by the physical distance along the chromosomal axes (micrometers) rather than the genomic (Mb) or genetic distance (cM) [Wang et al., 2015]. However, not all COs are subject to interference, leading to recognition of interfering (class I) and non-interfering (class II) COs in different organisms [Hollingsworth and Brill, 2004; Phadnis et al., 2011]. Non-interfering COs are Mus81-Mms4 dependent and distribute themselves randomly along the chromosomes independent of each other, whereas interfering COs have been found to be distributed according to a gamma distribution. In mice, most COs manifest interference and are controlled by proteins Msh4-Msh5 [Berchowitz and Copenhaver, 2010], although some Mus81 activity has been detected during meiosis [Holloway et al., 2008]. Despite the evolutionary rationale of CO interference is still unknown, spaced COs ensure faithful chromosomal segregation and might facilitate linkage of functionally related genes [Wang et al., 2015 and references therein].

% OBLIGATORY CO
% These cytological analyses also show that at least one CO is formed per chromosome arm both in humans and mice, apart from short heterochromatic arms from acrocentric chromosomes. This regulation of CO frequency is a manifestation of the rule of the obligatory CO. Interestingly, mouse strains carrying Robertsonian translocations have two CO per chro- mosome, one per euchromatic arm (Dumas & Britton- Davidian 2002). The rule of the obligatory CO is also observed between the X and Y, where one CO is always observed in the pseudoautosomal region. A second level of CO regulation is shown by the
% measures of distances between chiasmata or Mlh1 foci, or by genetic distances. These show that, both in mouse and human, CO are not randomly distributed and are more evenly spaced than expected if they occurred independently, a phenomenon defined as positive interference (Lawrie et al. 1995, Laurie & Hulten 1985, Anderson et al. 1999, Broman & Weber 2000, Broman et al. 2002).
%

% Voir aussi Neil Hunter: CO control

% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003294/ : CO control (Yanowitz)

% CHAPITRE 3
% An additional checkpoint that monitors whether each chromosome has received a crossover is thought to exist, although molecular insight into this checkpoint is lacking[84].
% Depuis Yanowitz
% % 84. Mehrotra S, McKim KS. Temporal analysis of meiotic DNA double-strand break formation and repair in Drosophila females. PLoS Genet. 2006;2:e200. [PMC free article] [PubMed] [Google Scholar]
%
%
% Aussi: il faudra parler du fait que, chez les femmes, il y a d'autres DSB qui se forment de facon non programmee. Et sont repares differemments (en particulier, plus longs + plus complex? voir chapitre 3). Plus, des problemes de trisomie plus souvent chez les femmes a cause du dictyate arrest.
%







% - CO SC independent: Kohl and de los Santos
% A mettre plus loin quand on parle des CO ET DE LA POSITIVE INTERFERENCE??
% Loidl
% which stabilizes the connection between partner chromosomes (151). It is also crucial
% for the maturation of a certain class of COs, the number and distribution of which are controlled
% by mutual local exclusion (positive interference).
% This CO class is predominant in most eukaryotes studied so far, but a small proportion of COs are SC independent and noninterfering (see 29, 66).
% 66. Kohl KP, Sekelsky J. 2013. Meiotic and mitotic recombination in meiosis. Genetics 194:327–34
% 29. de los Santos T, Hunter N, Lee C, Larkin B, Loidl J, Hollingsworth NM. 2003. The Mus81/Mms4
% endonuclease acts independently of double-Holliday junction resolution to promote a distinct subset of
% crossovers during meiosis in budding yeast. Genetics 164:81–94

% SUR LES CROSSOVERS (CLASSES)
% Hunter - synaptonemal complexities
% Recent data suggest that two classes of crossovers form in S. cerevisiae (and probably plants and mammals; e.g., de Los Santos et al., 2003). The majority, Class-I, is subject to distribution controls and dependent on Zip1 and at least five other meiosis-specific proteins (Börner, V., Kleckner, N. and N. Hunter, unpublished data). Loss of Class-I crossovers causes chromosomes to missegregate in most meioses, inferring a critical function for these events, and thus SC, in S. cerevisiae. In contrast, Class-II crossovers are randomly distributed and SC independent. Loss of these events does not affect SCs or cause chromosomes to missegregate.




%
% \section{Variation}
% isochores
% reparition variation des CO
% differences within species
% differences between male and females
% genomic and environmental
% differences between species
% evolutionary effects of recombination rate variation on structure of genome (ducleotide diversity and divergence, distinguish scenarii, codon bias, other)
% broad scale rate and PAR
% CO assurance + interference
% intragenomic variation in recombination and natural selection
% Distribution of meiotic recombination (broad scale, fine scale, PRDM9)
% role of genetic variation
%
%

\section{Evolvability of recombination rates (RR)}
\subsection{Inter- and intra-species comparison of fine-scale RR}
\subsection{PRDM9, the fast-evolving mammalian speciation gene}
\subsection{The Red Queen dynamics of hotspot evolution}
hotspot paradox et dBGC

PRDM9 (role hybrid sterility speciation)
positive sel prdm9
%exemple de l'un qui a perdu PRDM9 mais qui n'est pas stérile pour autant.

% \section{Evolution}
% PRDM9 (role hybrid sterility speciation)
% hotspots and evolution
% positive sel prdm9
% Rapid evolution of the landscape







% GENERAL

% Troisieme chapitre:
% Methodological approaches to study recombination
% Variation of recombination rates wtithin genomes and among species
% evolvability of recombination rates
%
% Plan Mammalian Meiotic Recombination: A Toolbox for Genome Evolution (https://www.karger.com/Article/FullText/452822):
% \begin{itemize}
%     \item Recombination and he repair of DSBs (Organization of meiotic chromosomes: importance of chromosomal axes, molecular events involved in he formation and repair of DSBs)
%     \item Methodological approaches to he study of recombination
%     \item Genetic and epigenetic marks of DSBs and recombination hotspots
%     \item Variation of recombination rates within genomes and among species (Variability at the chromosomal level, variation of fine-scale recombination maps)
%     \item Evolvability of recombination rates (Chromosomal rearrangements as recombination modifiers)
% \end{itemize}
%
% Plan de Hotposts for initiation of meiotic recombination (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6237102/)
% \begin{itemize}
%     \item Defining DSB hotspots
%     \item Chromatin shapes the meiotic DSB landscape (Nucleosome occupancy, meiotic chromosome architecture)
%     \item Meiotic DSB and crossover distributions
%     \item PRDM9 and H3K4me3
%     \item The hotspot paradox
%     \item Recombination initiation in repetitive sequences
%     \item Byond hotspots: DSB-dependent spatial regulation
% \end{itemize}
%
% Autre:
% \begin{itemize}
%     \item Recombining without hotspots (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4684701/)
%     \item Knockout of PRDM9 (http://science.sciencemag.org.inee.bib.cnrs.fr/content/352/6284/474)
% \end{itemize}








%%% THESES DES AUTRES
% these darrier - partie recombinaison et DL
% methodes detection
% repartition et variation des CO
% points chauds de recombi
% evolution des points chaud
% DL
%
% 2 theses droso rate variation Fine scale recombination variation in Drosophila melanogaster
% many levels of varation
% diff species (between within)
% sex-biased recombi
% genomic and environmental variation
% +
% evolutionary effects of recombination rate variation on structure of genome (ducleotide diversity and divergence, distinguish scenarii, codon bias, other)
%
%
% odenthal:
% rapid evolution of the landscape in human genomes
%
%
% these prdm9 bovins
% diversite de prdm9 (altemose)
%
%
% Hinch: Detect and measure
% sperm assays
% linage of alleles and detection in pedigrees
% genetic maps using LD
% immunoprecip followed by sequencing
%
% Hinch: localisation, control and evolution of recombination
% CO assurance + interference
% hotspots of recombination
% evolution of hotspots
% role of genetic variation
% differences between male and females
% broad scale rate and PAR
%
% Hussin:
% intragenomic variation in recombination and natural selection
% detecting
% patterns of variation in humans
%
% Lesecque: dynamique spatiale et temporelle de recombi et BGC
% mesurer recombi
% taux de recombinaison le long des genomes
% dynamique temporelle des points chauds
%
% POPA
% Genetic marker
% Genetc maps (ordering markers, genetic distance to calculate, sex average VS sex-specific)
% LD (quantify LD, hapmap, biases with LD)
% sperm-typing
% gene conversion rate
%
% VENN
% Distribution of meiotic recombination (broad scale, fine scale, PRDM9)
% Identify recombination events (pedigree + DeCODE, LD, count, models of recombination (approximate likelihood, approximate genealogy, african american admixture), typing in single gametes)
%
%





















%%%% NOTES CHAPITRE PRECEDENT




% CHAPITRE 4
% chapitre 4: BGC comme conseqeucne de la recombinaison
% Isochores
% (+ BGC comme sel nat)
% (+ preuves directes et indirectes du BGC)



%%% DANS GENE CONVERSION (CHAPITRE 4)
% Parler de gene conversion + conversion/restauration
% Parler de tract
% Parler de heteroduplex
% MMR et BER


%%%% CHAPITRE 4
% %% QUAND PARLERAI DU MISMATCH REPAIR
% % https://en.wikipedia.org/wiki/Holliday_junction
% Robin Holliday proposed the junction structure that now bears his name as part of his model of homologous recombination in 1964, based on his research on the organisms Ustilago maydis and Saccharomyces cerevisiae. The model provided a molecular mechanism that explained both gene conversion and chromosomal crossover. Holliday realized that the proposed pathway would create heteroduplex DNA segments with base mismatches between different versions of a single gene. He predicted that the cell would have a mechanism for mismatch repair, which was later discovered.[3] Prior to Holliday's model, the accepted model involved a copy-choice mechanism[26] where the new strand is synthesized directly from parts of the different parent strands.[27]
% 3.  Liu Y, West S (2004). "Happy Hollidays: 40th anniversary of the Holliday junction". Nature Reviews Molecular Cell Biology. 5 (11): 937–44. doi:10.1038/nrm1502. PMID 15520813.
% 27.  Advances in genetics. Academic Press. 1971. ISBN 9780080568027.
%


%% CHAPITRE 4
%% GENE CONVERSION DES NCO (SDSA)
% Wikipedia https://en.wikipedia.org/wiki/Synthesis-dependent_strand_annealing
%SDSA is unique in that D-loop translocation results in conservative rather than semiconservative replication, as the first extended strand is displaced from its template strand, leaving the homologous duplex intact. Therefore, although SDSA produces non-crossover products because flanking markers of heteroduplex DNA are not exchanged, gene conversion does occur, wherein nonreciprocal genetic transfer takes place between two homologous sequences.[10]

% CHAPITRE 4
% % https://books.google.fr/books?id=7V0N6Tt8fUwC&pg=PA43&lpg=PA43&dq=murray+1960+polarity&source=bl&ots=mtj-qfJ1ZM&sig=ACfU3U1rKTqzCqEtcJkNw4ex96F_KPI87Q&hl=fr&sa=X&ved=2ahUKEwiG0b39-tfgAhUJ0RoKHRn4CWsQ6AEwB3oECAkQAQ#v=onepage&q=murray%201960%20polarity&f=false
% Sur la polarité des gene conversion DONC des sites precis ou la recombinaison demarre (a mettre dans les points chauds de recombinaison).
%
% N. Saitou, Introduction to Evolutionary Genomics, Computational Biology 17,
% % DOI 10.1007/978-1-4471-5304-7_2, © Springer-Verlag London 2013
% file:///Users/maudgautier/Downloads/9781447153030-c2.pdf
%
% %% GENE CONVERSION (pargraphe de Whitehouse ou Saitou ou autre??)
% Early studies on gene conversion were mostly restricted to fungal genetics. As
% molecular evolutionary studies of multigene family started, unexpected similarity
% of tandemly arrayed rRNA genes was found [ 15  ]. This phenomenon was termed
% ‘concerted evolution,’ and gene conversion or unequal crossing-over was proposed
% to explain this characteristic of some multigene families (e.g., [ 16  ]). New statistical
% methods were developed to detect gene conversion between homologous
% sequences [ 17, 18  ]. Program GENECONV developed by Sawyer [ 19  ] became the
% standard tool for analyzing gene conversions. We now know that conversion can
% occur in any genomic region irrespective of genes (DNA regions having function)
% or nongenic regions (e.g., [ 20  ]). However, ‘gene conversion’ as technical jargon is
% currently widely accepted, and I follow this nomenclature. Gene conversion can be
% classifi ed into two types: intragenic or between alleles and intergenic or between
% duplicated genes.
%
%
% mismatch repair
% https://fr.wikipedia.org/wiki/Mismatch_repair


% Question à moi-même (pour Laurent): si DSB, pourquoi la partie cassée du chromosome ne part pas ailleurs dans le cytoplasme?



% Biblio souris - meiose
% O’Bryan, M. K. & Kretser, D. Mouse models for genes involved in impaired spermatogenesis. Int. J. Androl. 29, 76–89 (2006).







% % SISTER
% Hinch: Nevertheless, it appears that some fraction of programmed meiotic DSBs are repaired using the sister chromatid [Hyppa and Smith, 2010].
% sex chromosomes: no homology donc repaired via sister chromatids at late prophase. Silenced (cf Altemose)
%
% % SYNCHRONISATION CELL CYCLE
% Meiotic success also hinges on theability to synchronize the meiotic transcriptional programwith cell cycle progression and cell growth. This isachieved in yeast by coupling double strand break for-mation with progression of the replication fork
% % Borde V, Goldman AS, Lichten M:Direct coupling betweenmeiotic DNA replication and recombination initiation.Science2000,290:806-809.
%
% % REGULATION
% This step regulated: Pachytene checkpoint: avoid defects (Handel Schimenti)
% If error, meiotic silencing (https://en.wikipedia.org/wiki/Synapsis)
%
% % DIFF MEIOSE MITOSE
% attachement des chromosomes par les kinetochores differe de la meiose (https://cshperspectives.cshlp.org/content/7/5/a015859.long)
% Autre difference avec la mitose: le spindle qui peut etre asymetrique.







