\begin{savequote}[8cm]
	% ‘[…] the process of mutation itself is not adaptive. On the contrary, the mutants which arise are, with rare exceptions, deleterious to their carriers, at least in the environments which the species normally encounters. Some of them are deleterious apparently in all environments. Therefore, the mutation process alone, not corrected and guided by natural selection, would result in degeneration and extinction rather than in improved adaptedness.’
	% ‘[…] the mutants which arise are, with rare exceptions, deleterious to their carriers, at least in the environments which the species normally encounters.’
	% — Theodosius Dobzhansky
	% 'On Methods of Evolutionary Biology and Anthropology', American Scientist, 1957, 45, 385.


	%%%%% KIMURA
	% ‘[…] if the neutral or nearly neutral mutation is being produced in each generation at a much higher rate than has been considered before, then we must recognize the great importance of random genetic drift due to finite population number in forming the genetic structure of biological populations.’
	‘Finally, if my chief conclusion is correct, and if the neutral or nearly neutral mutation is being produced in each generation at a much higher rate than has been considered before, then we must recognize the great importance of random genetic drift due to finite population number in forming the genetic structure of biological populations.’
	\qauthor{--- Motoo Kimura, \textit{\usebibentry{kimura1968evolutionary}{title}} \citeyearpar{kimura1968evolutionary} }
	
	% ‘Of course, Darwinian change is necessary to explain change at the phenotypic level — fish becoming man — but in terms of molecules, the vast majority of them are not like that.’
	%
	% ‘This leads us to an important principle for the neutral theory stating that “the neutral mutants” are not the limit of selectively advantageous mutants but the limit of deleterious mutants when the effect of mutation on fitness becomes indefinitely small. This means that mutational pressure causes evolutionary change whenever the negative-selection barrier is lifted.’
	% ‘[…] “the neutral mutants” are not the limit of selectively advantageous mutants but the limit of deleterious mutants when the effect of mutation on fitness becomes indefinitely small. This means that mutational pressure causes evolutionary change whenever the negative-selection barrier is lifted.’
	% ‘[…] “the neutral mutants” are not the limit of selectively advantageous mutants but the limit of deleterious mutants when the effect of mutation on fitness becomes indefinitely small.’
	% \qauthor{--- Motoo Kimura, \textit{\usebibentry{kimura1983neutral}{title}} \citeyearpar{kimura1983neutral} }

	% FROM
	% https://books.google.fr/books?hl=fr&lr=&id=olIoSumPevYC&oi=fnd&pg=PR9&dq=%22The+neutral+theory+of+molecular+evolution%22+by+Motoo+Kimura&ots=P0R5memrff&sig=5w8VhVKBKCo2IxJIhcSPc-36vnA#v=onepage&q=deleterious&f=false
	% — avec recherche du mot "deleterious" (page 113)


	% DEPUIS L'AUTRE
	% ‘Unlike the Darwinian theory of evolution by natural selection, the neutral theory claims that the overwhelming majority of evolutionary changes at the molecular level are not caused by Darwinian natural selection acting on advantageous mutants, but by random fixation of selectively neutral or very nearly neutral mutants through the cumulative effect of sampling drift (due to finite population number) under continued input of new mutations.’
	% \qauthor{--- Motoo Kimura, \textit{\usebibentry{kimura1991neutral}{title}} \citeyearpar{kimura1991neutral} }

\end{savequote}

\chapter{\label{ch:4-gBGC}Biased gene conversion, a major designer of genomic lanscapes} 
%\otherpagedecoration

\minitoc{}

% DES TITRES POSSIBLES
% The evolution of genomic GC-content through biased gene conversion
% GC-biased gene conversion and the evolution of genomic landscapes (de Duret et Galtier)
% GC-biased gene conversion, the designer of genomic lanscapes
%
% The Role of GC-Biased Gene Conversion in Shaping the Fastest Evolving Regions of the Human Genome
% The role of GC-biased gene conversion in shaping genomic GC-content
% How GC-biased gene conversion shapes the genomic GC-content
%
%
% Biased gene conversion and GC-content evolution
% The evolution of genomic base composition through GC-biased gene conversion
% GC-biased gene conversion and the evolution of base composition
% Gene conversion and GC-content evolution in mammals
% GC-content evolution and biased gene conversion
% GC-biased gene conversion and the evolution of genomic content
% Biased gene conversion and GC-content evolution
% The genomic landscape of GC-content and biased gene conversion

% GC-biased gene conversion, the fourth force of genome evolution (depuis Lesecque) 




Gene conversion, i.e.\ the process through which one DNA sequence is non-reciprocally copied onto another one (the homologue in the case of allelic gene conversion), leads to the non-Mendelian segregation of genetic information at the locus where it occured. 
If the two alleles are equally likely to be converted, this has no incidence at the population scale: allelic frequencies remain constant over generations. 
If, however, one homologue preferentially converts the other, it is more frequent in the pool of gametes and the transmission of alleles is necessarily biased: the donor has an evolutionary advantage over the acceptor.

Such biased gene conversion (BGC) exists under two forms: DSB-induced BGC (dBGC) when the bias comes from a differential competency for homologues to host the double-strand break (see Chapter~\ref{ch:3-recombination-variation}), and GC-biased gene conversion (gBGC) when it comes from the nature of the nucleotides involved.
Indeed, the repair of the cut homologue involves the formation of heteroduplex DNA, i.e.\ a stretch of DNA where the two strands bear distinct alleles.
These mismatches are either ‘restored’ if the original allele of the cut sequence is reinstated, or ‘converted’ if it is supplanted by the allele of the homologue.
The position of these events delineate ‘conversion tracts’ (CTs) — which are designated as ‘complex CTs’ when they alternate conversion and restoration events \citep{borts1989length} and ‘simple CTs’ otherwise.

In some species, whether through conversions or restorations, the repair favours \textit{GC} over \textit{AT} alleles \citep{mancera2008highresolution,si2015widely,williams2015noncrossover,halldorsson2016rate,smeds2016highresolution}, hence the term ‘GC-biased gene conversion’ (gBGC).
%\citep{pessia2012evidence,glemin2014gc,glemin2015quantification,figuet2014biased,wallberg2015extreme,bolivar2016recombination}, hence the term ‘GC-biased gene conversion’ (gBGC). 
Because its consequences on genome evolution ressemble those of natural selection, the very existence of this recently discovered phenomenon has been questioned by many in the more global context of the controversy opposing selectionists to neutralists (see Chapter~\ref{ch:1-history-genetics}).
I will therefore start this chapter by reviewing the breakthrough of gBGC in the climate of this debate, then explore the similitudes of its implications for genome evolution with those of natural selection and finish by looking into the first studies that characterised it.



\section{Discovery of GC-biased gene conversion (gBGC)}
\subsection{The debated origin of isochores}

\begin{figure}[!b]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap4/isochores-human.eps}
	\caption[Overview of isochores on four human chromosomes]
	{\textbf{Overview of isochores on four human chromosomes.}
		\par Human chromosomes 1, 2, 3 and 4 are divided into 100-kb windows coloured according to their mean GC-content: the spectrum of GC-level was divided into five classes (indicated by broken horizontal lines) from ultra-marine blue (GC-poorest L1 isochores) to scarlet red (GC-richest H3 isochores). 
		Grey vertical lines correspond to gaps present in the sequences and grey vertical regions to centromeres.
		\par This figure was reproduced from \citet{costantini2006isochore} and corresponds to a subsample of the original figure (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:isochores-human}
\end{figure}


In double-stranded DNA, adenosine (A) and thymine (T) nucleotides pair up while cytosine (C) nucleotides mate guanine (G) bases \citep[reviewed in \citealp{kresge2005chargaff}]{chargaff1950chemical}.
Therefore, when studying the composition of a stretch of DNA, it is conventional to measure its GC-content.

Originally, this was done \textit{via} the ultra-centrifugation of DNA fragments \citep{meselson1957equilibrium,corneo1968isolation}.
Using this technique, a few studies have characterised GC-content distribution in several eukaryotes \citep{filipski1973analysis,thiery1976analysis,macaya1976approach,macaya1978analysis,cortadas1977analysis} and revealed that mammalian, avian and reptilian genomes — but not amphibians nor fishes \citep{bernardi1990compositional} — display a long-range compositional heterogeneity (Figure~\ref{fig:isochores-human}).
The long regions of 100 kb or more that carry a relatively homogeneous GC-content were later termed ‘isochores’ \citep{cuny1981major}.\\


GC-rich isochores are enriched in genes \citep[reviewed in \citealp{bernardi2005distribution}]{bernardi1985mosaic,mouchiroud1991distribution,lander2001initial} that are shorter and more compact than in GC-poor regions \citep{duret1995statistical}.
Regional GC-content further correlates with the timing of DNA replication \citep{federico1998generichest,watanabe2002chromosomewide,costantini2008replication}, the density in transposable elements (TEs) \citep{smit1999interspersed,lander2001initial,mousegenomesequencingconsortium2002initial} and the recombinational activity \citep{fullerton2001local,kong2002highresolution}.\\


Since base composition of homologous genomic regions correlate between the three amniotic lineages (mammals, birds and reptiles) \citep{kadi1993compositional,caccio1994singlecopy,hughes1999warmblooded}, it is thought that isochores were inherited from their last common ancestor (LCA).
Since then, certain lineages have undergone additional somehow steep changes.
For instance, the isochore GC-content of mice is less variable than that of other mammals — a pattern that is in the derived state as compared to nonrodents \citep{galtier1998isochore} and which likely reflects one \citep{mouchiroud1988compositional} or two \citep{smith2002compositional} extra ‘murid shifts’ since the LCA\@.

Originally, two main hypotheses had been proposed as for the origin of isochores \citep[reviewed in][]{duret2009biased}.
According to the mutational bias hypothesis, isochores would be caused by a variation along chromosomes in the mutational bias towards either AT or GC nucleotides \citep{filipski1988why,wolfe1989mutation,francino1999isochores,fryxell2000cytosine}.
If this were true, GC~$\rightarrow$~AT and AT~$\rightarrow$~GC mutations should have the same probability of fixation at neutral sites.
The finding that this was not the case \citep{eyre-walker1999evidence, smith2001synonymous, lercher2002evolution, webster2004fixation, spencer2006influence} ruled out this theory.

Another proposition involving natural selection has been thoroughly defended by one of the major discoverers of isochores \citep{bernardi2000isochores,bernardi2007neoselectionist,bernardi2012genome}. 
In his view, the fact that G and C bases are linked \textit{via} three hydrogen bonds (instead of two for A and T bases) would compensate for the purportedly instable nature of DNA in warm-blooded animals.
However, this does not explain why only a fraction of the genome is affected by higher GC-content \citep{duret2009biased}.
This theory was further invalidated by the facts that no correlation between body temperature and GC-content was found \citep{belle2002analysis,ream2003base} and that this isochore organisation also takes place in cold-blooded animals like reptiles \citep{hughes1999warmblooded,hamada2003presence,costantini2016anolis}.
In addition, a scenario according to which all sites are under selection has theoretical limitations: given the elevated rate of deleterious mutations in their protein-coding sequences \citep{eyre-walker1999high,keightley2000deleterious}, mammalian genomes would probably accumulate a mutation load too high to be coped with \citep{eyre-walker2001evolution}.

An alternative role for natural selection in causing isochore organisation would be its fine-tuning the expression of tissue-specific genes \citep{vinogradov2003isochores,vinogradov2005noncoding}.
This hypothesis may not hold, though, since the correlation between GC-content and gene expression is extremely weak \citep[reviewed in \citealp{duret2009biased}]{semon2005relationship,semon2006no,pouyet2017recombination}.\\


Since natural selection thus seems insufficient to explain, on its own, the bias towards the fixation of \textit{GC} alleles, another track has been considered: GC-biased gene conversion (gBGC).





\subsection{An alternative causation: the gBGC model}

The excess of AT~$\rightarrow$~GC substitutions in a context where GC~$\rightarrow$~AT mutations are preponderant can be explained in two non-mutually exclusive ways: either because of non-stationarity (i.e.\ the GC-content in GC-rich isochores would still be decreasing) or because of GC-biased gene conversion (gBGC).
This hypothesis, initially mentioned by \citet{holmquist1992chromosome} and \citet{eyre-walker1993recombination,eyre-walker1999evidence}, has been promoted by \citet{galtier2001gccontent}.

The latter model originates from the observation that the mismatch repair (MMR) system — \mccorrect{the main pathway active during recombination to correct base misalignments} \citep[reviewed in \citealp{evans2000roles} and \citealp{spies2015mismatch}]{alani1994interaction,nicolas1994polarity} which is also involved in the mending of base misincorporations during DNA replication \citep{surtees2004mismatch} — may favour \textit{G} and \textit{C} alleles \citep{brown1988different,bill1998efficient}. (Figure~\ref{fig:gBGC-galtier}). \\


\begin{figure}[t]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap4/gBGC-galtier.eps}
	\caption[Gene conversion during a recombination event involving a strong (G or C) \textit{versus} a  weak (A or T) base mismatch]
	{\textbf{Gene conversion during a recombination event involving a strong (G or C) \textit{versus} a  weak (A or T) base mismatch.}
		\par A pair of homologous chromosomes displaying a heterozygous site with a G:C pair represented as a black rectangle and a A:T pair as a white rectangle (\textbf{a}) undergoes a recombination event which materialises as a heteroduplex (\textbf{b}) containing a T:G mismatch (\textbf{c}).
		The T:G mismatch is repaired and results either in a G:C (\textbf{d}) or a A:T (\textbf{d'}) pair which yield a non-Mendelian segregation of alleles (\textbf{e} and \textbf{e'}).
		This has an incidence at the population-scale if the repair towards G:C (\textbf{d}) or A:T (\textbf{d'}) is more frequent than the other one. 
		It is called GC-biased gene conversion (gBGC) in the particular case where the repair towards G:C (\textbf{d}) occurs more often.
		\par This figure was reproduced from \citet{galtier2001gccontent} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:gBGC-galtier}
\end{figure}


% - Le gBGC, puisqu'il modifie la frequence de transmission des alleles GC, devrait avoir un effet sur la composition en base, mais eyre-walker montre que seulement u naroow range of parameters ou le taux
% the rate of biased conversion would be high enough to signif- icantly alter polymorphism patterns but low enough not to induce an extreme base com- position.

A predictable consequence of such alteration in the frequency of transmission of \textit{G} and \textit{C} alleles is the long-term evolution of base composition in regions undergoing gBGC\@.
% If the frequency of transmission of \textit{G} and \textit{C} alleles is indeed altered, base composition would be expected to
% Because the gBGC model predicts an altered transmission of \textit{GC} alleles, it should impact base composition in the long-term.
Though, at the time, one major argument against the gBGC model was that there was only a one-order-of-magnitude range of parameters for which the rate of biased gene conversion would be sufficiently high to alter polymorphism patterns significantly but remain sufficiently low not to induce an extreme base composition \citep{eyre-walker1999evidence}.
% At the time, one major argument against gBGC was that there was only a narrow range of parameters within which gBGC can significantly alter base composition\footnote{If $N_e \times w \ll 1$ (where $N_e$ is the effective population size and $w$ is the per nucleotide conversion rate times the average conversion bias), gBGC has little effect on base composition.} without making it extreme\footnote{If $N_e \times w \gg 1$ (where $N_e$ is the effective population size and $w$ is the per nucleotide conversion rate times the average conversion bias), gBGC leads to extreme base composition.}.
This objection was adressed by \citet{duret2008impact} who found that the gBGC model explains well the relationship between recombination and substitution rates.
Indeed, considering that gBGC acts only at recombination hotspots, the substitution rate increases greatly at these loci, but stops before their GC-content reaches 100\%, because hotspots generally have a short lifespan \citep{ptak2005finescale,winckler2005comparison}. 
In particular, as soon as a hotspot gets inactivated, its GC-content should start decreasing, consistently with what has been observed in the GC-rich regions\footnote{According to the gBGC hypothesis, GC-rich regions are those that host the hotspots.} of primates \citep{duret2002vanishing,belle2004decline,meunier2004recombination,duret2006gc}.


gBGC also provides an explanation for the higher heterogeneity of GC-rich isochores \citep{clay2001compositional,clay2001compositionala}: since recombination hotspots would locally display higher GC-levels than the genome-wide average, hotspot-dense regions would exhibit a particularly disparate GC-content.\\
% hotspot-dense regions would, under that hypothesis, correspond to genomic segments
% locally GC-enriched loci surrounded by normal GC-level regions.\\
% : GC-content would vary depending on the intensity and age of hotspots.
% In particular, as soon as a hotspot gets inactivated, its GC-content should start decreasing, consistently with what has been observed in the GC-rich regions\footnote{According to the gBGC hypothesis, GC-rich regions are those that host the hotspots.} of primates \citep{duret2002vanishing,belle2004decline,meunier2004recombination,duret2006gc}.
% In contrast, in avian genomes that have evolutionarily stable recombination hotspots, GC-content continues increasing \citep{webster2006strong,capra2011substitution,mugal2013twisted}.\\

Another objection to gBGC \citep{eyre-walker1999evidence} came with the observation that GC-content at the synonymous third position of codons (GC\textsubscript{3}) is generally greater than intronic GC-content \citep{clay1996human}.
But \citet{duret2001elevated} provided an explanation compatible with gBGC to this observation: assuming that transposons are GC-poorer than the GC-rich regions of the genome, their accumulation within introns (but not exons) would justify such difference between intronic GC-content and GC\textsubscript{3}.\\

Altogether, the presence of isochores seems to fit theoretically with gBGC \citep{duret2006new}.
But, under the gBGC hypothesis, a number of other consequences are expected and their footprints can be researched in genomes.



\subsection{Footprints of gBGC in mammalian genomes}

One strong prediction of the gBGC model is that highly recombining regions should be GC-rich, which happens to be the case in several instances.

For example, components of the genome that undergo ectopic gene conversion (i.e.\ conversion between copies of a gene family) — like transfer RNAs (tRNAs), introns of ribosomal RNAs (rRNAs) \citep{galtier2001gccontent}, human and mouse major histocompatibility complex (MHC) regions \citep{hogstrand1999gene} and other gene families \citep{backstrom2005gene,galtier2003gene,kudla2004gene} — are all GC-richer than the rest of the genome.

The human pseudoautosomal region (PAR) of X and Y chromosomes — the only portion of male sexual chromosomes which has homology and therefore recombines — provides another example of the association between recombination and GC-content.
Indeed, given its short size, the per-nucleotide recombination rate (RR) of the PAR is much higher than that of autosomes \citep{soriano1987high}, while the non-PAR sections of sex chromosomes recombine even less (X chromosome) or not at all (Y chromosome).
Under the gBGC model, the average GC\textsubscript{3} of these four genomic domains is expected to increase with their RR — which, as a matter of fact, is the case \citep{galtier2001gccontent}.

This relationship between recombination and GC-content is really impressive in the \textit{Fxy} gene that has been translocated onto the boundary of the mouse PAR a few million years ago: as compared to its X-linked portion, the PAR-side part of \textit{Fxy} has undergone an acceleration in substitution rates \citep{perry1999evolutionary} together with a strong increase in GC-content at both coding and non-coding positions \citep{montoya-burgos2003recombination,galtier2007adaptation} — a finding that is consistent with gBGC occuring at the highly recombining PAR-side of the gene.
Surprisingly however, the \textit{XG} gene overlapping the PAR boundary of primates does not show the same pattern \citep{yi2004recombination}.
Nevertheless, this observation does not necessarily conflict with the gBGC model: if \textit{XG} was wholly located within the PAR before displacing onto its boundary, — or rather, before the PAR boundary displaces onto the gene, since the mammalian PAR gradually erodes \citep{lahn1999four,marais2003sex}, — it would have accumulated a high GC-content and would now be undergoing a slow, mutation-driven decrease in GC-content that would not be detectable yet \citep{galtier2004recombination}.\\


At the genome-wide scale, GC-content correlates positively with recombination rate in many eukaryotes \citep{pessia2012evidence} including yeasts \citep{gerton2000global,birdsell2002integrating}, nematodes and flies \citep{marais2001does,marais2003neutral,marais2002hillrobertson}, birds \citep{internationalchickengenomesequencingconsortium2004sequence,mugal2013twisted}, turtles \citep{kuraku2006cdnabased}, paramecia \citep{duret2008analysis}, algae \citep{jancek2008clues}, plants \citep{glemin2006impact} and humans \citep{fullerton2001local,yu2001comparison,meunier2004recombination,khelifi2006gc,duret2008impact}.

But, since the evolution of GC-content is relatively slow as compared to that of recombination rates in mammalian clades, it has been claimed that these estimates should be measured on similar time scales to be correctly compared \citep{duret2009biased}.
To do this, the stationary GC-content (GC\textsuperscript{*}), i.e.\ the GC-content that sequences would reach at equilibrium if patterns of substitution remained constant over time, is generally used.
Under the assumption that all sites evolve independently from one another \citep{sueoka1962genetic}, this statistic can be calculated as:

\begin{equation*}
	GC\textsuperscript{*} = \frac{u}{u+v}
\end{equation*}

where $u$ and $v$ represent respectively the AT~$\rightarrow$~GC and the GC~$\rightarrow$~AT substitution rates.
But, because the latter assumption is not valid in vertebrates where the mutation rate of a given base depends on the nature of the neighbouring bases\footnote{For instance, CpG sites (i.e.\ CG dinucleotides) are hypermutable \citep{arndt2003distinct}.}, \citet{duret2008impact} used a maximum likelihood approach to improve the estimation of GC\textsuperscript{*} and showed that it correlated better with recombination rate than with the observed GC-content (Figure~\ref{fig:correlation-recombi-stationary-GC}). This further suggests that recombination acts upon GC-content, and not the other way round, as was proposed by \citet{gerton2000global}, \citet{blat2002physical} and \citet{petes2002context}.

In past primate lineages, GC\textsuperscript{*} also correlates well with the historical recombination rate \citep{munch2014finescale}.\\


% These correlations are in accordance with the gBGC model.

These correlations between GC-content and recombination appear to be greater in males than in females for several species including mice, dogs and sheeps \citep{popa2012sexspecific} as well as humans \citep{webster2005maledriven, dreszer2007biased, duret2008impact}.
% Also, GC\textsuperscript{*} is better correlated with male recombination rate (RR) than female RR in mice, dogs and sheeps \citep{popa2012sexspecific} as well as in human Alu repeats \citep{webster2005maledriven} and hotspots \citep{dreszer2007biased} and in 1-Mb windows of autosomes \citep{duret2008impact}.
Since \mccorrect{chiasmata persist many years in females} \citep{coop2007evolutionary}, it is possible that the repair of mismatches proceeds differently in the two sexes, which could explain the seemingly male-specific gBGC \citep{duret2009biased}.
Alternatively, the sex-specific strategies for the distribution of recombination events along chromosomes (and more specifically, as a distance to telomeres) seem to account for this difference between males and females \citep{popa2012sexspecific}.


\begin{figure}[h]
	\centering
	\includegraphics[width = 0.7\textwidth]{figures/chap4/correlation-recombi-stationary-GC.eps}
	\caption[Correlation between the stationary GC-content (GC\textsuperscript{*}) and the crossover rate (cM/Mb) in human autosomes]
	{\textbf{Correlation between the stationary GC-content (GC\textsuperscript{*}) and the crossover rate (cM/Mb) in human autosomes.}
		\par Each dot corresponds to a 1-Mb-long genomic region. Green dots correspond to the predictions of the gBGC model.
		\par This figure was reproduced from \citet{duret2009biased} and originally adapted from \citet{duret2008impact} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:correlation-recombi-stationary-GC}
\end{figure}





% Dans recombination: 
% longer in females humans (the map)
% A second-generation combined linkage physical map of the human genome.
% Matise TC, Chen F, Chen W, De La Vega FM, Hansen M, He C, Hyland FC, Kennedy GC, Kong X, Murray SS, Ziegle JS, Stewart WC, Buyske S
% Genome Res. 2007 Dec; 17(12):1783-6.
%
% longer in dogs
% A comprehensive linkage map of the dog genome.
% Wong AK, Ruhe AL, Dumont BL, Robertson KR, Guerrero G, Shull SM, Ziegle JS, Millon LV, Broman KW, Payseur BA, Neff MW
% Genetics. 2010 Feb; 184(2):595-605.







\section{Interference with natural selection}

Several of the aforementioned observations supporting gBGC would also be predicted under a natural selection model.
For instance, since linkage reduces the efficacy of selection \citep{hill1966effect}, a correlation between GC-content and recombination rate would be expected if there was a very high selection coefficient in favour of \textit{GC} alleles \citep{galtier2001gccontent}.
More generally, the dynamics of the fixation process for one locus is identical no matter which of the two forces (biased gene conversion or natural selection) is responsible for it \citep{nagylaki1983evolution}, which explains why the first observations were initially interpreted as resulting from natural selection \citep[e.g.][]{eyre-walker1999evidence}.
In this section, I review a few case studies in which such confounding patterns between gBGC and natural selection exist.


\subsection{The case of codon usage bias (CUB)}

Codon usage bias (CUB) corresponds to the observation that the frequency of use of synonymous codons (i.e.\ sequences of three nucleotides coding for the same amino acid (AA)) can vary across or within species \citep{fitch1976there}.
Both adaptative (natural selection) and non-adaptative (mutation \citep{marais2001synonymous} or biased gene conversion) forces account for CUB \citep{bulmer1991selectionmutationdrift,sharp1993codon,akashi1998translational}, but there remains a controversy about the quantitative contribution of each of these mechanisms to CUB \citep{pouyet2016etude}.\\

In \textit{Drosophila}, the CUB of each gene is correlated to transfer RNA (tRNA) content \citep{akashi1994synonymous,duret1999expression,bierne2006variation,behura2011coadaptation}, particularly for genes that are highly expressed \citep{chavancy1979adaptation,shields1988silent,moriyama1997codon,hey2002interactions}.
This association between CUB and gene expression also holds true in \textit{Caenorhabditis} \citep{duret1999expression,castillo-davis2002genome,marais2002hillrobertson}, \textit{Daphnia} \citep{lynch2017population}, \textit{Arabidopsis} \citep{duret1999expression,wright2004effects}, \textit{Oryza} \citep{muyle2011gcbiased} and single-celled organisms like \textit{Giardia} \citep{lafay1999synonymous}, \textit{Saccharomyces} \citep{bennetzen1982codon,akashi2003translational,harrison2011biased}, \textit{Dictyostelium} \citep{sharp1989codon} and bacteria \citep{gouy1982codon,ikemura1985codon,sharp1987rate}.
This has been interpreted as ‘translational selection’: the coevolution of tRNA content with codon usage would increase either the accuracy or the efficiency of translation \citep{sharp1995dna,duret2002evolution}.
Though, other processes, like messenger RNA (mRNA) stability, protein folding, splicing regulation and robustness to translational errors could also play a role \citep[reviewed in \citealp{clement2017evolutionary}]{chamary2006hearing,cusack2011preventing,plotkin2011synonymous}.\\

In contrast, in lowly recombining regions of \textit{Drosophila} \citep{kliman1993reduced} and in species with small effective population size ($Ne$) \citep{subramanian2008nearly,galtier2018codon}, like mammals \citep{urrutia2003signature,comeron2004selective,lavner2005codon}, selection for codon usage remains weak.
Instead, in mammals, codon usage is primarily governed by variations in GC-content \citep{semon2006no,rudolph2016codondriven,pouyet2017recombination}, which implies that gBGC could be one of the drivers of CUB in that clade.
In \textit{Drosophila} too, even if selection on codon usage predominates \citep{zeng2009estimating,zeng2010studying,zeng2010simple}, gBGC could also participate to CUB\@.
Indeed, one peculiar feature of codon usage in this species is that, for all 20 amino acids (AAs), the preferred codon systematically ends with a G or a C nucleotide \citep[reviewed in][]{duret2009biased}.
Even if the reason for this remains unknown, the finding that the base composition of the third position of 4-fold degenerate\footnote{A codon is said to be \textit{n}-fold degenerate if \textit{n} distinct three-nucleotide sequences result in the same amino acid (AA).} codons is similar to that of non-coding regions \citep{clay2011gc3} indicates that the patterns of CUB could (at least partly) come from evolutionary processes influencing base composition irrespectively of translational selection — such as gBGC \citep[but see \citealp{jackson2017variation}]{duret2002evolution,galtier2006gcbiased,lynch2007origins}. 
A similar observation made in plants was also interpreted as the consequence of gBGC \citep{clement2017evolutionary}.




\subsection{The case of human accelerated regions (HAR)}

gBGC has also been mistaken for positive selection in fast-evolving regions specific to the human genome \citep[reviewed in][]{duret2009biased}.
Such regions, — named human accelerated regions (HAR) or human accelerated conserved non-coding sequences (HACNS), — have been searched by several groups \citep{pollard2006forces,pollard2006rna,prabhakar2006accelerated,bird2007fastevolving,bush2008genomewide,lindblad-toh2011highresolution} in a quest to find the molecular adaptations that make the human genome distinct from other mammals.

HARs have first been interpreted as resulting from positive selection \citep[reviewed in][]{hubisz2014exploring} but, because they harbour an excess of AT~$\rightarrow$~GC substitutions, gBGC has been proposed as an alternative origin for these accelerated sequences \citep{galtier2007adaptation,berglund2009hotspots,duret2009comment,katzman2010gcbiased,ratnakumar2010detecting}.
And indeed, about one fifth of HARs seem to have evolved under gBGC alone \citep{kostka2012role}.\\


Thus, altogether, gBGC mimics natural selection in terms of consequences on the nucleotidic sequence \citep{bherer2014biased}, and this is likely to bring biases to molecular evolution and phylogenomics analyses \citep{berglund2009hotspots,ratnakumar2010detecting,webster2012direct,romiguier2013less,romiguier2016phylogenomics,romiguier2017analytical,bolivar2018biased,bolivar2019gcbiased,rousselle2019influence}.
Consequently, prior to concluding that positive selection explains a given observation, one should check that the extended null hypothesis of molecular evolution (i.e.\ both the neutral and the gBGC models) has been rejected \citep{galtier2007adaptation, duret2009biased}.
To check for this, three observations should be taken into consideration: first, whether AT~$\rightarrow$~GC substitutions are preponderant; second, whether the studied locus is in a highly recombining region; and third, whether both functional and non-functional sites are affected.
Whenever all three criteria are met, gBGC remains a likely explanation for any observed acceleration in substitution rates.

But, if gBGC interferes with natural selection, what happens when both forces drive evolution in the opposite direction? 




\subsection{The deleterious effects of gBGC}

The AT~$\rightarrow$~GC mutations whose fixation is favoured by gBGC can be either beneficial, inconsequential or detrimental to the fitness of the individual carrying it.
To quantify the fate of all these categories of mutations in presence of gBGC, \citet{duret2009biased} performed simulations with characteristics close to those of human populations (in terms of effective population size and mutation rate) and showed that gBGC mainly favours the fixation of slightly deleterious and neutral AT~$\rightarrow$~GC mutations.

Analysing the ratio ($\frac{d_N}{d_S}$) of the rate of nonsynonymous\footnote{A nonsynonymous substitution does not modify the amino acid (AA) produced.} ($d_N$) over that of synonymous\footnote{A synonymous substitution modifies the amino acid (AA) produced.} substitutions ($d_S$) in exon-specific episodes of accelerated amino acid (AA) evolution, \citet{galtier2009gcbiased} demonstrated that gBGC has been sufficiently strong to outdo the effect of purifying selection\footnote{Purifying selection (or negative selection) is the selective removal of deleterious alleles.} and promote, instead, the fixation of deleterious AT~$\rightarrow$~GC mutations within proteins.
In wheat too, the accumulation of deleterious AT~$\rightarrow$~GC mutations shown by the analysis of $\frac{d_N}{d_S}$ has been interpreted as originating from gBGC \citep{haudry2008mating}.
% A similar observation made in wheat was also interpreted as a better of efficiency of gBGC in outcrossing than in selfing species due to the smaller effective population size ($N_e$) of the latter \citep{haudry2008mating}.
More generally, gBGC maintains deleterious mutations associated to human diseases \citep{necsulea2011meiotic,capra2013modelbased,lachance2014biased,xue2016basebiased}.\\


But, if gBGC prejudices fitness, how come it has persisted over evolutionary times?
This question remains open as of today, but it has been claimed that gBGC could somehow counterbalance the mutational load \citep{bengtsson1986biased,marais2003biased,glemin2010surprising,arbeithuber2015crossovers} which favours GC~$\rightarrow$~AT mutations in both eukaryotes \citep{lynch2010rate} and procaryotes \citep{hershberg2009general}.
Alternatively, gBGC has been proposed to be a meiotical side-effect of the GC-biased base excision repair (BER) mechanism which is crucial in mitosis to reduce the number of somatic mutations \citep{marais2003sex,lesecque2014conversion}.
Though, a study aiming at characterising gBGC in \textit{Saccharomyces cerevisiae} ruled out the latter hypothesis for yeasts \citep{lesecque2014conversion}.








\section{Characterisation of gBGC}

Understanding the still-blurry reason for the evolutionary maintenance of gBGC requires to better quantify it in living beings and characterise its relationship with other parameters of genome evolution. 
I review the knowledge acquired so far on this topic in the last section of this chapter.



\subsection{Quantification \textit{via} site frequency spectra (SFS)}

Fundamentally, gBGC shifts the allelic frequency of strong (S) (i.e.\ G and C) and weak (W) (i.e.\ A and T) bases, since it favours the fixation of the former and hinders that of the latter.
Thus, comparing the distribution of the derived allele frequency (DAF) of S bases arising from W~$\rightarrow$~S (WS) mutations and of W bases arising from S~$\rightarrow$~W (SW) mutations can allow to estimate the intensity of gBGC\@.

In practice, this is done by analysing site frequency spectra (SFS), a.k.a.\ derived allele frequency spectra (DAFS).
Indeed, because the SFS provides the number of SNPs for each class of frequency, it summarises the information in a much more detailed manner than any existing statistics (such as the GC\textsubscript{3} content in the case of gBGC, the ratio of non-synonymous over synonymous diversity ($\frac{\pi_N}{\pi_S}$) in the case of polymorphism, or the ratio of non-synonymous over synonymous substitutions ($\frac{d_N}{d_S}$) in the case of divergence) \citep{rousselle2018estimation}.

In the particular case of gBGC, the spectra for WS and SW mutations must be compared.
This requires to polarise mutations from the ancestral to the derived state, thanks to an outgroup\footnote{An outgroup is a distantly related group of organisms that serves as the ancestral reference for the studied group (or ingroup).} giving the ancestral state.
But, because the increased propensity for transitional\footnote{A transition is a mutation between two nucleotidic bases of the same family (purine or pyrimidine), i.e.\ either a A~$\leftrightarrow$~G or a C~$\leftrightarrow$~T mutation.} over transversional\footnote{A transversion is a mutation involving a change of nucleotidic family (from a purine to a pyrimidine or the other way round), i.e.\ either a A~$\leftrightarrow$~C, a A~$\leftrightarrow$~T, a G~$\leftrightarrow$~C or a G~$\leftrightarrow$~T mutation.} mutations as well as the hypermutability of CpG sites and other context-dependent DNA replication errors \citep{hwang2004bayesian} are known to induce spurious signatures of gBGC \citep{hernandez2007contextdependent}, \citet{glemin2015quantification} developed a method correcting for such polarisation errors and thus allowing to better detect and quantify gBGC\@.
Indeed, if gBGC participates in the evolution of the genome studied, the SFS will present WS mutations shifted towards higher frequencies than SW mutations (e.g.\ in Figure~\ref{fig:DAFS-example}), and the intensity of the shift will reflect that of gBGC.\\


\begin{figure}[t!]
	\centering
	\includegraphics[width = 0.7\textwidth]{figures/chap4/DAFS-example.eps}
	\caption[Example of a derived allele frequency spectrum (DAFS) separated for AT~$\rightarrow$~GC (WS) and GC~$\rightarrow$~AT (SW) mutations]
	{\textbf{Example of a derived allele frequency spectrum (DAFS) separated for AT~$\rightarrow$~GC (WS) and GC~$\rightarrow$~AT (SW) mutations.}
		\par AT~$\rightarrow$~GC (WS) mutations are coloured in black and GC~$\rightarrow$~AT (SW) in white. 
		The spectrum for WS mutations is shifted towards higher frequencies, as compared to the spectrum for SW mutations, as predicted in the gBGC model.
		\par This figure was reproduced from \citet{glemin2015quantification} and corresponds to a subsample of the original figure (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:DAFS-example}
\end{figure}



As an alternative to SFS, comparative genomics approaches exist to quantify gBGC\@.
For instance, \citet{lartillot2013phylogenetic} created a method based on the analysis of substitution patterns to quantify gBGC in a whole phylogeny and \citet{capra2013modelbased} developed another one allowing to quantify gBGC along a given genome \citep[reviewed in][]{mugal2015gc}.

As such, gBGC has been quantified in several organisms \textit{via} theoretical approaches.
But empirical studies too have helped in better characterising this driver of genome evolution, as reviewed hereunder.




\subsection{Empirical studies of gBGC}%Characterisation \textit{via} empirical studies}


All in all, the gBGC model is in accordance with observations in countless metazoans \citep{capra2011substitution,galtier2018codon} including vertebrates \citep{figuet2014biased}, — among which mammals \citep{romiguier2010contrasting,katzman2011ongoing,lartillot2013phylogenetic,clement2013meiotic,glemin2015quantification,dutta20181000}, avians \citep{webster2006strong,weber2014evidence,bolivar2016recombination} and reptiles \citep{figuet2014biased}, — and also some invertebrates like bees \citep{kent2012recombination,wallberg2015extreme} and \textit{Daphnia} \citep{keith2016high}.
Though, not all invertebrates are subject to gBGC\@: \textit{Drosophila}, except for its X chromosome \citep{galtier2006gcbiased,haddrill2008nonneutral}, is not affected \citep{robinson2014population}.
Plants — both angiosperms \citep[but see \citealp{liu2018tetrad}]{escobar2011gcbiased, glemin2014gc,rodgers-melnick2016open,clement2017evolutionary,niu2017mutational} and gymnosperms \citep{serres-giardi2012patterns} — also show molecular characteristics compatible with gBGC\@.
Thus, these eukaryotes, as well as numerous others \citep{escobar2011gcbiased,pessia2012evidence} — but also certain prokaryotes \citep{lassalle2015gccontent,long2018evolutionary}, — likely undergo gBGC\@.\\

% eukaryotes \citep{escobar2011gcbiased,pessia2012evidence}
% animals \citep{galtier2018codon} aka metazoans \citep{capra2011substitution}
% vertebrates \citep{figuet2014biased}
% mammals \citep{romiguier2010contrasting,lartillot2013phylogenetic}
% humans \citep{glemin2015quantification}
% mice \citep{clement2013meiotic}
%
% avians \citep{webster2006strong,weber2014evidence,bolivar2016recombination}
%
% reptiles \citep{figuet2014biased}
%
% and invertebrates like bees \citep{kent2012recombination,wallberg2015extreme} and \textit{Daphnia} \citep{keith2016high}
% But not in drosophila \citep{robinson2014population} except for its X chromosome \citep{galtier2006gcbiased,haddrill2008nonneutral}
%
% also plants:
% angiosperms \citep{escobar2011gcbiased, glemin2014gc,clement2017evolutionary,niu2017mutational} and gymnosperms \citep{serres-giardi2012patterns} but see \citep{liu2018tetrad}
% maize \citep{rodgers-melnick2016open}
%
%
% some prokaryotes \citep{lassalle2015gccontent}

Nevertheless, in all the aforementioned cases, gBGC was only observed indirectly — for instance \textit{via} correlations between GC-content and recombination, or \textit{via} the analysis of patterns of substitutions between closely related species. 
A decade ago though, gBCG has been confirmed experimentally in yeasts thanks to the creation of the first high-resolution recombination map \citep{mancera2008highresolution}: this map allowed to precisely analyse conversion tracts (CTs) at the genome-wide scale and to demonstrate that S alleles are significantly overtransmitted, even if the effect is extremely weak (GC-bias: 50.065\%).
Further analyses of this dataset have revealed that, in yeasts, gBGC is only associated with COs (but not NCOs), and solely affects the markers at the extremities of CTs \citep{lesecque2013gcbiased}.

In contrast, the first experimental evidence for gBGC in humans was restricted to a few hotspots \citep{odenthal-hesse2014transmission,arbeithuber2015crossovers} and was found exclusively in NCOs.
Nonetheless, gBGC remains a pervasive driver of human genome evolution since it has been estimated to affect about 15\% of our genome \citep{pouyet2018background}.\\

% In both these species, the observations seem rather compatible with gBGC originating from the mismatch repair (MMR) machinery, rather than with the base excision repair (BER) machinery.
% The mechanism from which gBGC originate (the mismatch repair (MMR) machinery or the base excision repair (BER) machinery)
% So far, two mechanisms have been proposed to be at the origin of gBGC\@: the mismatch repair (MMR) machinery and the base excision repair (BER) machinery.
% And, in the two aforementioned species, the
The mechanism at the origin of gBGC may not be the same for these two species.
Indeed, in yeasts, gBGC is primarily associated to simple CTs \citep{lesecque2013gcbiased}, which rules out the hypothesis of gBGC originating from the base excision repair (BER) machinery (according to which gBGC should be associated mainly with complex CTs) and instead suggests that it would originate from the mismatch repair (MMR) machinery.
As for humans, \citet{halldorsson2016rate} found that gBGC was stronger at CpG than at non-CpG sites, which argues in favour of the BER hypothesis.
% As for humans, the finding that gBGC is not any stronger at CpG than at non-CpG sites argues against the BER hypothesis, since BER is known to act preferentially at CpG sites \citep{glemin2015quantification}.

Interestingly, the BER and the non-canonical MMR (i.e.\ MMR activated by DNA lesions) pathways have been shown to cooperate in the removal of mismatches in the context of DNA demethylation \citep{grin2016interplay}, and a similar interplay between the two machineries in the context of meiotic repair of programmed DSBs could alternatively be conceived.\\

More recently, direct observations of gBGC at a larger scale in humans have been reported by two independent studies \citep{williams2015noncrossover, halldorsson2016rate}.
They confirmed that gBGC affects NCOs (GC-bias: 68\%), but also COs displaying complex CTs (GC-bias: 70\%).
However, the framework used did not allow to test for gBGC in COs with simple CTs.
This phenomenon was also directly observed in NCO CTs of birds \citep{smeds2016highresolution} and rice \citep{si2015widely} (GC-bias: 59\% in both cases), but could not be tested either in CO CTs.

% RICE: 
% >>> 1.47/(1+1.47)
% 0.5951417004048584

% and selfers have a lower GC content than outcrossers (because recombination less efficient) muyle2011gcbiased, haudry2008mating.






% - MMR et conversion restoration

%% INFO MISMATCH REPAIR DANS CHAPITRE 4
% % POUR APRES SI SAUTE:
% MISMATCH REPAIR
% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC86394/
% https://academic-oup-com.inee.bib.cnrs.fr/hmg/article/11/15/1697/636769
% https://cshperspectives-cshlp-org.inee.bib.cnrs.fr/content/7/3/a022657
% https://www-ncbi-nlm-nih-gov.inee.bib.cnrs.fr/pmc/articles/PMC4856981/pdf/gkw059.pdf
%
% DNA mismatch repair in mammals: role in disease and meiosis
% Norman Arnheim* and Darryl Shibatat
%
% http://content.ebscohost.com.inee.bib.cnrs.fr/ContentServer.asp?EbscoContent=dGJyMMTo50SeprM4v%2BbwOLCmr1Gep7NSsKu4TLWWxWXS&ContentCustomer=dGJyMOzpsE21p7JOuePfgeyx9Yvf5ucA&T=P&P=AN&S=R&D=a9h&K=2800408
% Andrew B. Buermeyer1, Suzanne M. Deschenes ˆ 1,Sean M. Baker2, and R. Michael Liskay
% MAMMALIAN DNA MISMATCH REPAIR
%
% Voir Hinch 2019 pour des references en biblio
%
% SAIS PLUS POURQUOI
% https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1558-5646.1981.tb04864.x


% %% DEPUIS LES Notes_for_cahp4.txt (deleted then)
% Sur le BGC: parler de la restauration/conservation des alleles.
% Hastings and colleagues in S. cerevisiae (46, 125) and in A. immersus(45)totestthehypothesisthatmismatchrepair cancorrecteithertowardstheinformationontheinvading strand(togiveaconversionevent)ortowardstheinforma- tionontherecipientstrand(togivearestorationevent).
% (depuis Orr-weaver and Szostak 1985, fungal recombination).
%
%
% MMR avec role dans BGC
% It is hard to find a single mechanism that is consistent
% with all of the genetic and physical data obtained in
% fungi. In S. cerevisiae, at the ARG4 locus, there is a
% double-strand break located near the high end of the
% conversion gradient 5~ and there is a good correlation
% between the level of gene conversion and the amount of
% single-strand excision 51. Although these results suggest
% that the polarity gradient of ARG4 might reflect only
% heteroduplex formation, E. Alani, R. Reenan and R.
% Kolodner (pers. commun.) have found that the steepness of the polarity gradient at ARG4 is substantially
% reduced by a mutation affecting mismatch repair. At the
% HIS4 locus, the conversion gradient for high PMS alleles is much less steep than that observed for low PMS
% alleles, and the gradient is nearly eliminated by one of
% the mutations affecting mismatch repair ~a. These results
% indicate that mismatch repair strongly influences the
% shape of the conversion gradient at this locus.
% %%%% de conclu de Nicolas and Pete Polarity of meiotic gene conversion in fungi: contrasting views (1994)







\subsection{Relationship with parameters of genome evolution}

Provided that no evolutionary force acts upon its transmission, the allelic frequency of a heterozygous locus in a pool of gametes should equal the Mendelian frequency of 50\%.
In presence of gBGC however, the allelic frequency of the favoured allele in the gametic pool ($x$) increases proportionately to the gBGC coefficient ($b$) according to the following relationship:

\begin{equation*}
	x = \frac{1}{2} \times (1+b)
\end{equation*}

Since $x$ is a proportion and is thus necessarily bounded between 0 and 1, $b$ is bounded between -1 (when \textit{AT} alleles are systematically transmitted) and 1 (when \textit{GC} alleles are systematically transmitted).

The intensity of the gap between the observed transmission and the Mendelian frequency (and thus, the gBGC coefficient $b$) depends on the recombination rate $r$ (including both COs and NCOs), the length of gene conversion tracts $L$ and the transmission bias (a.k.a.\ mismatch repair bias) $b_0$, as such: 

\begin{equation*}
	b = r \times L \times b_0
\end{equation*}

% The gBGC coefficient ($b$) can be decomposed as $b = r \times L \times b_0$, where $r$ is the recombination rate (considering both COs and NCOs), $L$ the length of gene conversion tracts and $b_0$ the transmission bias (i.e.\ the mismatch repair bias).
Finally, the spread of the favoured allele in the population is represented by the population-scaled gBGC coefficient ($B$), which itself depends on both $b$ and the effective population size ($N_e$) in a fashion much similar to the probability of fixation under selection defined by \citet{kimura1962probability}:

\begin{equation*}
	B = 4 \times N_e \times b
\end{equation*}

% Pour comparer avec l'effet de la derive
%In the case where $4 \times N_e \times b$ is greater than 1, 

In human genomes, apart from recombination hotspots which display a mean $B$ value of 3 \citep{glemin2015quantification}, the average $B$ found in several independent studies circumscribes between 0.1 and 0.5 \citep{lartillot2013phylogenetic,demaio2013linking,glemin2015quantification}, which is a low estimate as compared to other mammalian genomes \citep{lartillot2013phylogenetic} (Figure~\ref{fig:B-phylogeny-estimates}).\\
% Information obtenue depuis https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773373/ et https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4510005/


\begin{figure}[p!]
	\centering
	\makebox[\textwidth][c]{\includegraphics[width = 1.2\textwidth]{figures/chap4/B-phylogeny-estimates.eps}}
	% OU: (si \usepackage[export]{adjustbox}[2011/08/13] dans main.tex)
	% \includegraphics[width = 1.2\textwidth, center]{figures/chap4/B-phylogeny-estimates.eps}
	\caption[Reconstructed phylogenetic history of $B = 4 \times N_e \times b$ in placental mammals]
	{\textbf{Reconstructed phylogenetic history of $B = 4 \times N_e \times b$ in placental mammals.}
		\par The names of orders are given on the right side of the tree and each branch is coloured according to its average genome-wide $B$.
		\par This figure was reproduced from \citet{lartillot2013phylogenetic} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:B-phylogeny-estimates}
\end{figure}




It has also been found that $B$ (approximated by the average GC\textsubscript{3} content) correlates with certain life history traits.
Indeed, it correlates negatively with genome size in mammals \citep{romiguier2010contrasting}, likely because the per-megabase recombination rate is greater in short chromosomes \citep{kaback1992chromosome,lander2001initial,internationalchickengenomesequencingconsortium2004sequence}.

$B$ also correlates negatively with body mass, longevity and age of sexual maturity in mammals \citep{romiguier2010contrasting,lartillot2013interaction} and birds \citep{weber2014evidence,figuet2015impact,figuet2016life}, which was interpreted in terms of effective population size ($N_e$), since body mass negatively correlates with $N_e$ in both mammals \citep{damuth1981population,white2007relationships} and birds \citep{nee1991relationship}.\\

Nevertheless, this relationship between life history traits and GC-content dynamics is not (or not entirely) mediated by $N_e$ since no direct correlation between $N_e$ and $B$ has been observed among animals \citep{galtier2018codon}.
This unexpected observation has been interpreted by two non-mutually exclusive possibilities.
One interpretation would be that, since gBGC is a deleterious process \citep{galtier2009gcbiased,necsulea2011meiotic,lachance2014biased}, there may be a selective pressure to minimise $b$ in species with large $N_e$.

Alternatively, there may be a ‘dilution effect’ if, as in yeasts \citep{lesecque2013gcbiased}, only the SNPs located at the extremities of conversion tracts (CTs) are converted: in that case where only one part of the CT markers are subject to gBGC, the mean $b$ would decrease with $N_e$ since polymorphism correlates positively with $N_e$ \citep{tajima1996amount,woolfit2009effective}.

% TRANSITION
% The hypothesis that $b$ is constant has been ruled out \citep{galtier2018codon}, thus $r$, $L$ and/or $b_0$ should vary inversively with $N_e$ or this effect would be detected.

% , which rules out a hypothesis of a constant $b$ and suggests that $r$, $L$ and/or $b_0$ should vary inversively with $N_e$.







