\begin{savequote}[8cm]
‘[…] if there is one event in the whole evolutionary sequence at which my own mind lets my awe still overcome my instinct to analyse, and where I might concede that there may be a difficulty in seeing a Darwinian gradualism hold sway throughout almost all, it is this event — the initiation of meiosis.’
% ‘[…] if there is one event in the whole evolutionary sequence at which my own mind lets my awe still overcome my instinct to analyse […], it is this event — the initiation of meiosis.’
	
\qauthor{--- W. D. ‘Bill’ Hamilton, \textit{\usebibentry{hamilton1996narrow}{title}} \citeyearpar{hamilton1996narrow} }
\end{savequote}

\chapter{\label{ch:2-recombination-mechanistics}Meiotic recombination, the essence of heredity} 
%\otherpagedecoration

\minitoc{}


\begin{quote}
\textit{‘Why all this silly rigmarole of sex? Why this gavotte of chromosomes? Why all these useless males, this striving and wasteful bloodshed, these grotesque horns, colors… and why, in the end, novels, like }Cancer Ward\textit{, about love?’}

\qauthor{--- W. D. Hamilton, \textit{\usebibentry{hamilton1975review}{title}} \citeyearpar{hamilton1975review}}
\end{quote}

This is how the fanciful Bill Hamilton (1936—2000) sums up the mystery of sexual reproduction (or simply, ‘sex’) that has been puzzling biologists for over a century and which, to this day, remains unanswered \citep{de2007evolution, otto2009evolutionary}.

This so-called ‘paradox of sex’ finds its roots in that most theoretical arguments plead an elevated cost of sex as compared to asexual modes of reproduction \citep{otto2002evolution,lehtonen2012many}.
First, females invest half their reproductive resources in the production of males which, in turn, invest minimally into the progeny, as epitomised by the uncommonness of paternal care when it is not beneficial to the male \citep{smith1977parental,fromhage2007stability} — a concept known as the ‘twofold cost of sex’ or ‘cost of meiosis’ \citep{bell1982masterpiece}.
Second, the sexual act itself wastes time and energy to find and attract a sexual partner, and exposes the individual to the risks of contracting diseases and of being predated (sometimes by the mate itself), thus making sex a pearilous and unprofitable endeavour.

Nevertheless, only 80 \citep{vrijenhoek1989list,neaves2011unisexual} of the 70,000 vertebrate species discovered so far \citep{iucn2019} and as little as 0.1\% of all named animals \citep{vrijenhoek1998animal} reproduce otherwise than sexually. 
Such pervasiveness of sex in nature constitutes indisputable proof of its evolutionary success. 

But, given its considerable drawkbacks, how come sex has superseded all other forms of reproduction?
Over 20 theories have been put forward to answer this question \citep{kondrashov1993classification}, but the most generally claimed advantages revolve around the idea that sex both eliminates deleterious mutations and brings up more favourable combinations of alleles \citep{normarck2003genomic, speijer2016can}. 
%— even if this sole benefit may not be sufficient \citep{keightley2000deleterious}.
This defensibly profitable reshuffling of alleles is called ‘recombination’ and occurs during meiosis, the cellular process leading to the formation of gametes.\\

This chapter — named after a review on the subject \citep{hunter2015meiotic} — explores the cytological features of meiosis and the mechanistic principles of homologous recombination (HR), before venturing into the body of molecular actors enacting in this complex process and the reasons why their performance is critical for heredity.










% \addtocontents{toc}{\protect\pagebreak}
\section{Meiosis in the context of gametogenesis}

\subsection{A two-step division process to form gametes}

% Most sexually-reproducing beings have diploid cells, i.e.\ cells with two sets of chromosomes: one from each parent.
% The transmission of half this genetic material to the progeny goes through the formation of specialised haploid cells (i.e.\ counting a single set of chromosomes) called ‘gametes’.
% Such diploidy-to-haploidy transition occurs during a special type of cell division called ‘meiosis’ (from the Greek word \textit{\textgreek{μ}\textepsilon\textgreek{ίωσις}}: ‘lessening’).
% The evolutionary origin of meiosis is still a mystery \citep{lenormand2016evolutionary} but its wide occurrence in eukaryotes suggests that their last common ancestor had already acquired it \citep{cavalier2002origins,ramesh2005phylogenomic,speijer2015sex} through a process still largely debated \citep{wilkins2009evolution,bernstein2010evolutionary,bernstein2011meiosis}.
% Despite its somewhat blurry origins, its cytological features are conserved.\\

Most sexually-reproducing organisms have diploid cells, i.e.\ cells counting two sets of chromosomes: one from each parent.
The transmission of half this genetic material to the progeny goes through the formation of specialised haploid cells (i.e.\ cells encompassing a single set of chromosomes) called ‘gametes’.
Such transition from diploidy to haploidy occurs during a particular type of cell division called ‘meiosis’ (from the Greek word \textit{\textgreek{μ}\textepsilon\textgreek{ίωσις}}: ‘lessening’).
The evolutionary origin of meiosis is still a mystery \citep{lenormand2016evolutionary} but its wide occurrence in eukaryotes suggests that their last common ancestor had already acquired it \citep{cavalier2002origins,ramesh2005phylogenomic,speijer2015sex} through a process that is still largely debated \citep{wilkins2009evolution,bernstein2010evolutionary,bernstein2011meiosis}.
Despite its somewhat blurry origins, its cytological features are conserved.\\

% Figure vient de: http://www.macmillanhighered.com/BrainHoney/Resource/6716/digital_first_content/trunk/test/hillis2e/hillis2e_ch07_5.html
% LEFT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-3.4cm
	\rightskip-2.7cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth, trim = 0cm 0.61cm 0cm 0cm, clip]{figures/chap2/meiosis-with-cytology.eps}
	\captionsetup{width=1.22\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Chromosome organisation diagrams and cytological pictures of the two meiotic divisions leading to the formation of four gametes]
	{\textbf{Chromosome organisation diagrams and cytological pictures of the two meiotic divisions leading to the formation of four gametes.}
		\par During the first meiotic division (top), homologous chromosomes condense and pair up \textit{via} chiasmata at prophase I, line up on the equatorial plate at metaphase I, segregate at anaphase I and decondense at telophase I, thus giving two secondary gametocytes.
		During the second meiotic division (bottom), the sister chromatids of the two secondary gametocytes go through the same steps.
		Chromosome segments of the same colour (blue or pink) come from the same parental origin and centrosomes are coloured in green.
		\par This figure was reproduced from \citet{hillis2012principles} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:meiosis-cytological-steps}
\end{sidewaysfigure}

% % RIGHT PAGE
% \begin{sidewaysfigure}[p]
%     \centering
%     \leftskip-2.4cm
%     \rightskip-2.4cm
%     \rotfloatpagestyle{empty}
%     \includegraphics[width = 1.25\textwidth, trim = 0cm 0.61cm 0cm 0cm, clip]{figures/chap2/meiosis-with-cytology.eps}
%     \captionsetup{width=1.25\textwidth, margin={-2.2cm,-3.3cm}}
%     \caption{MMMy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blap y caption
%     \citep{hillis2012principles}
% }
% \label{fig:meiosis-cytological-steps}
% \end{sidewaysfigure}



% \footnote{Except for species with holocentric chromosomes (i.e.\ chromosomes devoid of any major centromeric constriction), like Lepidoptera, aphids and nematodes.}
%https://urgi.versailles.inra.fr/Projects/Achieved-projects/Holocentrism
Concretely, meiosis is preceded by a unique round of chromosome duplication occurring during the interphase of diploid germinal cells (ovocytes in females and spermatocytes in males).
Thence, before entering meiosis, each homologous chromosome (or ‘homologue’) i.e.\ each parental copy, is formed of two identical double-helix DNA molecules called ‘sister chromatids’ which are physically attached at a point called the ‘centromere’\footnote{Except for species with holocentric chromosomes (i.e.\ chromosomes devoid of any major centromeric constriction), like Lepidoptera, aphids and nematodes.} and adjoined along their whole length by cohesins \citep{klein1999central}.
Therefrom, the two successive cell divisions that compose meiosis will result in the distribution of the chromatids into four gametes.

The first meiotic division is also known as the ‘reductional division’ because it reduces ploidy by setting apart the homologues of each pair.
It is classically divided into four stages: prophase, metaphase, anaphase and telophase (Figure~\ref{fig:meiosis-cytological-steps}, top).
Prophase I, described more extensively in Subsection~\ref{chap2:prophase-I}, stages the pairing of homologous chromosomes along with recombination.
Next, the meiotic spindle bonds the paired homologues and lines them up on the equatorial plate during metaphase I, before separating them during anaphase I. 
Such partition of homologues is achieved thanks to the existence of two opposite forces that stabilise the chromosomes until they are correctly oriented: first, the chiasmata that maintain the homologues attached and second, the meiotic spindle that creates a poleward tension \citep{petronczki2003menage}.
The co-segregation of sister chromatids is likely due to a physical jointure of their kinetochores \citep{nasmyth2015meiotic}.
Segregation \textit{per se} terminates at telophase I during which the chromosomes decondense and a nuclear enveloppe (NE) forms around the nuclei.
At the end of the first meiotic division, each of the two haploid daughter cells (‘secondary gametocytes’) contains one pair of sister chromatids corresponding either to the paternal or to the maternal homologue.

Following a short interkinesis during which DNA does not replicate, the second meiotic division splits sister chromatids in a manner much similar to a haploid mitosis. 
This division is termed ‘equational’ because the number of chromosomes stays equal before and after it.
Like the first one, it is partitioned into four stages (Figure~\ref{fig:meiosis-cytological-steps}, bottom) executed synchroneously in the two secondary gametocytes.
During prophase II, the NEs break down and the chromatids recondense. 
In the meantime, the centrosomes duplicated during interkinesis move towards opposite poles while a new meiotic spindle forms in between and starts to capture chromatids.
The single chromosomes line up across the equational plates of each cell during metaphase II and sister chromatids segregate towards opposing poles during anaphase II\@.
At telophase II, the chromosomes begin to decondense and new NEs form around them, thus producing the final set of four genetically-unique haploid gametes.\\

% process of meiosis:
% https://cnx.org/contents/GFy_h8cu@9.87:GYZS3DDP@8/The-Process-of-Meiosis
% https://eggsnchromosomes.com/what-is-meiosis/
% https://www.khanacademy.org/science/biology/cellular-molecular-biology/meiosis/a/phases-of-meiosis
% http://www.macmillanhighered.com/BrainHoney/Resource/6716/digital_first_content/trunk/test/hillis2e/hillis2e_ch07_5.html
% https://www.ncbi.nlm.nih.gov/books/NBK26840/
% https://www.khanacademy.org/science/high-school-biology/hs-reproduction-and-cell-division/hs-meiosis/a/hs-meiosis-review
% https://www.britannica.com/science/meiosis-cytology
% https://www.khanacademy.org/science/high-school-biology/hs-reproduction-and-cell-division/hs-meiosis/a/hs-meiosis-review
% https://www.khanacademy.org/science/biology/cellular-molecular-biology/meiosis/a/phases-of-meiosis
% https://www.ncbi.nlm.nih.gov/books/NBK26840/



Albeit these general features of meiosis are shared, its timing and the products it forges are sexually dimorphic in mammals \citep[reviewed in][]{handel2010genetics}. 
Indeed, male meiosis forms four gametes (spermatids) whereas female meiosis ends in a single functional gamete and three non-functional haploid cells called ‘polar bodies’.
As for the timing, spermatogonia mature into spermatocytes which initiate meiosis all along male adulthood, thus ensuring a continuous production of sperm.
In contrast, the common conception in females is that the integrality of oogonia mature into ovocytes during fetal development \citep{pearl1921studies,zuckerman1951number}, even though recent findings suggest that oocyte production may be sustained in postnatal ovaries \citep{johnson2004germline,johnson2005oocyte}.
In any case, female meiotic prophase I — initiated and arrested right after the production of ovocytes — is resumed in small batches of ovocytes at periodic intervals during the reproductive lifespan.
It halts once again at metaphase II, until fertilisation by a spermatozoid (if it ever occurs) triggers the completion of the process.\\

While the transition from plain cell cycle to meiotic entry is managed by a complex body of checkpoints \citep[reviewed in][]{marston2005meiosis}, the metronomic completion of meiotic subprocesses is abundantly warranted by the capacity of chromosomes to respond to cell cycle controls \citep[reviewed in][]{mckim1995chromosomal}.
But the most regulated — and perhaps most critical — meiotic step is the synapsis of homologous chromosomes which takes place during prophase I.



\subsection{The synapsis of homologues during prophase I}
\label{chap2:prophase-I}

\subsubsection{Four differential degrees of synapsis}
Prophase I is commonly subdivided into four stages (Figure~\ref{fig:prophase-I-stages}): leptotene (or leptonema), zygotene (or zygonema), pachytene (or pachynema) and diplotene (or diplonema).
Each is characterised by a particular chromosomal configuration that mirrors their degree of ‘synapsis’ i.e.\ pairing of homologues.

At leptotene, chromosome ends connect the cytoskeleton located outside the nucleus \citep{scherthan1996centromere} \textit{via} their binding a complex body of SUN-domain proteins of the inner nuclear membrane (INM) that have beforehand bridged KASH-domain proteins of the outer nuclear membrane (ONM) \citep{tzur2006sun, yanowitz2010meiosis}.
This allows cytoplasmic forces to animate the motion of chromosome ends at the surface of the INM \citep{penkner2009meiotic} and ends at late leptotene by the formation of a ‘bouquet’ (Figure~\ref{fig:prepairing}) \citep{zickler1998leptotene} which constrains the chromosomes to a limited nuclear area \citep{zickler2006early}.

At zygotene, the homologous chromosomes begin to synapse, starting with the telomeric regions tethered in the bouquet \citep{pfeifer2003sex}.
By the end of pachytene, synapsis is complete for all pairs of chromosomes, with the notable exception of the non-homologous male X and Y chromosomes.
Instead, the sex chromosomes are transcriptionnally inactivated (‘meiotic sex chromosome inactivation’: MSCI) by remodelling into heterochromatin \citep{fernandez2003h2ax} and are pushed to the periphery of the nucleus where they form the ‘sex body’ \citep{handel2004xy} (Figure~\ref{fig:prophase-I-stages}.e.).
Then, during diplotene, the homologous chromosomes desynapse but remain attached in pairs \textit{via} their chiasmata.


\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/prophase-I-stages.eps}
	\caption[Chromosome organisation and cytology during prophase I]
	{\textbf{Chromosome organisation and cytology during prophase I.}
		\par Top: Two pairs of duplicated homologous chromosomes (red and blue) display different configurations in the four substages of meiotic prophase I.
		Double-strand break (DSB) formation at leptotene triggers both synapsis and the DSB resolution materialising as chiasmata during zygotene. Synapsis is completed at the onset of pachytene. Diplotene stages desynapsis, with homologues held together \textit{via} chiasmata.
		\par Bottom: immunofluorescence staining of synaptonemal complex protein 3 (SYCP3) and stage-specific signals on mouse spermatocyte spreads.\
		\textbf{a |} Meiosis-specific MEI4-homologue (MEI4) colocalises with the synaptonemal complex (SC).
		\textbf{b |} H2AX is phosphorylated (\textgreek{γ}H2AX) following DSB formation.\
		\textbf{c |} DNA recombinases DMC1 and RAD51 localise at DSB repair sites.\
		\textbf{d |} MutL protein homologue 1 (MLH1) localises at DSB sites repaired as COs.\
		\textbf{e |} Unrepaired DSB sites in the sex body are marked by \textgreek{γ}H2AX\@.
		\par This figure was reproduced from \citet{baudat2013meiotic} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:prophase-I-stages}
\end{figure}


\subsubsection{Presynaptic pairing}
Matching homologous chromosomes into pairs constitutes the most critical event of synapsis.
This challenge is colossal: for human cells, it compares to finding a 20-cm stretch — other than the sister chromatid — throughout the London-Moscow distance, simulaneously for hundreds of sites and coordinately with higher-order cellular processes \citep{neale2006clarifying}.\\

This search is likely facilitated by the establishment of pre-meiotic physical contacts between homologues \citep[reviewed in][]{mckee2004homologous, zickler2006early}.
Such presynaptic pairing has been evidenced in mice \citep{boateng2013homologous} and, although its detailed mechanism remains ignored, several theories wrestle to explain its formation. 

According to one of them, presynaptic associations may occur through DNA-DNA duplexes \citep{danilowicz2009single}. 
This assumption relies on the observation that meiotic chromosomes pair only when they are transcriptionally active \citep{cook1997transcriptional}. 
DNA duplexes could thus momentarily form within the ‘transcription factory’ to which DNA loops are attached \citep{xu2008similar}.
Alternatively, these associations may be promoted by sequence-specific RNA molecules, in a manner similar to gene silencing in plants and fungi \citep[cited in \citealp{zickler2006early}]{bender2004dna}.
A third scenario suggests a mechanism analogous to the ‘pairing centres’ (PC) or ‘homologue recognition regions’ (HRR) described in \textit{Caenorhabditis elegans} \citep{villeneuve1994cis,macqueen2005chromosome}, \textit{Drosophila melanogaster} \citep{mckee1996license} and \textit{Saccharomyces cerevisiae} \citep{kemp2004role}.
Namely, the \textit{cis}-acting PCs (or HRRs) could initiate interactions between homologues \citep{gerton2005homologous}.\\


In any case, demonstrating the existence of such presynaptic pairing in mice has driven \citet{boateng2013homologous} to propose a new model for homology search (Figure~\ref{fig:prepairing}).
With it, they challenge the commonly accepted view that homology search is triggered by the need to repair newly-opened DNA double-strand breaks (DSBs).
Instead, they propose that DSBs occur after the pre-leptotene pairing and that their repair serves as a prophase checkpoint to proofread the initial connection.
If that were so, homology search for DSB repair would be restrained to a reduced territory and thus, much facilitated \citep{barzel2008finding,mirny2011fractal}.\\

Whether or not this view is correct, chromosomal movements allow random collisions between chromosomes \citep{fung1998homologous}, thus creating opportunities for homologues to encounter and, more importantly, to disrupt unwanted (non-homologous) associations \citep{koszul2009dynamic}.
Yet, at this stage, the interstitial interactions between homologues are transient and reversible \citep{boateng2013homologous}.
They thus need to be strengthened by a higher-order chromosomal structure: the synaptonemal complex (SC).



\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/prepairing.eps}
	\caption[Mouse preleptotene DSB-independent pairing model proposed by \citet{boateng2013homologous}]
	{\textbf{Mouse preleptotene DSB-independent pairing model proposed by \citet{boateng2013homologous}.}
		\par \citet{boateng2013homologous}'s model stipulates that the tethering of telomeres (green points) to the NE in late preleptotene facilitates the initiation of synapsis at subtelomeric regions by simplifying the search for the homologous chromosome (light and dark grey lines). 
		The authors also conjecture that, upon entry into prophase (leptotene), this DSB-independent pairing at non-telomeric sites is lost, but that telomeric pairing is maintained at least at one end until homologues recombine. 
		Ultimately, DSB repair and synapsis at zygotene and pachytene would progressively restore pairing at non-telomeric sites.
		\par This figure was reproduced from \citet{boateng2013homologous} (permission in Appendix~\ref{app:permissions}).
		}
\label{fig:prepairing}
\end{figure}




\subsubsection{The synaptonemal complex (SC)}
The synaptonemal complex (SC), discovered by \citet{fawcett1956fine} and \citet{moses1956chromosomal}, is a remarkably well-conserved ribbon-like proteinaceous structure composed of three units: two dense lateral (or axial) elements (LE) and — except in the green alga \textit{Ulva} \citep{braten1973ultrastructure} and \textit{Chlamydomonas} \citep{storms1977fine} — one less dense central element (CE) (Figure~\ref{fig:synaptonemal-complex}) \citep{schmekel1995central}.

LEs resemble axes along which the sister chromatids are loaded, binding short stretches of DNA to the LE and condensing the rest of it into long loops of tens to hundreds of kilo base pairs (kb). 
Generally, the loops closer to the telomeres are much shorter than the ones located elsewhere \citep{heng1996regulation}.

LE assembly begins at leptotene with the aggregation of both REC8 cohesins and axial proteins (SCP2 and SCP3 in mammals) into small fragments \citep{eijpe2003meiotic} which later fuse into full LEs \citep{schalk1998localization}.
At full synapsis, they are connected to the CE (formed of SYCE1 and SYCE2 proteins \citep{pera2013stem}) by transverse filaments (TFs), thus giving the SC a striated, zipper-like appearance.
The main constituent of TFs — the SCP1 protein, in mammals — has homologues in worms \citep{macqueen2002synapsis,colaiacovo2003synaptonemal}, flies \citep{mckim2002meiotic} and yeasts \citep[reviewed in][]{zickler1999meiotic} that, despite little sequence conservation, display a similar structure: two head-to-head homodimers of an ${\sim}80$~nm coiled coil flanked by globular C and N termini \citep{meuwissen1992coiled,liu1996localization}.
The polymerisation of these central region proteins between paired homologue axes results in the tight pairing (${\sim}100$~nm) of the bivalents\footnote{Homologous chromosomes} along their entire length at the end of pachytene \citep{page2004genetics}, as compared to their ${\sim}400$-nm spacing during presynaptic alignement \citep{tesse2003localization}.\\


\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/synaptonemal-complex.eps}
	\caption[Structure of the synaptonemal complex (SC)]
	{\textbf{Structure of the synaptonemal complex (SC).}
		\par Original legend by the author: ‘The SC consists of a pair of parallel strands, the lateral
		elements, that are linked by transversal filaments. The central element runs halfway between the lateral
		elements. Loops of sister chromatids are tethered both to each other and to a lateral element. Synapsis
		progresses along pairs of homologous chromosomes in a zipper-like fashion. The axes of unsynapsed
		portions are called axial elements. Initial homologous interactions may or may not need axial elements. The
		sites of crossing over are marked by recombination nodules, which are located between the axial elements.’
		\par This figure was reproduced from \citet{loidl2016conservation} (permission in Appendix~\ref{app:permissions}).
		}
\label{fig:synaptonemal-complex}
\end{figure}



Synapsis is indeed the most commonly acknowledged role of the SC, but it may also act to limit recombination with the sister chromatid.
Avoiding the sister may seem a trivial problem given the 2:1 odds ratio in favour of homologue templates \citep{lao2010trying}. 
However, an important guarantee of genome stability is the preferential use of the sister chromatid in mitotically dividing cells \citep{kadyk1992sister,bzymek2010double} which is likely promoted by their cohesin-dependent proximity \citep{sjogren2010sphase}.
Thus, switching this mitotic inter-sister bias to a meiotic inter-homologue bias is essential for synapsis.
Even though this could be ensured by other features of meiosis \citep[reviewed in \citealp{humphryes2014non}]{schwacha1997interhomolog, goldfarb2010frequent, hong2013logic}, recent evidence points that the components of the CE are effectively involved in template choice \citep{kim2010sister} as was suggested in the past \citep{haber1998meiosis}.\\

Microscopy observation of the SC reveals dense nodules where recombination occurs (‘recombination nodules’) \citep{carpenter1975electron, schmekel1998evidence}.
Indeed, the formation of DSBs is a prerequisite for SC formation in many species including plants and mammals \citep{zickler1999meiotic, henderson2004tying}.
Yet, the meiotic program seems to vary for other species: SC formation is recombination-independent in species with holocentric chromosomes like \textit{Caenorhabditis elegans} \citep{dernburg1998meiotic} and \textit{Bombyx mori} \citep{rasmussen1977transformation} but also in \textit{Drosophila} females \citep{mckim1998meiotic} (and recombination does not even occur in \textit{Drosophila} males, as reviewed in \citealp{tsai2011homologous}) whereas \textit{Schizosaccharomyces pombe} \citep{bahler1993unusual} and \textit{Aspergillus nidulans} \citep{egel1982meiosis} recombinate but have no SC \citep[reviewed in][]{zickler2015recombination}.

More generally, whenever SC is associated to recombination, it is clear that its correct formation is of paramount importance to create stable DNA connections between homologues \citep[reviewed in \citealp{hunter2003synaptonemal}]{hunter2001singleend}.
If, contrariwise, it builds improperly, the resulting asynapsis may have dramatic consequences on the fate of maturating gametes.







\subsection{Impaired meiosis-associated diseases}

\subsubsection{Asynapsis}
To prevent the formation of abnormal gametes, surveillance systems (a.k.a.\ ‘checkpoints’) chase after defects at several meiotic stages \citep[reviewed in][]{handel2010genetics}.
In particular, the ‘pachytene checkpoint’ \citep{roeder2000pachytene} monitors chromosome synapsis in \textit{Saccharomyces cerevisiae} \citep{wu2006two}, \textit{Drosophila melanogaster} \citep{ghabrial1999activation,abdu2002activation} and \textit{Caenorhabditis elegans} \citep{bhalla2005conserved}.
In mammals however, this one in multiple surveillance systems \citep{barchi2005surveillance} seems to be associated to the completion of recombination rather than to synapsis \textit{per se} \citep{li2007mouse}.\\

An early pachytene response to asynapsis in both mice \citep{baarends2005silencing,turner2005silencing} and humans \citep{ferguson2008silencing,sciurano2007asynaptic} is the meiotic silencing of unsynapsed chromatin (MSUC). 
In normal males, its specialisation, meiotic sex chromosome inactivation (MSCI), silences sex chromosomes in both mammals and birds \citep{schoenmakers2009female} and leads to their compartmentalisation into the sex body (Figure~\ref{fig:prophase-I-stages}).

MSUC of only one asynapsed chromosome (on top of the sex chromosomes) allows to escape apoptosis\footnote{Programmed cell death (from the Greek word \textgreek{ἀπόπτωσις}: ‘falling off’)} \citep{mahadevaiah2008extensive,jaramillo-lambert2010single}, the normal response to asynapsis \citep{hochwagen2006checking}.


\subsubsection{Infertility and cancer}
Because it leads to cell death or cell-cycle arrest in most cases, asynapsis is associated with impaired fertility and cancer.
Indeed, mice and yeasts that lack an enzyme decisive for proper synapsis (DMC1) are sterile \citep{bannister2007dominant} while a deficiency in other synapsis-associated proteins provokes tumors \citep{moynahan2002cancer, jasin2002homologous}.\\

Regarding sex effects, chromosomal anomalies associated with asynapsis are found in 3\% of infertile men \citep[cited in \citealp{burgoyne2009consequences}]{vincent2002cytogenetic} and, more generally, mammalian males are more severely affected by asynapsis-dependent sterility than females (reviewed in \citealp{burgoyne2009consequences} and \citealp{hunt2002sex}), likely because meiosis checkpoints are either less numerous or less efficient in females \citep{champion2002playing}.

The converse is true for aneuploidy: since female checkpoints interrupt a smaller proportion of abnormal meioses, they exhibit a higher rate of unbalanced conceptions.



\subsubsection{Aneuploidy}
In humans, aneuploidy is the primary cause of miscarriage and congenital birth defects \citep{hassold2007origin}.

As one studied chromosome proved to transmit properly even in the absence of chiasma \citep{fledel-alon2009broadscale}, the incapacity to control for proper disjunction, — rather than the effective number of recombination events, — may cause these irregularities.
These female-specific failures are likely due to the dictyate arrest: female chiasmata, formed at the fetal age, have to hold for decades until puberty resumes meiosis.
Consequently, they may degrade over time \citep{hassold2001err}.
In accordance with this hypothesis, the frequency of Down Syndrome (a.k.a.\ trisomy 21) \citep{penrose2009relative} and other human trisomies \citep[reviewed in \citealp{hassold1996human} and \citealp{smith1998recombination}]{morton1988maternal} are positively correlated with maternal age. 
In yeasts too, trisomies correlate with parental age \citep{boselli2009effects}.
% In accordance with this hypothesis, the frequency of Down Syndrome (a.k.a.\ trisomy 21) — as well as other trisomies in humans \citep{morton1988maternal} and yeasts \citep{boselli2009effects} — is positively correlated with maternal age \citep{penrose2009relative}.\\

These aneuploidy defects are caused by segregation errors, 80\% of which arising during the first meiotic division and many involving an achiasmate bivalent \citep{szekvolgyi2010meiosisa}.
Therefore, this suggests that one of the most crucial features of meiosis is that yielding chiasmata: homologous recombination (HR).










\addtocontents{toc}{\protect\pagebreak}
% % % % \addtocontents{toc}{\protect\newpage}
\section{Models of homologous recombination (HR)}

Ever since the unexpected observations on fungal products of meiosis (see Chapter~\ref{ch:1-history-genetics}), a few \textit{aficionados} with a craving to understand the exchange of genetic information between chromosomes have come up with theoretical models of homologous recombination (HR).

The Holliday model \citep{holliday1964mechanism} was the first widely accepted molecular explanation of the relationship between aberrant segregation and crossing-over. 
It has since then been refuted by posterior discoveries but one of its concepts, the ‘Holliday junction’ (HJ), remains a key feature in all current models of HR\@.


\subsection{The Holliday junction (HJ)}

One central tenet of the Holliday model lies in the idea that DNA can break, thus allowing complementary sequences to pair in a cruciform structure that was later designated as the ‘Holliday junction’ (HJ).
The HJ forms as a consequence of the single-end invasion (SEI) of a nicked DNA strand into the homologous, intact chromosome.

Double Holliday junctions (dHJs) have later been directly observed in recombination intermediates of yeasts \citep{schwacha1994identification, schwacha1995identification}.
However, these studies, like prior works \citep{sun1989double,cao1990pathway}, have shown that recombination does not start with single-strand nicks as enunciated in the Holliday model, but with double-strand breaks (DSBs) as posited in the DSB repair (DSBR) model.


\subsection{Double-strand break repair (DSBR)}

The double-strand break repair (DSBR) model \citep{szostak1983doublestrandbreak} was originally developped from yeast studies \citep{orr-weaver1981yeast, orr-weaver1983yeast} and postulates the formation of DSBs. 
The broken ends are then processed into two single-stranded DNA (ssDNA) tails. 
One of them invades the homologue by displacing one of its intact strands into a D-shaped loop designated as the ‘D-loop’. This forms the prime HJ (Figure~\ref{fig:pathways-resolution}).
Following DNA synthesis of the invading strand, the D-loop broadens sufficiently to anneal the opposite, free 5’ end.
This completes the formation of a second HJ, crisscrossed with the first one.
According to this model, the newly formed dHJ is later resolved into a crossing-over (CO) or a non-crossover (NCO) with a 50:50 odds-ratio.

Many of the predictions of this model revealed true and, as such, it is still used today (see Subsection~\ref{chap2:resolution-intermediates}).
But the prognosis regarding the equal number of COs and NCOs was never confirmed biologically \citep{bishop2004early} which implied the existence of an alternative mechanism yielding NCOs.

%Et parler de asymmetric VS symmetric heteroduplex DNA (orr-weaver and szostak — section Polarity)????????????




\subsection{Synthesis-dependent strand annealing (SDSA)}
\label{chap2:model-SDSA}

The synthesis-dependent strand annealing (SDSA) model \citep{resnick1976repair,nassif1994efficient,ferguson1996recombinational} shares its initial steps with the DSBR model: it begins with a DSB and involves a SEI that extends along the recipient strand, thus tangibly translocating the D-loop in a process called ‘bubble migration’ \citep[reviewed in][]{mcmahill2007synthesisdependent}.
Once it has elongated past the DSB site, the D-loop is disrupted and the invading strand anneals its original complementary ssDNA on the \textit{vis-\`a-vis} side of the DSB\@.
Last, the remaining gaps are filled in by DNA synthesis and ligation.
This generates NCOs \textit{prior} to the formation of dHJs in the DSBR pathway \citep{allers2001differential}.\\



In the past decades, many experimental studies have uncovered additional spatial and temporal features of meiotic recombination, many of which being in accordance with the aforementioned HR models. 
I review these findings in the upcoming section.













\section{Molecular mechanisms of recombination}

Homologous recombination (HR), which occurs during prophase I, leads to the formation of a (relatively) long-term connection that maintains the bivalents together until their separation at anaphase I\@.

It begins at leptotene with the formation of a DNA double-strand break (DSB) on one homologue. 
To repair properly, this crack needs a DNA strand to use as template. There begins a homology search accomplished at zygotene by the broken-strand invasion onto the mating chromosome.
The template-based repair process creates a transient structure, subsequently resolved into either a crossing-over (CO) or a non-crossover (NCO) during late zygotene and pachytene.

In mammals, each of these actions is executed by a complex body of proteins summarised in Figure~\ref{fig:recombination-mammalian-proteins}. 



\subsection{Initiation of recombination}

The evolutionarily conserved SPO11 endonuclease — observed in a wide range of species \citep{baudat2000chromosome,mckim1998meiw68,romanienko2000mouse,steiner2002meiotic,bowring2006chromosome,stacey2006arabidopsis} — catalyses the programmed formation of DSBs \citep{keeney1997meiosisspecific,bergerat1997atypical} that marks the beginning of HR \citep{sun1989double}.
Of the two isoforms found in mice \citep{metzler-guillemain2000identification}, SPO11\textgreek{β} is the one responsible for DSB formation \citep{bellani2010expression}.
DNA cleavage by this homodimeric protein leaves a two-nucleotide 5' overhang \citep{demassy1995nucleotide} onto which it remains trapped till the further processing of DSB ends (see Subsection~\ref{chap2:DSB-repair}) \citep[reviewed in][]{cole2010evolutionary}.

Several other proteins have been identified as essential for the correct formation of DSBs (extensively reviewed in \citealp{keeney2008spo11a} and \citealp{demassy2013initiation}).
Among them, the yeast Mer2-Mei4-Rec114 complex \citep{li2006saccharomyces,maleki2007interactions} and two of its mouse homologues (MEI4 and REC114) have been identified as functional and important for nick generation \citep{kumar2010functional,kumar2015mei4}, thus suggesting a conserved mechanism for recombination initiation. 
Nevertheless, the mammalian system has some specificities since MEI1 \citep{libby2002mouse,libby2003positional}, which does not set forth any yeast homologue, has been uncovered as essential for normal DSB levels, along with HORMAD1 (yeast homologue: Hop1) \citep{shin2010hormad1,daniel2011meiotic}.

Once DSBs have been generated, the ataxia telangiectasia mutated (ATM) kinase both phosphorylates the 139\textsuperscript{th} serine residue of histone H2AX variants located in their vicinity (then named \textgreek{γ}H2AX) \citep{rogakou1998dna,burma2001atm} and thwarts further DSB apertures of DNA \citep{lange2011atm,lukaszewicz2018control}.\\

In mice and humans, ${\sim}$200—400 DSBs initiated in this manner at early leptotene are required to avoid defects in synapsis \citep{kauppi2013numerical,smagulova2013suppression}.
From this point forward, they thus have to be repaired to secure the production of viable gametes.







\subsection{Repair of double-strand breaks (DSBs)}
\label{chap2:DSB-repair}

\begin{figure}[p]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/recombination-mammalian-proteins.eps}
	% \includegraphics[width = 0.7\textwidth, trim = 0cm 0cm 11.65cm 0cm, clip]{figures/chap1/morgan-drosophila-cross-results.eps}
	\caption[Proteins involved in mammalian meiotic recombination]
	{\textbf{Proteins involved in mammalian meiotic recombination.}
		\par \textbf{a |} DNA double-strand break (DSB) formation (blue triangles) is catalysed by SPO11 (purple spheres) on the chromosome axes and requires MEI1, MEI4, REC114 and HORMAD1.
		\textbf{b |} Endonucleolytic cleavage of DSB ends by MRE11, RAD50, NBS1, CTIP and Pol \textgreek{β} forms SPO11-oligonucleotide complexes of 12—36 nucleotides (purple spheres with tails). EXO11 further enacts a 5’-to-3’ resection of DSB tails.
		\textbf{c |} Strand invasion is catalysed by DMC1 and RAD51 recombinases in the presence of several co-factors: HOP2, MND1, RAD52, RAD54, BRIT1, BRCA1 and BRCA2. RPA and NBPA2 bind recombination intermediates. At this stage (zygotene), homologous chromosomal axes are synapsed at DSB repair sites by proteins of the synaptonemal complex, including SYCP1 (brown segments).
		\textbf{d |} Recombination intermediates are stabilised by BLM-RMI1-TOP3, TEX11, MCMC8, MSH4-MSH5 and RNF212. They are further processed into either double Holliday junctions (CO intermediates) or single-end strand invasions (non-crossover intermediates).
		\textbf{e |} Resolution into crossovers requires MLH1, MLH3 and EXO1 while non-crossovers are formed after strand displacement and annealing. Non-crossovers formed \textit{via} alternative pathways are not shown. Recombination products are generated at the end of pachytene. Gene conversion (unidirectional transfer of genetic information in the vicinity of DSB) is present in both products.
		\par Proteins marked with an asterisk (\textsuperscript{*}) are predicted to be involved, but not yet confirmed by experimental evidence.
		Chromatin loops and chromosome axes during zygotene are illustrated in the top right.
		\par This figure was reproduced from \citet{baudat2013meiotic} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:recombination-mammalian-proteins}
\end{figure}



\subsubsection{DSB-end processing}
The repair of DSBs begins with the processing of its ends: an endonucleolytic cleavage several nucleotides downstream of the 5' end \citep{neale2005endonucleolytic} is executed by the Mre11/MRE11 complex both in yeasts \citep[reviewed in][]{borde2009double} and mammals \citep[reviewed in][]{borde2007multiple}.
In \textit{Saccharomyces cerevisiae} and \textit{Caenorhabditis elegans}, Mre11/MRE11 acts collaboratively with Rad50/RAD50 and Xrs2/NBS1, two proteins required for DSB mending \citep[reviewed in][]{lam2015mechanism}. 
Both have mammalian homologues, but their putative role in DSB repair \citep[reviewed in][]{baudat2013meiotic} is hard to prove since knocking them out is lethal for mice \citep{luo1999disruption,zhu2001targeted}.
In contrast, that of a DNA polymerase (Pol \textgreek{β}) in releasing SPO11-bound oligonucleotides at DSB ends was unequivocally evidenced \citep[reviewed in \citealp{baudat2013meiotic}]{kidane2010dna}.



\subsubsection{Single-end invasion (SEI)}
As removal of SPO11 is paired with the 5’-to-3’ end resection of the DSB, 3’ single-stranded DNA (ssDNA) tails become accessible to the nuclear machinery (Figure~\ref{fig:recombination-mammalian-proteins}.b.).
As such, RPA proteins rapidly bind them \citep{he1995rpa} but are then displaced by RAD51 and/or DMC1 recombinases \citep{pittman1998meiotic,yoshida1998mouse} which catalyze the pairing and exchange between the ssDNA strand and the intact, homologous double-stranded DNA (dsDNA).
Their relationship is complex: RPA is necessary both for RAD51 filament formation and for DMC1-catalysed strand exchange, but notwithstandingly, it also competes with them for ssDNA binding \citep{sung2003rad51}.

The proper functioning of DMC1 and RAD51 in strand invasion requires several other proteins that interact with either one or both of them: HOP2 and MND1 \citep{bugreev2014hop2mnd1}, BRCA1 \citep{scully1997association} and BRCA2 \citep{thorslund2007interactions}.
This complex process also requires other, less well-characterised actors that I will not describe here for they are of little interest for the scope of this thesis \citep[but for review, see][and Figure~\ref{fig:recombination-mammalian-proteins}.c.]{neale2006clarifying}.

Next, the sensor proteins of the mismatch repair (MMR) system (MSH2-MSH3 and MSH2-MSH6 complexes in mammals) control the identity between the targeted strand and the invader.
When it is insufficient, the latter is rejected and repaired using the sister chromatid instead, thus preventing any potentially deleterious ectopic recombination (reviewed in \citealp{surtees2004mismatch} and \citealp{goldfarb2010frequent}).


\subsubsection{Recombination-intermediate processing}
The interaction between the invading strand and the homologue is subsequently stabilised by several proteins.
Indeed, BLM, TEX11 (yeast homologue: Zip4) and RNF212 (yeast homologue: Zip3) appear at zygotene at recombination foci and progressively decrease until the end of pachytene, i.e.\ when DSBs are repaired \citep[reviewed in][]{baudat2013meiotic}. 
In addition, together with MCM8 and MCM9 proteins \citep{lutzmann2012mcm8}, heterodimers of MSH4 and MSH5 \citep{scully1997association} are required for synapsis stabilisation in both mice \citep{devries1999mouse,kneitz2000muts} and humans \citep{snowden2004hmsh4hmsh5}.

Though, the role of MSH4 continues beyond synapsis establishment. 
Indeed, the stabilisation of the interaction between the two homologues creates an intertwined recombination intermediate structure, and MSH4 participates in its resolution when it leads to COs, but also, as argued by \citet{baudat2007regulating}, to NCOs.





\subsection{Resolution of recombination intermediates}
\label{chap2:resolution-intermediates}

Recombination intermediate structures may be resolved \textit{via} two main pathways (Figure~\ref{fig:pathways-resolution}).
In the pathway leading to COs, the non-invading strand of the broken chromosome interacts with the displaced homologue strand which forms the D-loop.
In constrast, in the pathway leading to NCOs, the non-invading strand anneals again the invading strand from the same chromatid, after the latter has elongated on the homologue and displaced from it.
Assertedly, these two pathways presuppose the production of distinct recombination intermediates (Figure~\ref{fig:recombination-mammalian-proteins}.d.\ and e.).


\begin{figure}[p]
	\centering
	% \includegraphics[width = 0.85\textwidth]{figures/chap2/baudat-pathways-resolution.eps}
	\includegraphics[width = 1\textwidth]{figures/chap2/Wyatt-west-2014-fig1.eps}
	\caption[Molecular mechanism of pathways leading to crossing-overs (COs) and non-crossovers (NCOs)]
	{\textbf{Molecular mechanism of pathways leading to crossing-overs (COs) and non-crossovers (NCOs).}
		\par Resected DSBs invade homologous duplex DNA to form a D-loop structure.
		The invading 3’ end then serves as a primer for DNA synthesis, which leads to the capture of the second end and, ultimately, to the formation of a double Holliday junction.
		This junction is then either dissolved into a NCO, or resolved symmetrically into a CO or asymmetrically into a NCO\@.
		\par This figure was reproduced from \citet{wyatt2014holliday} (permission in Appendix~\ref{app:permissions}).
	}
\label{fig:pathways-resolution}
\end{figure}


\subsubsection{The CO pathway}
In certain cases, the homologues are physically bound twice: one strand from each chromosome (the invading strand and the D-loop strand) displaces to bind the homologue, thus creating a double Holliday junction (dHJ) in step with the DSBR model.
TEX11 (yeast homologue: Zip4), RNF212 (yeast homologue: Zip3) and HFM1 (yeast homologue: Mer3) are thought to play a role in processing the dHJ, since knocking one of them out leads to a diminished level of chiasmata and COs \citep[reviewed in \citealp{baudat2013meiotic}]{adelman2008zip4h,guiraldelli2013mouse,reynolds2013rnf212}.
In yeasts, Mer3, one of the proteins of this ZMM complex, specifically interacts with Rad51 to stimulate heteroduplex extension \citep{mazina2004saccharomyces}.\\

The resolution of the dHJ \textit{per se} is catalysed by resolvases, i.e.\ enzymes that slice the interwound strands.
In mice, a pair of symmetrical nicks is introduced across the helical branchpoint of most (90\%) dHJs either by a heterodimer of MLH1 \citep{baker1996involvement, edelmann1996meiotic} and MLH3 \citep{lipkin2002meiotic} or by EXO1 \citep{wei2003inactivationa}.

Alternatively, the dHJ can be resolved by introducing two single-stranded incisions \citep{wyatt2014holliday}. 
In that case, the two nicks are asymmetric and can be located several nucleotides away from the branchpoint. 
This resolution is catalysed by MUS81 and EME1 (yeast homologue: Mms4).
In \textit{Schizosaccharomyces pombe} where it was first discovered, it is the only pathway to produce COs \citep{osman2003generating}.
However, in plants \citep{mercier2005two} and in mice \citep{holloway2008mus81}, it coexists with the MLH1-dependent CO pathway.\\


Of the 200—400 recombination foci in mice, only $\sim$20 (approximately one per chromosome) lead to a CO \citep{baudat2007regulating}.
This implies the existence of another repair pathway: that leading to NCO events.






\subsubsection{The NCO pathway}
Instead of being resolved, the dHJ is sometimes dissolved by the BLM helicase together with a topoisomerase \citep{wu2003bloom}. 
This pathway thus interferes with the formation of COs. Indeed, inactivating BLM leads to an increased number of chiasmata \citep{holloway2010mammalian}.

Though, most NCOs are formed \textit{via} another pathway that occurs before the resolution of dHJs: the synthesis-dependent strand annealing (SDSA) pathway (see Subsection~\ref{chap2:model-SDSA}).
In \textit{Saccharomyces cerevisiae}, it produces the large majority of NCOs \citep{martini2011genomewide} and the dissociation between the invading strand and the homologue is promoted by Srs2 \citep{ira2003srs2}.
However, the latter helicase does not have any mammalian homologue \citep{spell2004examination}. Therefore, the molecular operations of SDSA in mammals are still unclear.\\


Altogether, the resolution of a genetically programmed DSB into a CO \textit{versus} a NCO outcome seems to be decided early: in most species — with the notable exception of \textit{Drosophila} \citep{crown2014eliminating}, — they arise from distinct intermediates \citep[reviewed in][]{hunter2015meiotic}.
This intermediate structure involves the formation of a heteroduplex, which, in mammals, can spread over 500—2,000 bp for COs, but generally less than 300, and sometimes as little as tens of base pairs, for NCOs \citep{jeffreys2004intense,ng2008quantitative}.
Heterozygous markers located within the heteroduplex are either all converted in the same direction (in that case, the conversion tract of the CO or NCO is said to be ‘simple’) or alternate converted and unconverted markers (in that case, the conversion tract is said to be ‘complex’) \citep{borts1989length}.

In contrast, non-programmed DSBs, which correspond to DNA lesions, are repaired through different processes \citep[reviewed in][]{sung2006mechanism}.
Such spontaneous DSBs are frequent in mitotic cells where genome integrity is the absolute priority.
Consequently, mitotic breaks are mainly repaired by recombining with the genetically identical sister chromatid, or \textit{via} one of two repair systems that are more error-prone \citep{smith2001influence}: non-homologous end-joining (NHEJ), which consits in directly ligating the broken strands of DNA \citep{weterings2004mechanism} or single-strand annealing (both reviewed in \citealp{helleday2003pathways} and \citealp{moynahan2010mitotic}).

Recombination may also occur between non-allelic sequences located at different genomic locations — generally low copy repeats resulting from duplication events \citep{bailey2006primate}. 
This is called non-allelic homologous recombination (NAHR) (or ‘ectopic recombination’) and proceeds similarly to HR \citep{sasaki2010genome}.


Distinguishing between HR and NAHR implies knowing where recombination effectively takes place on the genome, which is the object of the next chapter.

% NOTE: Image pour pathwat: http://jcs.biologists.org/content/124/4/501 (fig1)



