\begin{savequote}[8cm]

	‘I suppose the process of acceptance will pass through the usual four stages:\\
	(i) this is worthless nonsense;\\
	(ii) this is an interesting, but perverse, point of view;\\
	(iii) this is true, but quite unimportant;\\
	(iv) I always said so.’
	
	\qauthor{--- John B. S. Haldane, \textit{\usebibentry{haldane1963truth}{title}} \citeyearpar{haldane1963truth} }
	
\end{savequote}

\chapter{\label{ch:9-discussion}Implications for mammalian genome evolution}
%\otherpagedecoration


\minitoc{}


The work presented in this thesis allowed to better comprehend recombination and its impact on genome evolution: 
in brief, we precisely characterised patterns of recombination in mice, brought preliminary answers to the specific role of one gene essential to recombination, and quantified the contribution of all types of recombination products to GC-content evolution \textit{via} biased gene conversion.

The progress we made on this topic principally rested on the analysis of the recombination events we could detect in mouse autosomal hotspots with the approach we developed. 
In this chapter, I will first discuss both the significance and the limitations of our method in the context of studies on recombination.
Next, I will try and give answers upon the original motivation for this work: figuring out the interplay between the effective population size ($N_e$) and the gBGC coefficient ($b$), by comparing our findings in mice to those of others in humans.
Last, I will provide more speculative interpretations about this interplay and the evolution of biased gene conversion in general.


\section{Significance and limitations of our method}
\subsection{Comparison with classical pedigree approaches}

The method we implemented to detect recombination events exhibits several advantages as compared to the more classical approach of pedigree analysis.

First, it allows to quantify and precisely characterise recombination events in a single individual whereas the events identified by pedigree analysis span at least several tens — and sometimes a few hundred — members of a given family.

Second, because we specifically targeted recombination hotspots, we only needed to sequence $\sim$244 Gb of DNA (as compared to the sequencing of between 500 and 50,000 Gb in comparable studies) and identified several thousands of events, whereas pedigree approaches cap at several hundreds \citep{halldorsson2016rate,smeds2016highresolution} or at a few thousands of events at best \citep{li2018highresolution}.
Thus, our method was much more powerful than pedigree analyses: the number of events detected per Gb sequenced with our method (77.1 events/Gb) was over 100 times as great as that of a recent study carried on mice by \citet{li2018highresolution} (0.604 events/Gb).
Despite the fact that the recombination rate is respectively twice and six times as high in humans and in flycatchers as in mice \citep{kawakami2014highdensity,kawakami2017wholegenome}, the power in detecting events in these two species \textit{via} pedigree analyses (0.00970 events/Gb and 1.18 events/Gb, respectively) was also largely lower than \textit{via} our method \citep{halldorsson2016rate, smeds2016highresolution}.

In addition, even if our approach was originally designed to study F1 hybrids, we showed that it could be extended to more complex designs to study recombination (see Chapter~\ref{ch:8-HFM1}).
Indeed, we managed to deal with the incorporation of a third genome and, theoretically, this adaptation should be possible with any other number of genomic introgressions.\\

As such, our approach indisputably outperforms classical pedigree analyses in detecting recombination.
This paves the way to study the individual role of genes that are essential to recombination, as we did in Chapter~\ref{ch:8-HFM1}.
Notwithstandingly, our method also encompasses a number of limitations, which will be discussed in the next two subsections. 



\subsection{A \textit{prior} knowledge of recombination hotspots in males}

To detect recombination events, one obvious prerequisite is the presence of polymorphic sites.
We thus selected hotspots that displayed a minimum of 4 markers in the 300-bp central region.
Yet, as many SNPs in F1 hybrid hotspots result from hotspot erosion in one parental lineage \citep{smagulova2016evolutionary}, this minimum-number-of-SNPs requirement led to a slightly greater proportion of asymmetric (i.e.\ eroded in one lineage) hotspots than would be expected with a random selection (Table~\ref{tab:classification-hotspots}).

As \citet{li2018highresolution} pointed out, such asymmetric hotspots display, on average, lower recombinational activities than expected on the basis of PRDM9 ChIP-seq binding (Figure~\ref{fig:correlation-PRDM9-standard-error}).
Thus, the overall recombination rate we extrapolated from our data is likely to be slightly underestimated (see Chapter~\ref{ch:6-recombination-parameters}).

What's more, by definition, hotspot asymmetry implies a haplotype bias for PRDM9 binding.
As such, our enrichment of targets in asymmetric hotspots likely amplified dBGC, and thus, the variations in dBGC intensity we observed may be somewhat more extreme than what would be expected on average hotspots.\\

Therefore, because of the hotspot selection step, the recombination events directly observable with our approach are those occurring in the close vicinity of highly polymorphic hotspots and, if their characteristics differ from those of the other, non-observable events, they may not be representative of the totality of recombination events.

In addition, our approach necessitates a large quantity of gametes to be analysed.
As such, it is much better suited to the study of recombination in males and, thus, does not permit to give insight into the process of recombination in the other sex.




\subsection{The issue of NCO detectability}

To minimise the rate of false positive calls, we filtered out all fragments that did not include a minimum of two \textit{B6}- and two \textit{CAST}-typed variants. 
This implies that, for a NCO to be detected, its conversion tract (CT) must be long enough to overlap at least two variants. 
Since NCO CTs are only a few base pairs to a few tens of base pairs long \citep{cole2014mouse}, one would \textit{a priori} expect a non-negligible proportion of them to be intrinsically undetectable, especially in regions with low marker density.

In particular, single-marker NCO (NCO-1) events cannot be detected directly with our approach.
As for multiple-marker NCO (NCO-2+) events, their level of detectability depends on marker density, which can vary across — but also along — hotspots.
To make this along-hotspot fluctuation visible, we added that information (the maximum number of Rec-1S and Rec-2S switch points detectable) for each existing marker-marker interval (Figure~\ref{fig:example-recombinants}).\\

Given that many events are undetectable, we used an approximate bayesian computation (ABC) approach to estimate the genuine values of certain recombination parameters: the lengths of CO and NCO conversion tracts (CTs) and the CO:NCO ratio.
Since the estimates that we obtained with the ABC were extremely close to the direct observations of CO and NCO CT lengths in a few mouse hotspots and to the CO:NCO ratio predicted on the basis of cytological estimates of DSBs (see Chapter~\ref{ch:6-recombination-parameters}), we are confident that this approach was globally valid.

Still, it should be noted that the validity of the ABC rests on the assumptions that were made to simulate recombination events. 
Notably, we hypothesised that the CO:NCO ratio was identical for all hotspots, that both CO and NCO CTs were centred on the DSB and that their CT lengths were arranged according to a unimodal distribution, i.e.\ that the resolution process was \textit{not} perceptibly different for any subclass of COs or NCOs.
But, since the process of recombination has not been completely elucidated yet, we cannot know whether the latter assumptions were biologically accurate nor whether other hypotheses could be more relevant to simulate these events.
Nevertheless, based on these assumptions, the ABC allowed us to assess that NCOs were approximately 3 times less detectable than COs.

As such, even if our sequencing fragments were relatively short (2$\times$250 bp) and if NCOs were less detectable than COs, our method allowed to detect an unprecendentedly large number of both types of events, and the ABC permitted to extrapolate the genuine average recombination parameters.\\


Altogether thus, our approach provides exceptional power to detect recombination at high resolution and at low cost in single individuals.
However, it it is only applicable to males and it requires a \textit{prior} knowledge of the position of recombination hotspots.
In addition, because the hotspots selected need to encompass multiple polymorphic sites, they may not be representative of the average hotspots, but even this high rate of heterozygous sites remains insufficient to totally erase NCO detectability issues.

Despite these few limitations, our approach was well suited to measure biased gene conversion in mice. 
To better understand how it evolved in mammals, we next compared our results with those found in another mammalian species: humans.


\section{Evolution of gBGC in mammals}
\subsection{Measure of the population-scaled gBGC coefficient}




Using the approach previously described by \citet{glemin2015quantification}, Brice Letcher, an intern in our lab, measured the population-scaled gBGC coefficient ($B$) in humans and in the two subspecies from which the parents of the hybrid mice we studied originated.
He found that $B$ is 1.5 to 3.5 times lower in humans than in mice (Table~\ref{tab:estimation-variation-b}).


Interestingly, the effective population size ($N_e$) is respectively 20- and 70-fold as high in \textit{Mus musculus domesticus} and in \textit{Mus musculus castaneus} as in humans \citep{charlesworth2009fundamental,phifer-rixey2012adaptive}.
Since $B = 4 \times N_e \times b$ (see Chapter~\ref{ch:4-gBGC}), this implies that $b$ is 6 to 10 times as high in humans as in the two mouse subspecies (Table~\ref{tab:estimation-variation-b}).\\

% >>> 0.355/(4*15000)
% 5.916666666666666e-06
% >>> 1.21/(4*466500)
% 6.484458735262593e-07
% >>> 0.465/(4*129000)
% 9.011627906976744e-07
% >>>
% >>> 5.916666666666666e-06/6.484458735262593e-07
% 9.124380165289256
% >>> 5.916666666666666e-06/9.011627906976744e-07
% 6.565591397849462





Different factors may contribute to this $b$ discrepancy between humans and mice.
Indeed, the gBGC coefficient ($b$) can be decomposed as: 


\begin{alignat*}{4}
	    b&={}&                     &b^{CO}                   &{}+{}&                      &b^{NCO}& \\
	     &={}& ( b_{0}^{CO} \times &r^{CO} \times L^{CO}  )  &{}+{}& ( b_{0}^{NCO} \times &r^{NCO}& \times L^{NCO}  )
\end{alignat*}

where $r^{i}$, $L^{i}$ and $b_0^{i}$ respectively represent the rate, conversion tract length and transmission bias on recombination events $i$ ($i$ corresponding either to CO or NCO events).\\

In the remaining portion of the discussion, I will go through all the parameters on which $b$ depends to try and identify which contribute more to the difference in $b$ between humans and mice.



\begin{table}[t]
	\centering
	\begin{adjustbox}{width=\textwidth}
	\begin{tabular}{rrrr}


		\toprule
		& $B$ [$CI_{95\%}$] & $N_e$ [min.-max.] & Predicted $b$ \\
		
		\midrule
		
		\textit{Homo sapiens} & 0.355 [0.282--0.445] & 15,000 [10,000--20,000]\textsuperscript{[1]} & $5.9 \times 10^{-6}$ \\
		\textit{M.\ m.\ domesticus} & 0.465 [0.337--0.603] & 129,000 [58,000--200,000]\textsuperscript{[2]} & $0.90 \times 10^{-6}$ \\
		\textit{M.\ m.\ castaneus} & 1.21 [1.13--1.26] & 466,500 [200,000--733,000]\textsuperscript{[2]} & $0.65 \times 10^{-6}$ \\

		\bottomrule

	\end{tabular}
	\end{adjustbox}
	\caption[Prediction of the gBGC coefficient ($b$) on the basis of the population-scaled gBGC coefficient ($B$) and the effective population size ($N_e$)]
	{\textbf{Prediction of the gBGC coefficient ($b$) on the basis of the population-scaled gBGC coefficient ($B$) and the effective population size ($N_e$).}
		\par A point estimate for the gBGC coefficient ($b$) was predicted based on the measurement of the population-scaled gBGC coefficient ($B$) using the approach described by \citet{glemin2015quantification} and a point estimate (arbitrarily chosen as the mid value between the minimum and the maximum reported values) for the effective population size ($N_e$).
		The sources providing the values reported in this table for $N_e$ are given with the following numbered superscript brackets. [1]: \citet{charlesworth2009fundamental}. [2]: \citet{phifer-rixey2012adaptive}.
	}
\label{tab:estimation-variation-b}
\end{table}





\subsection{Variation in recombination rate and tract length}


In this subsection, we will examine the contribution of the recombination parameters ($r^{CO}$, $r^{NCO}$, $L^{CO}$ and $L^{NCO}$) by considering the $r \times L$ parameter for COs and NCOs separately.\\

On the one hand, the $r^{CO} \times L^{CO}$ parameter is twice as small in male mice as in men (Table~\ref{tab:estimation-gBGC-values}).
This directly comes from the 2-fold difference in $r^{CO}$ between these two species, since the $L^{CO}$ we estimated in mice in this study (447 bp) was almost identical to that measured by others in human sperm (460 bp).
Therefore, the CO rate ($r^{CO}$) contributes to decreasing $b$ by 2-fold in mice as compared to humans.\\

On the other hand, the $r^{NCO} \times L^{NCO}$ parameter is three times as small in male mice as in men (Table~\ref{tab:estimation-gBGC-values}).
Since we found mouse $L^{NCO}$ to be at least 1.5 times as small as human $L^{NCO}$, the 3-fold difference on the $r^{NCO} \times L^{NCO}$ parameter is compatible with a 2-fold difference on the NCO rate ($r^{NCO}$) between male mice and humans.
We note that, since the mouse CO rate ($r^{CO}$) too is twice as small as the human CO rate, the 2-fold difference on the NCO rate is compatible with a human CO:NCO rate of 0.10, i.e.\ close to the known mouse CO:NCO rate \citep{cole2010comprehensive}.





\begin{sidewaystable}[p]
	\centering
	\begin{adjustbox}{width=\textwidth}
	\begin{tabular}{rrrrr}

		\toprule
		& \textbf{\textit{Mus musculus}} &  & \textbf{\textit{Homo sapiens}} &  \\
		\cmidrule(l){2-2} \cmidrule(l){3-5}
		Parameter (unit) & \textit{Male} & \textit{Male} & \textit{Female} & \textit{Sex-averaged} \\

		\cmidrule(l){1-1} \cmidrule(l){2-2} \cmidrule(l){3-5}

		$r^{CO}$ (cM/Mb)    & 0.42\textsuperscript{[3,4]} & 0.812\textsuperscript{[2]} & 1.398\textsuperscript{[2]} & 1.13\textsuperscript{[1,2]} \\
		$L^{CO}$ (bp)       & 447\textsuperscript{$\star$} [245--874]\textsuperscript{b} & 460\textsuperscript{[5]} [300--1,019]\textsuperscript{a} & \textit{-} & \textit{-} \\
		$r^{CO} \times L^{CO}$ ($\times 10^{-6}$)      & 1.88 [1.03--3.67]\textsuperscript{b} & 3.73 [2.44--8.27]\textsuperscript{a} & 6.43 [4.19--14.25]\textsuperscript{a} & 5.20 [3.39--11.51]\textsuperscript{a} \\
		$b_0^{CO}$          & 0.00\textsuperscript{$\star$} [-0.0407--0.00428]\textsuperscript{b} & 0.5\textsuperscript{[6]} & 0.404\textsuperscript{[6]} [0.262--0.572]\textsuperscript{b} & 0.402\textsuperscript{[6]} [0.262--0.576]\textsuperscript{b}  \\
		\textbf{$r^{CO} \times L^{CO} \times b_0^{CO}$ ($\times 10^{-6}$)}      & \textbf{0.00 [-0.0529--0.0198]\textsuperscript{c}} & \textbf{1.87 [1.22--4.135]\textsuperscript{b}} & \textbf{2.60 [1.10--8.15]\textsuperscript{c}} & \textbf{2.09 [0.882--6.63]\textsuperscript{c}} \\
		\\
		\midrule
		\\
		$r^{NCO}$ (cM/Mb)   & 3.78\textsuperscript{$\ddagger$} & \textit{-} & \textit{-} & \textit{-} \\
		$L^{NCO}$ (bp)      & 36\textsuperscript{$\star$} [4--54]\textsuperscript{b} & 55--290\textsuperscript{[5],$\mathsection$} & \textit{-} & \textit{-} \\
		$r^{NCO} \times L^{NCO}$ ($\times 10^{-6}$)    & 1.36 [0.151--2.04]\textsuperscript{b} & 3.9\textsuperscript{[6]} [3.5--4.4]\textsuperscript{b} & 10.0\textsuperscript{[6]} [8.5--11.6]\textsuperscript{b} & 7.0\textsuperscript{[6]} [6.0--8.0]\textsuperscript{b} \\
		$b_0^{NCO}$         & 0.310\textsuperscript{$\star$} & 0.262\textsuperscript{[7]} [0.142--0.382]\textsuperscript{b} & 0.450\textsuperscript{[7]} [0.382--0.522]\textsuperscript{b} & 0.36\textsuperscript{[7]} [0.16--0.56]\textsuperscript{b} \\
		\textbf{$r^{NCO} \times L^{NCO} \times b_0^{NCO}$ ($\times 10^{-6}$)}   & \textbf{0.422 [0.0469--0.633]\textsuperscript{b}} & \textbf{1.02 [0.497--1.68]\textsuperscript{c}} & \textbf{4.50 [3.25--6.06]\textsuperscript{c}} & \textbf{2.52 [0.96--4.48]\textsuperscript{c}} \\
		\\
		\midrule

		\textbf{$b$ ($\times 10^{-6}$)} & \textbf{0.422 [0.0469--0.633]\textsuperscript{b} } & \textbf{2.89 [1.72--5.81]\textsuperscript{c} } & \textbf{7.10 [4.35--14.21]\textsuperscript{c} } & \textbf{4.61 [1.84--11.1]\textsuperscript{c} } \\

		\bottomrule

	\end{tabular}
	\end{adjustbox}
	\caption[Estimation of biased gene conversion parameters in \textit{Homo sapiens} and \textit{Mus musculus}]
	{\textbf{Estimation of biased gene conversion parameters in \textit{Homo sapiens} and \textit{Mus musculus}.}
		\par The sources providing the values reported in this table are given with the following numbered superscript brackets. [1]: \citet{dumont2008evolution}. [2]: \citet{kong2002highresolution}. [3]: \citet{shifman2006highresolution}. [4]: \citet{paigen2008recombinational}. [5]: \citet{jeffreys2004intense}. [6]: \citet{halldorsson2016rate}. [7]:~\citet{williams2015noncrossover}.
		$\star$: Measured or estimated in this study.
		$\ddagger$: We assumed that the mouse $r^{NCO}$ was about 9 times the mouse $r^{CO}$, since the CO:NCO ratio is 1:10 in the mouse \citep{handel2010genetics}.
		$\mathsection$: Human $L^{NCO}$ values provided by \citet{jeffreys2004intense} correspond to the mean CT lengths of the two most extreme simulated distributions compatible with observed NCO events.
		Any value reported without any source was directly calculated by us on the basis of the other parameters in this table.
		Between brackets, we report uncertainty intervals on these values. As their types may differ according to the sources, we specify them explicitly with alphabetical characters: Minimal and maximal values (a); 95\% confidence interval (b); 90\% confidence interval (c).
	}
\label{tab:estimation-gBGC-values}
\end{sidewaystable}




\subsection{Confidence in the estimation of $b_{0}^{CO}$ and $b_{0}^{NCO}$}

Next, we wanted to examine whether the last parameters on which $b$ depends (the transmission bias $b_0$ of COs and NCOs) also changed in the same direction.
But prior to assessing the contribution of the latter to the intensity of gBGC, it is important to authenticate the validity of our estimates for $b_{0}^{CO}$ and $b_{0}^{NCO}$.
This will be the object of this subsection, while the extent of their contributions to decreasing $b$ in mice will be discussed in the last section of this chapter.\\

The estimates for $b_{0}^{CO}$ and $b_{0}^{NCO}$ were based on the direct observation of $b_0$ for Rec-1S, Rec-2S and NCO-1 events (see Chapter~\ref{ch:7-quantification-BGC}).
The latter depend largely on the correctness in the identification of the donor in the gene conversion event.
Indeed, if the inferred donor were \textit{not} accurate, results for the defective fragment would be reversed: all polymorphic sites within the CT\textsuperscript{$\star$} would be designated as being outside CTs\textsuperscript{$\star$}, and conversely.

Regarding Rec-2S events (NCO-2+ and NCO-1 events), since both edges of its CTs\textsuperscript{$\star$} were directly observable, there should not have been any mistake on their orientation.

As for Rec-1S events, we reduced their genuine CT to the segment (CT\textsuperscript{$\star$}) located between the switch point and the PRDM9 ChIP-seq peak summit (see Chapter~\ref{ch:6-recombination-parameters}).
But, if the DSB site were located outside this CT\textsuperscript{$\star$} (for example, in the portion of the CT on the opposite side of the unambiguous CT\textsuperscript{$\star$} edge), donor inference would be erroneous.
It was previously shown that the position of the DSB may vary by up to 30 bp from the consensus motif \citep{lange2016landscape} and we thus performed simulations in which the genuine position of the DSB was 30 bp away from its inference (the PRDM9 ChIP-seq peak summit).
Using biologically realistic values for all other parameters, we found that the inferred donor was incorrect in fewer than 1\% of all recombination events identified under that scenario (data not shown).
Therefore, the procedure we used to infer the donor in the recombination event was robust to the inferred position of the DSB\@.

Whatsoever, even under a worst-case scenario where the donor would be erroneously inferred for most Rec-1S, this would not change results for Rec-2S events and, since NCOs are, by far, the main contributors to $b$ (see below and in Table~\ref{tab:estimation-gBGC-values}), our main conclusions regarding the quantification of gBGC would not change drastically.

% Confidence b
% pour NCO
% $R^2 = 0.5630626$; \textit{p}-val $< 2.2 \times 10^{-16}$
% pour CO
% $R^2 = 0.4183843$; \textit{p}-val $< 2.2 \times 10^{-16}$




\section{Speculations on the evolution of BGC}
\subsection{Role of CO and NCO events in limiting $B$}



Since the transmission bias on NCOs ($b_0^{NCO}$) is similar for humans and mice (Table~\ref{tab:estimation-gBGC-values}), this parameter does not participate in the disparity regarding $b$ between the two species.

However, the transmission bias on COs ($b_0^{CO}$) could explain the remaining difference on $b$.
Indeed, we found in this study that, in mice, the transmission bias of COs is null (Table~\ref{tab:estimation-gBGC-values}).
In contrast, in humans, \citet{halldorsson2016rate} observed that the transmission bias equals 0.5.
It should be noted, however, that \citet{halldorsson2016rate} measured $b_0^{CO}$ only for COs displaying complex conversion tracts, which represent only about 0.31\% and 1.33\% of male and female COs, respectively \citep{webb2008sperm, halldorsson2016rate}.
As the repair mechanism which leads to the formation of these complex COs might be different from that leading to the formation of those with simple conversion tracts, whether or not simple COs display the same transmission bias remains unknown.

As such, aside from the recombination rate and conversion tract lengths, the factors explaining the $b$ difference between mice and humans are still unclear.






\subsection{A selective pressure restraining gBGC?}

All in all, both qualitative and quantitative differences exist between humans and mice for males, and likely between men and women too (but data is lacking to verify this in mice).
This suggests that the DSB repair machinery leading to gBGC proceeds differently in these two species and, thus, that this machinery evolved rapidly within the mammalian clade.\\

As gBGC is known to promote the fixation of \textit{G} and \textit{C} alleles even when they are deleterious \citep{galtier2009gcbiased, necsulea2011meiotic}, the burden of this force at the population-scale should be higher in species with large $N_e$. Nonetheless, $B$ remains in a small range, irrespective of the effective population size ($N_e$). It is thus tempting to suggest that there may be a selective pressure on the DSB repair machinery to minimise $b$ in species with large $N_e$, as has already been proposed by \citet{galtier2018codon}.

Given our observations, it seems that several parameters would allow to restrain $B$ in species — like mice — where the effective population size is high.
Indeed, both the recombination rate and the lengths of NCO CTs are smaller in mice than in humans and thus participate in lessening $b$.\\

In addition, since $b_0^{NCO_1}$ is much greater than $b_0^{NCO_{2+}}$ in mice, the relative proportion of NCO-1 and NCO-2+ events — which depends on the level of polymorphism — has an impact on $b$ (see Chapter~\ref{ch:6-recombination-parameters}): the more polymorphic, the greater proportion of NCO-2+ events, and thus the lower $b$. 

Interestingly, in mice, the transmission bias on multiple-marker NCOs ($b_0^{NCO_{2+}} = 0.042$) is extremely weak as compared to that for single-marker NCOs ($b_0^{NCO_{1}} = 0.40$) (see Chapter~\ref{ch:7-quantification-BGC}).
To find out whether this was the case in humans, we reanalysed data from \citet{halldorsson2016rate} and found that the transmission bias on multiple-marker NCOs is similar to the transmission bias on single-marker NCOs (Table~\ref{tab:analysis-halldorsson-all}).
This suggests that the repair mechanism leading to NCOs might differ between humans and mice.
However, as, in \textit{Homo sapiens}, most (84\%) of the NCO-2+ events come from women (Table~\ref{tab:analysis-halldorsson}), the difference between $b_0^{NCO_{2+}}$ and $b_0^{NCO_{1}}$ may reflect a sex-based rather than an interspecific discrepancy.
But it is presently impossible to discriminate between these two possible explanations since no data is yet available in female mice.

More generally, since species with large $N_e$ are more polymorphic and thus entail more NCO-2+ events, the fact that $b_0^{NCO_{2+}}$ is much smaller than $b_0^{NCO_1}$ may be interpreted as another manifestation of the existence of a selective pressure acting to restrain $B$ in large-$N_e$ populations.





\begin{table}[t]
	\centering
	\begin{adjustbox}{width=0.581\textwidth}
	\begin{tabular}{rrrrr}

		\toprule
		& \multicolumn{4}{c}{\textbf{All NCO events}} \\
		\cmidrule(l){2-5}
		CT class & \# CTs & \# W\textrightarrow{}S & \# S\textrightarrow{}W & $b_0$ \\

		\cmidrule(l){1-1} \cmidrule(l){2-5} 

		1 marker   & 1818 & 1192 & 554 & 0.37 \\
		2 markers  & 150  & 187  & 95  & 0.33 \\
		3 markers  & 53   & 106   & 47   & 0.39 \\
		>4 markers & 119  & 731  & 357  & 0.35 \\
		\midrule
		All CTs     & 2140 & 2216 & 1053 & 0.36 \\

		\bottomrule

	\end{tabular}
	\end{adjustbox}
	\caption[Transmission biases for all human NCOs]
	{\textbf{Transmission biases for all human NCOs.}
		\par The data used in this table correspond to the NCO events in the ChIP-seq dataset of \citet{halldorsson2016rate}.
		Similar results were obtained for the NCO events coming from the sequencing dataset of \citet{halldorsson2016rate} (data not shown).
	}
\label{tab:analysis-halldorsson-all}
\end{table}





\subsection{dBGC hitchhiking in structured populations}



Finally, the other type of biased gene conversion — dBGC — also seems to play a significant role in genome evolution, particularly in experimental designs such as ours, and this should also be discussed.
Indeed, the hybrid mice that we analysed descended from crosses between two strains derived from subspecies which displayed distinct \textit{Prdm9} alleles.
Thus, their respective hotspots specifically underwent gBGC and got GC-enriched as compared to the genome of the other (‘nonself’) strain:
in the B6 (resp.\ CAST) lineage, PRDM9\textsuperscript{Dom2}-targeted (resp.\ PRDM9\textsuperscript{Cst}-targeted) hotspots locally enriched in GC while these positions in the CAST (resp.\ B6) lineage did not.
In parallel, the targeted hotspots got eroded in the ‘self’ lineage, as predicted by the hotspot conversion paradox \citep{boulton1997hotspot}.


\begin{table}[t]
	\centering
	\begin{adjustbox}{width=\textwidth}
	\begin{tabular}{rrrrrrrrr}
	
		% AVEC b0
		\toprule
		& \multicolumn{4}{c}{\textbf{Paternal NCO events}} &  \multicolumn{4}{c}{\textbf{Maternal NCO events}}  \\
		\cmidrule(l){2-5} \cmidrule(l){6-9}
		CT class & \# CTs & \# W\textrightarrow{}S & \# S\textrightarrow{}W & $b_0$ & \# CTs & \# W\textrightarrow{}S & \# S\textrightarrow{}W & $b_0$ \\

		\cmidrule(l){1-1} \cmidrule(l){2-5} \cmidrule(l){6-9}

		1 marker   & 824 & 513 & 270 & 0.31 & 994 & 679 & 284 & 0.41 \\
		2 markers  & 34  & 34  & 23  & 0.19 & 116 & 153 & 72  & 0.36 \\
		3 markers  & 5   & 7   & 7   & 0.00 & 48  & 99  & 40  & 0.42 \\
		>4 markers & 12 & 51  & 46  & 0.05 & 107 & 680 & 311 & 0.37 \\
		\midrule
		All CTs     & 875 & 605 & 346 & 0.27 & 1265 & 1611 & 707 & 0.39 \\

		\bottomrule

	\end{tabular}
	\end{adjustbox}
	\caption[Transmission biases for human paternal and maternal NCOs]
	{\textbf{Transmission biases for human paternal and maternal NCOs.}
		\par The data used in this table correspond to the NCO events in the ChIP-seq dataset of \citet{halldorsson2016rate}.
		Similar results were obtained for the NCO events coming from the sequencing dataset of \citet{halldorsson2016rate} (data not shown).
	}
\label{tab:analysis-halldorsson}
\end{table}




Consequently, when the two strains were crossed into a hybrid, each hotspot had been eroded in the locally GC-enriched (self) haplotype.
Thus, the DSB initiated preferentially on the other (nonself), non-eroded and GC-poorer haplotype.
In turn, this led the eroded, GC-richer (self) haplotype to be the donor during the gene conversion event and its \textit{GC} alleles to be overtransmitted into the pool of gametes.\\

Such interplay between dBGC (targeting the non-eroded haplotype) and past gBGC (local enrichment in \textit{GC} alleles) can be extended to any more general case of structured population:
if two populations with distinct \textit{Prdm9} alleles have evolved independently during a length of time sufficient for the hotspots targeted by each allele to erode specifically in their lineage, crossing them together will end in dBGC hitchhiking past gBGC\@ (Figure~\ref{fig:dBGC-hitchhiking}).\\



\begin{figure}[p]
	\centering
	\includegraphics[width = 0.9\textwidth]{figures/inkscape/dBGC-hitchhiking.eps}
	\caption[dBGC hitchhiking in structured populations]
	{\textbf{dBGC hitchhiking in structured populations.}
		\par In the CAST lineage (yellow box) where PRDM9\textsuperscript{Cst} (yellow triangles) is present, PRDM9\textsuperscript{Cst}-targeted motifs (yellow square) undergo erosion and, because of gBGC, weak (W) bases (A or T) get supplanted by strong (S) bases (G or C) at WS polymorphic sites.
		In contrast, in the B6 lineage (red box) where only PRDM9\textsuperscript{Dom2} (red triangles) is present, PRDM9\textsuperscript{Cst}-targeted motifs do \textit{not} undergo biased gene conversion.
		As such, the CAST haplotype (yellow segment) is locally enriched in S bases as compared to the B6 haplotype (red segment).
		When populations cross into a hybrid (grey box), the non-eroded motif from the B6 lineage is targeted by PRDM9\textsuperscript{Cst} and the DSB initiates on the B6 haplotype (red thunderbolt).
		Consequently, the CAST haplotype with both the eroded motif and the local enrichment in S bases is the donor in the conversion event: past gBGC that occurred in the CAST lineage is hitchhiked by dBGC occurring in the hybrid.
	}
\label{fig:dBGC-hitchhiking}
\end{figure}




This phenomenon of dBGC hitchhiking brought a confounding effect to quantify gBGC and we thus decoupled the two processes by equalising, at every hotspot, the number of B6- and CAST-donor fragments to cancel the dBGC effect.
This allowed us to quantify the transmission bias ($b_0$) in mice, which was useful to comprehend how the gBGC coefficient ($b$) varies with the effective population size ($N_e$) and to show that the observations complied with the hypothesis of a selective pressure restraining gBGC at the individual-scale to limit its nefast consequences at the population-scale.










But, this hypothesis, — if it were true, — would surely bring other more conceptual questions like the following:
how can effects on a population drive the evolution of a molecular mechanism in single individuals?
And at what scale, — populational or individual, — should the concept of evolutionary forces be defined?
Rather than answering them, I will try and provide food for thought on these open questions in the following — and final — chapter of this thesis.



