\begin{savequote}[8cm]
	‘I don't claim to be a methodologist, but I act like one only because I do methodology to protect myself from crazy methodologists.’
	
	\qauthor{--- Ward Cunningham, \textit{\usebibentry{cunningham2004geek}{title}} \citeyearpar{cunningham2004geek} }
	
\end{savequote}

\chapter{\label{ch:5-methodology}High-resolution detection of recombination in single individuals}
% High-resolution recombination events in single individuals}
% Detection of high-resolution recombination events in single individuals}
%\otherpagedecoration

\minitoc{}

{\small{} \itshape{}

\paragraph{This chapter in brief —}
Un resume avec on a fait desgin qui consiste a croiser des souris et a faire du sperm typing sur 1000 hotspots de l'hybride.
La selection des hotspots permet un sequencage a haute profondeur.
Ensuite, il s'agit d'identifier les recombinants: on fait ca avec un double mapping puis une serie de filtres (couverture, frequence, 2+2 variants).
L'etape absolument cruciale est le double mapping: permet d'eliminer les erreurs d'alignement.
Un resume avec on a fait desgin qui consiste a croiser des souris et a faire du sperm typing sur 1000 hotspots de l'hybride.
La selection des hotspots permet un sequencage a haute profondeur.
Ensuite, il s'agit d'identifier les recombinants: on fait ca avec un double mapping puis une serie de filtres (couverture, frequence, 2+2 variants).
L'etape absolument cruciale est le double mapping: permet d'eliminer les erreurs d'alignement.

}




%
% Studies of recombination events
%
%
% Un mini-blabla sur pourquoi on cree cette approche? Car les autres etudes qui ont etudier les evenements de facon directe son limitees: soit sperm-typing avec peu de hotspots, soit pedigree mais dans plein d'individus pas un seul. (ou uniquement dans le paragraphe de resume en haut)
% Ici, je vais decrire d'abord le design experimental (et donc l'acquisition des données), puis comment on fait pour detecter les recombinants (pipeline bioinformatique), puis l'importance relative des differents aspects de la methode sur la detectabilite d'evenements de recombinaison.
%
% DISCUSSIOn (epistemo): necessite de regarder les donnees pour mettre en place le pipeline qui soit correct.
% DISCUSSion: redire l'aspect crucial du mapping et de tenir compte des biais existants. Ils ne sont pas problematiques quand on fait le mapping pour ce pour quoi il a ete fait. En revanche, dans notre cas de figure, vu qu'on detecte des evenements rares, c'est un biais. Il faudra aussi discuter de son implication quand on le fait sur l'autre approche: en particulier, sur le hotspots ou un individu est genotype B6/B6 et l'autre est genotype B6/CAST — dans ce cas de figure, ca permettrait de quantifier les erreurs sur B6/B6 (et, de memoire, on trouve peu de faux positifs) et s'il y a un lien avec le nombre de faux positifs et la densite en SNP, le nombre d'INDELs…
% limitations a redire sur la detectabilite donc transi vers discussion
% DISCUSSion: preciser que puisqu'on a mis un filtre sur le nombre de markers, on aura des hotspots plus asymetriques (plsu riches en markers).
%


\section{Overview of the experimental design}
\subsection{Acquisition of highly polymorphic individuals}

Detecting recombination events rests on one indispensable prerequisite: the presence of markers (i.e.\ polymorphic sites).

Therefore, we performed a cross between two subspecies of mice that present a high level of heterozygosity (1 SNP every 150 bp) \citep{keane2011mouse,yalcin2012nextgeneration} and that are known to hybridise naturally \citep{orth1998natural}: \textit{Mus musculus domesticus} (strain C57BL/6J, hereafter called B6) and \textit{Mus musculus castaneus} (strain CAST/EiJ, hereafter called CAST).
This cross resulted in F1 hybrid mice (B6xCAST), of which two males were selected. 
Sperm DNA was then collected from these two individuals and kindly given to us by D. Bourch'is (Institut Curie, Paris).

The extracted DNA from both biological replicates was then sonicated to produce fragments of a mean size of 350 bp.


\subsection{Enrichment in recombination events}%Selection of target loci and DNA capture}

\begin{figure}[p]
	\centering
	\missingfigure[figwidth=14cm, figheight = 15cm]{Schema du design experimental}
	\caption[Overview of the experimental design]
	{\textbf{Overview of the experimental design.}
		\par \textbf{LEGENDE A FAIRE}
		\par Cross B6xCAST $\rightarrow$ F1 hybrid
		\par Extract sperm from F1 hybrid
		\par DNA capture $\rightarrow$ pool of fragments enriched in recombination events
		\par Sequence with Illumina (2 $\times$ 250 pb)
	}
\label{fig:experimental-design}
\end{figure}


Since hotspots host the large majority of recombination events (see Chapter~\ref{ch:3-recombination-variation}), we wanted to enrich our pool of DNA in fragments originating from these regions (so as to increase the proportion of sequenced fragments corresponding to recombination events).
This required two steps: selecting hotspots, and performing DNA capture (i.e.\ hybridisation-based targeted-DNA enrichment) of these loci \citep[reviewed in \citealp{horn2012target}]{gnirke2009solution,hodges2007genomewide}.

\subsubsection{Selection of targets}

The recombination hotspots of B6xCAST mice had previously been identified \textit{via} chromatin immunoprecipitation followed by sequencing (ChIP-seq) of the PRDM9 protein \citep{baker2015prdm9}.
We restricted this known list of 6,758 hotspots to those (1) displaying a high marker density in the vicinity of the PRDM9 binding site (so as to increase the chance of detecting recombination events) and (2) aligning on their whole length on the CAST and B6 reference genomes (so as to restrain mapping artifacts).

In practice, the selection criterium on heterozygosity (minimum of 4 SNPs in the 300-bp central region of the locus centred on the PRDM9 peak summit) was the most stringent: it cut down the original list of 6,758 hotspots to only 1,261 hotspots.
The other two criteria on mappability (a strict maximum of 60 sites with low sequence quality in the 1-kb central region, and the absence of a large indel by ensuring that a minimum of 800 bp in the 1-kb from the B6 genome shared at least 90\% identity with the CAST genome) respectively discarded 205 and 38 additional loci.
Altogether thus, a total of 1,018 1-kb long regions centred on the summit of the PRDM9 ChIP-seq peaks were selected.
These were positioned randomly both across and along chromosomes (Figure~\ref{fig:hotspots-along-chromosomes}).



% RIGHT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-2.4cm
	\rightskip-2.4cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth]{figures/chap5/Recombination_events_along_chromosomes_bw.eps}
	\captionsetup{width=1.25\textwidth, margin={-2.2cm, -3.3cm}}
	\caption[Distribution of recombination events across the 1,018 selected hotspots positioned randomly along chromosomes]
	{\textbf{Distribution of recombination events across the 1,018 selected hotspots positioned randomly along chromosomes.}
		\par Chromosomes are represented in grey and oriented so that the centromere is on the bottom side of the figure (mouse chromosomes are acrocentric).
		The total number of recombination events identified is given by the length of the horizontal black bar at the position of each of the 1,018 selected hotspots.
	}
\label{fig:hotspots-along-chromosomes}
\end{sidewaysfigure}





In addition, we selected 500 1-kb control regions which displayed genomic characteristics similar to those of the 1,018 hotspots (in terms of GC-content, SNP density, sequence quality and TE content) but which did not belong to the list of known recombination hotspots.

\subsubsection{DNA capture}

To enrich the sequencing data in fragments coming from the 1,518 aforementioned loci (hereafter called targets), we performed either one or two rounds of DNA capture targeting them.
 % \citep[reviewed in \citealp{horn2012target}]{gnirke2009solution,hodges2007genomewide}.
Since our final aim was to detect recombination events, i.e.\ fragments bearing both a portion of the B6 haplotype and a portion of the CAST haplotype, it was essential that the efficiency of the capture be similar for both haplotypes.
We thus designed two baits (one for each of the two haplotypes) for every target.

We next monitored the existence of any capture bias by looking at the origin of all the non-recombinant fragments.
Indeed, as recombination is rare, the vast majority of sequenced fragments do not correspond to recombination events and thus, half of all non-recombinant fragments should come from the B6 haplotype (and thus, the other half from the CAST haplotype).
We found that this was indeed the case since the proportion of fragments containing only B6-typed markers (i.e.\ coming from the B6 haplotype) revolved around 50\% for nearly all targets: 95\% of targets in the [43.4\%; 55.6\%] range (Figure~\ref{fig:prop-B6-in-targets}).



\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap5/Absence_capture_bias_non_recombinant_B6.eps}
	\caption[Absence of capture bias between the B6 and CAST haplotypes]
	{\textbf{Absence of capture bias between the B6 and CAST haplotypes.}
		\par All fragments exclusively containing B6-typed markers were designed as non-recombining fragments coming from the B6 haplotype. 
		The distribution of the proportion of such fragments across targets is reported in this figure. 
		The dashed line corresponds to the median proportion of B6-genotyped fragments across targets and the two dotted lines correspond to the 2.5 and 97.5 percentiles (i.e.\ the delimitation of the proportion for 95\% of targets).
	}
\label{fig:prop-B6-in-targets}
\end{figure}





\subsection{Ultra deep-sequencing of the targeted sperm DNA}

Libraries were then sequenced by an Illumina device using a 250-bp paired-end protocol, except for 4 small libraries (out of 18) that contributed to 6\% of the total number of fragments and which were sequenced as a pilot experiment using a 100-bp paired-end protocol (Table~\ref{tab:characteristics-seq-mapping-capture}). 
% The mean sequencing depth on 1-kb targeted hotspots and control regions was 97,340 $x$ and raised up to above 926,800 $x$ in certain regions.
%% OU mean sequencing depth on 1-kb long loci: 95,810 $x$ ??)

% The sequencing data is available under the accession \textbf{AJOUTER LE NUMERO SRA}.

Overall, sequenced reads mapped equally well to both the B6 and the CAST reference genome assemblies (Table~\ref{tab:characteristics-seq-mapping-capture}).
In addition, DNA capture was efficient since 72\% of sequenced fragments mapped within the selected targets.
This resulted in a substantial coverage of the targeted loci: the mean sequencing depth on 1-kb long targets was 97,340 $x$, and it raised up to above 926,800 $x$ in certain regions.

The variation in coverage across hotspots was similar to that across control regions (data not shown) and it was rather limited as the variation in coverage of 90\% of hotspots held in a 7:1 ratio ([0.05; 0.95] quantiles = [18,877; 128,293] reads).
Nonetheless, we found that the variation in coverage across hotspots was highly correlated to the mean GC-content in the 1-kb-long sequence (Pearson correlation: ${r = -0.641}$; \textit{p}-val $< 2.2 \times 10^{-16}$).

Thus, apart from a GC-content effect, there was no large capture nor mapping bias across hotspots.


\begin{sidewaystable}[p]
    \centering
	\begin{adjustbox}{width = 1\textwidth}
    \begin{tabular}{rrrrrrrrrrrr}
        \toprule
        \textbf{} & \multicolumn{6}{c}{\textbf{Sample and sequencing characteristics}} & \multicolumn{2}{c}{\textbf{Mapping (\%)}} & \multicolumn{3}{c}{\textbf{Capture efficiency}} \\
		\cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-12}
        \textbf{Library} & \textbf{Mouse} & \textbf{\# DNA} & \textbf{Rep.} & \textbf{Lane} & \textbf{Read} & \textbf{Library} & \textbf{Ref.} & \textbf{Ref.} & \textbf{\# Filtered} & \textbf{\% in} & \textbf{\# in} \\
        \textbf{ID} & \textbf{ID} & \textbf{capture} & \textbf{} & \textbf{ID} & \textbf{length} & \textbf{size} & \textbf{B6} & \textbf{CAST} & \textbf{Fragments} & \textbf{targets} & \textbf{targets} \\
        % \midrule
		\cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-12}
        A-1 & 1 & 2 & NA & NA & 100 bp & 14,977,880 & 99.87 & 99.76 & 7,206,235 & 86.09 & 6,203,730 \\
        A-2 & 1 & 1 & NA & NA & 100 bp & 12,457,816 & 99.75 & 99.27 & 5,813,649 & 26.86 & 1,561,461 \\
        A-3 & 2 & 2 & NA & NA & 100 bp & 16,000,000 & 99.85 & 99.73 & 7,631,724 & 84.88 & 6,478,370 \\
        A-4 & 2 & 1 & NA & NA & 100 bp & 13,291,526 & 99.74 & 99.24 & 6,110,086 & 24.29 & 1,484,199 \\
        % \midrule
		\cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-12}
        B-1 & 1 & 2 & NA & NA & 250 bp & 51,923,148 & 99.76 & 99.74 & 24,887,319 & 85.95 & 21,391,551 \\
        B-2 & 1 & 1 & NA & NA & 250 bp & 64,260,092 & 99.72 & 99.66 & 29,732,709 & 26.67 & 7,927,136 \\
        B-3 & 2 & 2 & NA & NA & 250 bp & 98,238,822 & 99.64 & 99.61 & 46,831,049 & 84.87 & 39,749,391 \\
        B-4 & 2 & 1 & NA & NA & 250 bp & 130,482,992 & 99.60 & 99.52 & 59,942,764 & 24.52 & 14,700,518 \\
        % \midrule
		\cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-12}
        C-1 & 1 & 2 & 1 & 1 & 250 bp & 67,775,154 & 99.79 & 99.75 & 33,221,010 & 82.54 & 27,421,183 \\
        C-2 & 1 & 2 & 1 & 2 & 250 bp & 69,564,748 & 99.78 & 99.75 & 34,102,906 & 82.65 & 28,185,446 \\
        C-3 & 1 & 2 & 2 & 1 & 250 bp & 79,002,218 & 99.79 & 99.75 & 38,818,821 & 84.05 & 33,013,536 \\
        C-4 & 1 & 2 & 2 & 2 & 250 bp & 81,074,012 & 99.77 & 99.74 & 39,841,630 & 85.14 & 33,921,167 \\
        C-5 & 2 & 2 & 1 & 1 & 250 bp & 60,911,138 & 99.71 & 99.68 & 29,876,369 & 85.45 & 25,530,140 \\
        C-6 & 2 & 2 & 1 & 2 & 250 bp & 62,042,362 & 99.71 & 99.67 & 30,437,699 & 85.55 & 26,039,414 \\
        C-7 & 2 & 2 & 2 & 1 & 250 bp & 66,489,166 & 99.77 & 99.74 & 32,701,170 & 86.18 & 28,182,084 \\
        C-8 & 2 & 2 & 2 & 2 & 250 bp & 68,382,888 & 99.76 & 99.73 & 33,636,524 & 86.28 & 29,022,692 \\
        C-9 & NA & 2 & 1\&2 & 1 & 250 bp & 9,796,678 & 83.27 & 84.44 & 3,801,189 & 62.12 & 2,361,438 \\
        C-10 & NA & 2 & 1\&2 & 2 & 250 bp & 10,156,096 & 82.77 & 84.93 & 3,920,630 & 62.04 & 2,432,278 \\
		% \midrule
		\cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-12}
		\textbf{Total} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{976,826,736} & \textbf{99.68} & \textbf{99.37} & \textbf{468,513,483} & \textbf{71.63} & \textbf{335,605,734} \\
        \bottomrule
    \end{tabular}
	\end{adjustbox}
    \caption[Sequencing, mapping and capture-efficiency summary metrics]
	{\textbf{Sequencing, mapping and capture-efficiency summary metrics.}
		\par The sperm from the two individuals was sequenced three times (horizontal panels): ‘A-libraries’ correspond to a pilot where reads were sequenced with a 2 $\times$ 100-bp protocol at a low coverage; ‘B-libraries’ were enriched in recombination events \textit{via} either one or two rounds of DNA-capture and were then sequenced deeply with a 2 $\times$ 250-bp protocol; ‘C-libraries’ were sequenced \textit{a posteriori} (after having analysed results of the ‘B-libraries’), using only samples with two rounds of DNA-capture to increase the total number of recombination events.
		\par The first vertical panel recaps the biological and sequencing characteristics of the samples. 
		The library size is the number of sequenced reads.
		\par The second vertical panel recaps the percentage of reads mapping on either of the two reference genomes (B6 and CAST).
		\par The third vertical panel recaps statistics on the efficiency of DNA-capture.
		The capture efficiency corresponds to the percentage of filtered fragments (i.e.\ fragments remaining after the removal of unmapped and secondary-aligned reads) that map within the targeted sites.
	}
\label{tab:characteristics-seq-mapping-capture}
\end{sidewaystable}





\section{The unique-molecule genotyping pipeline}
\subsection{Identification of heterozygous markers}


\subsection{Genotyping of individual DNA fragments}
\subsection{Second-pass on the alternate parental genome} Ou overview GATK avant Ou second-pass on other parental genome OU Removal of PCR duplicates OU PLUTOT SELECTION DES RECOMBINANTS et suppression des duplicats de PCR
pourquoi on choisit 2+2

\section{The determinants of sensitivity and specificity}
% Impact of filters on the detection of events}%sensitivity and specificity of detection}
\subsection{The crucial step: mapping on both reference genomes}
\subsection{Impact of the filters on the false positive (FP) rate} OU Final rate of false positives  (CHANEGER LE TITRE)
DONT parler de la suppression des duplicats de PCR
Voir lequel supprimer dans cette liste


DIRE QQPART que plus de recombinants vus dans les cas ou 2 rounds de capture (ce qui justifie que dans les C-libraries on a utilise seulement 2 rounds caputre).
\subsection{Limitations on the detectability of events}
detectabilite depend de polym donc on ne detecte pas tout (plus on a un filtre 2+2)
Donc ca me fait ma TRANSITION vers chapitre qui suit: pour avoir les parametres reels (non observables), il faut passer par des methodes inferentielles




% intro sur GATK
experimental design (sperme + hotspots + controles + sequence a telle profondeur)
procedure utilisee + comment GATK fonctionne
Effet des filtres (quality, double mapping (crucial), deux genotypes, min coverage) et validation (faux positifs)
conclu: double mapping est reellement l'etape cruciale dans tout cela




% chap 6: (plan a revoir)
% intro: souris avec les infos connues dedans sur la recombi (DCM1, les cartes PRDM9 ChIP-seq Baker, Spo11…) — differences entre tous ces descripteurs de la recombinaison et ce qui est su chez la souris
% high confidence + determinants (trouves dans les hotspots intenses en relation avec DMC1) et inferrence des CT + donneur et description du jeu de donnees
% ABC et inference des parametres reels de recombinaison chez la souris (mise au point des simulations + selection des simul + resultats sur les param de recombi)

% chap7:
% clasification des hotspots cibles par B6 ou CAST
% population structuree hitchhinking
% quantification du BGC (g et d)
%
% chap8:
% design experimental avec l'introgression
% adaptation de la methode
% premiers resultats sur la recombinaison















% TABLES NON-UTILISEES MAIS A NE PAS PERDRE SI CHANGEMENT LAURENT

% \begin{table}[h]
%     \centering
%     \begin{adjustbox}{width = 1\textwidth}
%     \begin{tabular}{rrrrrrrrr}
%         \toprule
%         \textbf{} & \multicolumn{6}{c}{\textbf{Sample and sequencing characteristics}} & \multicolumn{2}{c}{\textbf{Mapping (\%)}} \\
%         \cmidrule(l){2-7} \cmidrule(l){8-9}
%         \textbf{Library} & \textbf{Mouse} & \textbf{\#DNA-} & \textbf{Rep.} & \textbf{Lane} & \textbf{Read} & \textbf{Library} & \textbf{Ref.} & \textbf{Ref.} \\
%         \textbf{ID} & \textbf{ID} & \textbf{capture} & \textbf{} & \textbf{ID} & \textbf{length} & \textbf{size} & \textbf{B6} & \textbf{CAST} \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9}
%         A-1 & 1 & 2 & NA & NA & 100 bp & 14,977,880 & 99.87 & 99.76 \\
%         A-2 & 1 & 1 & NA & NA & 100 bp & 12,457,816 & 99.75 & 99.27 \\
%         A-3 & 2 & 2 & NA & NA & 100 bp & 16,000,000 & 99.85 & 99.73 \\
%         A-4 & 2 & 1 & NA & NA & 100 bp & 13,291,526 & 99.74 & 99.24 \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9}
%         B-1 & 1 & 2 & NA & NA & 250 bp & 51,923,148 & 99.76 & 99.74 \\
%         B-2 & 1 & 1 & NA & NA & 250 bp & 64,260,092 & 99.72 & 99.66 \\
%         B-3 & 2 & 2 & NA & NA & 250 bp & 98,238,822 & 99.64 & 99.61 \\
%         B-4 & 2 & 1 & NA & NA & 250 bp & 130,482,992 & 99.60 & 99.52 \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9}
%         C-1 & 1 & 2 & 1 & 1 & 250 bp & 67,775,154 & 99.79 & 99.75 \\
%         C-2 & 1 & 2 & 1 & 2 & 250 bp & 69,564,748 & 99.78 & 99.75 \\
%         C-3 & 1 & 2 & 2 & 1 & 250 bp & 79,002,218 & 99.79 & 99.75 \\
%         C-4 & 1 & 2 & 2 & 2 & 250 bp & 81,074,012 & 99.77 & 99.74 \\
%         C-5 & 2 & 2 & 1 & 1 & 250 bp & 60,911,138 & 99.71 & 99.68 \\
%         C-6 & 2 & 2 & 1 & 2 & 250 bp & 62,042,362 & 99.71 & 99.67 \\
%         C-7 & 2 & 2 & 2 & 1 & 250 bp & 66,489,166 & 99.77 & 99.74 \\
%         C-8 & 2 & 2 & 2 & 2 & 250 bp & 68,382,888 & 99.76 & 99.73 \\
%         C-9 & NA & 2 & 1\&2 & 1 & 250 bp & 9,796,678 & 83.27 & 84.44 \\
%         C-10 & NA & 2 & 1\&2 & 2 & 250 bp & 10,156,096 & 82.77 & 84.93 \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9}
%         \textbf{Total} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{976,826,736} & \textbf{99.68} & \textbf{99.37} \\
%         \bottomrule
%     \end{tabular}
%     \end{adjustbox}
%     \caption[Sequencing characteristics and mapping summary metrics]
%     {\textbf{Sequencing characteristics and mapping summary metrics.}
%         \par The sperm from the two individuals was sequenced three times (horizontal panels): ‘A-libraries’ correspond to a pilot where reads were sequenced with a 2 $\times$ 100-bp protocol and at low coverage; ‘B-libraries’ correspond to the first part of the real experiment where we sequenced deeply with a 2 $\times$ 250-bp protocol and where samples were either enriched in recombination events \textit{via} either one or two rounds of DNA-capture; ‘C-libraries’ were sequenced after having analysed results of the ‘B-libraries’ to increase the total number of recombination events (using only the samples with two rounds of DNA-capture).
%         \par The second vertical panel recaps all the biological and sequencing characteristics of the samples studied. The library size corresponds to the total number of reads sequenced.
%         \par The third vertical panel recaps the percentage of reads mapping on either of the two reference genomes (B6 and CAST).
%     }
% \label{tab:characteristics-seq-mapping}
% \end{table}
%



% Table avant d'enlever la colonne des fragments (ATTENTION: legende incomplete)
% \begin{sidewaystable}[p]
%     \centering
%     \begin{adjustbox}{width = 1\textwidth}
%     \begin{tabular}{rrrrrrrrrrrrr}
%         \toprule
%         \textbf{} & \multicolumn{6}{c}{\textbf{Sample and sequencing characteristics}} & \multicolumn{2}{c}{\textbf{Mapping (\%)}} & \multicolumn{4}{c}{\textbf{Capture efficiency}} \\
%         \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-13}
%         \textbf{Library} & \textbf{Mouse} & \textbf{\#DNA-} & \textbf{Rep.} & \textbf{Lane} & \textbf{Read} & \textbf{Library} & \textbf{Ref.} & \textbf{Ref.} & \textbf{\# Potential} & \textbf{\# Selected} & \textbf{Capture} & \textbf{\# Fragments} \\
%         \textbf{ID} & \textbf{ID} & \textbf{capture} & \textbf{} & \textbf{ID} & \textbf{length} & \textbf{size} & \textbf{B6} & \textbf{CAST} & \textbf{Fragments} & \textbf{Fragments} & \textbf{Efficiency (\%)} & \textbf{in targets} \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-13}
%         A-1 & 1 & 2 & NA & NA & 100 bp & 14,977,880 & 99.87 & 99.76 & 7,488,940 & 7,206,235 & 86.09 & 6,203,730 \\
%         A-2 & 1 & 1 & NA & NA & 100 bp & 12,457,816 & 99.75 & 99.27 & 6,228,908 & 5,813,649 & 26.86 & 1,561,461 \\
%         A-3 & 2 & 2 & NA & NA & 100 bp & 16,000,000 & 99.85 & 99.73 & 8,000,000 & 7,631,724 & 84.88 & 6,478,370 \\
%         A-4 & 2 & 1 & NA & NA & 100 bp & 13,291,526 & 99.74 & 99.24 & 6,645,763 & 6,110,086 & 24.29 & 1,484,199 \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-13}
%         B-1 & 1 & 2 & NA & NA & 250 bp & 51,923,148 & 99.76 & 99.74 & 25,961,574 & 24,887,319 & 85.95 & 21,391,551 \\
%         B-2 & 1 & 1 & NA & NA & 250 bp & 64,260,092 & 99.72 & 99.66 & 32,130,046 & 29,732,709 & 26.67 & 7,927,136 \\
%         B-3 & 2 & 2 & NA & NA & 250 bp & 98,238,822 & 99.64 & 99.61 & 49,119,411 & 46,831,049 & 84.87 & 39,749,391 \\
%         B-4 & 2 & 1 & NA & NA & 250 bp & 130,482,992 & 99.60 & 99.52 & 65,241,496 & 59,942,764 & 24.52 & 14,700,518 \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-13}
%         C-1 & 1 & 2 & 1 & 1 & 250 bp & 67,775,154 & 99.79 & 99.75 & 33,887,577 & 33,221,010 & 82.54 & 27,421,183 \\
%         C-2 & 1 & 2 & 1 & 2 & 250 bp & 69,564,748 & 99.78 & 99.75 & 34,782,374 & 34,102,906 & 82.65 & 28,185,446 \\
%         C-3 & 1 & 2 & 2 & 1 & 250 bp & 79,002,218 & 99.79 & 99.75 & 39,501,109 & 38,818,821 & 84.05 & 33,013,536 \\
%         C-4 & 1 & 2 & 2 & 2 & 250 bp & 81,074,012 & 99.77 & 99.74 & 40,537,006 & 39,841,630 & 85.14 & 33,921,167 \\
%         C-5 & 2 & 2 & 1 & 1 & 250 bp & 60,911,138 & 99.71 & 99.68 & 30,455,569 & 29,876,369 & 85.45 & 25,530,140 \\
%         C-6 & 2 & 2 & 1 & 2 & 250 bp & 62,042,362 & 99.71 & 99.67 & 31,021,181 & 30,437,699 & 85.55 & 26,039,414 \\
%         C-7 & 2 & 2 & 2 & 1 & 250 bp & 66,489,166 & 99.77 & 99.74 & 33,244,583 & 32,701,170 & 86.18 & 28,182,084 \\
%         C-8 & 2 & 2 & 2 & 2 & 250 bp & 68,382,888 & 99.76 & 99.73 & 34,191,444 & 33,636,524 & 86.28 & 29,022,692 \\
%         C-9 & NA & 2 & 1\&2 & 1 & 250 bp & 9,796,678 & 83.27 & 84.44 & 4,898,339 & 3,801,189 & 62.12 & 2,361,438 \\
%         C-10 & NA & 2 & 1\&2 & 2 & 250 bp & 10,156,096 & 82.77 & 84.93 & 5,078,048 & 3,920,630 & 62.04 & 2,432,278 \\
%         % \midrule
%         \cmidrule(l){1-1} \cmidrule(l){2-7} \cmidrule(l){8-9} \cmidrule(l){10-13}
%         \textbf{Total} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{-} & \textbf{976,826,736} & \textbf{99.68} & \textbf{99.37} & \textbf{488,413,368} & \textbf{468,513,483} & \textbf{71.63} & \textbf{335,605,734} \\
%         \bottomrule
%     \end{tabular}
%     \end{adjustbox}
%     \caption[Sequencing characteristics and mapping summary metrics]
%     {\textbf{Sequencing characteristics and mapping summary metrics.}
%         \par The sperm from the two individuals was sequenced three times (horizontal panels): ‘A-libraries’ correspond to a pilot where reads were sequenced with a 2 $\times$ 100-bp protocol and at low coverage; ‘B-libraries’ correspond to the first part of the real experiment where we sequenced deeply with a 2 $\times$ 250-bp protocol and where samples were either enriched in recombination events \textit{via} either one or two rounds of DNA-capture; ‘C-libraries’ were sequenced after having analysed results of the ‘B-libraries’ to increase the total number of recombination events (using only the samples with two rounds of DNA-capture).
%         \par The second vertical panel recaps all the biological and sequencing characteristics of the samples studied. The library size corresponds to the total number of reads sequenced.
%         \par The third vertical panel recaps the percentage of reads mapping on either of the two reference genomes (B6 and CAST).
%     }
% \label{tab:characteristics-seq-mapping}
% \end{sidewaystable}




