\begin{savequote}[8cm]
“[…] if there is one event in the whole evolutionary sequence at which my own mind lets my awe still overcome my instinct to analyse, and where I might concede that there may be a difficulty in seeing a Darwinian gradualism hold sway throughout almost all, it is this event — the initiation of meiosis.”
% “[…] if there is one event in the whole evolutionary sequence at which my own mind lets my awe still overcome my instinct to analyse […], it is this event — the initiation of meiosis.”
	
\qauthor{--- W. D. “Bill” Hamilton, \textit{\usebibentry{hamilton1996narrow}{title}} \citeyearpar{hamilton1996narrow} }
\end{savequote}

\chapter{\label{ch:2-recombination-mechanistics}Meiotic recombination, the essence of heredity} 
%\otherpagedecoration

\minitoc{}


\begin{quote}
\textit{‘Why all this silly rigmarole of sex? Why this gavotte of chromosomes? Why all these useless males, this striving and wasteful bloodshed, these grotesque horns, colors… and why, in the end, novels, like }Cancer Ward\textit{, about love?’}

\qauthor{--- W. D. Hamilton, \textit{\usebibentry{hamilton1975review}{title}} \citeyearpar{hamilton1975review}}
\end{quote}

This is how the fanciful Bill Hamilton (1936—2000) sums up the mystery of sexual reproduction (or simply, “sex”) that has been puzzling biologists for over a century and which, to this day, remains unanswered \citep{de2007evolution, otto2009evolutionary}.

This so-called “paradox of sex” finds its roots in that most theoretical arguments plead an elevated cost of sex as compared to asexual modes of reproduction \citep{otto2002evolution,lehtonen2012many}.
First, females invest half their reproductive resources in the production of males which, in turn, invest minimally into the progeny, as epitomised by the uncommonness of paternal care when it is not beneficial to the male \citep{smith1977parental,fromhage2007stability} — a concept known as the “twofold cost of sex” or “cost of meiosis” \citep{bell1982masterpiece}.
Second, the sexual act itself wastes time and energy to find and attract a sexual partner, and exposes the individual to the risks of contracting diseases and of being predated (sometimes by the mate itself), thus making sex a pearilous and unprofitable endeavour.

Nevertheless, only 80 \citep{vrijenhoek1989list,neaves2011unisexual} of the 70,000 vertebrate species discovered so far \citep{iucn2019} and as little as 0.1\% of all named animals \citep{vrijenhoek1998animal} reproduce otherwise than sexually. %More generally, although asexuality often arises, it rarely persists for long, thus suggesting that it is an evolutionary dead-end or that sex is hard to quit \citep{normarck2003genomic}.
Such pervasiveness of sex in nature constitutes indisputable proof of its evolutionary success. 

But, given its considerable drawkbacks, how come sex has superseded all other forms of reproduction?
% What then, given all the costs of sex, explains its pervasiveness in nature and where does its evolutionary success come from?
% How come then, given all its costs, that sex is so widespread in nature and what explains its evolutionary success?
Over 20 theories have been put forward to answer this question \citep{kondrashov1993classification}, but the most generally claimed advantages revolve around the idea that sex both eliminates deleterious mutations and brings up more favourable combinations of alleles \citep{normarck2003genomic, speijer2016can}.
This defensibly profitable reshuffling of alleles is called “recombination” and occurs during meiosis, the cellular process leading to the formation of gametes.\\

This chapter — named after a review on the subject \citep{hunter2015meiotic} — explores the details of the cytological features of meiosis and recombination with a particular focus on mammals, before venturing into the multiple models of meiotic recombination which still demand to be evidenced.

% \textbf{+ dire que tout n'a pas ete elucie et toujours du travail dessus + dire que les choses sont conservees chez les eucaryotes + meiose vient du grec. + citation victor hensen dans weisman? + parler des origines controversees de la meiose? (mais mecanisme assez bien elucide) + evidence pour l'avantage sont toujours scarce + focus sur mammalian meiosis and recombination}
% \textbf{citation victor hensen dans weisman?}



% After over a century of investigation, one persisting mystery of biology is why sexual reproduction (or simply, “sex”) is so widespread in nature \citep{de2007evolution, otto2009evolutionary}:
% % After over a century of investigation, the reason why sexual reproduction (or simply, “sex”) is so widespread in nature remains a mystery \citep{de2007evolution, otto2009evolutionary}:
% % After over a century of investigation, the mystery remains on the reason why sexual reproduction (or simply, “sex”) is so widespread in nature \citep{de2007evolution, otto2009evolutionary}:
% only 80 \citep{vrijenhoek1989list,neaves2011unisexual} of the 70,000 vertebrate species discovered so far \citep{iucn2019} and as little as 0.1\% of all named animals \citep{vrijenhoek1998animal} reproduce exclusively asexually.
% This is puzzling because most theoretical arguments plead an elevated cost of sex as compared to asexual modes of reproduction \citep{otto2002evolution,lehtonen2012many}.
% Indeed, females invest half their reproductive resources in the production of males which, in turn, only invest minimally into the progeny: paternal care is uncommon in many — though not all \citep{smith1977parental} — species. This is known as the



% - Sexual reproduction widespread in most eukaryotes.
% - Meiosis forms the gametes, and then are fused through fertilisation.
% % - However, largely costly, because search for mate... + requires to have a doublecost.
% - So, why so widespread? One largely shared thought on this matter is the fact that recombination, which occurs during, confers a selective advantage.
% - In this chapter, which I termed after a review by Neil Hunter, I will go through the details of the mechanistic features of meiosis, and of that of recombination. So far, not elucidated and several models to explain the recombination process. I will go through all the known recombination mechanisms so far. Last, go through a rapid overview of the importance of recombination by showing the defects that can occur in cases where it does not work.
%



% INTRO
% titre de Neil Hunter.
% Une fois les bases de la génétique posées, qui sont une comprehension de l'hérédité => pour comprendre l'hérédité, besoin de comprendre les mécanismes (meiose + recombinaison).
%
% OU
% Ceux qui ont étudié la génétique, sont partis sur des observations sur l'hérédité. Et la base de l'éhérédité est la recombinaison et la méiose. Donc j'explicite ce qu'on sait dessus d'abord.
% Le titre vient d'une revue de Neil Hunter.
%
% D'abord parle de la méiose, qui est l'événement qui permet de mener à la formation des gamètes, et donc qui est l'étape clé qui explique l'hérédité.
% Cette méiose assez complexe, avec un grand nombre d'étapes.
%
% Ensuite, parle d'un aspect essentiel de la méiose: la recombinaison homologue. Quelle est la chronologie de cet événement complexe (qui est encore loin d'être totalement élucidé). Et quels sont les modèles proposés pour le comprendre (mais qui ne sont que des modèles).
%
%
% Question à moi-même (pour Laurent): si DSB, pourquoi la partie cassée du chromosome ne part pas ailleurs dans le cytoplasme?
%
%









% Biblio souris - meiose
% O’Bryan, M. K. & Kretser, D. Mouse models for genes involved in impaired spermatogenesis. Int. J. Androl. 29, 76–89 (2006).

\section{Meiosis in the context of gametogenesis}

% \subsection{The production of gametes via two cellular divisions}
\subsection{A two-step division process to form gametes}

Most sexually-reproducing organisms have diploid cells, \textit{i.e.} cells that contain two sets of chromosomes: one from each parent.
The transmission of half this genetic material to their progeny goes through the formation of specialised haploid cells (\textit{i.e.} cells containing a single set of chromosomes) called “gametes”.
%, which later fuse with another coming from the second parent during a stage called “fertilisation” to form a “zygote”, the fertilised cell that will lead to the formation of a new individual. 
Such transition from diploidy to haploidy occurs during a particular type of cell division called “meiosis” (from the Greek word \textit{\textgreek{μ}\textepsilon\textgreek{ίωσις}}: “lessening”).

% Despite some clear differences between organisms \citep{hunter2003synaptonemal, loidl2016conservation}, the most salient features of meiosis are conserved among eukaryotes.
% This suggests that their last common ancestor had already acquired it \citep{cavalier2002origins,ramesh2005phylogenomic,speijer2015sex} through a process that is still largely debated \citep{wilkins2009evolution,bernstein2010evolutionary,bernstein2011meiosis}.
The evolutionary origin of meiosis is still a mystery \citep{lenormand2016evolutionary} but its wide occurrence in eukaryotes suggests that their last common ancestor had already acquired it \citep{cavalier2002origins,ramesh2005phylogenomic,speijer2015sex} through a process that is still largely debated \citep{wilkins2009evolution,bernstein2010evolutionary,bernstein2011meiosis}.
Despite its somewhat blurry origins, its mechanistic details have been uncovered in several model organisms.\\
% its mechanistic details have been uncovered in several model organisms and, despite some clear differences between organisms \citep{hunter2003synaptonemal, loidl2016conservation}, the most salient features are conserved among eukaryotes.


% IMPORTANTS
% https://www.ncbi.nlm.nih.gov/books/NBK26840/
% Figure vient de: http://www.macmillanhighered.com/BrainHoney/Resource/6716/digital_first_content/trunk/test/hillis2e/hillis2e_ch07_5.html
% https://www.khanacademy.org/science/high-school-biology/hs-reproduction-and-cell-division/hs-meiosis/a/hs-meiosis-review
% https://www.khanacademy.org/science/biology/cellular-molecular-biology/meiosis/a/phases-of-meiosis

% LEFT PAGE
\begin{sidewaysfigure}[p]
	\centering
	\leftskip-3.4cm
	\rightskip-2.7cm
	\rotfloatpagestyle{empty}
	\includegraphics[width = 1.25\textwidth, trim = 0cm 0.61cm 0cm 0cm, clip]{figures/chap2/meiosis-with-cytology.eps}
	\captionsetup{width=1.22\textwidth, margin={-2.2cm, -3.3cm}}
	\caption{MMy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blap
		\citep{hillis2012principles}
	}
\label{fig:meiosis-cytological-steps}
\end{sidewaysfigure}

% % RIGHT PAGE
% \begin{sidewaysfigure}[p]
%     \centering
%     \leftskip-2.4cm
%     \rightskip-2.4cm
%     \rotfloatpagestyle{empty}
%     \includegraphics[width = 1.25\textwidth, trim = 0cm 0.61cm 0cm 0cm, clip]{figures/chap2/meiosis-with-cytology.eps}
%     \captionsetup{width=1.25\textwidth, margin={-2.2cm,-3.3cm}}
%     \caption{MMMy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blapy caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla bla y caption bla bla blap y caption
%     \citep{hillis2012principles}
% }
% \label{fig:meiosis-cytological-steps}
% \end{sidewaysfigure}





Concretely, meiosis is preceded by a unique round of chromosome duplication occurring during the interphase of diploid germinal cells (ovocytes in females and spermatocytes in males).
% Basically, meiosis consists of two successive cell divisions preceded by a unique round of chromosome duplication occurring during the interphase of diploid germinal cells (ovocytes in females and spermatocytes in males).
Thence, before entering meiosis, each homologous chromosome (or “homologue”) \textit{i.e.} each parental copy, is formed of two identical double-helix DNA molecules called “sister chromatids” which are physically attached at a point called the “centromere” and adjoined along their whole length by cohesins \citep{klein1999central}.
Therefrom, the two successive cell divisions that compose meiosis will result in the distribution of the chromatids into four gametes.

The first meiotic division is also known as the “reductional division” because it separates the pairs of homologues to form two haploid cells. 
It is classically divided into four stages: prophase, metaphase, anaphase and telophase (Figure~\ref{fig:meiosis-cytological-steps}, top).
Prophase I, described more extensively in Section~\ref{chap2:prophase-I}, stages the pairing of homologous chromosomes along with recombination.
Next, the meiotic splindle bonds the paired homologues and lines them up on the equatorial plate during metaphase I, before separating them during anaphase I. 
Such partition of homologues is achieved thanks to the existence of two opposite forces that stabilise the chromosomes until they are correctly oriented: first, the chiasmata that maintain the homologues attached and second, the meiotic spindle that creates a poleward tension \citep{petronczki2003menage}.
The co-segregation of sister chromatids is likely due to a physical jointure of their kinetochores \citep{nasmyth2015meiotic}.
Segregation \textit{per se} terminates at telophase I during which the chromosomes decondense and a nuclear enveloppe (NE) forms around the nuclei.
At the end of the first meiotic division, each of the two haploid daughter cells (“secondary gametocytes”) contains one pair of sister chromatids corresponding either to the paternal or to the maternal homologue.

Following a short interkinesis during which DNA does not replicate, the second meiotic division splits sister chromatids in a manner much similar to a haploid mitosis. 
This division is termed “equational” because the number of chromosomes stays equal before and after it.
Like the first one, it is partitioned into four stages (Figure~\ref{fig:meiosis-cytological-steps}, bottom) executed synchroneously in the two secondary gametocytes.
During prophase II, the NEs break down and the chromatids recondense. 
In the meantime, the centrosomes duplicated during interkinesis move towards opposite poles while a new meiotic spindle forms in between and starts to capture chromatids.
The single chromosomes line up across the equational plates of each cell during metaphase II and sister chromatids segregate towards opposing poles during anaphase II\@.
At telophase II, the chromosomes begin to decondense and new NEs form around them, thus producing the final set of four genetically-unique haploid gametes.\\

% process of meiosis:
% https://cnx.org/contents/GFy_h8cu@9.87:GYZS3DDP@8/The-Process-of-Meiosis
% https://eggsnchromosomes.com/what-is-meiosis/
% https://www.khanacademy.org/science/biology/cellular-molecular-biology/meiosis/a/phases-of-meiosis
% http://www.macmillanhighered.com/BrainHoney/Resource/6716/digital_first_content/trunk/test/hillis2e/hillis2e_ch07_5.html
% https://www.ncbi.nlm.nih.gov/books/NBK26840/
% https://www.khanacademy.org/science/high-school-biology/hs-reproduction-and-cell-division/hs-meiosis/a/hs-meiosis-review
% https://www.britannica.com/science/meiosis-cytology


Even though these general features of meiosis are shared, its timing and the products it forms are sexually dimorphic in mammals \citep[reviewed in][]{handel2010genetics}. 
Indeed, male meiosis ends in the formation of four gametes (spermatids) whereas female meiosis ends in a single functional gamete and three non functional haploid cells called “polar bodies”.

As for the timing, all along male adulthood, spermatogonia mature into spermatocytes which then initiate meiosis. As such, this results in a continuous production of sperm.
In contrast, one common conception in females is that the integrality of oogonia mature into ovocytes during fetal development \citep{pearl1921studies,zuckerman1951number}, even though recent findings suggest that oocyte production may be sustained in the postnatal ovaries \citep{johnson2004germline,johnson2005oocyte}.
In any case, female meiotic prophase I — initiated and arrested right after the production of ovocytes — is resumed in small batches of ovocytes at periodic intervals during the reproductive lifespan.
It halts once again at metaphase II, until fertilisation by a spermatozoid (if it ever occurs) triggers the finalisation of the process.\\

While the transition from plain cell cycle to meiotic entry is managed by a complex body of checkpoints \citep[reviewed in][]{marston2005meiosis}, the metronomic completion of meiotic subprocesses is abundantly warranted by the capacity of chromosomes to respond to cell cycle controls \citep[reviewed in][]{mckim1995chromosomal}.
But the most regulated — and perhaps most critical — meiotic step is the synapsis of homologous chromosomes which takes place during prophase I.



\subsection{The synapsis of homologues during prophase I}
\label{chap2:prophase-I}

\subsubsection{Four differential degrees of synapsis}
Prophase I is commonly subdivided into four stages (Figure~\ref{fig:prophase-I-stages}): leptotene (or leptonema), zygotene (or zygonema), pachytene (or pachynema) and diplotene (or diplonema).
Each is characterised by a particular chromosomal configuration that mirrors their degree of “synapsis” \textit{i.e.} pairing of homologues.

At leptotene, chromosome ends connect the cytoskeleton located outside the nucleus \citep{scherthan1996centromere} \textit{via} their binding a complex body of SUN-domain proteins of the inner nuclear membrane (INM) that have beforehand bridged KASH-domain proteins of the outer nuclear membrane (ONM) \citep{tzur2006sun, yanowitz2010meiosis}.
This allows cytoplasmic forces to animate the motion of chromosome ends at the surface of the INM \citep{penkner2009meiotic} and ends at late leptotene by the formation of a “bouquet” (Figure~\ref{fig:prepairing}) \citep{zickler1998leptotene} which constrains the chromosomes to a limited nuclear area \citep{zickler2006early}.

At zygotene, the homologous chromosomes start to synapse, starting with the telomeric regions tethered in the bouquet \citep{pfeifer2003sex}.
By the end of pachytene, synapsis is complete for all pairs of chromosomes, with the notable exception of the non-homologous male X and Y chromosomes.
Instead, the sex chromosomes are transcriptionnally inactivated (“meiotic sex chromosome inactivation”, or MSCI) by remodelling into heterochromatin \citep{fernandez2003h2ax} and are pushed to the periphery of the nucleus where they form the “sex body” \citep{handel2004xy} (Figure~\ref{fig:prophase-I-stages}.e.).
Then, during diplotene, the homologous chromosomes desynapse but remain attached in pairs \textit{via} their chiasmata.


\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/prophase-I-stages.eps}
	% \includegraphics[width = 0.7\textwidth, trim = 0cm 0cm 11.65cm 0cm, clip]{figures/chap1/morgan-drosophila-cross-results.eps}
	\caption[Chromosome organisation and cytology during prophase I]
	{\textbf{Chromosome organisation and cytology during prophase I.}
		\par Top: Two pairs of duplicated homologous chromosomes (red and blue) display different configurations in the four substages of meiotic prophase I.
		Double-strand break (DSB) formation at leptotene triggers both synapsis and the DSB resolution materialising as chiasmata during zygotene. Synapsis is completed at the onset of pachytene. Diplotene stages desynapsis, with homologues held together \textit{via} chiasmata.
		\par Bottom: immunofluorescence staining of synaptonemal complex protein 3 (SYCP3) and stage-specific signals on mouse spermatocyte spreads.\
		a: Meiosis-specific MEI4-homologue (MEI4) colocalises with the synaptonemal complex (SC).
		b: H2AX is phosphorylated (\textgreek{γ}H2AX) following DSB formation.\
		c: DNA recombinases DMC1 and RAD51 localise at DSB repair sites.\
		d: MutL protein homologue 1 (MLH1) localises at DSB sites repaired as COs.\
		e: Unrepaired DSB sites in the sex body are marked by \textgreek{γ}H2AX\@.
		\par This figure was reproduced from \citet{baudat2013meiotic}.
	}
\label{fig:prophase-I-stages}
\end{figure}


\subsubsection{Presynaptic pairing}
Matching homologous chromosomes into pairs constitutes the most critical event of synapsis.
This challenge is colossal: for human cells, it compares to finding a 20-cm stretch — other than the sister chromatid — throughout the London-Moscow distance, simulaneously for hundreds of sites and coordinately with higher-order cellular processes \citep{neale2006clarifying}.\\

This search is likely facilitated by the establishment of pre-meiotic physical contacts between homologues \citep[reviewed in][]{mckee2004homologous, zickler2006early}.
Such presynaptic pairing has been evidenced in mice \citep{boateng2013homologous} and, although its detailed mechanism remains ignored, several theories wrestle to explain its formation. 

According to one of them, presynaptic associations may occur through DNA-DNA duplexes \citep{danilowicz2009single}. 
This assumption relies on the observation that meiotic chromosomes pair only when they are transcriptionally active \citep{cook1997transcriptional}. 
DNA duplexes could thus momentarily form within the “transcription factory” to which DNA loops are attached \citep{xu2008similar}.
Alternatively, these associations may be promoted by sequence-specific RNA molecules, in a manner similar to gene silencing in plants and fungi \citep[cited in \citealp{zickler2006early}]{bender2004dna}.
A third scenario suggests a mechanism analogous to the “pairing centres” (PC) or “homologue recognition regions” (HRR) described in \textit{Caenorhabditis elegans} \citep{villeneuve1994cis,macqueen2005chromosome}, \textit{Drosophila melanogaster} \citep{mckee1996license} and \textit{Saccharomyces cerevisiae} \citep{kemp2004role}.
Namely, the \textit{cis}-acting PCs (or HRRs) could initiate interactions between homologues \citep{gerton2005homologous}.\\


In any case, demonstrating the existence of such presynaptic pairing in mice has driven \citet{boateng2013homologous} to propose a new model for homology search (Figure~\ref{fig:prepairing}).
With it, they challenge the commonly accepted view that homology search is triggered by the need to repair newly-opened DNA double-strand breaks (DSBs).
Instead, they propose that DSBs occur after the pre-leptotene pairing and that their repair serves as a prophase checkpoint to proofread the initial connection.
If that were so, homology search for DSB repair would be restrained to a reduced territory and thus, much facilitated \citep{barzel2008finding,mirny2011fractal}.\\
%\citep[cited in \citealp{boateng2013homologous}]{mirny2011fractal}

Whether or not this view is correct, chromosomal movements allow random collisions between chromosomes \citep{fung1998homologous}, thus creating opportunities for homologues to encounter and, more importantly, to disrupt unwanted (non-homologous) associations \citep{koszul2009dynamic}.
Yet, at this stage, the interstitial interactions between homologues are transient and reversible \citep{boateng2013homologous}.
They thus need to be strengthened by a higher-order chromosomal structure: the synaptonemal complex (SC).

% Mutations that disrupt chromosome movements also result in loss of this nuclear reorganization during leptotene/zygotene (MacQueen and Villeneuve, 2001; Couteau et al., 2004; Couteau and Zetka, 2005; Martinez-Perez and Villeneuve, 2005; Penkner et al., 2007b).



\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/prepairing.eps}
	% \includegraphics[width = 0.7\textwidth, trim = 0cm 0cm 11.65cm 0cm, clip]{figures/chap1/morgan-drosophila-cross-results.eps}
	\caption[Mouse preleptotene DSB-independent pairing model proposed by \citet{boateng2013homologous}]
	{\textbf{Mouse preleptotene DSB-independent pairing model proposed by \citet{boateng2013homologous}.}
		\par \citet{boateng2013homologous}'s model stipulates that the tethering of telomeres (green points) to the NE in late preleptotene facilitates the initiation of synapsis at subtelomeric regions by simplifying the search for the homologous chromosome (light and dark grey lines). 
		The authors also conjecture that, upon entry into prophase (leptotene), this DSB-independent pairing at non-telomeric sites is lost, but that telomeric pairing is maintained at least at one end until homologues recombine. 
		Ultimately, DSB repair and synapsis at zygotene and pachytene would progressively restore pairing at non-telomeric sites.
		This figure was reproduced from \citet{boateng2013homologous}.
		}
\label{fig:prepairing}
\end{figure}




\subsubsection{The synaptonemal complex (SC)}
The synaptonemal complex (SC), discovered by \citet{fawcett1956fine} and \citet{moses1956chromosomal}, is a remarkably well-conserved ribbon-like proteinaceous structure composed of three units: two dense lateral (or axial) elements (LE) and — except in the green alga \textit{Ulva} \citep{braten1973ultrastructure} and \textit{Chlamydomonas} \citep{storms1977fine} — one less dense central element (CE) (Figure~\ref{fig:synaptonemal-complex}) \citep{schmekel1995central}.

LEs resemble axes along which the sister chromatids are loaded, binding short stretches of DNA to the LE and condensing the rest of it into long loops of tens to hundreds of kilo bases (kb). 
Generally, the loops closer to the telomeres are much shorter than the ones located elsewhere \citep{heng1996regulation}.

LE assembly begins at leptotene with the aggregation of both REC8 cohesins and axial proteins (SCP2 and SCP3 in mammals) into small fragments \citep{eijpe2003meiotic} which later fuse into full LEs \citep{schalk1998localization}.
At full synapsis, they are connected to the CE (formed of SYCE1 and SYCE2 proteins \citep{pera2013stem}) by transverse filaments (TFs), thus giving the SC a striated, zipper-like appearance.
The main constituent of TFs — the SCP1 protein, in mammals — has homologues in worms \citep{macqueen2002synapsis,colaiacovo2003synaptonemal}, flies \citep{mckim2002meiotic} and yeasts \citep[reviewed in][]{zickler1999meiotic} that, despite little sequence conservation, display a similar structure: two head-to-head homodimers of a ${\sim}80$~nm coiled coil flanked by globular C and N termini \citep{meuwissen1992coiled,liu1996localization}.
The polymerisation of these central region proteins between paired homologue axes results in the tight pairing (${\sim}100$~nm) of the bivalents\footnote{Homologous chromosomes} along their entire length at the end of pachytene \citep{page2004genetics}, as compared to their ${\sim}400$-nm spacing during presynaptic coalignement \citep{tesse2003localization}.\\


\begin{figure}[h]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/synaptonemal-complex.eps}
	% \includegraphics[width = 0.7\textwidth, trim = 0cm 0cm 11.65cm 0cm, clip]{figures/chap1/morgan-drosophila-cross-results.eps}
	\caption[Structure of the synaptonemal complex (SC)]
	{\textbf{Structure of the synaptonemal complex (SC) by \citet{loidl2016conservation}.}
		\par Original legend the author: “The SC consists of a pair of parallel strands, the lateral
		elements, that are linked by transversal filaments. The central element runs halfway between the lateral
		elements. Loops of sister chromatids are tethered both to each other and to a lateral element. Synapsis
		progresses along pairs of homologous chromosomes in a zipper-like fashion. The axes of unsynapsed
		portions are called axial elements. Initial homologous interactions may or may not need axial elements. The
		sites of crossing over are marked by recombination nodules, which are located between the axial elements.”
		This figure was reproduced from \citet{loidl2016conservation}.
		}
\label{fig:synaptonemal-complex}
\end{figure}



Synapsis is indeed the most commonly acknowledged role of the SC, but it may also act to limit recombination with the sister chromatid.
Avoiding the sister may seem a trivial problem given the 2:1 odds ratio in favour of homologue templates \citep{lao2010trying}. 
However, an important guarantee of genome stability is the preferential use of the sister chromatid in mitotically dividing cells \citep{kadyk1992sister,bzymek2010double} which is likely promoted by their cohesin-dependent proximity \citep{sjogren2010sphase}.
Thus, switching this mitotic inter-sister bias to a meiotic inter-homologue bias is essential for synapsis.
Even though this could be ensured by other features of meiosis \citep[reviewed in \citealp{humphryes2014non}]{schwacha1997interhomolog, goldfarb2010frequent, hong2013logic}, recent evidence points that the components of the CE are effectively involved in template choice \citep{kim2010sister} as was suggested in the past \citep{haber1998meiosis}.\\

Microscopy observation of the SC show dense nodules where recombination occurs (“recombination nodules”: RN) \citep{carpenter1975electron, schmekel1998evidence}.
Indeed, the formation of DSBs is a prerequisite for SC formation in many species including plants and mammals \citep{zickler1999meiotic, henderson2004tying}.
Yet, the meiotic program seems to vary for other species: SC formation is recombination-independent in \textit{Caenorhabditis elegans} \citep{dernburg1998meiotic}, \textit{Bombyx mori} \citep{rasmussen1977transformation} and \textit{Drosophila} females \citep{mckim1998meiotic} (and recombination does not even occur in \textit{Drosophila} males \citep[reviewed in][]{tsai2011homologous}) whereas \textit{Schizosaccharomyces pombe} \citep{bahler1993unusual} and \textit{Aspergillus nidulans} \citep{egel1982meiosis} recombinate but have no SC \citep[reviewed in][]{zickler2015recombination}.

More generally, whenever recombination is associated with the SC, it is clear that its correct formation is of paramount importance to create stable DNA connections between homologues \citep[reviewed in \citealp{hunter2003synaptonemal}]{hunter2001singleend}.
These form \textit{via} the process of homologous recombination.
\textbf{AJOUTER LIEN VERS MODELS\@: Several models have been proposed to explain meiotic recombination. Reviewed in the next section. Aussi, dans chronology of meiotic recombination (appeler molecular mechanisms of meiotic recombination), dire que les evidence moleculaires collent avec DSBR et que le lien entre le role et les proteines est basee sur ce modele la.}

- voir les modeles de recombinaison pour comprendre comment l'echange se fait entre l'ADN des deux chromosomes. Pas vraiment su, des modeles. Ceci etant dit, on observe un ensemble de proteines dont on predit, sur la base de ces modeles, un role, et on observe egalement des CO et des NCO\@.



% - CO SC independent: Kohl and de los Santos
% A mettre plus loin quand on parle des CO ET DE LA POSITIVE INTERFERENCE??
% Loidl
% which stabilizes the connection between partner chromosomes (151). It is also crucial
% for the maturation of a certain class of COs, the number and distribution of which are controlled
% by mutual local exclusion (positive interference).
% This CO class is predominant in most eukaryotes studied so far, but a small proportion of COs are SC independent and noninterfering (see 29, 66).
% 66. Kohl KP, Sekelsky J. 2013. Meiotic and mitotic recombination in meiosis. Genetics 194:327–34
% 29. de los Santos T, Hunter N, Lee C, Larkin B, Loidl J, Hollingsworth NM. 2003. The Mus81/Mms4
% endonuclease acts independently of double-Holliday junction resolution to promote a distinct subset of
% crossovers during meiosis in budding yeast. Genetics 164:81–94

% SUR LES CROSSOVERS (CLASSES)
% Hunter - synaptonemal complexities
% Recent data suggest that two classes of crossovers form in S. cerevisiae (and probably plants and mammals; e.g., de Los Santos et al., 2003). The majority, Class-I, is subject to distribution controls and dependent on Zip1 and at least five other meiosis-specific proteins (Börner, V., Kleckner, N. and N. Hunter, unpublished data). Loss of Class-I crossovers causes chromosomes to missegregate in most meioses, inferring a critical function for these events, and thus SC, in S. cerevisiae. In contrast, Class-II crossovers are randomly distributed and SC independent. Loss of these events does not affect SCs or cause chromosomes to missegregate.



% % SISTER
% Hinch: Nevertheless, it appears that some fraction of programmed meiotic DSBs are repaired using the sister chromatid [Hyppa and Smith, 2010].
% sister attached by cohesins Klein 1999. Why not homology?
% sex chromosomes: no homology donc repaired via sister chromatids at late prophase. Silenced (cf Altemose)
%
% % SYNCHRONISATION CELL CYCLE
% Meiotic success also hinges on theability to synchronize the meiotic transcriptional programwith cell cycle progression and cell growth. This isachieved in yeast by coupling double strand break for-mation with progression of the replication fork
% % Borde V, Goldman AS, Lichten M:Direct coupling betweenmeiotic DNA replication and recombination initiation.Science2000,290:806-809.
%
% % REGULATION
% This step regulated: Pachytene checkpoint: avoid defects (Handel Schimenti)
% If error, meiotic silencing (https://en.wikipedia.org/wiki/Synapsis)
%
% % DIFF MEIOSE MITOSE
% attachement des chromosomes par les kinetochores differe de la meiose (https://cshperspectives.cshlp.org/content/7/5/a015859.long)
% Autre difference avec la mitose: le spindle qui peut etre asymetrique.
%







\section{Chronology of meiotic recombination}

Homologous recombination (HR), which occurs during prophase I, is the process through which homologous chromosomes physically interact and exchange strands of DNA\@. 
In certain cases, it leads to the formation of a (relatively) long-term connection that maintains the bivalents together until their separation at anaphase I\@.

HR begins at leptotene with the formation of a DNA double-strand break (DSB) on one homologue. 
To repair properly, this crack needs a DNA strand to use as template. There begins a homology search accomplished at zygotene by the broken-strand invasion of the mating chromosome.
The template-based repair process creates a transient structure, subsequently resolved into either a crossing-over (CO) or a non-crossover (NCO) during late zygotene and pachytene.

In mammals, each of these actions is executed by a complex body of proteins summarised in Figure~\ref{fig:recombination-mammalian-proteins}. 



\subsection{Initiation of recombination}

The evolutionarily conserved SPO11 endonuclease — observed in a wide range of species \citep{baudat2000chromosome,mckim1998meiw68,romanienko2000mouse,steiner2002meiotic,bowring2006chromosome,stacey2006arabidopsis} — catalyses the programmed formation of DSBs \citep{keeney1997meiosisspecific,bergerat1997atypical} that marks the beginning of HR \citep{sun1989double}.
Of the two isoforms found in mice \citep{metzler-guillemain2000identification}, SPO11\textgreek{β} is the one responsible for DSB formation \citep{bellani2010expression}.
DNA cleavage by this homodimeric protein leaves a two-nucleotide 5' overhang \citep{demassy1995nucleotidea}	onto which it remains trapped till the further processing of DSB ends (see Section~\ref{chap2:DSB-repair}) \citep[reviewed in][]{cole2010evolutionarya}.

Several other proteins have been identified as essential for the correct formation of DSBs (extensively reviewed in \citealp{keeney2008spo11a} and \citealp{demassy2013initiation}).
Among them, the yeast MER2-MEI4-REC114 complex \citep{li2006saccharomyces,maleki2007interactions} and two mouse homologues (MEI4 and REC114) have been identified as functional and important for nick generation \citep{kumar2010functional,kumar2015mei4}, thus suggesting a conserved mechanism for recombination initiation. 
Nevertheless, the mammalian system has some specificities since MEI1 \citep{libby2002mouse,libby2003positional}, which does not set forth any yeast homologue, has been uncovered as essential for normal DSB levels, along with HORMAD1 (yeast homologue: HOP1) \citep{shin2010hormad1,daniel2011meiotic}.

Once DSBs have been generated, the ataxia telangiectasia mutated (ATM) kinase both phosphorylates the 139\textsuperscript{th} serine residue of histone H2AX variants located in their vicinity (then named \textgreek{γ}H2AX) \citep{rogakou1998dna,burma2001atm} and thwarts further DSB apertures of DNA \citep{lange2011atm,lukaszewicz2018control}.\\

In mice and humans, ${\sim}$200—400 DSBs are initiated in this manner at early leptotene and, from this point forward, have to be repaired to secure the production of viable gametes.







\subsection{Repair of double-strand breaks (DSBs)}
\label{chap2:DSB-repair}

\begin{figure}[p]
	\centering
	\includegraphics[width = 1\textwidth]{figures/chap2/recombination-mammalian-proteins.eps}
	% \includegraphics[width = 0.7\textwidth, trim = 0cm 0cm 11.65cm 0cm, clip]{figures/chap1/morgan-drosophila-cross-results.eps}
	\caption[Proteins involved in mammalian meiotic recombination]
	{\textbf{Proteins involved in mammalian meiotic recombination.}
		\par blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blabla
		\par r blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blabla
		\par r blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blablar blabla blabla blabla
		\par This figure was reproduced from \citet{baudat2013meiotic}.
	}
\label{fig:recombination-mammalian-proteins}
\end{figure}



\subsubsection{DSB-end processing}
The repair of DSBs begins with the processing of its ends: an endonucleolytic cleavage several nucleotides downstream of the 5' end \citep{neale2005endonucleolytic} is executed by the MRE11 complex both in yeasts \citep[reviewed in][]{borde2009double} and mammals \citep[reviewed in][]{borde2007multiple}.
In \textit{Saccharomyces cerevisiae} and \textit{Caenorhabditis elegans}, MRE11 acts collaboratively with RAD50 and NBS1, two proteins required for DSB mending \citep[reviewed in][]{lam2015mechanism}. 
Both have mammalian homologues, but their putative role in DSB repair \citep[reviewed in][]{baudat2013meiotic} is hard to prove since knocking them out is lethal for mice \citep{luo1999disruption,zhu2001targeted}.
In contrast, that of a DNA polymerase (Pol \textgreek{β}) in releasing SPO11-bound oligonucleotides at DSB ends was unequivocally evidenced \citep[reviewed in \citealp{baudat2013meiotic}]{kidane2010dna}.



\subsubsection{Strand invasion}
As removal of SPO11 is paired with the 5’-to-3’ end resection of the DSB, 3’ single-stranded DNA (ssDNA) tails become accessible to the nuclear machinery (Figure~\ref{fig:recombination-mammalian-proteins}.b.).
As such, RPA proteins rapidly bind them \citep{he1995rpa} but are then displaced by RAD51 and/or DMC1 recombinases \citep{pittman1998meiotic,yoshida1998mouse} which catalyze the pairing and exchange between the ssDNA strand and the intact, homologous double-stranded DNA (dsDNA).
Their relationship is complex: RPA is necessary both for RAD51 filament formation and for DMC1-catalysed strand exchange, but notwithstandingly, it also competes with them for ssDNA binding \citep{sung2003rad51}.

The proper functioning of DMC1 and RAD51 in strand invasion requires several other proteins that interact with either one or both of them: HOP2 and MND1 \citep{bugreev2014hop2mnd1}, BRCA1 \citep{scully1997association} and BRCA2 \citep{thorslund2007interactions}.
This complex process also requires other, less well-characterised actors that I will not describe here for they are of little interest for the scope of this thesis \citep[but for review, see][and Figure~\ref{fig:recombination-mammalian-proteins}.c.]{neale2006clarifying}.\\

\textbf{CF LESECQUE aussi parler de SEI et du fait que peut etre rejete si pas d'homologie pour eviter la recombinaison ectopique}

\subsubsection{Recombination-intermediate processing}
The interaction between the invading strand and the homologue is subsequently stabilised by several proteins.
Indeed, BLM, TEX11 (yeast homologue: ZIP4) and RNF212 (yeast homologue: ZIP3) appear at zygotene at recombination foci and progressively decrease until the end of pachytene, \textit{i.e.} when DSBs are repaired \citep[reviewed in][]{baudat2013meiotic}. 
In addition, together with MCM8 and MCM9 proteins \citep{lutzmann2012mcm8}, heterodimers of MSH4 and MSH5 \citep{scully1997association} are requisiste for synapsis stabilisation in both mice \citep{devries1999mouse,kneitz2000muts} and humans \citep{snowden2004hmsh4hmsh5}.

Though, the role of MSH4 continues beyond synapsis establishment. 
Indeed, the stabilisation of the interaction between the two homologues creates an intertwined recombination intermediate structure, and MSH4 participates in its resolution when it leads to a certain class of COs, but also, as argued by \citep{baudat2007regulating}, when it leads to NCOs.
% The stabilisation of the interaction between the two homologues creates an intertwined recombination intermediate structure, — a Holliday junction (HJ), — which needs to be resolved.





\subsection{Resolution of recombination intermediates}

Recombination intermediate structures may be resolved \textit{via} two main pathways (Figure~\ref{fig:recombination-mammalian-proteins}.d.\ and e.).
In the pathway leading to COs, the non-invading strand of the broken chromosome interacts with the displaced homologue strand which forms the D-loop.
In constrast, in the pathway leading to NCOs, the non-invading strand anneals again the invading strand from the same chromatid, after the latter has elongated on the homologue and displaced from it.


\begin{figure}[p]
	\centering
	\includegraphics[width = 0.85\textwidth]{figures/chap2/baudat-pathways-resolution.eps}
	% \includegraphics[width = 0.7\textwidth, trim = 0cm 0cm 11.65cm 0cm, clip]{figures/chap1/morgan-drosophila-cross-results.eps}
	\caption[Molecular mechanism of pathways leading to crossing-overs (COs) and non-crossovers (NCOs)]
	{\textbf{Molecular mechanism of pathways leading to crossing-overs (COs) and non-crossovers (NCOs).}
		\par Refaire une figure qui ressemble a celle-ci \citep{baudat2007regulating} mais qui incluera en plus l'information sur la resolution symmetrique et asymmetrique (cf \citep{wyatt2014holliday} — Figure 1). Il faudra ajouter dans le texte les ref a cette figure.
	}
\label{fig:pathways-resolution}
\end{figure}


\subsubsection{The CO pathway}
In certain cases, the homologues are physically bound twice: one strand from each chromosome (the invading strand and the D-loop strand) displaces to bind the homologue, thus creating a double Holliday junction (dHJ).
TEX11 (yeast homologue: ZIP4), RNF212 (yeast homologue: ZIP3) and HFM1 (yeast homologue: MER3) are though to play a role in processing the dHJ, since knocking one of them out leads to a diminished level of chiasmata and COs \citep[reviewed in \citealp{baudat2013meiotic}]{adelman2008zip4h,guiraldelli2013mouse,reynolds2013rnf212}.
In yeasts, MER3, one of the proteins of this ZMM complex, specifically interacts with RAD51 to stimulate heteroduplex extension \citep{mazina2004saccharomyces}.\\

The resolution of the dHJ \textit{per se} is catalysed by resolvases, \textit{i.e.} enzymes that slice the interwound strands.
In mice, a pair of symmetrical nicks is introduced across the helical branchpoint of most (90\%) dHJs either by a heterodimer of MLH1 \citep{baker1996involvement, edelmann1996meiotic} and MLH3 \citep{lipkin2002meiotic} or by EXO1 \citep{wei2003inactivationa}.

Alternatively, the dHJ can be resolved by introducing two single-stranded incisions \citep{wyatt2014holliday}. 
In that case, the two nicks are asymmetric and can be located several nucleotides away from the branchpoint. 
This resolution is catalysed by MUS81 and EME1 (yeast homologue: MMS4).
In \textit{Saccharomyces pombe} where it was first discovered, it is the only pathway to produce COs \citep{osman2003generating}.
However, in plants \citep{mercier2005two} and in mice \citep{holloway2008mus81}, it coexists with the MLH1-dependent CO pathway.\\


Of the 200—400 recombination foci in mice, only 20 (one per chromosome) lead to a CO \citep{baudat2007regulating}.
This implies the existence of another repair pathway: that leading to NCO events.


\subsubsection{The NCO pathway}
Instead of being resolved, the dHJ is sometimes dissolved by the BLM helicase together with a topoisomerase \citep{wu2003bloom}. 
This pathway thus interferes with the formation of COs and, indeed, inactivating BLM leads to an increased number of chiasmata \citep{holloway2010mammalian}.

Though, most NCOs are formed \textit{via} another pathway that occurs before the resolution of dHJs: the synthesis-dependent strand-annealing (SDSA) pathway \citep{allers2001differential}.
In it, after the invading strand has extended past the site of the DSB, the D-loop is disrupted and the invading strand anneals its complementary ssDNA on the other side of the DSB\@.
Intermediates of this pathway have been directly observed in \textit{Saccharomyces cerevisiae} \citep{mcmahill2007synthesisdependent} in which it produces the large majority of NCOs \citep{martini2011genomewide}.
Molecularly, the dissociation between the invading strand and the homologue is promoted by the SRS2 helicase \citep{ira2003srs2}.\\


Everything that has been described so far corresponds to recombination between the homologous chromosomes. 
Though, recombination occurs in two additional forms.

First, the broken strand can interact with the sister chromatid, as is the case in mitotic cells. 
Indeed, mitotic DSBs are not programmed but occur spontaneously when DNA is damaged.
In such cases, it is important for genome integrity to recombine with an identical DNA\@: that of the sister chromatid. 
Such breaks are mainly repaired \textit{via} non-homologous end-joining (NHEJ), which consits in directly ligating the broken strands of DNA \citep{weterings2004mechanism} and which is highly prone to errors \citep{smith2001influence}.

Second, non-allelic sequences located at different genomic locations — generally low copy repeats resulting from duplication events \citep{bailey2006primate} — can recombine together. 
This is called non-allelic homologous recombination (NAHR) (or “ectopic recombination”) and proceeds similarly to HR \citep{sasaki2010genome}.

Since both these processes go far beyond the scope of this thesis, and I will not detail them any further.\\

In all three cases (mitotic recombination, NAHR and HR), errors in the recombination process sometimes arise. 
Since HR is crucial for the correct segregation of homologues, any error arising in this process can have dramatic consequences for the forthcoming gametes.




\textbf{NOTES\@: notions non abordees ici qui le seront dans d'autres chapitres. CO interference + obligatory CO (chap3 avec la distribution des CO/NCO/DSB sur les chromosomes. Il faudra a ce moment faire le lien avec CO interferents par Mlh1 et CO non interferents par Mus81 + au moins un dnas le PAR), conversion tract + MMR + BER + conversion/restauration (chap4 avec conversion genique). Voir ou je place le brassage genetique par les CO qui font des recombinants + la segregation aleatoire des chroms. Pas sur que ce soit necessaire d'en parler.}
% DANS CROSSING-OVER: generate genetic diversity along with independent assortment (http://www.macmillanhighered.com/BrainHoney/Resource/6716/digital_first_content/trunk/test/hillis2e/hillis2e_ch07_5.html)
Parler de NHEJ



DANS LES PROBLEMES
Aussi: il faudra parler du fait que, chez les femmes, il y a d'autres DSB qui se forment de facon non programmee. Et sont repares differemments (en particulier, plus longs + plus complex? voir chapitre 3). Plus, des problemes de trisomie plus souvent chez les femmes a cause du dictyate arrest.

In females, chiasmata maintained for decades because stop at the end of diplotene (dictyate stage). May explain why higher aneuploidy and especially with maternal age.





% https://www.ncbi.nlm.nih.gov/books/NBK21986/ — tetrad analysis utilisee pour mapper les doubles CO


%%%%%% chap3: DSBs: In humans and mice, DSBs tend to avoid centromeres and heterochromatic acrocentric short arms, although they are heavily enriched near telomeres in human males [17]. % ALTEMOSE

% Spatial distribution of DSBs: Lam and Keeney 2015
% CHAPITRE 3: Revue de Massy initiation recombinaison parle des cartes de CO chez plein d'especes + les differences de recombinaison selon les regions genomiques
% POUR CHAPITRE 3: DONC au total pour observer, on peut regarder ATM, Spo11. Egalement generalement RAD51 et DMC1. mais donc plus large car dans processing. 200 a 400 foci dans les souris.

% CHAPITRE 3
% BAUDAT MASSY 2013
% In addition, the number of SPO11-dependent DSBs formed in every meiocyte is regulated by several factors. In contrast to crossovers, meiotic DSBs are difficult to quantify, especially because they are transient. Counting the foci formed by the recombinases RAD51 and DMC1 (see section below) has been widely used as the best available estimate of the number of DSBs in individual cells (oocytes or spermatocytes). It gives a count of 200–400 foci per cell in mice and humans3. A method for comparing the relative level of meiotic DSBs between mice of different genotypes was recently developed by quantifying the global level of oligonucleotides that are covalently attached to SPO11 (REF. 65). The role of the ataxia telangiectasia mutated (ATM) kinase as a nega- tive regulator of meiotic DSBs has been demonstrated using this method66. Situations in which SPO11 is unable to generate a wild-type number of DSBs correlate with defects in synapsis, suggesting that a minimum number of DSBs is needed to allow proper interactions between homologues67,68.
%


% Aussi des hotspots (chapitre 3) - depuis Baudat et de Massy CO
% CO are not randomly distrib- uted, but occur in multiple specific regions of the genome called CO hotspots (de Massy 2003, Kauppi et al. 2004). A hotspot is a region 1Y2 kb wide where CO are clustered, as a result of localized initiation events (see below). The average spacing between hotspots is 50Y100 kb and hotspot activities vary over three to four orders of magnitude, from 0.9 to 0.0005 cM as determined in the human genome. In the human genome the number of hotspots is estimated around 25 000Y50 000 (Myers et al. 2005). The variations of hotspot density and activity along chromosomes result in domains with high (jungles) or low (desert) recombination activity. Most sub-telomeric regions are recombination jun- gles in male meiosis whereas centromeric regions are recombination deserts. However, the determinants of CO variation along chromosomes are not known, even though some factors correlated with CO density are beginning to be analyzed (Buard & de Massy 2007, TIG in revision).
%


%% INTERFERENCE
% Cf lesecque revue + POPA Un paragraphe tres complet
% + dans Baudat 2007 et 2013: interfering par dHJ, non-interfering Ms81.
% Pas d'interference (ou peu? Baudat CO) dans les NCO

%% INTERFERENCE (CAPILLO)
% Since only a small fraction of DSBs are eventually processed as COs, a highly regulated genetic control determines both CO homeostasis and chromosomal distribution. In this way, if a CO occurs in a certain position, the probability for a new CO to take place nearby increases with chromosomal length. As a consequence, COs tend to follow an evenly spaced distribution across chromosome axes [Jones, 1967; Kleckner et al., 2003; Wang et al., 2015]. Importantly, this CO interference is influenced by the physical distance along the chromosomal axes (micrometers) rather than the genomic (Mb) or genetic distance (cM) [Wang et al., 2015]. However, not all COs are subject to interference, leading to recognition of interfering (class I) and non-interfering (class II) COs in different organisms [Hollingsworth and Brill, 2004; Phadnis et al., 2011]. Non-interfering COs are Mus81-Mms4 dependent and distribute themselves randomly along the chromosomes independent of each other, whereas interfering COs have been found to be distributed according to a gamma distribution. In mice, most COs manifest interference and are controlled by proteins Msh4-Msh5 [Berchowitz and Copenhaver, 2010], although some Mus81 activity has been detected during meiosis [Holloway et al., 2008]. Despite the evolutionary rationale of CO interference is still unknown, spaced COs ensure faithful chromosomal segregation and might facilitate linkage of functionally related genes [Wang et al., 2015 and references therein].

% OBLIGATORY CO
% These cytological analyses also show that at least one CO is formed per chromosome arm both in humans and mice, apart from short heterochromatic arms from acrocentric chromosomes. This regulation of CO frequency is a manifestation of the rule of the obligatory CO. Interestingly, mouse strains carrying Robertsonian translocations have two CO per chro- mosome, one per euchromatic arm (Dumas & Britton- Davidian 2002). The rule of the obligatory CO is also observed between the X and Y, where one CO is always observed in the pseudoautosomal region. A second level of CO regulation is shown by the
% measures of distances between chiasmata or Mlh1 foci, or by genetic distances. These show that, both in mouse and human, CO are not randomly distributed and are more evenly spaced than expected if they occurred independently, a phenomenon defined as positive interference (Lawrie et al. 1995, Laurie & Hulten 1985, Anderson et al. 1999, Broman & Weber 2000, Broman et al. 2002).
%

% Voir aussi Neil Hunter: CO control

% https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003294/ : CO control (Yanowitz)

%%% DANS GENE CONVERSION (CHAPITRE 4)
% Parler de gene conversion + conversion/restauration
% Parler de tract
% Parler de heteroduplex
% MMR et BER






\section{Models of recombination}

Voir aussi: orr-weaver et szostak
Et parler de asymmetric VS symmetric heteroduplex DNA (orr-weaver and szostak — section Polarity)


% https://books.google.fr/books?id=7V0N6Tt8fUwC&pg=PA43&lpg=PA43&dq=murray+1960+polarity&source=bl&ots=mtj-qfJ1ZM&sig=ACfU3U1rKTqzCqEtcJkNw4ex96F_KPI87Q&hl=fr&sa=X&ved=2ahUKEwiG0b39-tfgAhUJ0RoKHRn4CWsQ6AEwB3oECAkQAQ#v=onepage&q=murray%201960%20polarity&f=false
Sur la polarité des gene conversion DONC des sites precis ou la recombinaison demarre (a mettre dans les points chauds de recombinaison).

N. Saitou, Introduction to Evolutionary Genomics, Computational Biology 17,
% DOI 10.1007/978-1-4471-5304-7_2, © Springer-Verlag London 2013
file:///Users/maudgautier/Downloads/9781447153030-c2.pdf
Recombination was discovered by Thomas Hunt Morgan and his colleagues in the
early twentieth century. The concept of “gene conversion” was fi rst proposed by
Winkler in 1930 [ 11  ], but it was not fully accepted for a long time, until studies on
fungi clearly showed conversion events [ 12, 13  ]. Holliday (1964; [ 14  ]) proposed the
“Holliday structure” model (Fig. 2.14) to connect gene conversion, or nonreciprocal
transfer of DNA fragment, and recombination.


Early studies on gene conversion were mostly restricted to fungal genetics. As
molecular evolutionary studies of multigene family started, unexpected similarity
of tandemly arrayed rRNA genes was found [ 15  ]. This phenomenon was termed
“concerted evolution,” and gene conversion or unequal crossing-over was proposed
to explain this characteristic of some multigene families (e.g., [ 16  ]). New statistical
methods were developed to detect gene conversion between homologous
sequences [ 17, 18  ]. Program GENECONV developed by Sawyer [ 19  ] became the
standard tool for analyzing gene conversions. We now know that conversion can
occur in any genomic region irrespective of genes (DNA regions having function)
or nongenic regions (e.g., [ 20  ]). However, “gene conversion” as technical jargon is
currently widely accepted, and I follow this nomenclature. Gene conversion can be
classifi ed into two types: intragenic or between alleles and intergenic or between
duplicated genes. 


When Winkler [ 11  ] proposed gene conversion in 1930, it was a deviation from
the Mendelian ratio. Later, detailed observations on baker’s yeast and Neurospora
[ 12, 13  ] established gene conversion, and Holliday’s [ 14  ] model transformed
gene conversion from phenomenon to mechanism. Nowadays several enzymes
are known to be involved in DNA strand exchanges [ 30  ]. Abundant genome
sequence data and their computational analyses again turned gene conversion or
more fl atly homogenization of homologous sequences from mechanism to phenomenon. We should be careful of any prejudice to a particular phenomenon when we
try to interpret them with certain mechanism. One phenomenon, such as homologous sequence homogenization, may occur not only via gene conversion but with
some other mechanisms, including one unknown to us at this moment. It is obvious
that we should grasp molecular mechanism of gene conversion, including enzymatic
machineries. 


\section{The importance of meiotic recombination}

Meiose essentiels, car certains mutatns qui empechent la mutation sont non viables (orr-weaver and szostak)

+ Parler de Gene conversion
% + Parler de Male vs Female meiosis  https://cellbiology.med.unsw.edu.au/cellbiology/index.php/Meiosis
+ errors in meiosis
+ regulation of meiois (cyclins)



Quand parle des maps de linkage: voir citation Muller %1920:98-101
“[I]t has never been claimed, in the theory of linear linkage, that the per cents of crossing over are actually proportional to the map distances: what has been stated is that the per cents of crossing overs are calculable from the map distances…”





\section*{Sorte de plan / Liste d'idees}
historic
initiation of recombination: 
- Homologue pairing / interhomolg interactions
- determinsation localisation
- formation DSB / programmed DSB formation
Meiotic DSB repair:
- Homoology search
- synapsis between homologues
- models of recombination
- Resolution into CO/NCO
- Dissolution of the SC
Crossover control: 
- assurance / interference
- differentiation CO/NCO
Chromatin state: shapes the recombination landscape (ou dans position des hotspots): nucleosome occupancy + meiotic chromosome architecture. 
Importance of meiotic recombination: Genetic disorders otherwise + exemple de l'un qui a perdu PRDM9 mais qui n'est pas stérile pour autant. 
Checkpoints
strand asymmetry
DNA polymerases
Gene conversion ici? + non-allelic gene conversion (et un impact sur la détection de)



Deuxieme chapitre:
Methodological approaches to study recombination
Variation of recombination rates wtithin genomes and among species
evolvability of recombination rates



\section*{Notes temporaires}

%%%%% Modele de Odenthal-Hesse (chapitre sur recombinaison)

% \section{Chronology of meiotic recombination}
% \subsection{Programmed DSB formation}
% \subsection{Strand invasion and junction molecule formation}
% \subsection{Mismatch repair}
% \subsection{Resolution}
%
% \section{Models of recombination}




%%%%% Modele de Papier Baudat de Massy 2013

% \section{Initiation of recombination}
% \subsection{Homologue pairing}
% \subsection{Programmed DSB formation}
%
% \section{Meiotic DSB repair}
% \subsection{Homology search}
% \subsection{Synapsis between homologues}
% \subsection{Models of DSB repair}
%
% \section{Resolution CO/NCO}
% \subsection{Differentiation CO/NCO}
% \subsection{CO interference}



Plan chronologique
\begin{itemize}
	\item Mammalian meiosis (overview of the cycle)
	\item (Zoom sur Prophase 1)
	\item Leptotene stage: Initiation of recombination (Homologue pairing before DSB + Determination localisation DSB + DSB) + Miotic DSB repair (homology search)
	\item Zygotene stage: Meiotic DSB repair (Synapsis between homologues + Start resolution CO/NCO)
	\item Pachytene stage: Meiotic DSB repair (Resolution CO/NCO)
	\item Pachytene + Zygotene stages: Preparation to metaphase I (dissolution of SC)

\end{itemize}

Plan Neil Hunter the essence of heredity
\begin{itemize}
	\item Meiosis and the roots of recombination research
	\item Molecular models of meiotic recombination
	\item Interhomolog interactions
	\item Programmed DSB formation
	\item Crossover control (Assurance and interference, differentiation CO/NCO, pro-CO role of the synaptonemal complex, recombination associated DNA synthesis)
	\item Resolving, disolving and unwinding joint molecules to implement CO and NCO fates (differential timing and regulation of CO and NCO formation, MutL and EXO1=CO-specific resolving factor, MUS81 enzymes = role in meiotic joint molecule processing, STR/BTR ensemble as master regulators of meiotic joint molecule metabolism, SLX4-associated endonucleases and he GEN1 resolvase, SMC complex facilitates joint molecules formation and resolution, implementing NCO formation)
	\item Clinical significance of meiotic recombination (Aneuploidy CO and advancing maternal age, meiotic recombination and genomic disorders, defective recombination and infertility)


\end{itemize}

Plan Mammalian Meiotic Recombination: A Toolbox for Genome Evolution (https://www.karger.com/Article/FullText/452822):
\begin{itemize}
	\item Recombination and he repair of DSBs (Organization of meiotic chromosomes: importance of chromosomal axes, molecular events involved in he formation and repair of DSBs)
	\item Methodological approaches to he study of recombination
	\item Genetic and epigenetic marks of DSBs and recombination hotspots 
	\item Variation of recombination rates within genomes and among species (Variability at the chromosomal level, variation of fine-scale recombination maps)
	\item Evolvability of recombination rates (Chromosomal rearrangements as recombination modifiers)
\end{itemize}

Plan de Hotposts for initiation of meiotic recombination (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6237102/)
\begin{itemize}
	\item Defining DSB hotspots
	\item Chromatin shapes the meiotic DSB landscape (Nucleosome occupancy, meiotic chromosome architecture)
	\item Meiotic DSB and crossover distributions
	\item PRDM9 and H3K4me3
	\item The hotspot paradox
	\item Recombination initiation in repetitive sequences
	\item Byond hotspots: DSB-dependent spatial regulation


\end{itemize}


Mechanismes moleculaires precis + molecules impliquees
\begin{itemize}
	\item Appariement des chromosomes
	\item Formation du DSB
	\item Reparation CO/NCO
	\item Tous les modeles de resynthese des brins
	\item Observation des parametres de recombinaison chez la levure, souris
	% \item https://www.cell.com/current-biology/pdf/S0960-9822(06)01257-7.pdf: surveillance of breaks = checkpoints
	\item interference des CO
	\item Notion de strand asymmetry
	\item DNA polymerases (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5295669/)
	\item https://www.karger.com/Article/FullText/452822: mammalian eiotic recombination: a toolbox for genome evolution

\end{itemize}


Molecules tres importantes sur lesquelles insister:
\begin{itemize}
	\item DMC1
	\item RAD51
	\item (PRDM9)
	\item Spo11
	\item MUS81
	\item MLH1
	\item HFM1
\end{itemize}

Autre:
\begin{itemize}
	\item Non-allelic gene conversion
	\item Recombining without hotspots (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4684701/)
	\item Knockout of PRDM9 (http://science.sciencemag.org.inee.bib.cnrs.fr/content/352/6284/474)

\end{itemize}




