\begin{savequote}[8cm]
‘Pour résoudre des problèmes, il faut d'abord se les poser. Et pour pouvoir se les poser, il faut se créer des difficultés personnelles à se casser le ciboulot.’
\qauthor{--- Roy Lewis, \textit{\usebibentry{lewis1960evolution}{title}} \citeyearpar{lewis1960evolution} }

\end{savequote}

\chapter*{\label{ch:objectives}}


% \subsection*{} % Just to start where I want to

% \par

% Add blank lines before starting
\begin{parse lines}[\noindent]{#1\\}



\end{parse lines}

One striking result that came with the quantification of the population-scaled GC-biased gene conversion (gBGC) coefficient ($B$) across metazoans \citep{galtier2018codon} is that its intensity restricts to a very limited scope.
For instance, in placental mammals, $B$ settles in a [0; 7] range \citep{lartillot2013phylogenetic}.
Given that $B$ is nothing but the product of the effective population size ($N_e$) by the gBGC coefficient ($b$) (see Chapter~\ref{ch:4-gBGC}) and that $N_e$ fluctuates over orders of magnitude across metazoans, any theory according to which the intensity of gBGC ($b$) would be evolutionarily stable has to be ruled out \citep{galtier2018codon}.
% the effective population sizes ($N_e$) of the species in that clade fluctuate over orders of magnitude, $b$ cannot be stable \citep{galtier2018codon}.
Instead, one or several of the parameters on which $b$ depends (the recombination rate $r$, the length of conversion tracts $L$ and the transmission bias $b_0$) necessarily vary inversely with $N_e$.
% for $B$ to remain constant across animals.

However, data still lack to understand the basis of the dependency between $N_e$ and $b$: the transmission bias ($b_0$) has only been measured in a handful of species \citep{mancera2008highresolution, si2015widely, williams2015noncrossover, halldorsson2016rate, keith2016high, smeds2016highresolution} and, among mammals, the only species for which $b_0$ has been quantified is one with a very low $N_e$ of 10,000 \citep{takahata1993allelic,erlich1996hla,harding1997archaic,charlesworth2009fundamental,yu2004nucleotide}: \textit{Homo sapiens}.

% and, among mammals, only in humans, which have a very low $N_e$ of 10,000 \citep{,,}.
% Among mammals, $b_0$ has been quantified only in humans, which have a very low $N_e$ of 10,000 \citep{,,}.
% Among mammals, the only species for which $b_0$ has been quantified is one with a very low $N_e$ of 10,000 \citep{,,}: \textit{Homo sapiens}.

In order to shed new light on the interplay between $b$ and $N_e$, we thus aimed at quantifying gBGC in another mammalian species displaying an effective population size much larger than that of humans \citep{geraldes2008inferring,phifer-rixey2012adaptive,davies2015factors}: \textit{Mus musculus}.\\

Such endeavour calls for a large number of recombination events on which gBGC could be measured. 
Though, the method prominently used to detect recombination — pedigree analysis — is extremely resource-intensive: it requires a considerable number of individuals sequenced genome-wide and results in the detection of a limited amount of recombination events (see Chapter~\ref{ch:3-recombination-variation}).
Thus, we implemented a novel approach allowing to detect thousands of such events at high resolution in single individuals. I describe it in Chapter~\ref{ch:5-methodology}.
% Our approach showed unprecented power to detect recombination events

Then, in Chapter~\ref{ch:6-recombination-parameters}, I describe how these tens of thousands of events allowed us to precisely characterise recombination in over 1,000 autosomal recombination hotspots and how we could infer the genuine parameters of mouse meiotic recombination (in particular the recombination rate $r$ and the length of conversion tracts $L$) through inferential methods.

Next, after distinguishing the effects of GC-biased gene conversion (gBGC) from those of DSB-induced biased gene conversion (dBGC) on the observable transmission of alleles, we managed to quantify the transmission bias ($b_0$) of \textit{GC} alleles in the conversion tracts of our detected recombination events as well as the intensity of dBGC in several hundreds of recombination hotspots. 
I describe these findings in Chapter~\ref{ch:7-quantification-BGC}.

Last, because the approach presented in Chapter~\ref{ch:5-methodology} showed unprecedented power to detect recombination events, the logical follow-up was to re-use it in other studies involving the knock-in (i.e.\ inactivation) of recombination-dependent genes.
In Chapter~\ref{ch:8-HFM1}, I describe the methodological adaptations of our procedure to such investigations and the primary results of our analysis.

The results described in the four aforementioned chapters will then be discussed in Chapter~\ref{ch:9-discussion}.


% \chapter{\label{ch:5-methodology}}
% \chapter{\label{ch:6-recombination-parameters}}
% \chapter{\label{ch:7-quantification-BGC}}
% \chapter{\label{ch:8-HFM1}}
% \chapter{\label{ch:9-discussion}}

